{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# FINDING, ACCESSING, INTEGRATING, and REUSING data (FAIR)\n",
    "\n",
    "During this course, we will explore the history of bioinformatics data publication and reuse.  We will explore data technologies from the most primitive through the past ~20 years to the most contemporary.  \n",
    "\n",
    "The overview of our lessons for the next ~2 months is:\n",
    "\n",
    "* \"Screen Scraping\" data from pages directly accessed on the Web\n",
    "* Application Programming Interfaces (APIs) for data retrieval and integration\n",
    "    * Custom APIs\n",
    "    * \"REST\" APIs\n",
    "    * BioRuby APIs\n",
    "* Linked Data & SPARQL query\n",
    "* FAIR Data\n",
    "\n",
    "At the end of the course, we will explore a very simple (very primitive!) way to create Web pages that are connected to databases.  \n",
    "\n",
    "In the end, you will be required to show that you can publish data that is FAIR for both humans and machines - you will be making a Website that can be queried by both people and by computers (this is your final assignment of the year)\n",
    "<pre>\n",
    "\n",
    "\n",
    "</pre>\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# The Painful Truth\n",
    "\n",
    "\n",
    "Getting data from the Web is still far more painful than it should be.\n",
    "\n",
    "\n",
    "Most resources do not offer data in a format that is easily accessible by machines.\n",
    "\n",
    "\n",
    "Many resources claim to provide “RESTful” access, but even they often make it difficult (we will discuss REST in a future lecture).\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# One example of a \"friendly\" provider\n",
    "\n",
    "There are (too) many examples of Bioinformatics data providers that do not make it easy to retrieve their data.  In your future careers, you will probably need to access data from one of these providers.   Good luck!\n",
    "\n",
    "In this course, we are going to mainly focus on \"nice\" data providers.  Nevertheless, we begin with a ~difficult case, so that you learn how to deal with these situations.  After that, and over the next weeks, we will explore increasingly better, friendlier, more powerful, and easier-to-explore data sources.\n",
    "\n",
    "For today, we are going to explore the European Bioinformatics Institute (EBI) dbFetch interface (**NOTE: I am NOT saying that this is a BAD interface!  I am just using it as an example of a difficult case from the perspective of software!**).\n",
    "\n",
    "http://www.ebi.ac.uk/Tools/dbfetch/\n",
    "\n",
    "dbFetch provides data in a variety of formats, using a predictable URL structure.\n",
    "\n",
    "<pre>\n",
    "\n",
    "\n",
    "</pre>\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# From Within a Script\n",
    "\n",
    "Scroll down the dbFetch web page until you get to the section titled \"from within a script\":\n",
    "\n",
    "------------------\n",
    "\n",
    "* from within a script - examples of the URL for all styles and formats\n",
    "\n",
    "For people interested in programmatic access to the Dbfetch functionality, we recommend using our new Web Services version of Dbfetch: WSDbfetch.\n",
    "\n",
    "Alternatively, you can use dbfetch for direct access:\n",
    "\n",
    "Making scripted http requests to dbfetch is very simple, the parameters which can be used are db, id, format and style. Of these parameters only db and id are required fields. When omitting to use format and/or style, the defaults for the chosen database will be used (the default style is always html).\n",
    "\n",
    "The URL to dbfetch is always of this format:\n",
    "**dbfetch?db=DB_NAME&id=IDS&format=FORMAT_NAME&style=STYLE_NAME**\n",
    "\n",
    "    DB_NAME - Must be chosen from the table below\n",
    "    IDS - Single id/acc or comma/white-space separated list (id1 or id1 id2 id3 or id1,id2,id3)\n",
    "    FORMAT_NAME - Name of the output format, varies between databases\n",
    "    STYLE_NAME - Name of the output style, available styles are raw and html\n",
    "\n",
    "\n",
    "For details of the available databases, formats and styles see the list of databases [http://www.ebi.ac.uk/Tools/dbfetch/dbfetch/dbfetch.databases]. Additional examples are provided in the syntax guide [http://www.ebi.ac.uk/Tools/dbfetch/syntax.jsp].\n",
    "\n",
    "Examples:\n",
    "\n",
    "https://www.ebi.ac.uk/Tools/dbfetch/dbfetch?db=ena_sequence&id=J00231,K00650,D87894,AJ242600\n",
    "\n",
    "Instead of the default raw (plain ASCII) style, entries can also be retrieved in plain text (raw):\n",
    "https://www.ebi.ac.uk/Tools/dbfetch/dbfetch?db=ena_sequence&id=J00231,K00650,D87894,AJ242600&style=raw\n",
    "\n",
    "It is also possible to retrieve Fasta formatted sequences:\n",
    "https://www.ebi.ac.uk/Tools/dbfetch/dbfetch?db=ena_sequence&id=J00231,K00650,D87894,AJ242600&format=fasta\n",
    "\n",
    "Because of backward compatibility issues the program can be simply called by giving one or more INSDC accession numbers or entry names:\n",
    "https://www.ebi.ac.uk/Tools/dbfetch/dbfetch?J00231\n",
    "\n",
    "-------------------"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# The dbFetch URL Structure\n",
    "\n",
    "## dbfetch?db=<span style='color:red;'>DB_NAME</span>&id=IDS&format=FORMAT_NAME&style=STYLE_NAME\n",
    "\n",
    "The first element to notice in the dbFetch URL is the \"DB_NAME\" field.  The value of this field can be one of many of the databaases hosted at EBI.  The list is below.   I have highlighted \"Ensembl Genomes Gene\" because it is one of the few databases that understands *Arabodiopsis* gene IDs\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# The List of Databases\n",
    "\n",
    "\n",
    "An overview of each database is also provided, which includes a short description of the database, a link to the database, a collection of example identifiers and details of the available data formats and result styles.\n",
    "Databases\n",
    "\n",
    "* EDAM (edam)\n",
    "* ENA Coding (ena_coding)\n",
    "* ENA Geospatial (ena_geospatial)\n",
    "* ENA Non-coding (ena_noncoding)\n",
    "* ENA Sequence (ena_sequence)\n",
    "* ENA Sequence Constructed (ena_sequence_con)\n",
    "* ENA Sequence Constructed Expanded (ena_sequence_conexp)\n",
    "* ENA/SVA (ena_sva)\n",
    "* Ensembl Gene (ensemblgene)\n",
    "* **Ensembl Genomes Gene (ensemblgenomesgene)**[ http://www.ebi.ac.uk/Tools/dbfetch/dbfetch/dbfetch.databases#ensemblgenomesgene ]\n",
    "* Ensembl Genomes Transcript (ensemblgenomestranscript)\n",
    "* Ensembl Transcript (ensembltranscript)\n",
    "* EPO Proteins (epo_prt)\n",
    "* HGNC (hgnc)\n",
    "* IMGT/HLA (imgthla)\n",
    "* IMGT/LIGM-DB (imgtligm)\n",
    "* InterPro (interpro)\n",
    "* IPD-KIR (ipdkir)\n",
    "* IPD-MHC (ipdmhc)\n",
    "* IPRMC (iprmc)\n",
    "* IPRMC UniParc (iprmcuniparc)\n",
    "* JPO Proteins (jpo_prt)\n",
    "* KIPO Proteins (kipo_prt)\n",
    "* MEDLINE (medline)\n",
    "* Patent DNA NRL1 (nrnl1)\n",
    "* Patent DNA NRL2 (nrnl2)\n",
    "* Patent Protein NRL1 (nrpl1)\n",
    "* Patent Protein NRL2 (nrpl2)\n",
    "* Patent Equivalents (patent_equivalents)\n",
    "* PDB (pdb)\n",
    "* RefSeq (nucleotide) (refseqn)\n",
    "* RefSeq (protein) (refseqp)\n",
    "* SGT (sgt)\n",
    "* Taxonomy (taxonomy)\n",
    "* Trace Archive (tracearchive)\n",
    "* UniParc (uniparc)\n",
    "* UniProtKB (uniprotkb)\n",
    "* UniRef100 (uniref100)\n",
    "* UniRef50 (uniref50)\n",
    "* UniRef90 (uniref90)\n",
    "* UniSave (unisave)\n",
    "* USPTO Proteins (uspto_prt)\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Ensembl Genomes Gene (ensemblgenomesgene)\n",
    "\n",
    "http://www.ensemblgenomes.org/\n",
    "\n",
    "Ensembl Genomes genome databases for metazoa, plants, fungi, protists and bacteria, for vertebrate species and **model organisms** [1] see Ensembl instead of Ensembl Genomes. Gene sequences and annotations.\n",
    "\n",
    "<span style='font-size: 50%;'>[1]  Apparently, EnsEMBL doesn't consider Arabidopsis a \"model organism\" ;-)</span>\n",
    "\n",
    "| <span style='color: blue;'>Format</span> |\t<span style='color: green;'>Styles</span> | \tExample Identifiers | \n",
    "| ------ | ------- | -------\n",
    "| default  | \tdefault, raw \t | Id: AAEL000001, AGAP006864, GB46163, b2736\n",
    "| csv  | \tdefault, raw  | \tId: AAEL000001, AGAP006864, GB46163, b2736\n",
    "| **embl**  | \tdefault, raw  | \tId: AAEL000001, AGAP006864, GB46163, b2736\n",
    "| fasta  | \tdefault, raw  | \tId: AAEL000001, AGAP006864, GB46163, b2736\n",
    "| genbank  | \tdefault, raw  | \tId: AAEL000001, AGAP006864, GB46163, b2736\n",
    "| gff2  | \tdefault, raw  | \tId: AAEL000001, AGAP006864, GB46163, b2736\n",
    "| gff3  | \tdefault, raw  | \tId: AAEL000001, AGAP006864, GB46163, b2736\n",
    "| tab |  \tdefault, raw  | \tId: AAEL000001, AGAP006864, GB46163, b2736 \n",
    "\n",
    "\n",
    "\n",
    "## dbfetch?db=DB_NAME&id=IDS&format=<span style='color: blue;'>FORMAT_NAME</span>&style=<span style='color: green;'>STYLE_NAME</span>\n",
    "\n",
    "The columns for Format and Style are highlighted, together with their position in our structured URL.  I have also highlighted **\"embl\"**, because that is the data format that I want us to use for this example.  "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# So far we have...\n",
    "\n",
    "## dbfetch?db=<span style='color: red;'>ensemblgenomesgene</span>&format=<span style='color: blue;'>embl</span>&id=.....\n",
    "\n",
    "These elements are called \"GET String Parameters\".  **db** is one parameter, with a value of \"ensemblgenomesgene\".  **format** is another parameter, with the value \"embl\".  In a URL, Get String parameters have one of these two patterns:\n",
    "\n",
    "    ?parameter1=value1;parameter2=value2;.....\n",
    "    ?parameter1=value1&parameter2=value2&.....\n",
    "   \n",
    "i.e., at the end of teh URL, the parameter/value pairs follow a '?' symbol, and parameter/value pairs are separated by '&' or ';'. (We will discuss what \"GET\" means when we talk about REST in a future lecture.)\n",
    "\n",
    "What is the complete URL?  What is before the '?'?\n",
    "\n",
    "Go back to the main dbFetch page and look at the examples:\n",
    "\n",
    "Examples:\n",
    "\n",
    "**https://www.ebi.ac.uk/Tools/dbfetch/dbfetch**?db=ena_sequence&id=J00231,K00650,D87894,AJ242600\n",
    "\n",
    "Instead of the default raw (plain ASCII) style, entries can also be retrieved in plain text (raw):\n",
    "\n",
    "**https://www.ebi.ac.uk/Tools/dbfetch/dbfetch**?db=ena_sequence&id=J00231,K00650,D87894,AJ242600&style=raw\n",
    "\n",
    "It is also possible to retrieve Fasta formatted sequences:\n",
    "\n",
    "**https://www.ebi.ac.uk/Tools/dbfetch/dbfetch**?db=ena_sequence&id=J00231,K00650,D87894,AJ242600&format=fasta\n",
    "<pre>\n",
    "\n",
    "\n",
    "</pre>\n",
    "\n",
    "\n",
    "\n",
    "##  So, the URL for the Arabidopsis gene At3g54340, in ensembl format, is:\n",
    "\n",
    "### https://www.ebi.ac.uk/Tools/dbfetch/dbfetch?db=ensemblgenomesgene&format=embl&id=At3g54340\n",
    "\n",
    "Open this in your browser (open in a new window!) and examine the structure of that file.  How would you extract information from that file, for example, the name of the gene, or the number of exons?"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<pre>\n",
    "  \n",
    "\n",
    "\n",
    "\n",
    "</pre>\n",
    "  \n",
    "# Prove that you understand:\n",
    "\n",
    "* Retrieve the EMBL record for AT4G36920 (what gene name is it?)\n",
    "* Retrieve the FASTA file for AT4G36920\n",
    "* Retrieve the UniProt record for AP3_ARATH \n",
    "    * DO NOT retrieve it in HTML - i.e. you want the RECORD, not the web page containing the record --> you will need to use the “style” GET string parameter (&style=...).  Try the different \"style\" options until you get what you want.  Look at the table(s) of styles for each database to decide\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<pre>\n",
    "\n",
    "\n",
    "\n",
    "</pre>\n",
    "\n",
    "# Your first step towards massively integrative informatics - Deja el navegador!\n",
    "\n",
    "It is impossible to do modern bioinformatics through your browser!  To achieve data discovery and integration at the scale of contemporary systems biology, it is necessary to do all of the data retrieval and integration inside of your software.\n",
    "\n",
    "**How do you access The Web in your code?   Use a softeware library called \"net/http\"**\n",
    "\n",
    "\n",
    "<pre>\n",
    "\n",
    "</pre>\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ID   3    standard; DNA; HTG; 2175 BP.\n",
      "XX\n",
      "AC   chromosome:TAIR10:3:20119140:20121314:1\n",
      "XX\n",
      "SV   chromosome:TAIR10:3:20119140:20121314:1\n",
      "XX\n",
      "DT   30-SEP-2019\n",
      "XX\n",
      "DE   Arabidopsis thaliana chromosome 3 TAIR10 partial sequence\n",
      "DE   20119140..20121314 annotated by The Arabidopsis Information Resource\n",
      "XX\n",
      "KW   .\n",
      "XX\n",
      "OS   Arabidopsis thaliana (thale-cress)\n",
      "OC   Eukaryota; Viridiplantae; Streptophyta; Embryophyta; Tracheophyta;\n",
      "OC   Spermatophyta; Magnoliophyta; eudicotyledons; core eudicots; rosids;\n",
      "OC   eurosids II; Brassicales; Brassicaceae; Arabidopsis.\n",
      "XX\n",
      "CC   This sequence displays annotation from Ensembl Genomes, based on underlying\n",
      "CC   annotation from The Arabidopsis Information Resource (\n",
      "CC   https://www.araport.org/ ). See http://www.ensemblgenomes.org for more\n",
      "CC   information.\n",
      "XX\n",
      "CC   All feature locations are relative to the first (5') base of the sequence\n",
      "CC   in this file.  The sequence presented is always the forward strand of the\n",
      "CC   assembly. Features that lie outside of the sequence contained in this file\n",
      "CC   have clonal location coordinates in the format: .:..\n",
      "XX\n",
      "CC   The /gene indicates a unique id for a gene, /note=\"transcript_id=...\" a\n",
      "CC   unique id for a transcript, /protein_id a unique id for a peptide and\n",
      "CC   note=\"exon_id=...\" a unique id for an exon. These ids are maintained\n",
      "CC   wherever possible between versions.\n",
      "XX\n",
      "FH   Key             Location/Qualifiers\n",
      "FT   source          1..2175\n",
      "FT                   /organism=\"Arabidopsis thaliana\"\n",
      "FT                   /db_xref=\"taxon:3702\"\n",
      "FT   gene            complement(1..2175)\n",
      "FT                   /gene=AT3G54340\n",
      "FT                   /locus_tag=\"AP3\"\n",
      "FT                   /note=\"Floral homeotic protein APETALA 3\n",
      "FT                   [Source:UniProtKB/Swiss-Prot;Acc:P35632]\"\n",
      "FT   mRNA            join(complement(1761..2175),complement(1593..1659),\n",
      "FT                   complement(1448..1509),complement(1264..1363),\n",
      "FT                   complement(1050..1091),complement(729..773),\n",
      "FT                   complement(1..483))\n",
      "FT                   /gene=\"AT3G54340\"\n",
      "FT                   /standard_name=\"AT3G54340.1\"\n",
      "FT   CDS             join(complement(1761..1948),complement(1593..1659),\n",
      "FT                   complement(1448..1509),complement(1264..1363),\n",
      "FT                   complement(1050..1091),complement(729..773),\n",
      "FT                   complement(289..483))\n",
      "FT                   /gene=\"AT3G54340\"\n",
      "FT                   /protein_id=\"AT3G54340.1\"\n",
      "FT                   /note=\"transcript_id=AT3G54340.1\"\n",
      "FT                   /db_xref=\"Uniprot/SWISSPROT:P35632\"\n",
      "FT                   /db_xref=\"RefSeq_peptide:NP_191002.1\"\n",
      "FT                   /db_xref=\"RefSeq_dna:NM_115294.6\"\n",
      "FT                   /db_xref=\"Uniprot/SPTREMBL:A0A178VHZ2\"\n",
      "FT                   /db_xref=\"TAIR_LOCUS_MODEL:AT3G54340.1\"\n",
      "FT                   /db_xref=\"EMBL:AY070397\"\n",
      "FT                   /db_xref=\"EMBL:AY087369\"\n",
      "FT                   /db_xref=\"EMBL:AY142590\"\n",
      "FT                   /db_xref=\"EMBL:M86357\"\n",
      "FT                   /db_xref=\"GO:0000977\"\n",
      "FT                   /db_xref=\"GO:0000977\"\n",
      "FT                   /db_xref=\"GO:0000977\"\n",
      "FT                   /db_xref=\"GO:0003677\"\n",
      "FT                   /db_xref=\"GO:0003677\"\n",
      "FT                   /db_xref=\"GO:0003677\"\n",
      "FT                   /db_xref=\"GO:0003700\"\n",
      "FT                   /db_xref=\"GO:0003700\"\n",
      "FT                   /db_xref=\"GO:0003700\"\n",
      "FT                   /db_xref=\"GO:0005634\"\n",
      "FT                   /db_xref=\"GO:0005634\"\n",
      "FT                   /db_xref=\"GO:0005634\"\n",
      "FT                   /db_xref=\"GO:0005634\"\n",
      "FT                   /db_xref=\"GO:0005634\"\n",
      "FT                   /db_xref=\"GO:0006351\"\n",
      "FT                   /db_xref=\"GO:0006355\"\n",
      "FT                   /db_xref=\"GO:0006355\"\n",
      "FT                   /db_xref=\"GO:0007275\"\n",
      "FT                   /db_xref=\"GO:0007275\"\n",
      "FT                   /db_xref=\"GO:0008134\"\n",
      "FT                   /db_xref=\"GO:0009908\"\n",
      "FT                   /db_xref=\"GO:0010093\"\n",
      "FT                   /db_xref=\"GO:0010093\"\n",
      "FT                   /db_xref=\"GO:0030154\"\n",
      "FT                   /db_xref=\"GO:0030154\"\n",
      "FT                   /db_xref=\"GO:0043565\"\n",
      "FT                   /db_xref=\"GO:0044212\"\n",
      "FT                   /db_xref=\"GO:0045944\"\n",
      "FT                   /db_xref=\"GO:0045944\"\n",
      "FT                   /db_xref=\"GO:0046983\"\n",
      "FT                   /db_xref=\"GO:0046983\"\n",
      "FT                   /db_xref=\"protein_id:AAD41557.1\"\n",
      "FT                   /db_xref=\"protein_id:AAD51887.1\"\n",
      "FT                   /db_xref=\"protein_id:AAD51888.1\"\n",
      "FT                   /db_xref=\"protein_id:AAD51889.1\"\n",
      "FT                   /db_xref=\"protein_id:AAD51890.1\"\n",
      "FT                   /db_xref=\"protein_id:AAD51891.1\"\n",
      "FT                   /db_xref=\"protein_id:AAD51892.1\"\n",
      "FT                   /db_xref=\"protein_id:AAD51893.1\"\n",
      "FT                   /db_xref=\"protein_id:AAD51894.1\"\n",
      "FT                   /db_xref=\"protein_id:AAD51895.1\"\n",
      "FT                   /db_xref=\"protein_id:AAD51896.1\"\n",
      "FT                   /db_xref=\"protein_id:AAD51897.1\"\n",
      "FT                   /db_xref=\"protein_id:AAD51898.1\"\n",
      "FT                   /db_xref=\"protein_id:AAD51899.1\"\n",
      "FT                   /db_xref=\"protein_id:AAD51900.1\"\n",
      "FT                   /db_xref=\"protein_id:AAD51901.1\"\n",
      "FT                   /db_xref=\"protein_id:AAD51902.1\"\n",
      "FT                   /db_xref=\"protein_id:AAD51903.1\"\n",
      "FT                   /db_xref=\"protein_id:AEE79216.1\"\n",
      "FT                   /db_xref=\"protein_id:BAA04665.1\"\n",
      "FT                   /db_xref=\"protein_id:CAB81799.1\"\n",
      "FT                   /db_xref=\"protein_id:OAP05456.1\"\n",
      "FT                   /db_xref=\"TAIR_TRANSLATION:AT3G54340\"\n",
      "FT                   /db_xref=\"PO:PO:0000037\"\n",
      "FT                   /db_xref=\"PO:PO:0000229\"\n",
      "FT                   /db_xref=\"PO:PO:0001078\"\n",
      "FT                   /db_xref=\"PO:PO:0001081\"\n",
      "FT                   /db_xref=\"PO:PO:0001185\"\n",
      "FT                   /db_xref=\"PO:PO:0004507\"\n",
      "FT                   /db_xref=\"PO:PO:0007600\"\n",
      "FT                   /db_xref=\"PO:PO:0007611\"\n",
      "FT                   /db_xref=\"PO:PO:0007616\"\n",
      "FT                   /db_xref=\"PO:PO:0009009\"\n",
      "FT                   /db_xref=\"PO:PO:0009010\"\n",
      "FT                   /db_xref=\"PO:PO:0009029\"\n",
      "FT                   /db_xref=\"PO:PO:0009030\"\n",
      "FT                   /db_xref=\"PO:PO:0009031\"\n",
      "FT                   /db_xref=\"PO:PO:0009032\"\n",
      "FT                   /db_xref=\"PO:PO:0009046\"\n",
      "FT                   /db_xref=\"PO:PO:0009052\"\n",
      "FT                   /db_xref=\"PO:PO:0025022\"\n",
      "FT                   /db_xref=\"PO:PO:0025281\"\n",
      "FT                   /db_xref=\"STRING:3702.AT3G54340.1\"\n",
      "FT                   /db_xref=\"UniParc:UPI0000001796\"\n",
      "FT                   /translation=\"MARGKIQIKRIENQTNRQVTYSKRRNGLFKKAHELTVLCDARVSI\n",
      "FT                   IMFSSSNKLHEYISPNTTTKEIVDLYQTISDVDVWATQYERMQETKRKLLETNRNLRTQ\n",
      "FT                   IKQRLGECLDELDIQELRRLEDEMENTFKLVRERKFKSLGNQIETTKKKNKSQQDIQKN\n",
      "FT                   LIHELELRAEDPHYGLVDNGGDYDSVLGYQIEGSRAYALRFHQNHHHYYPNHGLHAPSA\n",
      "FT                   SDIITFHLLE\"\n",
      "FT   exon            complement(1448..1509)\n",
      "FT                   /note=\"exon_id=AT3G54340.1.exon3\"\n",
      "FT   exon            complement(1593..1659)\n",
      "FT                   /note=\"exon_id=AT3G54340.1.exon2\"\n",
      "FT   exon            complement(1..483)\n",
      "FT                   /note=\"exon_id=AT3G54340.1.exon7\"\n",
      "FT   exon            complement(729..773)\n",
      "FT                   /note=\"exon_id=AT3G54340.1.exon6\"\n",
      "FT   exon            complement(1761..2175)\n",
      "FT                   /note=\"exon_id=AT3G54340.1.exon1\"\n",
      "FT   exon            complement(1264..1363)\n",
      "FT                   /note=\"exon_id=AT3G54340.1.exon4\"\n",
      "FT   exon            complement(1050..1091)\n",
      "FT                   /note=\"exon_id=AT3G54340.1.exon5\"\n",
      "FT   mRNA            join(complement(AL132971.2:19369..19757),\n",
      "FT                   complement(AL132971.2:19047..19281),\n",
      "FT                   complement(AL132971.2:18782..18927),\n",
      "FT                   complement(AL132971.2:18060..18147),\n",
      "FT                   complement(AL132971.2:17832..17978),\n",
      "FT                   complement(AL132971.2:16910..17059),\n",
      "FT                   complement(AL132971.2:14556..14644),\n",
      "FT                   complement(AL132971.2:14227..14290),\n",
      "FT                   complement(AL132971.2:13793..13853),\n",
      "FT                   complement(AL132971.2:13177..13184),\n",
      "FT                   complement(AL132971.2:11590..13131),\n",
      "FT                   complement(AL132971.2:11453..11504),\n",
      "FT                   complement(AL132971.2:11074..11222),\n",
      "FT                   complement(AL132971.2:10884..10970),\n",
      "FT                   complement(AL132971.2:10633..10761),complement(1448..1509),\n",
      "FT                   complement(1264..1363),complement(1050..1091),\n",
      "FT                   complement(729..773),complement(289..483))\n",
      "FT                   /product=\"MTTTSVCPFSKAARPDDGSTRKQGEITASGCPFSKAARPDDASARKQGE\n",
      "FT                   TTASGCPFSKSARPDENGSKGCPEQEGNLNKDSTDSATVPAKCPFGYDSQTFKLGPFSC\n",
      "FT                   MLCQALLYESSRCVPCTHVFCKVCLTRFKDCPLCGADIESIEVDENLQKMVDQFIEGHA\n",
      "FT                   RIKRSVVNGTEKEEVENDNKKVIYADVSMERGSFLVQQAMRAFSAQNYESAKSRLAMCT\n",
      "FT                   EDIRDQLGREGNTPELCSQLGAVLGMLGDCSRAMGDSSSAVKHFEESVEFLMKLPLNDL\n",
      "FT                   EITHTLSVSLNKIGDLKYYDEDLQAARSYYDRALNVRRDAMKHHPNAPSQILDVAVSLA\n",
      "FT                   KVADIDRTLQNEVAATDGFKEGMRLLESLKLDSEDSALEQRFCDMVTVARLLNLTLVVP\n",
      "FT                   EIDKTSFFADSRFCVLSPSLRPLLRYAFLREVKLLVSIVDFEGIGEYDISFEAACLAGA\n",
      "FT                   SLESLAKGAVQFSRRFSIRELQDRWHALLYDPVVSVEAAFRMAELERTNPNFPTKFGRT\n",
      "FT                   GYSKENKSSSRKRNAERLRSTYHSLRKKFRTEPFNSLDLGFLVPPNDSHFMDNGDATHL\n",
      "FT                   GLEDSHMDIIHNAFPEILAEGGCVTTHVLPEDNLQGDIPYVEGENLTFTEHAGLSVCDV\n",
      "FT                   VHQDSEQKLENTAHEAKNTMASTDFLAQLSTSLFEEDMEPFMEVDGKEVDKSYYDGLSS\n",
      "FT                   LLVNSTNDTNREAFPNPTEQEPSIAPTHPGEATLDDHVMLELDGTIALDPHPEIVGGVI\n",
      "FT                   CCLLNEEDPDIPCNDDIFLSNNSRPMSVSSLARRNFKDTNSPITTCVRDVSASKEKSEG\n",
      "FT                   YSLQAQKKKPGRLQGSTQGKPEMGQPSKGSKFRASTSTELKNTVAPGGSSSAQACSNTL\n",
      "FT                   LSTGTGAKDGKKETATGTLFVGSDGHGNHPEKDSENCKEKNVVPPVNESPHAKDTDDGL\n",
      "FT                   IEITVPELEITRAEAEAEAEAHVCESDEDLPNYSDIEAMILDMDLEPDDQDNFDLEVSK\n",
      "FT                   YQSQDMKRTIIRLEQAAHSYMQRAIASRGAFAVLYGRYSKHYIKKPEVLVGRSTEDLAV\n",
      "FT                   DIDLGREKRGSKISRRQAIIRLGDDGSFHIKNLGKYSISVNEKEVDPGQSLILKSDCLV\n",
      "FT                   ERMQETKRKLLETNRNLRTQIKQRLGECLDELDIQELRRLEDEMENTFKLVRERKFKSL\n",
      "FT                   GNQIETTKKKNKSQQDIQKNLIHELELRAEDPHYGLVDNGGDYDSVLGYQIEGSRAYAL\n",
      "FT                   RFHQNHHHYYPNHGLHAPSASDIITFHLLE\"\n",
      "FT                   /note=\"identifier=.1\"\n",
      "FT                   /note=\"Derived by automated computational analysis using\n",
      "FT                   gene prediction method:fgenesh\"\n",
      "FT   variation       18..18\n",
      "FT                   /replace=\"C/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH06313434\"\n",
      "FT   variation       22..22\n",
      "FT                   /replace=\"T/A\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH02503635\"\n",
      "FT   variation       23..23\n",
      "FT                   /replace=\"A/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20119162_A_G\"\n",
      "FT   variation       38..38\n",
      "FT                   /replace=\"G/A\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH06313435\"\n",
      "FT   variation       40..40\n",
      "FT                   /replace=\"A/-\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH15197512\"\n",
      "FT   variation       52..52\n",
      "FT                   /replace=\"T/A\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20119191_T_A\"\n",
      "FT   variation       68..68\n",
      "FT                   /replace=\"T/C\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH02503636\"\n",
      "FT   variation       73..73\n",
      "FT                   /replace=\"A/C\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH00418964\"\n",
      "FT   variation       89..89\n",
      "FT                   /replace=\"T/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20119228_T_G\"\n",
      "FT   variation       102..102\n",
      "FT                   /replace=\"G/A\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH14432852\"\n",
      "FT   variation       145..145\n",
      "FT                   /replace=\"C/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20119284_C_G\"\n",
      "FT   variation       179..179\n",
      "FT                   /replace=\"C/A\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20119318_C_A\"\n",
      "FT   variation       185..185\n",
      "FT                   /replace=\"A/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH06313438\"\n",
      "FT   variation       189..189\n",
      "FT                   /replace=\"T/C\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH02503637\"\n",
      "FT   variation       195..194\n",
      "FT                   /replace=\"-/T\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20119333_G_GT\"\n",
      "FT   variation       210..210\n",
      "FT                   /replace=\"A/C\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH06313439\"\n",
      "FT   variation       225..224\n",
      "FT                   /replace=\"-/T\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20119363_G_GT\"\n",
      "FT   variation       249..249\n",
      "FT                   /replace=\"G/A\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20119388_G_A\"\n",
      "FT   variation       264..264\n",
      "FT                   /replace=\"C/T\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20119403_C_T\"\n",
      "FT   variation       333..333\n",
      "FT                   /replace=\"C/T\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH00418965\"\n",
      "FT   variation       352..352\n",
      "FT                   /replace=\"G/A\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH02503638\"\n",
      "FT   variation       357..357\n",
      "FT                   /replace=\"A/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH02503639\"\n",
      "FT   variation       380..380\n",
      "FT                   /replace=\"C/T\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH02503640\"\n",
      "FT   variation       385..385\n",
      "FT                   /replace=\"A/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH12755904\"\n",
      "FT   variation       400..400\n",
      "FT                   /replace=\"C/A\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20119539_C_A\"\n",
      "FT   variation       454..454\n",
      "FT                   /replace=\"T/A\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20119593_T_A\"\n",
      "FT   variation       455..455\n",
      "FT                   /replace=\"C/T\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20119594_C_T\"\n",
      "FT   variation       502..501\n",
      "FT                   /replace=\"-/T\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20119640_G_GT\"\n",
      "FT   variation       537..537\n",
      "FT                   /replace=\"T/A\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20119676_T_A\"\n",
      "FT   variation       590..590\n",
      "FT                   /replace=\"C/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH12755925\"\n",
      "FT   variation       603..603\n",
      "FT                   /replace=\"A/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH06313441\"\n",
      "FT   variation       624..624\n",
      "FT                   /replace=\"T/C\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH00418966\"\n",
      "FT   variation       627..627\n",
      "FT                   /replace=\"T/C\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20119766_T_C\"\n",
      "FT   variation       654..654\n",
      "FT                   /replace=\"T/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20119793_T_G\"\n",
      "FT   variation       700..700\n",
      "FT                   /replace=\"C/A\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH06313442\"\n",
      "FT   variation       708..708\n",
      "FT                   /replace=\"A/-\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20119846_GA_G\"\n",
      "FT   variation       760..760\n",
      "FT                   /replace=\"T/A\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH12755926\"\n",
      "FT   variation       786..786\n",
      "FT                   /replace=\"A/C\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH02503641\"\n",
      "FT   variation       792..792\n",
      "FT                   /replace=\"C/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH14432854\"\n",
      "FT   variation       802..802\n",
      "FT                   /replace=\"G/T\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH02503642\"\n",
      "FT   variation       819..819\n",
      "FT                   /replace=\"T/A\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH02503643\"\n",
      "FT   variation       820..820\n",
      "FT                   /replace=\"C/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH12755929\"\n",
      "FT   variation       822..822\n",
      "FT                   /replace=\"C/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20119961_C_G\"\n",
      "FT   variation       825..825\n",
      "FT                   /replace=\"A/C\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20119964_A_C\"\n",
      "FT   variation       845..844\n",
      "FT                   /replace=\"-/C\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH04251560\"\n",
      "FT   variation       848..847\n",
      "FT                   /replace=\"-/A\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20119986_C_CA\"\n",
      "FT   variation       857..857\n",
      "FT                   /replace=\"T/A\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20119996_T_A\"\n",
      "FT   variation       867..867\n",
      "FT                   /replace=\"A/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20120006_A_G\"\n",
      "FT   variation       876..876\n",
      "FT                   /replace=\"A/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH12755932\"\n",
      "FT   variation       884..885\n",
      "FT                   /replace=\"TT/-\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH15365420\"\n",
      "FT   variation       906..909\n",
      "FT                   /replace=\"ACAC/AC/ACACAC/-\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH15365421\"\n",
      "FT   variation       915..915\n",
      "FT                   /replace=\"C/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20120054_C_G\"\n",
      "FT   variation       926..926\n",
      "FT                   /replace=\"A/C\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20120065_A_C\"\n",
      "FT   variation       980..980\n",
      "FT                   /replace=\"A/T\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH06313444\"\n",
      "FT   variation       984..984\n",
      "FT                   /replace=\"A/T\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH00418967\"\n",
      "FT   variation       1063..1063\n",
      "FT                   /replace=\"G/T\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH00418968\"\n",
      "FT   variation       1119..1119\n",
      "FT                   /replace=\"A/C\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20120258_A_C\"\n",
      "FT   variation       1140..1140\n",
      "FT                   /replace=\"T/C\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH14432875\"\n",
      "FT   variation       1144..1144\n",
      "FT                   /replace=\"T/C\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20120283_T_C\"\n",
      "FT   variation       1162..1162\n",
      "FT                   /replace=\"A/T\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH06313446\"\n",
      "FT   variation       1174..1174\n",
      "FT                   /replace=\"A/C\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH02503644\"\n",
      "FT   variation       1183..1183\n",
      "FT                   /replace=\"T/A\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH02503645\"\n",
      "FT   variation       1185..1185\n",
      "FT                   /replace=\"T/C\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH02503646\"\n",
      "FT   variation       1186..1186\n",
      "FT                   /replace=\"A/T\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH02503647\"\n",
      "FT   variation       1188..1193\n",
      "FT                   /replace=\"TAATTC/GAATTC/T\"\n",
      "FT                   /db_xref=\"The 1001 Genomes\n",
      "FT                   Project:tmp_3_20120327_TAATTC_GAATTC,T\"\n",
      "FT   variation       1198..1198\n",
      "FT                   /replace=\"C/A\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH12755933\"\n",
      "FT   variation       1209..1209\n",
      "FT                   /replace=\"C/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20120348_C_G\"\n",
      "FT   variation       1223..1223\n",
      "FT                   /replace=\"A/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20120362_A_G\"\n",
      "FT   variation       1239..1239\n",
      "FT                   /replace=\"G/-\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20120377_AG_A\"\n",
      "FT   variation       1275..1275\n",
      "FT                   /replace=\"G/T\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH12755934\"\n",
      "FT   variation       1276..1276\n",
      "FT                   /replace=\"A/C\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH02503648\"\n",
      "FT   variation       1289..1289\n",
      "FT                   /replace=\"G/T\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20120428_G_T\"\n",
      "FT   variation       1318..1318\n",
      "FT                   /replace=\"C/T\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20120457_C_T\"\n",
      "FT   variation       1320..1320\n",
      "FT                   /replace=\"G/A\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH12755935\"\n",
      "FT   variation       1323..1323\n",
      "FT                   /replace=\"C/T\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20120462_C_T\"\n",
      "FT   variation       1329..1329\n",
      "FT                   /replace=\"T/C\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH06313448\"\n",
      "FT   variation       1338..1338\n",
      "FT                   /replace=\"C/T\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH02503649\"\n",
      "FT   variation       1363..1363\n",
      "FT                   /replace=\"C/T\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH14432876\"\n",
      "FT   variation       1374..1374\n",
      "FT                   /replace=\"A/T\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH12755936\"\n",
      "FT   variation       1382..1382\n",
      "FT                   /replace=\"G/C/A\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH12755937\"\n",
      "FT   variation       1390..1390\n",
      "FT                   /replace=\"T/C\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH14432877\"\n",
      "FT   variation       1395..1395\n",
      "FT                   /replace=\"C/T\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20120534_C_T\"\n",
      "FT   variation       1408..1408\n",
      "FT                   /replace=\"G/A\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH06313449\"\n",
      "FT   variation       1441..1441\n",
      "FT                   /replace=\"C/A/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH06313450\"\n",
      "FT   variation       1450..1450\n",
      "FT                   /replace=\"G/T\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20120589_G_T\"\n",
      "FT   variation       1453..1453\n",
      "FT                   /replace=\"C/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH12755938\"\n",
      "FT   variation       1571..1571\n",
      "FT                   /replace=\"A/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20120710_A_G\"\n",
      "FT   variation       1585..1585\n",
      "FT                   /replace=\"A/-\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20120723_GA_G\"\n",
      "FT   variation       1586..1586\n",
      "FT                   /replace=\"A/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH06313451\"\n",
      "FT   variation       1628..1628\n",
      "FT                   /replace=\"T/C\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH12755939\"\n",
      "FT   variation       1647..1647\n",
      "FT                   /replace=\"G/A\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH12755940\"\n",
      "FT   variation       1670..1670\n",
      "FT                   /replace=\"C/A\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH02503650\"\n",
      "FT   variation       1675..1675\n",
      "FT                   /replace=\"T/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20120814_T_G\"\n",
      "FT   variation       1685..1685\n",
      "FT                   /replace=\"A/T\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH14432878\"\n",
      "FT   variation       1704..1704\n",
      "FT                   /replace=\"A/-\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20120842_GA_G\"\n",
      "FT   variation       1712..1712\n",
      "FT                   /replace=\"G/C\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20120851_G_C\"\n",
      "FT   variation       1735..1735\n",
      "FT                   /replace=\"G/A\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20120874_G_A\"\n",
      "FT   variation       1747..1747\n",
      "FT                   /replace=\"G/T\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20120886_G_T\"\n",
      "FT   variation       1811..1811\n",
      "FT                   /replace=\"G/A\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH02503651\"\n",
      "FT   variation       1868..1868\n",
      "FT                   /replace=\"A/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20121007_A_G\"\n",
      "FT   variation       1874..1874\n",
      "FT                   /replace=\"T/C\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH02503652\"\n",
      "FT   variation       1889..1889\n",
      "FT                   /replace=\"C/T\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20121028_C_T\"\n",
      "FT   variation       1916..1916\n",
      "FT                   /replace=\"T/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20121055_T_G\"\n",
      "FT   variation       1954..1954\n",
      "FT                   /replace=\"T/C\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH14432879\"\n",
      "FT   variation       1980..1980\n",
      "FT                   /replace=\"A/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20121119_A_G\"\n",
      "FT   variation       1987..1987\n",
      "FT                   /replace=\"T/C\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20121126_T_C\"\n",
      "FT   variation       2000..2000\n",
      "FT                   /replace=\"C/A\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH12755941\"\n",
      "FT   variation       2012..2012\n",
      "FT                   /replace=\"T/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20121151_T_G\"\n",
      "FT   variation       2016..2016\n",
      "FT                   /replace=\"A/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20121155_A_G\"\n",
      "FT   variation       2131..2131\n",
      "FT                   /replace=\"G/C\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH14432880\"\n",
      "FT   variation       2138..2138\n",
      "FT                   /replace=\"A/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH02503653\"\n",
      "FT   variation       2145..2144\n",
      "FT                   /replace=\"-/A\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20121283_T_TA\"\n",
      "FT   variation       2165..2165\n",
      "FT                   /replace=\"C/G/T\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH02503654\"\n",
      "FT   variation       2168..2168\n",
      "FT                   /replace=\"A/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH02503655\"\n",
      "FT   variation       2169..2169\n",
      "FT                   /replace=\"T/A\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH06313453\"\n",
      "FT   repeat_region   complement(823..829)\n",
      "FT                   /note=\"dust repeat: matches 20119962..20119968(1) of\n",
      "FT                   consensus\"\n",
      "FT   repeat_region   complement(906..924)\n",
      "FT                   /note=\"dust repeat: matches 20120045..20120063(1) of\n",
      "FT                   consensus\"\n",
      "FT   repeat_region   complement(1399..1406)\n",
      "FT                   /note=\"dust repeat: matches 20120538..20120545(1) of\n",
      "FT                   consensus\"\n",
      "FT   repeat_region   complement(1558..1591)\n",
      "FT                   /note=\"dust repeat: matches 20120697..20120730(1) of\n",
      "FT                   consensus\"\n",
      "FT   repeat_region   complement(1979..2063)\n",
      "FT                   /note=\"dust repeat: matches 20121118..20121202(1) of\n",
      "FT                   consensus\"\n",
      "FT   repeat_region   complement(2169..2175)\n",
      "FT                   /note=\"dust repeat: matches 20121308..20121314(1) of\n",
      "FT                   consensus\"\n",
      "FT   misc_feature    1..2175\n",
      "FT                   /note=\"contig AL132971.2 7316..9490(1)\"\n",
      "XX\n",
      "SQ   Sequence       2175 BP;        774 A;        367 C;        377 G;        657 T;          0 other;\n",
      "     GACAATATAT ATGCATTCAA GTAGAAAATC AAATCGAGTA AATTTGTAGC ATAATTAAGT        60\n",
      "     AGAGTGGTCA ATATAATAAT ACGATACATG ATAAAAGTCT TGAAAACATT CCACACACAA       120\n",
      "     TAAGCCTAAG CACTACTGAT CACACAATCC ATATTTCTTT AGGCTTTAAA ATACATAACT       180\n",
      "     AAGAAACATG ACAGTTTAAG GTTTTAGCAA CACCATGCCT TATGTTTTAA GTAGTTACTA       240\n",
      "     AATAATTAGA TAGACAATGA TGGCACCAGC AAACCTTTTA GCCTTTAATT ATTCAAGAAG       300\n",
      "     ATGGAAGGTA ATGATGTCAG AGGCAGAGGG TGCATGAAGG CCATGGTTGG GGTAATAGTG       360\n",
      "     GTGATGGTTC TGGTGGAAAC GAAGAGCGTA AGCACGTGAC CCTTCGATTT GGTATCCAAG       420\n",
      "     AACTGAGTCG TAATCTCCTC CATTGTCTAC TAGTCCATAG TGAGGATCTT CAGCTCTTAG       480\n",
      "     TTCCTAATTC ACCAAAACAT GTTTTAAGCC ATATTCCACA TAACTAAAAC CAATATTATT       540\n",
      "     AATTCGAAAT GACATGACAT TTGATATATG ATTTAACTAG GAGATATTTC GATAAGAGAA       600\n",
      "     TCACAAGAAA ACATAAGAGT GACTGATCAA TCTACACTAC ATTTCTAGAT ATCTTTTTAA       660\n",
      "     CTGGCTCCGT ATTCCATGAA TGTATTATGC CTAAACCAAC ATTTGAGAAA AACAGAAAGA       720\n",
      "     GATATTACCA GCTCATGTAT GAGATTCTTT TGTATGTCTT GTTGACTTTT GTTCTGCAAT       780\n",
      "     TGTGTAAACC ACAAGAGAAA TGATGAAGAA ATATTCCCTC CCAAAAAAAT TATGAAGATA       840\n",
      "     TATACCCAAG TTATTATGAC ATCACTATAT AGTATACTTG GTATTTTCTT TATCCATTAG       900\n",
      "     CTTAAACACA CACACACACA CACAAATATA ATTGTCCAAG AAAAGTGTTA TCAACAGTTT       960\n",
      "     TAGTAAGTTG ATAAATTCAA TTGACACAAA ATTGTTGAGA CAAATAAATA TGATAAAATG      1020\n",
      "     GAAAAGTGAG TGGACATAGA AGATGTGACC TTTTTCTTGG TGGTCTCGAT CTGATTCCCA      1080\n",
      "     AGAGATTTGA ACTGTACAGT ACAATACAAT TAACATATAT TATCCAACAG TAAACAAAAT      1140\n",
      "     GTTTCTTTTT CTAAAACATA TATACAACAA TAAAATTCAT TTTTTAATAA TTCAATTCAA      1200\n",
      "     AACTGAAGCT TGGGTATATA GGATTAATTG AAACCCTAGG TAAGGGAAAA GTATGAAGAG      1260\n",
      "     AACCTTGCGC TCGCGAACGA GTTTGAAAGT GTTTTCCATT TCATCCTCAA GACGACGCAG      1320\n",
      "     CTCCTGAATG TCAAGCTCGT CCAAACACTC ACCTAGCCTC TGCCTACAAT TTTAAAAATA      1380\n",
      "     TGTATATTTT CGTACTCATT TTTTTTAGTA AAACTCCTCA TCTAAGAGAA GAGAGAGAAA      1440\n",
      "     CAAATACTTG ATCTGAGTCC GGAGATTTCT ATTTGTCTCC AACAGTTTCC TCTTGGTTTC      1500\n",
      "     TTGCATTCGC TGCAGCAAAA CGCATCAAGA ATTTAACCAA CCAGCGAAAC ACATATCAAG      1560\n",
      "     AAGAAGAAGA AAGATCTAAG AAGGAAAAAA ACCTCATATT GAGTGGCCCA AACATCGACA      1620\n",
      "     TCAGAAATAG TTTGGTACAG ATCTACGATC TCCTTCGTTC TATAACCAAC AAAATCACCA      1680\n",
      "     AAAAAGTAGT GGTTAATTAT AAGAAAACAA AGCAACTCAA TTAGCTTAAT TTAAGAGTGG      1740\n",
      "     TGTTTAGAGA GATGGTGTAC GTGGTGTTAG GGCTGATATA CTCATGAAGC TTGTTGGAGC      1800\n",
      "     TAGAGAACAT GATAATCGAA ACCCTAGCAT CACACAAAAC CGTGAGCTCA TGTGCTTTCT      1860\n",
      "     TGAATAAACC ATTTCTTCTC TTTGAATACG TCACTTGTCT GTTTGTCTGG TTCTCTATCC      1920\n",
      "     TCTTGATCTG GATCTTCCCT CTCGCCATAT TCTTCTCTCT TTGTTTAATC TTTTTGTTGA      1980\n",
      "     AGAGATTTGG TGGAGAGGAC AAGAGATATA ATGAGAAGAA AGAAGTGAAA TAGAAAGAGA      2040\n",
      "     GAGATGGAGT TGAAGAAGTA AAGGGTCCAC TTGAGTTACT AAAAATGGAA AGTATTGCCT      2100\n",
      "     AATCCATGAA AGGTAAGTTC AGAAGTTGAT GAAAACTAAG TGATAAATTT TCCATTTATG      2160\n",
      "     TAAACTGATT TTTTT                                                       2175\n",
      "//\n",
      "\n",
      "\n",
      "\n",
      "\n"
     ]
    }
   ],
   "source": [
    "\n",
    "\n",
    "require 'rest-client'   # this is how you access the Web\n",
    "\n",
    "address = 'http://www.ebi.ac.uk/Tools/dbfetch/dbfetch?db=ensemblgenomesgene&format=embl&id=At3g54340'\n",
    "\n",
    "response = RestClient::Request.execute(\n",
    "  method: :get,\n",
    "  url: address)  # use the Net::HTTP Class \"get_response\" method\n",
    "                                               # to call that address\n",
    "puts response.body\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "\n",
    "<pre>\n",
    "\n",
    "\n",
    "</pre>\n",
    "\n",
    "## That was easy!  \n",
    "\n",
    "\n",
    "That was easy!   ....too easy!  This is a good time to learn about how to handle errors in Ruby.  A lot of the time when you try to access the Web, something will go wrong... maybe the Web page doesn't exist, or you have lost your WiFi connection, or you have made a mistake in your code, or... or... or...\n",
    "\n",
    "\n",
    "The code below does the same thing as the code above, but it is more careful.  It uses a function called \"fetch\" to attempt to contact the website, and if it fails, it does something useful (returns False).  This is the first time you have seen error-handling.  Look at the examples in the code, and try to understand them.  See also:  http://rubylearning.com/satishtalim/ruby_exceptions.html \n",
    "<pre>\n",
    "\n",
    "\n",
    "</pre>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ID   3    standard; DNA; HTG; 2175 BP.\n",
      "XX\n",
      "AC   chromosome:TAIR10:3:20119140:20121314:1\n",
      "XX\n",
      "SV   chromosome:TAIR10:3:20119140:20121314:1\n",
      "XX\n",
      "DT   30-SEP-2019\n",
      "XX\n",
      "DE   Arabidopsis thaliana chromosome 3 TAIR10 partial sequence\n",
      "DE   20119140..20121314 annotated by The Arabidopsis Information Resource\n",
      "XX\n",
      "KW   .\n",
      "XX\n",
      "OS   Arabidopsis thaliana (thale-cress)\n",
      "OC   Eukaryota; Viridiplantae; Streptophyta; Embryophyta; Tracheophyta;\n",
      "OC   Spermatophyta; Magnoliophyta; eudicotyledons; core eudicots; rosids;\n",
      "OC   eurosids II; Brassicales; Brassicaceae; Arabidopsis.\n",
      "XX\n",
      "CC   This sequence displays annotation from Ensembl Genomes, based on underlying\n",
      "CC   annotation from The Arabidopsis Information Resource (\n",
      "CC   https://www.araport.org/ ). See http://www.ensemblgenomes.org for more\n",
      "CC   information.\n",
      "XX\n",
      "CC   All feature locations are relative to the first (5') base of the sequence\n",
      "CC   in this file.  The sequence presented is always the forward strand of the\n",
      "CC   assembly. Features that lie outside of the sequence contained in this file\n",
      "CC   have clonal location coordinates in the format: .:..\n",
      "XX\n",
      "CC   The /gene indicates a unique id for a gene, /note=\"transcript_id=...\" a\n",
      "CC   unique id for a transcript, /protein_id a unique id for a peptide and\n",
      "CC   note=\"exon_id=...\" a unique id for an exon. These ids are maintained\n",
      "CC   wherever possible between versions.\n",
      "XX\n",
      "FH   Key             Location/Qualifiers\n",
      "FT   source          1..2175\n",
      "FT                   /organism=\"Arabidopsis thaliana\"\n",
      "FT                   /db_xref=\"taxon:3702\"\n",
      "FT   gene            complement(1..2175)\n",
      "FT                   /gene=AT3G54340\n",
      "FT                   /locus_tag=\"AP3\"\n",
      "FT                   /note=\"Floral homeotic protein APETALA 3\n",
      "FT                   [Source:UniProtKB/Swiss-Prot;Acc:P35632]\"\n",
      "FT   mRNA            join(complement(1761..2175),complement(1593..1659),\n",
      "FT                   complement(1448..1509),complement(1264..1363),\n",
      "FT                   complement(1050..1091),complement(729..773),\n",
      "FT                   complement(1..483))\n",
      "FT                   /gene=\"AT3G54340\"\n",
      "FT                   /standard_name=\"AT3G54340.1\"\n",
      "FT   CDS             join(complement(1761..1948),complement(1593..1659),\n",
      "FT                   complement(1448..1509),complement(1264..1363),\n",
      "FT                   complement(1050..1091),complement(729..773),\n",
      "FT                   complement(289..483))\n",
      "FT                   /gene=\"AT3G54340\"\n",
      "FT                   /protein_id=\"AT3G54340.1\"\n",
      "FT                   /note=\"transcript_id=AT3G54340.1\"\n",
      "FT                   /db_xref=\"Uniprot/SWISSPROT:P35632\"\n",
      "FT                   /db_xref=\"RefSeq_peptide:NP_191002.1\"\n",
      "FT                   /db_xref=\"RefSeq_dna:NM_115294.6\"\n",
      "FT                   /db_xref=\"Uniprot/SPTREMBL:A0A178VHZ2\"\n",
      "FT                   /db_xref=\"TAIR_LOCUS_MODEL:AT3G54340.1\"\n",
      "FT                   /db_xref=\"EMBL:AY070397\"\n",
      "FT                   /db_xref=\"EMBL:AY087369\"\n",
      "FT                   /db_xref=\"EMBL:AY142590\"\n",
      "FT                   /db_xref=\"EMBL:M86357\"\n",
      "FT                   /db_xref=\"GO:0000977\"\n",
      "FT                   /db_xref=\"GO:0000977\"\n",
      "FT                   /db_xref=\"GO:0000977\"\n",
      "FT                   /db_xref=\"GO:0003677\"\n",
      "FT                   /db_xref=\"GO:0003677\"\n",
      "FT                   /db_xref=\"GO:0003677\"\n",
      "FT                   /db_xref=\"GO:0003700\"\n",
      "FT                   /db_xref=\"GO:0003700\"\n",
      "FT                   /db_xref=\"GO:0003700\"\n",
      "FT                   /db_xref=\"GO:0005634\"\n",
      "FT                   /db_xref=\"GO:0005634\"\n",
      "FT                   /db_xref=\"GO:0005634\"\n",
      "FT                   /db_xref=\"GO:0005634\"\n",
      "FT                   /db_xref=\"GO:0005634\"\n",
      "FT                   /db_xref=\"GO:0006351\"\n",
      "FT                   /db_xref=\"GO:0006355\"\n",
      "FT                   /db_xref=\"GO:0006355\"\n",
      "FT                   /db_xref=\"GO:0007275\"\n",
      "FT                   /db_xref=\"GO:0007275\"\n",
      "FT                   /db_xref=\"GO:0008134\"\n",
      "FT                   /db_xref=\"GO:0009908\"\n",
      "FT                   /db_xref=\"GO:0010093\"\n",
      "FT                   /db_xref=\"GO:0010093\"\n",
      "FT                   /db_xref=\"GO:0030154\"\n",
      "FT                   /db_xref=\"GO:0030154\"\n",
      "FT                   /db_xref=\"GO:0043565\"\n",
      "FT                   /db_xref=\"GO:0044212\"\n",
      "FT                   /db_xref=\"GO:0045944\"\n",
      "FT                   /db_xref=\"GO:0045944\"\n",
      "FT                   /db_xref=\"GO:0046983\"\n",
      "FT                   /db_xref=\"GO:0046983\"\n",
      "FT                   /db_xref=\"protein_id:AAD41557.1\"\n",
      "FT                   /db_xref=\"protein_id:AAD51887.1\"\n",
      "FT                   /db_xref=\"protein_id:AAD51888.1\"\n",
      "FT                   /db_xref=\"protein_id:AAD51889.1\"\n",
      "FT                   /db_xref=\"protein_id:AAD51890.1\"\n",
      "FT                   /db_xref=\"protein_id:AAD51891.1\"\n",
      "FT                   /db_xref=\"protein_id:AAD51892.1\"\n",
      "FT                   /db_xref=\"protein_id:AAD51893.1\"\n",
      "FT                   /db_xref=\"protein_id:AAD51894.1\"\n",
      "FT                   /db_xref=\"protein_id:AAD51895.1\"\n",
      "FT                   /db_xref=\"protein_id:AAD51896.1\"\n",
      "FT                   /db_xref=\"protein_id:AAD51897.1\"\n",
      "FT                   /db_xref=\"protein_id:AAD51898.1\"\n",
      "FT                   /db_xref=\"protein_id:AAD51899.1\"\n",
      "FT                   /db_xref=\"protein_id:AAD51900.1\"\n",
      "FT                   /db_xref=\"protein_id:AAD51901.1\"\n",
      "FT                   /db_xref=\"protein_id:AAD51902.1\"\n",
      "FT                   /db_xref=\"protein_id:AAD51903.1\"\n",
      "FT                   /db_xref=\"protein_id:AEE79216.1\"\n",
      "FT                   /db_xref=\"protein_id:BAA04665.1\"\n",
      "FT                   /db_xref=\"protein_id:CAB81799.1\"\n",
      "FT                   /db_xref=\"protein_id:OAP05456.1\"\n",
      "FT                   /db_xref=\"TAIR_TRANSLATION:AT3G54340\"\n",
      "FT                   /db_xref=\"PO:PO:0000037\"\n",
      "FT                   /db_xref=\"PO:PO:0000229\"\n",
      "FT                   /db_xref=\"PO:PO:0001078\"\n",
      "FT                   /db_xref=\"PO:PO:0001081\"\n",
      "FT                   /db_xref=\"PO:PO:0001185\"\n",
      "FT                   /db_xref=\"PO:PO:0004507\"\n",
      "FT                   /db_xref=\"PO:PO:0007600\"\n",
      "FT                   /db_xref=\"PO:PO:0007611\"\n",
      "FT                   /db_xref=\"PO:PO:0007616\"\n",
      "FT                   /db_xref=\"PO:PO:0009009\"\n",
      "FT                   /db_xref=\"PO:PO:0009010\"\n",
      "FT                   /db_xref=\"PO:PO:0009029\"\n",
      "FT                   /db_xref=\"PO:PO:0009030\"\n",
      "FT                   /db_xref=\"PO:PO:0009031\"\n",
      "FT                   /db_xref=\"PO:PO:0009032\"\n",
      "FT                   /db_xref=\"PO:PO:0009046\"\n",
      "FT                   /db_xref=\"PO:PO:0009052\"\n",
      "FT                   /db_xref=\"PO:PO:0025022\"\n",
      "FT                   /db_xref=\"PO:PO:0025281\"\n",
      "FT                   /db_xref=\"STRING:3702.AT3G54340.1\"\n",
      "FT                   /db_xref=\"UniParc:UPI0000001796\"\n",
      "FT                   /translation=\"MARGKIQIKRIENQTNRQVTYSKRRNGLFKKAHELTVLCDARVSI\n",
      "FT                   IMFSSSNKLHEYISPNTTTKEIVDLYQTISDVDVWATQYERMQETKRKLLETNRNLRTQ\n",
      "FT                   IKQRLGECLDELDIQELRRLEDEMENTFKLVRERKFKSLGNQIETTKKKNKSQQDIQKN\n",
      "FT                   LIHELELRAEDPHYGLVDNGGDYDSVLGYQIEGSRAYALRFHQNHHHYYPNHGLHAPSA\n",
      "FT                   SDIITFHLLE\"\n",
      "FT   exon            complement(1050..1091)\n",
      "FT                   /note=\"exon_id=AT3G54340.1.exon5\"\n",
      "FT   exon            complement(1..483)\n",
      "FT                   /note=\"exon_id=AT3G54340.1.exon7\"\n",
      "FT   exon            complement(729..773)\n",
      "FT                   /note=\"exon_id=AT3G54340.1.exon6\"\n",
      "FT   exon            complement(1448..1509)\n",
      "FT                   /note=\"exon_id=AT3G54340.1.exon3\"\n",
      "FT   exon            complement(1761..2175)\n",
      "FT                   /note=\"exon_id=AT3G54340.1.exon1\"\n",
      "FT   exon            complement(1593..1659)\n",
      "FT                   /note=\"exon_id=AT3G54340.1.exon2\"\n",
      "FT   exon            complement(1264..1363)\n",
      "FT                   /note=\"exon_id=AT3G54340.1.exon4\"\n",
      "FT   mRNA            join(complement(AL132971.2:19369..19757),\n",
      "FT                   complement(AL132971.2:19047..19281),\n",
      "FT                   complement(AL132971.2:18782..18927),\n",
      "FT                   complement(AL132971.2:18060..18147),\n",
      "FT                   complement(AL132971.2:17832..17978),\n",
      "FT                   complement(AL132971.2:16910..17059),\n",
      "FT                   complement(AL132971.2:14556..14644),\n",
      "FT                   complement(AL132971.2:14227..14290),\n",
      "FT                   complement(AL132971.2:13793..13853),\n",
      "FT                   complement(AL132971.2:13177..13184),\n",
      "FT                   complement(AL132971.2:11590..13131),\n",
      "FT                   complement(AL132971.2:11453..11504),\n",
      "FT                   complement(AL132971.2:11074..11222),\n",
      "FT                   complement(AL132971.2:10884..10970),\n",
      "FT                   complement(AL132971.2:10633..10761),complement(1448..1509),\n",
      "FT                   complement(1264..1363),complement(1050..1091),\n",
      "FT                   complement(729..773),complement(289..483))\n",
      "FT                   /product=\"MTTTSVCPFSKAARPDDGSTRKQGEITASGCPFSKAARPDDASARKQGE\n",
      "FT                   TTASGCPFSKSARPDENGSKGCPEQEGNLNKDSTDSATVPAKCPFGYDSQTFKLGPFSC\n",
      "FT                   MLCQALLYESSRCVPCTHVFCKVCLTRFKDCPLCGADIESIEVDENLQKMVDQFIEGHA\n",
      "FT                   RIKRSVVNGTEKEEVENDNKKVIYADVSMERGSFLVQQAMRAFSAQNYESAKSRLAMCT\n",
      "FT                   EDIRDQLGREGNTPELCSQLGAVLGMLGDCSRAMGDSSSAVKHFEESVEFLMKLPLNDL\n",
      "FT                   EITHTLSVSLNKIGDLKYYDEDLQAARSYYDRALNVRRDAMKHHPNAPSQILDVAVSLA\n",
      "FT                   KVADIDRTLQNEVAATDGFKEGMRLLESLKLDSEDSALEQRFCDMVTVARLLNLTLVVP\n",
      "FT                   EIDKTSFFADSRFCVLSPSLRPLLRYAFLREVKLLVSIVDFEGIGEYDISFEAACLAGA\n",
      "FT                   SLESLAKGAVQFSRRFSIRELQDRWHALLYDPVVSVEAAFRMAELERTNPNFPTKFGRT\n",
      "FT                   GYSKENKSSSRKRNAERLRSTYHSLRKKFRTEPFNSLDLGFLVPPNDSHFMDNGDATHL\n",
      "FT                   GLEDSHMDIIHNAFPEILAEGGCVTTHVLPEDNLQGDIPYVEGENLTFTEHAGLSVCDV\n",
      "FT                   VHQDSEQKLENTAHEAKNTMASTDFLAQLSTSLFEEDMEPFMEVDGKEVDKSYYDGLSS\n",
      "FT                   LLVNSTNDTNREAFPNPTEQEPSIAPTHPGEATLDDHVMLELDGTIALDPHPEIVGGVI\n",
      "FT                   CCLLNEEDPDIPCNDDIFLSNNSRPMSVSSLARRNFKDTNSPITTCVRDVSASKEKSEG\n",
      "FT                   YSLQAQKKKPGRLQGSTQGKPEMGQPSKGSKFRASTSTELKNTVAPGGSSSAQACSNTL\n",
      "FT                   LSTGTGAKDGKKETATGTLFVGSDGHGNHPEKDSENCKEKNVVPPVNESPHAKDTDDGL\n",
      "FT                   IEITVPELEITRAEAEAEAEAHVCESDEDLPNYSDIEAMILDMDLEPDDQDNFDLEVSK\n",
      "FT                   YQSQDMKRTIIRLEQAAHSYMQRAIASRGAFAVLYGRYSKHYIKKPEVLVGRSTEDLAV\n",
      "FT                   DIDLGREKRGSKISRRQAIIRLGDDGSFHIKNLGKYSISVNEKEVDPGQSLILKSDCLV\n",
      "FT                   ERMQETKRKLLETNRNLRTQIKQRLGECLDELDIQELRRLEDEMENTFKLVRERKFKSL\n",
      "FT                   GNQIETTKKKNKSQQDIQKNLIHELELRAEDPHYGLVDNGGDYDSVLGYQIEGSRAYAL\n",
      "FT                   RFHQNHHHYYPNHGLHAPSASDIITFHLLE\"\n",
      "FT                   /note=\"identifier=.1\"\n",
      "FT                   /note=\"Derived by automated computational analysis using\n",
      "FT                   gene prediction method:fgenesh\"\n",
      "FT   variation       18..18\n",
      "FT                   /replace=\"C/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH06313434\"\n",
      "FT   variation       22..22\n",
      "FT                   /replace=\"T/A\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH02503635\"\n",
      "FT   variation       23..23\n",
      "FT                   /replace=\"A/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20119162_A_G\"\n",
      "FT   variation       38..38\n",
      "FT                   /replace=\"G/A\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH06313435\"\n",
      "FT   variation       40..40\n",
      "FT                   /replace=\"A/-\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH15197512\"\n",
      "FT   variation       52..52\n",
      "FT                   /replace=\"T/A\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20119191_T_A\"\n",
      "FT   variation       68..68\n",
      "FT                   /replace=\"T/C\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH02503636\"\n",
      "FT   variation       73..73\n",
      "FT                   /replace=\"A/C\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH00418964\"\n",
      "FT   variation       89..89\n",
      "FT                   /replace=\"T/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20119228_T_G\"\n",
      "FT   variation       102..102\n",
      "FT                   /replace=\"G/A\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH14432852\"\n",
      "FT   variation       145..145\n",
      "FT                   /replace=\"C/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20119284_C_G\"\n",
      "FT   variation       179..179\n",
      "FT                   /replace=\"C/A\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20119318_C_A\"\n",
      "FT   variation       185..185\n",
      "FT                   /replace=\"A/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH06313438\"\n",
      "FT   variation       189..189\n",
      "FT                   /replace=\"T/C\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH02503637\"\n",
      "FT   variation       195..194\n",
      "FT                   /replace=\"-/T\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20119333_G_GT\"\n",
      "FT   variation       210..210\n",
      "FT                   /replace=\"A/C\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH06313439\"\n",
      "FT   variation       225..224\n",
      "FT                   /replace=\"-/T\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20119363_G_GT\"\n",
      "FT   variation       249..249\n",
      "FT                   /replace=\"G/A\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20119388_G_A\"\n",
      "FT   variation       264..264\n",
      "FT                   /replace=\"C/T\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20119403_C_T\"\n",
      "FT   variation       333..333\n",
      "FT                   /replace=\"C/T\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH00418965\"\n",
      "FT   variation       352..352\n",
      "FT                   /replace=\"G/A\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH02503638\"\n",
      "FT   variation       357..357\n",
      "FT                   /replace=\"A/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH02503639\"\n",
      "FT   variation       380..380\n",
      "FT                   /replace=\"C/T\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH02503640\"\n",
      "FT   variation       385..385\n",
      "FT                   /replace=\"A/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH12755904\"\n",
      "FT   variation       400..400\n",
      "FT                   /replace=\"C/A\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20119539_C_A\"\n",
      "FT   variation       454..454\n",
      "FT                   /replace=\"T/A\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20119593_T_A\"\n",
      "FT   variation       455..455\n",
      "FT                   /replace=\"C/T\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20119594_C_T\"\n",
      "FT   variation       502..501\n",
      "FT                   /replace=\"-/T\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20119640_G_GT\"\n",
      "FT   variation       537..537\n",
      "FT                   /replace=\"T/A\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20119676_T_A\"\n",
      "FT   variation       590..590\n",
      "FT                   /replace=\"C/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH12755925\"\n",
      "FT   variation       603..603\n",
      "FT                   /replace=\"A/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH06313441\"\n",
      "FT   variation       624..624\n",
      "FT                   /replace=\"T/C\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH00418966\"\n",
      "FT   variation       627..627\n",
      "FT                   /replace=\"T/C\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20119766_T_C\"\n",
      "FT   variation       654..654\n",
      "FT                   /replace=\"T/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20119793_T_G\"\n",
      "FT   variation       700..700\n",
      "FT                   /replace=\"C/A\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH06313442\"\n",
      "FT   variation       708..708\n",
      "FT                   /replace=\"A/-\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20119846_GA_G\"\n",
      "FT   variation       760..760\n",
      "FT                   /replace=\"T/A\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH12755926\"\n",
      "FT   variation       786..786\n",
      "FT                   /replace=\"A/C\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH02503641\"\n",
      "FT   variation       792..792\n",
      "FT                   /replace=\"C/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH14432854\"\n",
      "FT   variation       802..802\n",
      "FT                   /replace=\"G/T\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH02503642\"\n",
      "FT   variation       819..819\n",
      "FT                   /replace=\"T/A\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH02503643\"\n",
      "FT   variation       820..820\n",
      "FT                   /replace=\"C/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH12755929\"\n",
      "FT   variation       822..822\n",
      "FT                   /replace=\"C/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20119961_C_G\"\n",
      "FT   variation       825..825\n",
      "FT                   /replace=\"A/C\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20119964_A_C\"\n",
      "FT   variation       845..844\n",
      "FT                   /replace=\"-/C\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH04251560\"\n",
      "FT   variation       848..847\n",
      "FT                   /replace=\"-/A\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20119986_C_CA\"\n",
      "FT   variation       857..857\n",
      "FT                   /replace=\"T/A\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20119996_T_A\"\n",
      "FT   variation       867..867\n",
      "FT                   /replace=\"A/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20120006_A_G\"\n",
      "FT   variation       876..876\n",
      "FT                   /replace=\"A/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH12755932\"\n",
      "FT   variation       884..885\n",
      "FT                   /replace=\"TT/-\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH15365420\"\n",
      "FT   variation       906..909\n",
      "FT                   /replace=\"ACAC/AC/ACACAC/-\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH15365421\"\n",
      "FT   variation       915..915\n",
      "FT                   /replace=\"C/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20120054_C_G\"\n",
      "FT   variation       926..926\n",
      "FT                   /replace=\"A/C\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20120065_A_C\"\n",
      "FT   variation       980..980\n",
      "FT                   /replace=\"A/T\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH06313444\"\n",
      "FT   variation       984..984\n",
      "FT                   /replace=\"A/T\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH00418967\"\n",
      "FT   variation       1063..1063\n",
      "FT                   /replace=\"G/T\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH00418968\"\n",
      "FT   variation       1119..1119\n",
      "FT                   /replace=\"A/C\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20120258_A_C\"\n",
      "FT   variation       1140..1140\n",
      "FT                   /replace=\"T/C\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH14432875\"\n",
      "FT   variation       1144..1144\n",
      "FT                   /replace=\"T/C\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20120283_T_C\"\n",
      "FT   variation       1162..1162\n",
      "FT                   /replace=\"A/T\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH06313446\"\n",
      "FT   variation       1174..1174\n",
      "FT                   /replace=\"A/C\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH02503644\"\n",
      "FT   variation       1183..1183\n",
      "FT                   /replace=\"T/A\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH02503645\"\n",
      "FT   variation       1185..1185\n",
      "FT                   /replace=\"T/C\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH02503646\"\n",
      "FT   variation       1186..1186\n",
      "FT                   /replace=\"A/T\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH02503647\"\n",
      "FT   variation       1188..1193\n",
      "FT                   /replace=\"TAATTC/GAATTC/T\"\n",
      "FT                   /db_xref=\"The 1001 Genomes\n",
      "FT                   Project:tmp_3_20120327_TAATTC_GAATTC,T\"\n",
      "FT   variation       1198..1198\n",
      "FT                   /replace=\"C/A\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH12755933\"\n",
      "FT   variation       1209..1209\n",
      "FT                   /replace=\"C/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20120348_C_G\"\n",
      "FT   variation       1223..1223\n",
      "FT                   /replace=\"A/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20120362_A_G\"\n",
      "FT   variation       1239..1239\n",
      "FT                   /replace=\"G/-\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20120377_AG_A\"\n",
      "FT   variation       1275..1275\n",
      "FT                   /replace=\"G/T\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH12755934\"\n",
      "FT   variation       1276..1276\n",
      "FT                   /replace=\"A/C\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH02503648\"\n",
      "FT   variation       1289..1289\n",
      "FT                   /replace=\"G/T\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20120428_G_T\"\n",
      "FT   variation       1318..1318\n",
      "FT                   /replace=\"C/T\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20120457_C_T\"\n",
      "FT   variation       1320..1320\n",
      "FT                   /replace=\"G/A\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH12755935\"\n",
      "FT   variation       1323..1323\n",
      "FT                   /replace=\"C/T\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20120462_C_T\"\n",
      "FT   variation       1329..1329\n",
      "FT                   /replace=\"T/C\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH06313448\"\n",
      "FT   variation       1338..1338\n",
      "FT                   /replace=\"C/T\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH02503649\"\n",
      "FT   variation       1363..1363\n",
      "FT                   /replace=\"C/T\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH14432876\"\n",
      "FT   variation       1374..1374\n",
      "FT                   /replace=\"A/T\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH12755936\"\n",
      "FT   variation       1382..1382\n",
      "FT                   /replace=\"G/C/A\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH12755937\"\n",
      "FT   variation       1390..1390\n",
      "FT                   /replace=\"T/C\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH14432877\"\n",
      "FT   variation       1395..1395\n",
      "FT                   /replace=\"C/T\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20120534_C_T\"\n",
      "FT   variation       1408..1408\n",
      "FT                   /replace=\"G/A\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH06313449\"\n",
      "FT   variation       1441..1441\n",
      "FT                   /replace=\"C/A/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH06313450\"\n",
      "FT   variation       1450..1450\n",
      "FT                   /replace=\"G/T\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20120589_G_T\"\n",
      "FT   variation       1453..1453\n",
      "FT                   /replace=\"C/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH12755938\"\n",
      "FT   variation       1571..1571\n",
      "FT                   /replace=\"A/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20120710_A_G\"\n",
      "FT   variation       1585..1585\n",
      "FT                   /replace=\"A/-\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20120723_GA_G\"\n",
      "FT   variation       1586..1586\n",
      "FT                   /replace=\"A/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH06313451\"\n",
      "FT   variation       1628..1628\n",
      "FT                   /replace=\"T/C\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH12755939\"\n",
      "FT   variation       1647..1647\n",
      "FT                   /replace=\"G/A\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH12755940\"\n",
      "FT   variation       1670..1670\n",
      "FT                   /replace=\"C/A\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH02503650\"\n",
      "FT   variation       1675..1675\n",
      "FT                   /replace=\"T/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20120814_T_G\"\n",
      "FT   variation       1685..1685\n",
      "FT                   /replace=\"A/T\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH14432878\"\n",
      "FT   variation       1704..1704\n",
      "FT                   /replace=\"A/-\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20120842_GA_G\"\n",
      "FT   variation       1712..1712\n",
      "FT                   /replace=\"G/C\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20120851_G_C\"\n",
      "FT   variation       1735..1735\n",
      "FT                   /replace=\"G/A\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20120874_G_A\"\n",
      "FT   variation       1747..1747\n",
      "FT                   /replace=\"G/T\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20120886_G_T\"\n",
      "FT   variation       1811..1811\n",
      "FT                   /replace=\"G/A\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH02503651\"\n",
      "FT   variation       1868..1868\n",
      "FT                   /replace=\"A/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20121007_A_G\"\n",
      "FT   variation       1874..1874\n",
      "FT                   /replace=\"T/C\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH02503652\"\n",
      "FT   variation       1889..1889\n",
      "FT                   /replace=\"C/T\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20121028_C_T\"\n",
      "FT   variation       1916..1916\n",
      "FT                   /replace=\"T/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20121055_T_G\"\n",
      "FT   variation       1954..1954\n",
      "FT                   /replace=\"T/C\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH14432879\"\n",
      "FT   variation       1980..1980\n",
      "FT                   /replace=\"A/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20121119_A_G\"\n",
      "FT   variation       1987..1987\n",
      "FT                   /replace=\"T/C\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20121126_T_C\"\n",
      "FT   variation       2000..2000\n",
      "FT                   /replace=\"C/A\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH12755941\"\n",
      "FT   variation       2012..2012\n",
      "FT                   /replace=\"T/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20121151_T_G\"\n",
      "FT   variation       2016..2016\n",
      "FT                   /replace=\"A/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20121155_A_G\"\n",
      "FT   variation       2131..2131\n",
      "FT                   /replace=\"G/C\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH14432880\"\n",
      "FT   variation       2138..2138\n",
      "FT                   /replace=\"A/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH02503653\"\n",
      "FT   variation       2145..2144\n",
      "FT                   /replace=\"-/A\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:tmp_3_20121283_T_TA\"\n",
      "FT   variation       2165..2165\n",
      "FT                   /replace=\"C/G/T\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH02503654\"\n",
      "FT   variation       2168..2168\n",
      "FT                   /replace=\"A/G\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH02503655\"\n",
      "FT   variation       2169..2169\n",
      "FT                   /replace=\"T/A\"\n",
      "FT                   /db_xref=\"The 1001 Genomes Project:ENSVATH06313453\"\n",
      "FT   repeat_region   complement(823..829)\n",
      "FT                   /note=\"dust repeat: matches 20119962..20119968(1) of\n",
      "FT                   consensus\"\n",
      "FT   repeat_region   complement(906..924)\n",
      "FT                   /note=\"dust repeat: matches 20120045..20120063(1) of\n",
      "FT                   consensus\"\n",
      "FT   repeat_region   complement(1399..1406)\n",
      "FT                   /note=\"dust repeat: matches 20120538..20120545(1) of\n",
      "FT                   consensus\"\n",
      "FT   repeat_region   complement(1558..1591)\n",
      "FT                   /note=\"dust repeat: matches 20120697..20120730(1) of\n",
      "FT                   consensus\"\n",
      "FT   repeat_region   complement(1979..2063)\n",
      "FT                   /note=\"dust repeat: matches 20121118..20121202(1) of\n",
      "FT                   consensus\"\n",
      "FT   repeat_region   complement(2169..2175)\n",
      "FT                   /note=\"dust repeat: matches 20121308..20121314(1) of\n",
      "FT                   consensus\"\n",
      "FT   misc_feature    1..2175\n",
      "FT                   /note=\"contig AL132971.2 7316..9490(1)\"\n",
      "XX\n",
      "SQ   Sequence       2175 BP;        774 A;        367 C;        377 G;        657 T;          0 other;\n",
      "     GACAATATAT ATGCATTCAA GTAGAAAATC AAATCGAGTA AATTTGTAGC ATAATTAAGT        60\n",
      "     AGAGTGGTCA ATATAATAAT ACGATACATG ATAAAAGTCT TGAAAACATT CCACACACAA       120\n",
      "     TAAGCCTAAG CACTACTGAT CACACAATCC ATATTTCTTT AGGCTTTAAA ATACATAACT       180\n",
      "     AAGAAACATG ACAGTTTAAG GTTTTAGCAA CACCATGCCT TATGTTTTAA GTAGTTACTA       240\n",
      "     AATAATTAGA TAGACAATGA TGGCACCAGC AAACCTTTTA GCCTTTAATT ATTCAAGAAG       300\n",
      "     ATGGAAGGTA ATGATGTCAG AGGCAGAGGG TGCATGAAGG CCATGGTTGG GGTAATAGTG       360\n",
      "     GTGATGGTTC TGGTGGAAAC GAAGAGCGTA AGCACGTGAC CCTTCGATTT GGTATCCAAG       420\n",
      "     AACTGAGTCG TAATCTCCTC CATTGTCTAC TAGTCCATAG TGAGGATCTT CAGCTCTTAG       480\n",
      "     TTCCTAATTC ACCAAAACAT GTTTTAAGCC ATATTCCACA TAACTAAAAC CAATATTATT       540\n",
      "     AATTCGAAAT GACATGACAT TTGATATATG ATTTAACTAG GAGATATTTC GATAAGAGAA       600\n",
      "     TCACAAGAAA ACATAAGAGT GACTGATCAA TCTACACTAC ATTTCTAGAT ATCTTTTTAA       660\n",
      "     CTGGCTCCGT ATTCCATGAA TGTATTATGC CTAAACCAAC ATTTGAGAAA AACAGAAAGA       720\n",
      "     GATATTACCA GCTCATGTAT GAGATTCTTT TGTATGTCTT GTTGACTTTT GTTCTGCAAT       780\n",
      "     TGTGTAAACC ACAAGAGAAA TGATGAAGAA ATATTCCCTC CCAAAAAAAT TATGAAGATA       840\n",
      "     TATACCCAAG TTATTATGAC ATCACTATAT AGTATACTTG GTATTTTCTT TATCCATTAG       900\n",
      "     CTTAAACACA CACACACACA CACAAATATA ATTGTCCAAG AAAAGTGTTA TCAACAGTTT       960\n",
      "     TAGTAAGTTG ATAAATTCAA TTGACACAAA ATTGTTGAGA CAAATAAATA TGATAAAATG      1020\n",
      "     GAAAAGTGAG TGGACATAGA AGATGTGACC TTTTTCTTGG TGGTCTCGAT CTGATTCCCA      1080\n",
      "     AGAGATTTGA ACTGTACAGT ACAATACAAT TAACATATAT TATCCAACAG TAAACAAAAT      1140\n",
      "     GTTTCTTTTT CTAAAACATA TATACAACAA TAAAATTCAT TTTTTAATAA TTCAATTCAA      1200\n",
      "     AACTGAAGCT TGGGTATATA GGATTAATTG AAACCCTAGG TAAGGGAAAA GTATGAAGAG      1260\n",
      "     AACCTTGCGC TCGCGAACGA GTTTGAAAGT GTTTTCCATT TCATCCTCAA GACGACGCAG      1320\n",
      "     CTCCTGAATG TCAAGCTCGT CCAAACACTC ACCTAGCCTC TGCCTACAAT TTTAAAAATA      1380\n",
      "     TGTATATTTT CGTACTCATT TTTTTTAGTA AAACTCCTCA TCTAAGAGAA GAGAGAGAAA      1440\n",
      "     CAAATACTTG ATCTGAGTCC GGAGATTTCT ATTTGTCTCC AACAGTTTCC TCTTGGTTTC      1500\n",
      "     TTGCATTCGC TGCAGCAAAA CGCATCAAGA ATTTAACCAA CCAGCGAAAC ACATATCAAG      1560\n",
      "     AAGAAGAAGA AAGATCTAAG AAGGAAAAAA ACCTCATATT GAGTGGCCCA AACATCGACA      1620\n",
      "     TCAGAAATAG TTTGGTACAG ATCTACGATC TCCTTCGTTC TATAACCAAC AAAATCACCA      1680\n",
      "     AAAAAGTAGT GGTTAATTAT AAGAAAACAA AGCAACTCAA TTAGCTTAAT TTAAGAGTGG      1740\n",
      "     TGTTTAGAGA GATGGTGTAC GTGGTGTTAG GGCTGATATA CTCATGAAGC TTGTTGGAGC      1800\n",
      "     TAGAGAACAT GATAATCGAA ACCCTAGCAT CACACAAAAC CGTGAGCTCA TGTGCTTTCT      1860\n",
      "     TGAATAAACC ATTTCTTCTC TTTGAATACG TCACTTGTCT GTTTGTCTGG TTCTCTATCC      1920\n",
      "     TCTTGATCTG GATCTTCCCT CTCGCCATAT TCTTCTCTCT TTGTTTAATC TTTTTGTTGA      1980\n",
      "     AGAGATTTGG TGGAGAGGAC AAGAGATATA ATGAGAAGAA AGAAGTGAAA TAGAAAGAGA      2040\n",
      "     GAGATGGAGT TGAAGAAGTA AAGGGTCCAC TTGAGTTACT AAAAATGGAA AGTATTGCCT      2100\n",
      "     AATCCATGAA AGGTAAGTTC AGAAGTTGAT GAAAACTAAG TGATAAATTT TCCATTTATG      2160\n",
      "     TAAACTGATT TTTTT                                                       2175\n",
      "//\n",
      "\n",
      "\n",
      "\n",
      "\n"
     ]
    }
   ],
   "source": [
    "require 'rest-client'   # this is how you access the Web\n",
    "\n",
    "\n",
    "def fetch(url, headers = {accept: \"*/*\"}, user = \"\", pass=\"\")\n",
    "  response = RestClient::Request.execute({\n",
    "    method: :get,\n",
    "    url: url.to_s,\n",
    "    user: user,\n",
    "    password: pass,\n",
    "    headers: headers})\n",
    "  return response\n",
    "  \n",
    "  rescue RestClient::ExceptionWithResponse => e\n",
    "    $stderr.puts e.response\n",
    "    response = false\n",
    "    return response  # now we are returning 'False', and we will check that with an \\\"if\\\" statement in our main code\n",
    "  rescue RestClient::Exception => e\n",
    "    $stderr.puts e.response\n",
    "    response = false\n",
    "    return response  # now we are returning 'False', and we will check that with an \\\"if\\\" statement in our main code\n",
    "  rescue Exception => e\n",
    "    $stderr.puts e\n",
    "    response = false\n",
    "    return response  # now we are returning 'False', and we will check that with an \\\"if\\\" statement in our main code\n",
    "end \n",
    "\n",
    "\n",
    "\n",
    "    \n",
    "\n",
    "    \n",
    "res = fetch('http://www.ebi.ac.uk/Tools/dbfetch/dbfetch?db=ensemblgenomesgene&format=embl&id=At3g54340');\n",
    "#puts res.class\n",
    "#puts res.public_methods\n",
    "  \n",
    "if res  # res is either the response object (Class Net::HTTPSuccess), or False, so you can test it with 'if'\n",
    "  body = res.body  # get the \"body\" of the response\n",
    "  puts body\n",
    "end\n",
    "  \n",
    "  \n",
    "  "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<pre>\n",
    "\n",
    "</pre>\n",
    "# Extracting data from the Web - \"screen scraping\"!\n",
    "\n",
    "You can now use tools that you already know - like regular expressions - to extract specific pieces of data from this Web response.  For example, the gene name:\n",
    "\n",
    "(note that this code contains more error handling.  It checks if the returned content of the web page contains a match to your regular expression, and it tries to \"rescue\" the situation if it fails to find a match.)\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "the name of the gene is AP3\n"
     ]
    }
   ],
   "source": [
    "    \n",
    "res = fetch('http://www.ebi.ac.uk/Tools/dbfetch/dbfetch?db=ensemblgenomesgene&format=embl&id=At3g54340');\n",
    "#puts res.class\n",
    "#puts res.public_methods\n",
    "  \n",
    "if res  # res is either the response object, or False, so you can test it with 'if'\n",
    "  body = res.body  # get the \"body\" of the response\n",
    "  \n",
    "  if body =~ /locus_tag=\"([^\"]+)\"/ \n",
    "    gene_name = $1\n",
    "    puts \"the name of the gene is #{gene_name}\"\n",
    "  #puts body\n",
    "  #gene_name_regexp = Regexp.new(/locus_tag=\"([^\"]+)\"/)  # this is one way to do Regular Expressions in Ruby.  There are several!\n",
    "  #match = gene_name_regexp.match(body)\n",
    "  #if match\n",
    "  #  gene_name = match[1]  # matches act like an array, so the first match is [1]\n",
    "                          # (try for yourself, what is match[0]?)\n",
    "  #  puts \"the name of the gene is #{gene_name}\"\n",
    "  else\n",
    "    begin  # use a \"begin\" block to handle errors\n",
    "      puts \"There was no gene name in this record\"  # print a friendly message\n",
    "      raise \"this is an error\" # raise an exception\n",
    "    rescue # some code to rescue the situation... for example, maybe a different regexp?\n",
    "      puts \"exiting gracefully\"  # in this case, we are just going to stop trying\n",
    "    end\n",
    "  end\n",
    "end\n",
    "  "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "\n",
    "<pre>\n",
    "\n",
    "\n",
    "\n",
    "\n",
    "</pre>\n",
    "# Prove that you understand:\n",
    "\n",
    "### to do this, you should switch to Komodo and write the code as an independent file that uses the objects you created for your assignment - don't try to do this in Jupyter!\n",
    "\n",
    " 1.  (easy) read the gene_information.tsv file from your assignment; retrieve the EMBL record for each gene in that file  (please retrieve them one-at-a-time, not all together)\n",
    "\n",
    " 2.  (medium) Create an “AnnotatedGene” Class, that extends the Gene Class by adding the DNA sequence and the Protein sequence attributes (as Strings).\n",
    "\n",
    " 3.  (hard) For every EMBL file retrieved from gene_information.tsv, find the cross-referenced UniProt identifier (needs a regular expression). \n",
    "\n",
    "    * Retrieve the UniProt Protein record as FASTA using dbfetch.  \n",
    "    * Retrieve the DNA (FASTA) sequence using dbfetch.   \n",
    "    * Create an AnnotatedGene object for each one.  \n",
    "    * Loop over every AnnotatedGene object and create a report:\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<pre><code>\n",
    "AGI_Locus        At3GXXXXX\n",
    "GeneName            Ap3\n",
    "Protein_ID        P11234\n",
    "DNA Sequence\n",
    ">Cosa|Cosa|At3GXXXXX    (keep the original FASTA header)\n",
    "ACTGCTAGCTGATGCTGATGCATGCTAGC\n",
    "Protein Sequence\n",
    ">Cosa|Cosa|P111234  (keep the original FASTA header)\n",
    "MYLLMSSPIOO\n",
    "</code></pre>\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Prove that you understand by using a different Web resource\n",
    "\n",
    "__The Gene Ontology Consortium provides two kinds of files:__\n",
    "* Ontology Files (Terms, Term ID, and Descriptions)\n",
    "* Associations (The GO terms used to annotate every gene)\n",
    "\n",
    " 4. (easy) In your code, retrieve the GO Slim Plant Ontology:\n",
    "http://www.geneontology.org/ontology/subsets/goslim_plant.obo\n",
    "\n",
    " 5. (~easy) In your code, parse that file, and for a GO identifier (e.g. GO:0006950) print the GO Term name to the screen (e.g. “response to stress”) (there are MANY different solutions for this!  All of them are regular expressions...)\n",
    "\n",
    " 6. (hard) Add “GO_Annotation” attribute to your AnnotatedGene Class (array of strings), then:\n",
    " \n",
    "For every gene in the gene_information.tsv file, \n",
    "* Retrieve the UniProt Record\n",
    "* Retrieve the GO annotations (GO_NNNNNN) from the UniProt record\n",
    "* Retreive the GO Term name from UniProt, and Term definition from the goslim_plant.obo file\n",
    "* Add those annotations to your AnnotatedGene object (think about this…..)\n",
    "* create a report like:\n",
    "\n",
    "\n",
    "<code>Gene_ID: \n",
    "Term1 (def)\n",
    "Term2 (def)\n",
    "Term3 (def)\n",
    "</code>"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<pre>\n",
    "  \n",
    "\n",
    "</pre>\n",
    "\n",
    "  ## These were examples of difficult cases\n",
    "  \n",
    "If you have to \"screen scrape\", you are facing a very difficult situation.  Your code will often break (when the data provider changes their Web page, for example).  You also probably have not seen EVERY possible case, so your RegularExpression will probably miss some data sometimes.  \n",
    "\n",
    "If you can find a BETTER way to get the data, you should do that!  In the next lesson we will learn about REST and about other data formats like JSON and XML, that make it easier to extract the data.\n"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Ruby 2.6.3",
   "language": "ruby",
   "name": "ruby"
  },
  "language_info": {
   "file_extension": ".rb",
   "mimetype": "application/x-ruby",
   "name": "ruby",
   "version": "2.6.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
