{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Sequence Alignments \n",
    "\n",
    "It is common to want to align sequences.  BioRuby can do this for you in simple cases:\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "require 'bio'\n",
    "\n",
    "seqs = [ 'atgca', 'aagca', 'acgca', 'acgcg' ]\n",
    "seqs = seqs.collect{ |x| Bio::Sequence::NA.new(x) }   # look up the documentation for .collect.  How is it different from .each?\n",
    "# creates alignment object\n",
    "a = Bio::Alignment.new(seqs)\n",
    "puts a.consensus \n",
    "puts a.consensus_iupac"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<pre>\n",
    "  \n",
    "</pre>\n",
    "Try adding a few nucleotides to one of the sequences.  What happens in the alignment?  Add the same nucleotides to the other sequences, one-by-one... what happens in the alignment?   What do you learn from this?\n",
    "\n",
    "\n",
    "### more complicated alignments, e.g. with gaps, requires an alignment algorithm.\n",
    "\n",
    "A common way to do alignments is ClustalW.  This is installed on your virtual machine.  \n",
    "\n",
    "To access ClustalW from BioRuby's Alignment object, you do this:\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "require 'bio'\n",
    "\n",
    "seqs = [ 'ctggtatt', 'aagcatt', 'acgcagtt', 'acgcgtt' ]\n",
    "seqs = seqs.collect{ |x| Bio::Sequence::NA.new(x) }   # look up the documentation for .collect.  How is it different from .each?\n",
    "# creates alignment object\n",
    "a = Bio::Alignment.new(seqs)\n",
    "puts \"BioRuby simple alignment\"\n",
    "puts a.consensus_string\n",
    "puts a.consensus_iupac\n",
    "puts\n",
    "puts \"ClustalW alignment\"\n",
    "factory = Bio::ClustalW.new\n",
    "a2 = a.do_align(factory)\n",
    "puts a2.consensus_string\n",
    "puts a2.consensus_iupac\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Sequence Searching\n",
    "\n",
    "We will look at sequence searches using FASTA and BLAST.  Both of these have been installed for you.\n",
    "\n",
    "See docs at http://bioruby.org/rdoc/Bio/Fasta and ....\n",
    "\n",
    "I have downloaded 160 Ebola virus sequences for you in the \"Sequences\" folder of your home directory.  You need to concatenate all of the single-sequence FASTA files into a single FASTA file, which is used as the database for a FASTA search:\n",
    "\n",
    "    $ cat *.fa  >  EBOLA.fa\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "require 'bio'\n",
    "require 'stringio'\n",
    "\n",
    "#puts Dir.pwd\n",
    "\n",
    "\n",
    "factory = Bio::Fasta.local('/home/osboxes/Anaconda/bin/fasta/fasta36', '../Databases/EBOLA.fa') \n",
    "\n",
    "# you haven't seen this before!  The second argument must be a stream (respond to \".read\", \n",
    "# so make the string look like an opened file\n",
    "find = StringIO.new(\">myseq\\nAGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\")\n",
    "\n",
    "ff = Bio::FlatFile.auto(find)   # 'find' is a streamable object\n",
    "\n",
    "# Iterates over each entry. the variable \"entry\" is a \n",
    "# Bio::FastaFormat object:\n",
    "ff.each do |entry|\n",
    "  \n",
    "  $stderr.puts \"Searching ... \" + entry.definition # output to \"standard error\", this is to output comments to yourself, or error messages\n",
    "                                                   # that are not intended to be part of the output of the software\n",
    "\n",
    "\n",
    "  # executes homology search. Returns Bio::Fasta::Report object.\n",
    "  report = factory.query(entry)\n",
    "\n",
    "  #puts factory.output\n",
    "  #puts report.hits\n",
    "  # Iterates over each hit\n",
    "  report.each do |hit|\n",
    "      print \"#{hit.query_id} : evalue #{hit.evalue}\\t#{hit.target_id} at \"\n",
    "      p hit.lap_at\n",
    "  end\n",
    "end"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Prove you understand\n",
    "modify the script above so that it only prints hits with evalue less than 10e-3\n",
    "<pre>\n",
    "\n",
    "\n",
    "</pre>\n",
    "\n",
    "# BLAST searches  \n",
    "\n",
    "## (I have done this installation for you! This is just for your information)\n",
    "\n",
    "to install Blast for yourself:  \n",
    "    \n",
    "    $  sudo apt-get install ncbi-blast+\n",
    "    \n",
    "    $  sudo apt-get install ncbi-blast+-legacy   (this is required for some versions of BioRuby!)\n",
    "    \n",
    "### Creating custom BLAST databases\n",
    "    \n",
    "Blast requires:\n",
    "\n",
    "    A sequence to be used as the query\n",
    "\n",
    "    A BLAST-formatted database\n",
    "\n",
    "Common BLAST databases can be downloaded from NCBI:\n",
    "ftp://ftp.ncbi.nlm.nih.gov/blast/db/ \n",
    "\n",
    "We will create our own small database for this class using the EBOLA data.\n",
    "\n",
    "First, create a new folder \"Databases\" in your home directory, and **copy** `Lectures/Databases/EBOLA.fa`  into that location.\n",
    "\n",
    "Now, go to that Databases folder and execute this command:\n",
    "\n",
    "    $  makeblastdb -in EBOLA.fa -dbtype 'nucl' -out EBOLA\n",
    "\n",
    "This creates some index files that are used by BLAST to speed-up the search.  You can see those files in the ./Databases folder.\n",
    "\n",
    "    \n",
    "## Using BLAST for a sequence search\n",
    "\n",
    "There are several blast algorithms.  You pick the correct one for your situation (e.g. comparing a nucleotide sequence to a nuclotide sequence uses blastn, but you can also compare protein to protein, or translations to proteins, etc.).  See the table of BLAST algorithms here:  https://etutorials.org/Misc/blast/Part+III+Practice/Chapter+5.+BLAST/5.1+The+Five+BLAST+Programs/\n",
    "\n",
    "The search itself is very similar to the FASTA search above, but a little bit easier because the search query can be passed as a string:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "require 'bio'\n",
    "require 'stringio'\n",
    "\n",
    "# blast is installed globally, so you don't need to include the full path to blastn\n",
    "# in this case, we are using blastn because it is nucleotide to nucleotide search\n",
    "factory = Bio::Blast.local('blastn', '../Databases/EBOLA')   # wherever your database is....a BLAST \"factory\" \n",
    "#factory = Bio::Blast.local('tblastx', '/home/osboxes/Databases/EBOLA') # \n",
    "\n",
    "$stderr.puts \"Searching ... \"  # output to \"standard error\", this is to output comments to yourself, or error messages\n",
    "                               # that are not intended to be part of the output of the software (pink background)\n",
    "\n",
    "# executes homology search. Returns Bio::Blast::Report object.\n",
    "report = factory.query(\">myseq\\nAGCCCCGTTTTTTGCCACTGGAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\") # this is a Bio::Blast::Report\n",
    "\n",
    "#puts factory.output   # to see the blast report, verbatim\n",
    "#puts report.hits\n",
    "# Iterates over each hit\n",
    "report.each do |hit|\n",
    "  print \"#{hit.hit_id} : evalue #{hit.evalue}\\t#{hit.target_id} at \"\n",
    "  puts \"#{hit.lap_at}\"   # this tells you start and end of both the query and the hit sequences\n",
    "  hit.each do |hsp|\n",
    "    puts hsp.qseq  # this is the gapped Alignment as text of the query\n",
    "    puts hsp.hseq  # this is the gapped Alignment as text of the hit\n",
    "  end\n",
    "end\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<pre>\n",
    "  \n",
    "\n",
    "\n",
    "\n",
    "</pre>\n",
    "  \n",
    "#  Put together things that we have learned\n",
    "\n",
    "## The Plan\n",
    "\n",
    "* For each gene on the Arabidopsis gene list, we need the FASTA formatted sequence\n",
    "* We can get this FASTA from DB fetch (we think!)\n",
    "* We then index this FASTA file using makeblastdb to create a BLAST database\n",
    "* We go for aperitivos to celebrate how smart we are :-)\n",
    "\n",
    "##### unfortunately, things are seldom as easy as you think they will be!\n",
    "\n",
    "\n",
    "**Your first task:  use dbfetch to retrieve the FASTA sequences for the <span style=\"color:red;\">first 3-4 Arabidopsis gene IDs</span> on the list we have been using (NOT all of them, yet!)**  \n",
    "\n",
    "* Go back in your course notes (or look-up using Google) the format for dbfetch URLs.  We need the ensemblgenomesgene database and fasta format.\n",
    "\n",
    "* In your script, put all of the IDs into a single URL (i.e. don't call DBFetch over and over again)\n",
    "\n",
    "* write the output to a file called \"ARA.fa\"\n",
    "\n",
    "TASK 7A and TASK 7B are in separate Lecutre files.  Open them now and complete the tasks.\n",
    "\n",
    "### be sure you understand Lesson 7 Task Solutions A and B before you continue!\n",
    "<pre>\n",
    "\n",
    "\n",
    "</pre>\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Accessing non-sequence data from NCBI - eUtils\n",
    "\n",
    "Before we begin, take note of the following announcement from NCBI:\n",
    "\n",
    "-----------\n",
    "<em>Coming in May 2018: API Keys\n",
    "\n",
    "On May 1, 2018, NCBI will begin enforcing the use of API keys that will offer enhanced levels of supported access to the E-utilities. After that date, any site (IP address) posting more than 3 requests per second to the E-utilities without an API key will receive an error message. By including an API key, a site can post up to 10 requests per second by default. Higher rates are available by request (vog.hin.mln.ibcn@seitilitue). Users can obtain an API key now from the Settings page of their NCBI account (to create an account, visit www.ncbi.nlm.nih.gov/account/). After creating the key, users should include it in each E-utility request by assigning it to the new api_key parameter.\n",
    "\n",
    "Example request including an API key:\n",
    "esummary.fcgi?db=pubmed&id=123456&api_key=ABCDE12345\n",
    "\n",
    "Example error message if rates are exceeded:\n",
    "{\"error\":\"API rate limit exceeded\",\"count\":\"11\"}\n",
    "\n",
    "Only one API key is allowed per NCBI account; however, a user may request a new key at any time. Such a request will invalidate any existing API key associated with that NCBI account.\n",
    "\n",
    "We encourage regular E-utility users to obtain an API key as soon as possible and begin the process of incorporating it into code. We also encourage users to monitor their request rates to determine if they will require rates higher than 10 per second. As stated above, we can potentially have higher rates negotiated prior to the beginning of enforcement on May 1, 2018.</em>\n",
    "\n",
    "----------------\n",
    "\n",
    "\n",
    "We may need to have all students register with NCBI; however, I will try to avoid this by providing examples that make only single requests.  \n",
    "\n",
    "<pre>\n",
    "\n",
    "\n",
    "</pre>\n",
    "\n",
    "# OVERVIEW \n",
    "\n",
    "**The Nine E-utilities in Brief**\n",
    "\n",
    "EUtilities consists of 9 \"REST\" style interfaces (i.e. parameterized URLs):\n",
    "\n",
    "\n",
    "* **EInfo** (database statistics)\n",
    "\n",
    "    http://eutils.ncbi.nlm.nih.gov/entrez/eutils/einfo.fcgi\n",
    "\n",
    "    Provides the number of records indexed in each field of a given database, the date of the last update of the database, and the available links from the database to other Entrez databases.\n",
    "\n",
    "* **ESearch** (text searches)\n",
    "\n",
    "    http://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi\n",
    "\n",
    "    Responds to a text query with the list of matching UIDs in a given database (for later use in ESummary, EFetch or ELink), along with the term translations of the query.\n",
    "\n",
    "* **EPost** (UID uploads)\n",
    "\n",
    "    http://eutils.ncbi.nlm.nih.gov/entrez/eutils/epost.fcgi\n",
    "\n",
    "    Accepts a list of UIDs from a given database, stores the set on the History Server, and responds with a query key and web environment for the uploaded dataset.\n",
    "\n",
    "* **ESummary** (document summary downloads)\n",
    "\n",
    "    http://eutils.ncbi.nlm.nih.gov/entrez/eutils/esummary.fcgi\n",
    "\n",
    "    Responds to a list of UIDs from a given database with the corresponding document summaries.\n",
    "\n",
    "* **EFetch** (data record downloads)\n",
    "\n",
    "    http://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi\n",
    "\n",
    "    Responds to a list of UIDs in a given database with the corresponding data records in a specified format.\n",
    "\n",
    "* **ELink** (Entrez links)\n",
    "\n",
    "    http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi\n",
    "\n",
    "    Responds to a list of UIDs in a given database with either a list of related UIDs (and relevancy scores) in the same database or a list of linked UIDs in another Entrez database; checks for the existence of a specified link from a list of one or more UIDs; creates a hyperlink to the primary LinkOut provider for a specific UID and database, or lists LinkOut URLs and attributes for multiple UIDs.\n",
    "\n",
    "* **EGQuery** (global query)\n",
    "\n",
    "    http://eutils.ncbi.nlm.nih.gov/entrez/eutils/egquery.fcgi\n",
    "\n",
    "    Responds to a text query with the number of records matching the query in each Entrez database.\n",
    "\n",
    "* **ESpell** (spelling suggestions)\n",
    "\n",
    "    http://eutils.ncbi.nlm.nih.gov/entrez/eutils/espell.fcgi\n",
    "\n",
    "    Retrieves spelling suggestions for a text query in a given database.\n",
    "\n",
    "* **ECitMatch** (batch citation searching in PubMed)\n",
    "\n",
    "    http://eutils.ncbi.nlm.nih.gov/entrez/eutils/ecitmatch.cgi\n",
    "\n",
    "    Retrieves PubMed IDs (PMIDs) corresponding to a set of input citation strings.\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<pre>\n",
    "  \n",
    "\n",
    "</pre>\n",
    "\n",
    "# Getting Started with EUtils\n",
    "\n",
    "Before we look at the BioRuby Classes for EUtils, we will look at the EUtils REST interface.  The complete tutorial for EUtils is at https://www.ncbi.nlm.nih.gov/books/NBK25497/\n",
    "\n",
    "I will take you through a (small!) subset of these functionalities in class.  With that background, you should be able to understand the rest of the functionality yourself.\n",
    "\n",
    "To begin, click on the link for EInfo:\n",
    "\n",
    "\n",
    "http://eutils.ncbi.nlm.nih.gov/entrez/eutils/einfo.fcgi\n",
    "\n",
    "This lists all of the databases that you can query/access from NCBI.  Each of those databases uses a **different type of identifier.**  Those are listed [here](https://www.ncbi.nlm.nih.gov/books/NBK25497/table/chapter2.T._entrez_unique_identifiers_ui/?report=objectonly)\n",
    "\n",
    "To find out more about any of those databases (in particular, the fields that can be queried, and the database-to-database linkages that are available!) the pattern is like what we learned for dbFetch - a set of key/value pairs after a '?' (separated by ';' if there is more than one)\n",
    "\n",
    "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/einfo.fcgi?db=nuccore\n",
    "\n",
    "\n",
    "From the output of nuccore we can see, for example, that you can query the field \"Title\" (TITL) in the nuccore database:\n",
    "\n",
    "    <Field>\n",
    "        <Name>TITL</Name>\n",
    "        <FullName>Title</FullName>\n",
    "\n",
    "\n",
    "The important part of that is the field name:  **TITL** and we will now see how to use it...\n",
    "\n",
    "<pre>\n",
    "\n",
    "\n",
    "</pre>\n",
    "\n",
    "# Searching NCBI Databases\n",
    "\n",
    "\n",
    "NCBI databases use a query language called ENTREZ.  ENTREZ queries are structured as \"value\"[FIELD]\n",
    "\n",
    "To do queries on the, for example, the Nucleotide database ('nuccore'), you would use the ESearch utility.  If you were looking for nucleotide entries with titles including the phrase \"apoptosis inhibitor\", you might create a URL with the following structure:\n",
    "\n",
    "[http://eutils.ncbi.nlm.nih.gov/entrez/eutils/**esearch.fcgi?db=nuccore;term=\"apoptosis+inhibitor\"[TITL]**](http://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?db=nuccore;term=\"apoptosis+inhibitor\"[TITL])\n",
    "\n",
    "The output of that call is a list of identifiers (what kind of identifiers?  See the \"nuccore\" entry in the linked table above) for the first 20 records that have \"apoptosis inhibitor\" in their title.\n",
    "\n",
    "<code>\n",
    "\n",
    "    <eSearchResult>\n",
    "    <Count>2138</Count>   <----   The number of matches in the database\n",
    "    <RetMax>20</RetMax>   <----   The maximum number of records returned\n",
    "    <RetStart>0</RetStart>  <---- The \"offset\" - i.e. what record am I starting with?  (in this case the first)\n",
    "    <IdList>\n",
    "    <Id>219283154</Id><Id>219283151</Id><Id>219283149</Id><Id>197098447</Id><Id>148230698</Id><Id>148229867</Id><Id>57529973</Id><Id>1279832269</Id><Id>1279832267</Id><Id>1279831517</Id><Id>1279711068</Id><Id>1279105385</Id><Id>1279090872</Id><Id>1279061012</Id><Id>1279061010</Id><Id>1279048499</Id><Id>1279048498</Id><Id>1279048496</Id><Id>1279048494</Id><Id>816197788</Id>\n",
    "    </IdList>\n",
    "    <TranslationSet/>\n",
    "    <TranslationStack>\n",
    "    <TermSet>\n",
    "    <Term>\"apoptosis inhibitor\"[TITL]</Term>\n",
    "    <Field>TITL</Field>\n",
    "    <Count>2138</Count>\n",
    "    <Explode>N</Explode>\n",
    "    </TermSet>\n",
    "    <OP>GROUP</OP>\n",
    "    </TranslationStack>\n",
    "    <QueryTranslation>\"apoptosis inhibitor\"[TITL]</QueryTranslation>\n",
    "    </eSearchResult>\n",
    "\n",
    "</code>\n",
    "\n",
    "To get more results, you change/add parameters.  For example, to get the next 50 records AFTER these records, the URL would be:\n",
    "\n",
    "\n",
    "[http://eutils.ncbi.nlm.nih.gov/entrez/eutils/**esearch.fcgi?<span style=\"color:red;\">retstart=20;retmax=50</span>;db=nuccore;term=\"apoptosis+inhibitor\"[TITL]**](http://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?retstart=20;retmax=50;db=nuccore;term=\"apoptosis+inhibitor\"[TITL])\n",
    "\n",
    "Finally, to increase the complexity of the search, you use AND, OR, and NOT operators, with the same \"value\"[FIELD] structure.  So to limit the current query to only those from the genus \"Spodoptera\" (\"ORGN = Organism) (NOTE that we reset the retstart back to zero!!)\n",
    "\n",
    "[http://eutils.ncbi.nlm.nih.gov/entrez/eutils/**esearch.fcgi?retstart=0;retmax=50;db=nuccore;<span style=\"color:red;\">term=\"apoptosis+inhibitor\"[TITL]+AND+\"Spodoptera\"[ORGN]**</span>](http://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?retstart=0;retmax=50;db=nuccore;term=\"apoptosis+inhibitor\"[TITL]+AND+\"Spodoptera\"[ORGN])\n",
    "\n",
    "The result is a single DNA sequence:  gi:1274128812\n",
    "\n",
    "<pre>\n",
    "\n",
    "\n",
    "</pre>\n",
    "\n",
    "# Retrieving from search results\n",
    "\n",
    "To retrieve that sequence, we use the EFetch utility ([Documentation Here](https://www.ncbi.nlm.nih.gov/books/NBK25499/#chapter4.EFetch)).  The process is exactly the same - create URL parameters after <code>efetch.fcgi?</code>  The two parameters of greatest importance to us are \"db=\" (which database am I fetching from?) and \"id=\" (which records am I fetching?).  So our next URL looks like this:\n",
    "\n",
    "[http://eutils.ncbi.nlm.nih.gov/entrez/eutils/**efetch.fcgi?db=nuccore;id=1274128812**](http://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=nuccore;id=1274128812)\n",
    "\n",
    "Note:  if you click that, it will pop-up a window.  That is because the default data-type returned from the Nucleotide sequence database (nuccore) is called \"ASN.1\", and you probably don't have a utility that will read that format.  You **do** have a utility that will read GenBank format, though!  **BioRuby!!**   So is there a way to get that record in GenBank format?  Yes.  There are two additional parameters for EFetch:  retmode and rettype.  Each of the databases supports a subset of possible values for retmode and rettype.  You can look at the possibilities [HERE](https://www.ncbi.nlm.nih.gov/books/NBK25499/table/chapter4.T._valid_values_of__retmode_and/?report=objectonly).  For us to get a GenBank formatted file, we have to set retmode=gb and rettype=text:\n",
    "\n",
    "[http://eutils.ncbi.nlm.nih.gov/entrez/eutils/**efetch.fcgi?db=nuccore;id=1274128812;<span style=\"color:red;\">rettype=gb;retmode=text**</span>](http://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=nuccore;rettype=gb;retmode=text;id=1274128812)\n",
    "\n",
    "Of course, if you need more than one record, you can request them all at the same time by adding more GI numbers separated by commas:\n",
    "\n",
    "\n",
    "[http://eutils.ncbi.nlm.nih.gov/entrez/eutils/**efetch.fcgi?db=nuccore;rettype=gb;retmode=text;<span style=\"color:red;\">id=1274128812,1274128812,1274128812,1274128812**</span>](http://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=nuccore;rettype=gb;retmode=text;id=1274128812,1274128812,1274128812,1274128812)\n",
    "\n",
    "From this point, you know what to do!  This is exactly the same as what we learned with EFetch and using BioRuby to parse the output.\n",
    "\n",
    "**BE CAREFUL!!**   NCBI is very very strict about their limits on number of requests!  For an account with no \"license\", you can only call 3 times per second.  With a license, only 10 times per second!  **So do NOT do this:**\n",
    "\n",
    "   \n",
    "\n",
    "    ids.each do |id|\n",
    "     one_record = fetch(\"http://eutils.../efetch.fcgi?db=nuccore;rettype=gb;retmode=text;id=#{id}\")\n",
    "     process(one_record)\n",
    "    end\n",
    "\n",
    " \n",
    "\n",
    "**That is how you get your entire institute BANNED from using NCBI utilities!**  \n",
    "\n",
    "<span style=\"color:red;\">YOU HAVE BEEN WARNED!!  :-)</span>\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Prove that you understand\n",
    "\n",
    "* Find the PubMed publications written by an author named Mark D Wilkinson who studies \"Reproducibility of Results\" (find the [MESH keyword](https://meshb.nlm.nih.gov/search) and use it in the [MESH] field of the search query.  Retrieve the abstract for the first 3 of these, in text format.\n",
    "\n",
    "(You will need to search for how ENTREZ represents author names!)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<pre>\n",
    "  \n",
    "\n",
    "</pre>\n",
    "\n",
    "# Enough of this REST stuff, let's use BioRuby!\n",
    "\n",
    "The BioRuby interface is very \"thin\" - it simply makes it easier to construct the EUtilities REST URLs.\n",
    "\n",
    "http://bioruby.org/rdoc/Bio/NCBI/REST\n",
    "\n",
    "\n",
    "A typical call is like this:\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "require 'bio'\n",
    "\n",
    "ncbi = Bio::NCBI::REST.new\n",
    "max = 5\n",
    "\n",
    "# note that you can now use spaces in your search terms - BioRuby will format the URL correctly\n",
    "ids = ncbi.esearch('\"Wilkinson MD\"[AUTH]', {\"db\"=>\"nucleotide\", \"rettype\"=>\"gb\", \"retmax\" => max})\n",
    "\n",
    "idlist = ids.join(\",\")\n",
    "puts idlist\n",
    "puts \"\\n\\nRecords are\\n\\n\"\n",
    "records = ncbi.efetch(idlist, {\"db\"=>\"nucleotide\", \"rettype\"=>\"gb\", \"retmode\"=>\"text\", \"retmax\"=> max})\n",
    "puts records\n",
    "\n",
    "# and its just that easy! \n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Better use of PubMed\n",
    "\n",
    "Most reference managers (EndNote, Mendeley, and even LaTex) are able to work with files in BibTex format.  Let's do one last BioRuby call to PubMed to retrieve articles in that format.\n",
    "\n",
    "Our task is to get the abstracts for the nucleotide sequence files that were discovered in the search above.  This is called a \"Link\".  Earlier in this Lesson, we looked at the output from the EInfo tool - it told you what FIELDs you could query for each database, but also what LINKS existed between one database and another.  Let's look at that again - the EInfo about the nuccore database:\n",
    "\n",
    "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/einfo.fcgi?db=nuccore\n",
    "\n",
    "Near the end of that page you will find:\n",
    "\n",
    "    <Link>\n",
    "        <Name>nuccore_pubmed</Name>\n",
    "        <Menu>PubMed Links</Menu>\n",
    "        <Description>PubMed articles cited by Nucleotide sequence record</Description>\n",
    "        <DbTo>pubmed</DbTo>\n",
    "    </Link>\n",
    "\n",
    "This describes that there is a link between nuccore (the sequence database) and pubmed (the publications database).  The tool that we need to find the links between the list of IDs for Database A and the list of IDs for Database B is the ELink tool\n",
    "\n",
    "http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi\n",
    "\n",
    "It works like all other EUtils; however, there is no BioRuby wrapper for this one, so we must do the hard work ourselves.  To see some example code in Perl go to:  https://www.ncbi.nlm.nih.gov/books/NBK25498/#chapter3.Application_4_Finding_unique_se.  We will now do the same thing in Ruby.\n",
    "\n",
    "\n",
    "### ELink parameters are:  \n",
    "* dbfrom\n",
    "* db  (i.e. which db are you linking TO)\n",
    "* id = ...list of ids...\n",
    "\n",
    "So, like all other EUtils, we can construct a URL that calls the ELink service:\n",
    "\n",
    "http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=nuccore;db=pubmed;id=732693005,732693505,1129728243,1129728241,1129728239\n",
    "\n",
    "If you click on that, you will see the XML document that contains the PubMed IDs that correspond to the nucleotide sequence IDs we sent to the **elink** tool.  They are separated into \"Link Sets\", corresponding to different subsections of pubmed (e.g. **nuccore_pubmed_refseq**).  \n",
    "\n",
    "HOWEVER:  If you read a little bit further in the documentation (for all EUtilities!), the instructions say \"use POST when you have many IDs (>200)\" \n",
    "\n",
    ">**Required Parameter â€“ Used only when input is from a UID list**\n",
    "\n",
    ">**id**\n",
    "\n",
    ">UID list. Either a single UID or a comma-delimited list of UIDs may be provided. All of the UIDs must be from the database specified by dbfrom. There is no set maximum for the number of UIDs that can be passed to ELink, but **if more than about 200 UIDs are to be provided, the request should be made using the HTTP POST method.**\n",
    "\n",
    "\n",
    "What does \"HTTP POST\" mean?   We will discuss it in detail in a later lecture, but for now, you only need to understand that it is another way of sending information to a Web server.  Using HTTP GET, you put the information at the end of the URL (<code>scriptname?a=b;c=d;...</code>).   With HTTP POST you send it in the \"body\" of a message.  \n",
    "\n",
    "It is important to learn how to create an HTTP POST request, so even though we _can_ use the URL above (because we only have 5 ID numbers), we should look at and understand the other solution.  An example of an HTTP POST is below, where we are sending a request to the ELink service of NCBI:\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "require 'net/http'\n",
    "require 'bio'\n",
    "\n",
    "ncbi = Bio::NCBI::REST.new\n",
    "max = 5\n",
    "\n",
    "# note that you can now use spaces in your search terms - BioRuby will format the URL correctly\n",
    "ids = ncbi.esearch('\"Wilkinson MD\"[AUTH]', {\"db\"=>\"nucleotide\", \"rettype\"=>\"gb\", \"retmax\" => max})\n",
    "\n",
    "idlist = ids.join(\",\")\n",
    "puts idlist\n",
    "\n",
    "# prepare for HTTP POST - this is very similar to what is in the \"fetch\" routine\n",
    "# from your other scripts.\n",
    "# Note:  some of what we are doing here is a bit hard to understand, because\n",
    "# NCBI uses \"https\" for its requests (encrypted messages) instead of simple http\n",
    "# HTTPS runs on port 443, and has to be explicitly activated\n",
    "elink = URI(\"https://eutils.ncbi.nlm.nih.gov:443/entrez/eutils/elink.fcgi\")\n",
    "request = Net::HTTP::Post.new(elink)\n",
    "https = Net::HTTP.new(elink.host,elink.port)\n",
    "https.use_ssl = true  # tell the HTTP connection to use ecryption (SSL = Secure Socket Layer)\n",
    "\n",
    "\n",
    "# here is where we create the content of the body - called \"form data\".  We will see what this\n",
    "# looks like in a later lecture.  For now, it is just treated like a Hash in Ruby.\n",
    "request.set_form_data(\n",
    "  'email' => 'markw@illuminae.com',\n",
    "  'dbfrom' => 'nuccore', \n",
    "  'db' => 'pubmed',\n",
    "  'retmode' => 'xml',\n",
    "  'id' => idlist)\n",
    "\n",
    "\n",
    "\n",
    "output = https.request(request)   # make the request (we really should check if it is successful)\n",
    "\n",
    "puts output.body\n",
    "\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "\n",
    "#  Parsing XML\n",
    "\n",
    "In an earlier lesson we learned how to extract information from JSON.  In the example above the output of the EUtils service is in XML format, so we now need to learn how to extract information from XML.\n",
    "\n",
    "The most popular Ruby library for XML parsing is called 'nokogiri' (this translates to English as \"saw\" and Spanish as \"sierra\" como \"sierra circular\").  So that's what we will use.  The XML output from EUtils is quite simple (we will see more complex examples later in the course!) so let's use it as our learning example\n",
    "\n",
    "\n",
    "    <eLinkResult>\n",
    "        <LinkSet>\n",
    "            <LinkSetDb>\n",
    "                  <DbTo>pubmed</DbTo>\n",
    "                  <LinkName>nuccore_pubmed</LinkName>\n",
    "\n",
    "                  <Link>\n",
    "                        <Id>25896417</Id>\n",
    "                  </Link>\n",
    "                  <Link>\n",
    "                        <Id>20610895</Id>\n",
    "                  </Link>\n",
    "                  <Link>\n",
    "                        <Id>19056867</Id>\n",
    "                  </Link>\n",
    "                  <Link>\n",
    "                        <Id>15498460</Id>\n",
    "                  </Link>\n",
    "                  <Link>\n",
    "                        <Id>12421765</Id>\n",
    "                  </Link>\n",
    "\n",
    "            </LinkSetDb>\n",
    "        </LinkSet>\n",
    "     <eLinkResut>\n",
    "\n",
    "This is the relevant section we want to extract IDs from.  XML documents are \"hierarchical\", sub-sections are enclosed in parent sections in the pattern:\n",
    "        \n",
    "        <A> <B> data </B>  </A>\n",
    "\n",
    "Given this structure, it is normal to talk about the \"path\" to the data.  In this case, the data is A-->B-->data.  \n",
    "\n",
    "In the output from ELink, the path is:  LinkSetDb-->Link-->Id-->data\n",
    "\n",
    "These \"paths\" are used in Ruby to define the piece of data we want to extract.  For example:\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "require 'nokogiri'\n",
    "\n",
    "xml = <<EOF\n",
    "\n",
    "    <eLinkResult>\n",
    "        <LinkSet>\n",
    "\n",
    "            <LinkSetDb>\n",
    "                <Link>\n",
    "                      <Id>25896417</Id>\n",
    "                </Link>\n",
    "                <Link>\n",
    "                      <Id>20610895</Id>\n",
    "                </Link>\n",
    "            </LinkSetDb>\n",
    "        </LinkSet>\n",
    "    <eLinkResut>\n",
    "\n",
    "EOF\n",
    "\n",
    "\n",
    "doc = Nokogiri::XML(xml)\n",
    "# puts doc.class  #  You should read the documentation for yourself one day!  \n",
    "                  # Also read about \"XPath\" for more complex XML documents\n",
    "\n",
    "doc.xpath('//LinkSet/LinkSetDb/Link').each do |node|    # the \"path\" (\"xpath\")to the data, starts with '//'\n",
    "  # puts node.class\n",
    "  puts \"PubMed = \" + node.at_xpath('Id').content\n",
    "end\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#  Putting it all together!\n",
    "\n",
    "Finally, we will create a script that:\n",
    "\n",
    "* retrieves the nucleotide database ids\n",
    "* retrieves the cross-reference links to PubMed\n",
    "* extracts the PubMed IDs\n",
    "* retrieves the PubMed Abstracts for those IDs\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "require 'net/http'\n",
    "require 'bio'\n",
    "require 'nokogiri'\n",
    "\n",
    "\n",
    "ncbi = Bio::NCBI::REST.new\n",
    "max = 5\n",
    "\n",
    "# note that you can now use spaces in your search terms - BioRuby will format the URL correctly\n",
    "ids = ncbi.esearch('\"Wilkinson MD\"[AUTH]', {\"db\"=>\"nucleotide\", \"rettype\"=>\"gb\", \"retmax\" => max})\n",
    "\n",
    "idlist = ids.join(\",\")\n",
    "\n",
    "elink = URI(\"https://eutils.ncbi.nlm.nih.gov:443/entrez/eutils/elink.fcgi\")\n",
    "request = Net::HTTP::Post.new(elink)\n",
    "https = Net::HTTP.new(elink.host,elink.port)\n",
    "https.use_ssl = true  # tell the HTTP connection to use ecryption (SSL = Secure Socket Layer)\n",
    "\n",
    "request.set_form_data(\n",
    "  'email' => 'markw@illuminae.com',\n",
    "  'dbfrom' => 'nuccore', \n",
    "  'db' => 'pubmed',\n",
    "  'retmode' => 'xml',\n",
    "  'id' => idlist)\n",
    "\n",
    "output = https.request(request)   # make the request (we really should check if it is successful)\n",
    "\n",
    "linkset = output.body\n",
    "\n",
    "\n",
    "doc = Nokogiri::XML(linkset)\n",
    "\n",
    "pubmedids = []  # force it to be a string so that we can use '<<' operator\n",
    "\n",
    "doc.xpath('//LinkSet/LinkSetDb/Link').each do |node|    # the \"path\" (\"xpath\")to the data, starts with '//'\n",
    "   pubmedids << node.at_xpath('Id').content.to_s\n",
    "end\n",
    "puts \"PUBMED IDs are: #{pubmedids}\"\n",
    "\n",
    "pubmedrecords = ncbi.efetch(pubmedids, {\"db\"=>\"pubmed\", \"rettype\"=>\"abstract\", \"retmode\"=>\"text\"})\n",
    "\n",
    "puts pubmedrecords.force_encoding(\"utf-8\")  # NOTE - this is a useful trick for text that has international characters!\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "\n",
    "# Bio::PubMed , Bio::MEDLINE and bibtex format\n",
    "\n",
    "The output above is great for a human, but not very useful for a machine.  Most reference managers can accept new records in bibtex format.  Can we get bibtex format for these records?  Sure!\n",
    "\n",
    "Bio::PubMed is a shortcut to the ESearch and EFetch of utilities for the pubmed database.  It is used like this:\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#===========\n",
    "#  This is only for your information\n",
    "# we already have a list of PubMed IDs from our LinkList above\n",
    "# but you can use Bio::PubMed.esearch to get them \n",
    "# from a search against the pubmed database\n",
    "# ===========\n",
    "#keywords = \"apoptosis death\"\n",
    "#options = {\n",
    "#  'maxdate' => '2016/05/31',\n",
    "#  'retmax' => 2,\n",
    "#}\n",
    "#entries = Bio::PubMed.esearch(keywords, options)  # entries is a list of pubmed ids\n",
    "# ===========\n",
    "\n",
    "\n",
    "pubmedids = [\"25896417\", \"20610895\", \"19056867\"]\n",
    "\n",
    "Bio::PubMed.efetch(pubmedids).each do |pubmedrecord|\n",
    "  # pubmedrecord is a string of the record, in pubmed format\n",
    "  # we can use that to create a new Bio::MEDLINE object\n",
    "  medline = Bio::MEDLINE.new(pubmedrecord)\n",
    "  reference = medline.reference  # reference is a Bio::Reference object\n",
    "  puts reference.bibtex   # the .bibtex method creates the bibtex string of the Bio::Reference\n",
    "end"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<pre>\n",
    "  \n",
    "\n",
    "</pre>\n",
    "  \n",
    "# The Final Code\n",
    "\n",
    "So finally we put the BibTex code at the end of our script, and...\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "require 'net/http'\n",
    "require 'bio'\n",
    "require 'nokogiri'\n",
    "\n",
    "\n",
    "ncbi = Bio::NCBI::REST.new\n",
    "max = 5\n",
    "\n",
    "# note that you can now use spaces in your search terms - BioRuby will format the URL correctly\n",
    "ids = ncbi.esearch('\"Wilkinson MD\"[AUTH]', {\"db\"=>\"nucleotide\", \"rettype\"=>\"gb\", \"retmax\" => max})\n",
    "\n",
    "idlist = ids.join(\",\")\n",
    "\n",
    "elink = URI(\"https://eutils.ncbi.nlm.nih.gov:443/entrez/eutils/elink.fcgi\")\n",
    "request = Net::HTTP::Post.new(elink)\n",
    "https = Net::HTTP.new(elink.host,elink.port)\n",
    "https.use_ssl = true  # tell the HTTP connection to use ecryption (SSL = Secure Socket Layer)\n",
    "\n",
    "request.set_form_data(\n",
    "  'email' => 'markw@illuminae.com',\n",
    "  'dbfrom' => 'nuccore', \n",
    "  'db' => 'pubmed',\n",
    "  'retmode' => 'xml',\n",
    "  'id' => idlist)\n",
    "\n",
    "output = https.request(request)   # make the request (we really should check if it is successful)\n",
    "\n",
    "linkset = output.body\n",
    "\n",
    "\n",
    "doc = Nokogiri::XML(linkset)\n",
    "\n",
    "pubmedids = []  # force it to be a string so that we can use '<<' operator\n",
    "\n",
    "doc.xpath('//LinkSet/LinkSetDb/Link').each do |node|    # the \"path\" (\"xpath\")to the data, starts with '//'\n",
    "   pubmedids << node.at_xpath('Id').content.to_s\n",
    "end\n",
    "$stderr.puts \"PUBMED IDs are: #{pubmedids}\"\n",
    "\n",
    "# we remove these lines because now we are going to output the BibTex instead of the Abstract\n",
    "#pubmedrecords = ncbi.efetch(pubmedids, {\"db\"=>\"pubmed\", \"rettype\"=>\"abstract\", \"retmode\"=>\"text\"})\n",
    "#puts pubmedrecords.force_encoding(\"utf-8\")  # NOTE - this is a useful trick for text that has international characters!\n",
    "\n",
    "Bio::PubMed.efetch(pubmedids).each do |pubmedrecord|\n",
    "  # pubmedrecord is a string of the record, in pubmed format\n",
    "  # we can use that to create a new Bio::MEDLINE object\n",
    "  medline = Bio::MEDLINE.new(pubmedrecord)\n",
    "  reference = medline.reference  # reference is a Bio::Reference object\n",
    "  puts reference.bibtex   # the .bibtex method creates the bibtex string of the Bio::Reference\n",
    "end"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "...and you can write that to a file and import it into EndNote, or Mendeley, or PaperPile, or whatever reference manager you prefer!\n",
    "\n",
    "\n",
    "DONE!\n"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Ruby 2.7.2",
   "language": "ruby",
   "name": "ruby"
  },
  "language_info": {
   "file_extension": ".rb",
   "mimetype": "application/x-ruby",
   "name": "ruby",
   "version": "2.7.2"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
