{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Sequence Searching\n",
    "\n",
    "We will look at sequence searches using FASTA and BLAST.  Both of these have been installed for you.\n",
    "\n",
    "See docs at http://bioruby.org/rdoc/Bio/Fasta and ....\n",
    "\n",
    "I have downloaded 160 Ebola virus sequences for you in the \"Sequences\" folder of your home directory.  You need to concatenate all of the single-sequence FASTA files into a single FASTA file, which is used as the database for a FASTA search:\n",
    "\n",
    "    $ cat *.fa  >  EBOLA.fa\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#  <span style=\"color:red;\">NOTE:  BioRuby's FASTA report parser is broken</span>\n",
    "\n",
    "<span style=\"color:red;\">The code below WILL NOT WORK right now. However, you can look at the structure of the code to understand what it is supposed to do...</span>\n",
    "<pre>\n",
    "\n",
    "</pre>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 99,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/home/osboxes/UPM_BioinfoCourse/Lectures\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Searching ... myseq\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[]\n"
     ]
    }
   ],
   "source": [
    "require 'bio'\n",
    "require 'stringio'\n",
    "\n",
    "#puts Dir.pwd\n",
    "\n",
    "\n",
    "factory = Bio::Fasta.local('/home/osboxes/fasta-36.3.8d/bin/fasta36', '/home/osboxes/Sequences/EBOLA.fa') \n",
    "\n",
    "# you haven't seen this before!  The second argument must be a stream (respond to \".read\", \n",
    "# so make the string look like an opened file\n",
    "find = StringIO.new(\">myseq\\nAGCCCCGTTTTTTGCCACTGGTTATTTAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\")\n",
    "\n",
    "ff = Bio::FlatFile.new(Bio::FastaFormat, find)   # 'find' is a streamable object\n",
    "\n",
    "# Iterates over each entry. the variable \"entry\" is a \n",
    "# Bio::FastaFormat object:\n",
    "ff.each do |entry|\n",
    "  \n",
    "  $stderr.puts \"Searching ... \" + entry.definition # output to \"standard error\", this is to output comments to yourself, or error messages\n",
    "                                                   # that are not intended to be part of the output of the software\n",
    "\n",
    "\n",
    "  # executes homology search. Returns Bio::Fasta::Report object.\n",
    "  report = factory.query(entry)\n",
    "\n",
    "  #puts factory.output\n",
    "  puts report.hits\n",
    "  # Iterates over each hit\n",
    "  report.each do |hit|\n",
    "      print \"#{hit.query_id} : evalue #{hit.evalue}\\t#{hit.target_id} at \"\n",
    "      p hit.lap_at\n",
    "  end\n",
    "end"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Prove you understand\n",
    "modify the script above so that it only prints hits with evalue less than 10e-3\n",
    "<pre>\n",
    "\n",
    "\n",
    "</pre>\n",
    "\n",
    "# BLAST searches\n",
    "\n",
    "to install Blast for yourself:  \n",
    "    \n",
    "    $  sudo apt-get install ncbi-blast+\n",
    "    \n",
    "    $  sudo apt-get install ncbi-blast+-legacy   (this is required for BioRuby!)\n",
    "    \n",
    "### Custom BLAST databases\n",
    "    \n",
    "Blast requires:\n",
    "\n",
    "    A sequence to be used as the query\n",
    "\n",
    "    A BLAST-formatted database\n",
    "\n",
    "Common BLAST databases can be downloaded from NCBI:\n",
    "ftp://ftp.ncbi.nlm.nih.gov/blast/db/ \n",
    "\n",
    "We will create our own small database for this class using the EBOLA data.\n",
    "\n",
    "First, create a new folder \"Databases\" in your home directory, and **copy** EBOLA.fa into that location.\n",
    "\n",
    "Now, execute this command:\n",
    "\n",
    "    $  makeblastdb -in EBOLA.fa -dbtype 'nucl' -out EBOLA\n",
    "\n",
    "This creates some index files that are used by BLAST to speed-up the search.    \n",
    "    \n",
    "The search itself is very similar to the FASTA search above, but a little bit easier because the search query can be passed as a string:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "require 'bio'\n",
    "require 'stringio'\n",
    "\n",
    "# blast is installed globally, so you don't need to include the full path to blastn\n",
    "factory = Bio::Blast.local('blastn', '../Databases/EBOLA')   # wherever your database is....a BLAST \"factory\" \n",
    "#factory = Bio::Blast.local('tblastx', '/home/osboxes/Databases/EBOLA') # \n",
    "$stderr.puts \"Searching ... \"  # output to \"standard error\", this is to output comments to yourself, or error messages\n",
    "                               # that are not intended to be part of the output of the software (pink background)\n",
    "\n",
    "# executes homology search. Returns Bio::Blast::Report object.\n",
    "report = factory.query(\">myseq\\nAGCCCCGTTTTTTGCCACTGGAATTAAGCCAATAACGTCAAGTGCTAGATCTAGTGAGTGGTAT\") # this is a Bio::Blast::Report\n",
    "\n",
    "#puts factory.output   # to see the blast report, verbatim\n",
    "#puts report.hits\n",
    "# Iterates over each hit\n",
    "report.each do |hit|\n",
    "  print \"#{hit.hit_id} : evalue #{hit.evalue}\\t#{hit.target_id} at \"\n",
    "  puts \"#{hit.lap_at}\"   # this tells you start and end of both the query and the hit sequences\n",
    "  hit.each do |hsp|\n",
    "    puts hsp.qseq  # this is the gapped Alignment as text of the query\n",
    "    puts hsp.hseq  # this is the gapped Alignment as text of the hit\n",
    "  end\n",
    "end\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<pre>\n",
    "  \n",
    "\n",
    "\n",
    "\n",
    "</pre>\n",
    "  \n",
    "#  Put together things that we have learned\n",
    "\n",
    "## The Plan\n",
    "\n",
    "* For each gene on the Arabidopsis gene list, we need the FASTA formatted sequence\n",
    "* We can get this FASTA from DB fetch (we think!)\n",
    "* We then index this FASTA file using makeblastdb to create a BLAST database\n",
    "* We go for aperitivos to celebrate how smart we are :-)\n",
    "\n",
    "##### unfortunately, things are seldom as easy as you think they will be!\n",
    "\n",
    "\n",
    "**Your first task:  use dbfetch to retrieve the FASTA sequences for the <span style=\"color:red;\">first 3-4 Arabidopsis gene IDs</span> on the list we have been using (NOT all of them, yet!)**  \n",
    "\n",
    "* Go back in your course notes (or look-up using Google) the format for dbfetch URLs.  We need the ensemblgenomesgene database and fasta format.\n",
    "\n",
    "* In your script, put all of the IDs into a single URL (i.e. don't call DBFetch over and over again)\n",
    "\n",
    "* write the output to a file called \"ARA.fa\"\n",
    "\n",
    "TASK 7A and TASK 7B are in separate Lecutre files.  Open them now and complete the tasks.\n",
    "\n",
    "### be sure you understand Lesson 7 Task Solutions A and B before you continue!\n",
    "<pre>\n",
    "\n",
    "\n",
    "</pre>\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Accessing non-sequence data from NCBI - eUtils\n",
    "\n",
    "Before we begin, take note of the following announcement from NCBI:\n",
    "\n",
    "-----------\n",
    "<em>Coming in May 2018: API Keys\n",
    "\n",
    "On May 1, 2018, NCBI will begin enforcing the use of API keys that will offer enhanced levels of supported access to the E-utilities. After that date, any site (IP address) posting more than 3 requests per second to the E-utilities without an API key will receive an error message. By including an API key, a site can post up to 10 requests per second by default. Higher rates are available by request (vog.hin.mln.ibcn@seitilitue). Users can obtain an API key now from the Settings page of their NCBI account (to create an account, visit www.ncbi.nlm.nih.gov/account/). After creating the key, users should include it in each E-utility request by assigning it to the new api_key parameter.\n",
    "\n",
    "Example request including an API key:\n",
    "esummary.fcgi?db=pubmed&id=123456&api_key=ABCDE12345\n",
    "\n",
    "Example error message if rates are exceeded:\n",
    "{\"error\":\"API rate limit exceeded\",\"count\":\"11\"}\n",
    "\n",
    "Only one API key is allowed per NCBI account; however, a user may request a new key at any time. Such a request will invalidate any existing API key associated with that NCBI account.\n",
    "\n",
    "We encourage regular E-utility users to obtain an API key as soon as possible and begin the process of incorporating it into code. We also encourage users to monitor their request rates to determine if they will require rates higher than 10 per second. As stated above, we can potentially have higher rates negotiated prior to the beginning of enforcement on May 1, 2018.</em>\n",
    "\n",
    "----------------\n",
    "\n",
    "\n",
    "We may need to have all students register with NCBI; however, I will try to avoid this by providing examples that make only single requests.  \n",
    "\n",
    "<pre>\n",
    "\n",
    "\n",
    "</pre>\n",
    "\n",
    "# OVERVIEW \n",
    "\n",
    "**The Nine E-utilities in Brief**\n",
    "\n",
    "EUtilities consists of 9 \"REST\" style interfaces (i.e. parameterized URLs):\n",
    "\n",
    "\n",
    "* **EInfo** (database statistics)\n",
    "\n",
    "    http://eutils.ncbi.nlm.nih.gov/entrez/eutils/einfo.fcgi\n",
    "\n",
    "    Provides the number of records indexed in each field of a given database, the date of the last update of the database, and the available links from the database to other Entrez databases.\n",
    "\n",
    "* **ESearch** (text searches)\n",
    "\n",
    "    http://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi\n",
    "\n",
    "    Responds to a text query with the list of matching UIDs in a given database (for later use in ESummary, EFetch or ELink), along with the term translations of the query.\n",
    "\n",
    "* **EPost** (UID uploads)\n",
    "\n",
    "    http://eutils.ncbi.nlm.nih.gov/entrez/eutils/epost.fcgi\n",
    "\n",
    "    Accepts a list of UIDs from a given database, stores the set on the History Server, and responds with a query key and web environment for the uploaded dataset.\n",
    "\n",
    "* **ESummary** (document summary downloads)\n",
    "\n",
    "    http://eutils.ncbi.nlm.nih.gov/entrez/eutils/esummary.fcgi\n",
    "\n",
    "    Responds to a list of UIDs from a given database with the corresponding document summaries.\n",
    "\n",
    "* **EFetch** (data record downloads)\n",
    "\n",
    "    http://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi\n",
    "\n",
    "    Responds to a list of UIDs in a given database with the corresponding data records in a specified format.\n",
    "\n",
    "* **ELink** (Entrez links)\n",
    "\n",
    "    http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi\n",
    "\n",
    "    Responds to a list of UIDs in a given database with either a list of related UIDs (and relevancy scores) in the same database or a list of linked UIDs in another Entrez database; checks for the existence of a specified link from a list of one or more UIDs; creates a hyperlink to the primary LinkOut provider for a specific UID and database, or lists LinkOut URLs and attributes for multiple UIDs.\n",
    "\n",
    "* **EGQuery** (global query)\n",
    "\n",
    "    http://eutils.ncbi.nlm.nih.gov/entrez/eutils/egquery.fcgi\n",
    "\n",
    "    Responds to a text query with the number of records matching the query in each Entrez database.\n",
    "\n",
    "* **ESpell** (spelling suggestions)\n",
    "\n",
    "    http://eutils.ncbi.nlm.nih.gov/entrez/eutils/espell.fcgi\n",
    "\n",
    "    Retrieves spelling suggestions for a text query in a given database.\n",
    "\n",
    "* **ECitMatch** (batch citation searching in PubMed)\n",
    "\n",
    "    http://eutils.ncbi.nlm.nih.gov/entrez/eutils/ecitmatch.cgi\n",
    "\n",
    "    Retrieves PubMed IDs (PMIDs) corresponding to a set of input citation strings.\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<pre>\n",
    "  \n",
    "\n",
    "</pre>\n",
    "\n",
    "# Getting Started with EUtils\n",
    "\n",
    "Before we look at the BioRuby Classes for EUtils, we will look at the EUtils REST interface.  The complete tutorial for EUtils is at https://www.ncbi.nlm.nih.gov/books/NBK25497/\n",
    "\n",
    "I will take you through a (small!) subset of these functionalities in class.  With that background, you should be able to understand the rest of the functionality yourself.\n",
    "\n",
    "To begin, click on the link for EInfo:\n",
    "\n",
    "\n",
    "http://eutils.ncbi.nlm.nih.gov/entrez/eutils/einfo.fcgi\n",
    "\n",
    "This lists all of the databases that you can query/access from NCBI.  Each of those databases uses a **different type of identifier.**  Those are listed [here](https://www.ncbi.nlm.nih.gov/books/NBK25497/table/chapter2.T._entrez_unique_identifiers_ui/?report=objectonly)\n",
    "\n",
    "To find out more about any of those databases (in particular, the fields that can be queried, and the database-to-database linkages that are available!) the pattern is like what we learned for dbFetch - a set of key/value pairs after a '?' (separated by ';' if there is more than one)\n",
    "\n",
    "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/einfo.fcgi?db=nuccore\n",
    "\n",
    "\n",
    "From the output of nuccore we can see, for example, that you can query the field \"Title\" (TITL) in the nuccore database:\n",
    "\n",
    "    <Field>\n",
    "        <Name>TITL</Name>\n",
    "        <FullName>Title</FullName>\n",
    "\n",
    "\n",
    "The important part of that is the field name:  **TITL** and we will now see how to use it...\n",
    "\n",
    "<pre>\n",
    "\n",
    "\n",
    "</pre>\n",
    "\n",
    "# Searching NCBI Databases\n",
    "\n",
    "\n",
    "NCBI databases use a query language called ENTREZ.  ENTREZ queries are structured as \"value\"[FIELD]\n",
    "\n",
    "To do queries on the, for example, the Nucleotide database ('nuccore'), you would use the ESearch utility.  If you were looking for nucleotide entries with titles including the phrase \"apoptosis inhibitor\", you might create a URL with the following structure:\n",
    "\n",
    "[http://eutils.ncbi.nlm.nih.gov/entrez/eutils/**esearch.fcgi?db=nuccore;term=\"apoptosis+inhibitor\"[TITL]**](http://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?db=nuccore;term=\"apoptosis+inhibitor\"[TITL])\n",
    "\n",
    "The output of that call is a list of identifiers (what kind of identifiers?  See the \"nuccore\" entry in the linked table above) for the first 20 records that have \"apoptosis inhibitor\" in their title.\n",
    "\n",
    "<code>\n",
    "\n",
    "    <eSearchResult>\n",
    "    <Count>2138</Count>   <----   The number of matches in the database\n",
    "    <RetMax>20</RetMax>   <----   The maximum number of records returned\n",
    "    <RetStart>0</RetStart>  <---- The \"offset\" - i.e. what record am I starting with?  (in this case the first)\n",
    "    <IdList>\n",
    "    <Id>219283154</Id><Id>219283151</Id><Id>219283149</Id><Id>197098447</Id><Id>148230698</Id><Id>148229867</Id><Id>57529973</Id><Id>1279832269</Id><Id>1279832267</Id><Id>1279831517</Id><Id>1279711068</Id><Id>1279105385</Id><Id>1279090872</Id><Id>1279061012</Id><Id>1279061010</Id><Id>1279048499</Id><Id>1279048498</Id><Id>1279048496</Id><Id>1279048494</Id><Id>816197788</Id>\n",
    "    </IdList>\n",
    "    <TranslationSet/>\n",
    "    <TranslationStack>\n",
    "    <TermSet>\n",
    "    <Term>\"apoptosis inhibitor\"[TITL]</Term>\n",
    "    <Field>TITL</Field>\n",
    "    <Count>2138</Count>\n",
    "    <Explode>N</Explode>\n",
    "    </TermSet>\n",
    "    <OP>GROUP</OP>\n",
    "    </TranslationStack>\n",
    "    <QueryTranslation>\"apoptosis inhibitor\"[TITL]</QueryTranslation>\n",
    "    </eSearchResult>\n",
    "\n",
    "</code>\n",
    "\n",
    "To get more results, you change/add parameters.  For example, to get the next 50 records AFTER these records, the URL would be:\n",
    "\n",
    "\n",
    "[http://eutils.ncbi.nlm.nih.gov/entrez/eutils/**esearch.fcgi?<span style=\"color:red;\">retstart=20;retmax=50</span>;db=nuccore;term=\"apoptosis+inhibitor\"[TITL]**](http://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?retstart=20;retmax=50;db=nuccore;term=\"apoptosis+inhibitor\"[TITL])\n",
    "\n",
    "Finally, to increase the complexity of the search, you use AND, OR, and NOT operators, with the same \"value\"[FIELD] structure.  So to limit the current query to only those from the genus \"Spodoptera\" (\"ORGN = Organism) (NOTE that we reset the retstart back to zero!!)\n",
    "\n",
    "[http://eutils.ncbi.nlm.nih.gov/entrez/eutils/**esearch.fcgi?retstart=0;retmax=50;db=nuccore;<span style=\"color:red;\">term=\"apoptosis+inhibitor\"[TITL]+AND+\"Spodoptera\"[ORGN]**</span>](http://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?retstart=0;retmax=50;db=nuccore;term=\"apoptosis+inhibitor\"[TITL]+AND+\"Spodoptera\"[ORGN])\n",
    "\n",
    "The result is a single DNA sequence:  gi:1274128812\n",
    "\n",
    "<pre>\n",
    "\n",
    "\n",
    "</pre>\n",
    "\n",
    "# Retrieving from search results\n",
    "\n",
    "To retrieve that sequence, we use the EFetch utility ([Documentation Here](https://www.ncbi.nlm.nih.gov/books/NBK25499/#chapter4.EFetch)).  The process is exactly the same - create URL parameters after <code>efetch.fcgi?</code>  The two parameters of greatest importance to us are \"db=\" (which database am I fetching from?) and \"id=\" (which records am I fetching?).  So our next URL looks like this:\n",
    "\n",
    "[http://eutils.ncbi.nlm.nih.gov/entrez/eutils/**efetch.fcgi?db=nuccore;id=1274128812**](http://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=nuccore;id=1274128812)\n",
    "\n",
    "Note:  if you click that, it will pop-up a window.  That is because the default data-type returned from the Nucleotide sequence database (nuccore) is called \"ASN.1\", and you probably don't have a utility that will read that format.  You **do** have a utility that will read GenBank format, though!  **BioRuby!!**   So is there a way to get that record in GenBank format?  Yes.  There are two additional parameters for EFetch:  retmode and rettype.  Each of the databases supports a subset of possible values for retmode and rettype.  You can look at the possibilities [HERE](https://www.ncbi.nlm.nih.gov/books/NBK25499/table/chapter4.T._valid_values_of__retmode_and/?report=objectonly).  For us to get a GenBank formatted file, we have to set retmode=gb and rettype=text:\n",
    "\n",
    "[http://eutils.ncbi.nlm.nih.gov/entrez/eutils/**efetch.fcgi?db=nuccore;id=1274128812;<span style=\"color:red;\">rettype=gb;retmode=text**</span>](http://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=nuccore;rettype=gb;retmode=text;id=1274128812)\n",
    "\n",
    "Of course, if you need more than one record, you can request them all at the same time by adding more GI numbers separated by commas:\n",
    "\n",
    "\n",
    "[http://eutils.ncbi.nlm.nih.gov/entrez/eutils/**efetch.fcgi?db=nuccore;rettype=gb;retmode=text;<span style=\"color:red;\">id=1274128812,1274128812,1274128812,1274128812**</span>](http://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=nuccore;rettype=gb;retmode=text;id=1274128812,1274128812,1274128812,1274128812)\n",
    "\n",
    "From this point, you know what to do!  This is exactly the same as what we learned with EFetch and using BioRuby to parse the output.\n",
    "\n",
    "**BE CAREFUL!!**   NCBI is very very strict about their limits on number of requests!  For an account with no \"license\", you can only call 3 times per second.  With a license, only 10 times per second!  **So do NOT do this:**\n",
    "\n",
    "   \n",
    "\n",
    "    ids.each do |id|\n",
    "     one_record = fetch(\"http://eutils.../efetch.fcgi?db=nuccore;rettype=gb;retmode=text;id=#{id}\")\n",
    "     process(one_record)\n",
    "    end\n",
    "\n",
    " \n",
    "\n",
    "**That is how you get your entire institute BANNED from using NCBI utilities!**  \n",
    "\n",
    "<span style=\"color:red;\">YOU HAVE BEEN WARNED!!  :-)</span>\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Prove that you understand\n",
    "\n",
    "* Find the PubMed publications written by an author named Mark D Wilkinson who studies \"Reproducibility of Results\" (find the [MESH keyword](https://meshb.nlm.nih.gov/search) and use it in the [MESH] field of the search query.  Retrieve the abstract for the first 3 of these, in text format.\n",
    "\n",
    "(You will need to search for how ENTREZ represents author names!)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<pre>\n",
    "  \n",
    "\n",
    "</pre>\n",
    "\n",
    "# Enough of this REST stuff, let's use BioRuby!\n",
    "\n",
    "The BioRuby interface is very \"thin\" - it simply makes it easier to construct the EUtilities REST URLs.\n",
    "\n",
    "http://bioruby.org/rdoc/Bio/NCBI/REST\n",
    "\n",
    "\n",
    "A typical call is like this:\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 100,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "732693005,732693505,1129728243,1129728241,1129728239\n",
      "\n",
      "\n",
      "Records are\n",
      "\n",
      "\n",
      "LOCUS       NM_015378              16360 bp    mRNA    linear   PRI 03-OCT-2017\n",
      "DEFINITION  Homo sapiens vacuolar protein sorting 13 homolog D (VPS13D),\n",
      "            transcript variant 1, mRNA.\n",
      "ACCESSION   NM_015378 XM_044546\n",
      "VERSION     NM_015378.3\n",
      "KEYWORDS    RefSeq.\n",
      "SOURCE      Homo sapiens (human)\n",
      "  ORGANISM  Homo sapiens\n",
      "            Eukaryota; Metazoa; Chordata; Craniata; Vertebrata; Euteleostomi;\n",
      "            Mammalia; Eutheria; Euarchontoglires; Primates; Haplorrhini;\n",
      "            Catarrhini; Hominidae; Homo.\n",
      "REFERENCE   1  (bases 1 to 16360)\n",
      "  AUTHORS   Nakada TA, Boyd JH, Russell JA, Aguirre-Hernandez R, Wilkinson MD,\n",
      "            Thair SA, Nakada E, McConechy MK, Fjell CD and Walley KR.\n",
      "  TITLE     VPS13D Gene Variant Is Associated with Altered IL-6 Production and\n",
      "            Mortality in Septic Shock\n",
      "  JOURNAL   J Innate Immun 7 (5), 545-553 (2015)\n",
      "   PUBMED   25896417\n",
      "  REMARK    GeneRIF: genetic polymorphism is associated with il-6 production\n",
      "            and mortality in patients with septic shock\n",
      "REFERENCE   2  (bases 1 to 16360)\n",
      "  AUTHORS   Koriyama H, Nakagami H, Katsuya T, Sugimoto K, Yamashita H, Takami\n",
      "            Y, Maeda S, Kubo M, Takahashi A, Nakamura Y, Ogihara T, Rakugi H,\n",
      "            Kaneda Y and Morishita R.\n",
      "  TITLE     Identification of evidence suggestive of an association with\n",
      "            peripheral arterial disease at the OSBPL10 locus by genome-wide\n",
      "            investigation in the Japanese population\n",
      "  JOURNAL   J. Atheroscler. Thromb. 17 (10), 1054-1062 (2010)\n",
      "   PUBMED   20610895\n",
      "  REMARK    GeneRIF: Observational study and genome-wide association study of\n",
      "            gene-disease association. (HuGE Navigator)\n",
      "REFERENCE   3  (bases 1 to 16360)\n",
      "  AUTHORS   Gonzales PA, Pisitkun T, Hoffert JD, Tchapyjnikov D, Star RA, Kleta\n",
      "            R, Wang NS and Knepper MA.\n",
      "  TITLE     Large-scale proteomics and phosphoproteomics of urinary exosomes\n",
      "  JOURNAL   J. Am. Soc. Nephrol. 20 (2), 363-379 (2009)\n",
      "   PUBMED   19056867\n",
      "REFERENCE   4  (bases 1 to 16360)\n",
      "  AUTHORS   Velayos-Baeza A, Vettori A, Copley RR, Dobson-Stone C and Monaco\n",
      "            AP.\n",
      "  TITLE     Analysis of the human VPS13 gene family\n",
      "  JOURNAL   Genomics 84 (3), 536-549 (2004)\n",
      "   PUBMED   15498460\n",
      "REFERENCE   5  (bases 1 to 16360)\n",
      "  AUTHORS   Nakayama M, Kikuno R and Ohara O.\n",
      "  TITLE     Protein-protein interactions between large proteins: two-hybrid\n",
      "            screening using a functionally classified library composed of long\n",
      "            cDNAs\n",
      "  JOURNAL   Genome Res. 12 (11), 1773-1784 (2002)\n",
      "   PUBMED   12421765\n",
      "COMMENT     REVIEWED REFSEQ: This record has been curated by NCBI staff. The\n",
      "            reference sequence was derived from HY121734.1, AJ608774.1,\n",
      "            BX682532.4 and BC051804.1.\n",
      "            On Dec 15, 2014 this sequence version replaced gi:54607138.\n",
      "            \n",
      "            Summary: This gene encodes a protein belonging to the\n",
      "            vacuolar-protein-sorting-13 gene family. In yeast,\n",
      "            vacuolar-protein-sorting-13 proteins are involved in trafficking of\n",
      "            membrane proteins between the trans-Golgi network and the\n",
      "            prevacuolar compartment. While several transcript variants may\n",
      "            exist for this gene, the full-length natures of only two have been\n",
      "            described to date. These two represent the major variants of this\n",
      "            gene and encode distinct isoforms. [provided by RefSeq, Jul 2008].\n",
      "            \n",
      "            Transcript Variant: This variant (1) represents the longer\n",
      "            transcript and encodes the longer isoform (1).\n",
      "            \n",
      "            ##Evidence-Data-START##\n",
      "            Transcript exon combination :: AJ608774.1 [ECO:0000332]\n",
      "            RNAseq introns              :: single sample supports all introns\n",
      "                                           SAMEA1968968, SAMEA1970526\n",
      "                                           [ECO:0000348]\n",
      "            ##Evidence-Data-END##\n",
      "            COMPLETENESS: complete on the 3' end.\n",
      "PRIMARY     REFSEQ_SPAN         PRIMARY_IDENTIFIER PRIMARY_SPAN        COMP\n",
      "            1-28                HY121734.1         1-28\n",
      "            29-8504             AJ608774.1         1-8476\n",
      "            8505-8505           BX682532.4         21312-21312\n",
      "            8506-8815           AJ608774.1         8478-8787\n",
      "            8816-8816           BX682532.4         29838-29838\n",
      "            8817-14662          AJ608774.1         8789-14634\n",
      "            14663-16360         BC051804.1         4065-5762\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..16360\n",
      "                     /organism=\"Homo sapiens\"\n",
      "                     /mol_type=\"mRNA\"\n",
      "                     /db_xref=\"taxon:9606\"\n",
      "                     /chromosome=\"1\"\n",
      "                     /map=\"1p36.22-p36.21\"\n",
      "     gene            1..16360\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /note=\"vacuolar protein sorting 13 homolog D\"\n",
      "                     /db_xref=\"GeneID:55187\"\n",
      "                     /db_xref=\"HGNC:HGNC:23595\"\n",
      "                     /db_xref=\"MIM:608877\"\n",
      "     exon            1..82\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            83..255\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     misc_feature    132..134\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /note=\"upstream in-frame stop codon\"\n",
      "     CDS             159..13325\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /note=\"isoform 1 is encoded by transcript variant 1;\n",
      "                     vacuolar protein sorting-associated protein 13D; vacuolar\n",
      "                     protein sorting 13D\"\n",
      "                     /codon_start=1\n",
      "                     /product=\"vacuolar protein sorting-associated protein 13D\n",
      "                     isoform 1\"\n",
      "                     /protein_id=\"NP_056193.2\"\n",
      "                     /db_xref=\"CCDS:CCDS30588.1\"\n",
      "                     /db_xref=\"GeneID:55187\"\n",
      "                     /db_xref=\"HGNC:HGNC:23595\"\n",
      "                     /db_xref=\"MIM:608877\"\n",
      "                     /translation=\"MLEGLVAWVLNTYLGKYVNNLNTDQLSVALLKGAVELENLPLKK\n",
      "                     DALKELELPFEVKAGFIGKVTLQIPFYRPHVDPWVISISSLHLIGAPEKIQDFNDEKE\n",
      "                     KLLERERKKALLQALEEKWKNDRQQKGESYWYSVTASVVTRIVENIELKIQDVHLRFE\n",
      "                     DGVTNPSHPFAFGICIKNVSMQNAVNEPVQKLMRKKQLDVAEFSIYWDVDCTLLGDLP\n",
      "                     QMELQEAMARSMESRSHHYVLEPVFASALLKRNCSKKPLRSRHSPRIDCDIQLETIPL\n",
      "                     KLSQLQYRQIMEFLKELERKERQVKFRRWKPKVAISKNCREWWYFALNANLYEIREQR\n",
      "                     KRCTWDFMLHRARDAVSYTDKYFNKLKGGLLSTDDKEEMCRIEEEQSFEELKILRELV\n",
      "                     HDRFHKQEELAESLREPQFDSPGACPGAPEPGGGSGMLQYLQSWFPGWGGWYGQQTPE\n",
      "                     GNVVEGLSAEQQEQWIPEEILGTEEFFDPTADASCMNTYTKRDHVFAKLNLQLQRGTV\n",
      "                     TLLHKEQGTPQMNESAFMQLEFSDVKLLAESLPRRNSSLLSVRLGGLFLRDLATEGTM\n",
      "                     FPLLVFPNPQKEVGRVSQSFGLQTTSADRSDHYPAADPDGPVFEMLYERNPAHSHFER\n",
      "                     RLNVSTRPLNIIYNPQAIKKVADFFYKGKVHTSGFGYQSELELRVAEAARRQYNKLKM\n",
      "                     QTKAEIRQTLDRLLVGDFIEESKRWTVRLDISAPQVIFPDDFKFKNPVLVVVDLGRML\n",
      "                     LTNTQDNSRRKSRDGSASEETQFSDDEYKTPLATPPNTPPPESSSSNGEKTPPFSGVE\n",
      "                     FSEEQLQAHLMSTKMYERYSLSFMDLQIMVGRVKDNWKHVQDIDVGPTHVVEKFNVHL\n",
      "                     QLERRLIYTSDPKYPGAVLSGNLPDLKIHINEDKISALKNCFALLTTPEMKTSDTQIK\n",
      "                     EKIFPQEEQRGSLQDSVMNLTQSIVLLEQHTREVLVESQLLLAEFKVNCMQLGVESNG\n",
      "                     RYISVLKVFGTNAHFVKRPYDAEVSLTVHGLLLVDTMQTYGADFDLLMASHKNLSFDI\n",
      "                     PTGSLRDSRAQSPVSGPNVAHLTDGATLNDRSATSVSLDKILTKEQESLIKLEYQFVS\n",
      "                     SECPSMNLDSTLQVISLQVNNLDIILNPETIVELIGFLQKSFPKEKDDLSPQPLMTDF\n",
      "                     ERSFREQGTYQSTYEQNTEVAVEIHRLNLLLLRTVGMANREKYGRKIATASIGGTKVN\n",
      "                     VSMGSTFDMNGSLGCLQLMDLTQDNVKNQYVVSIGNSVGYENIISDIGYFESVFVRME\n",
      "                     DAALTEALSFTFVERSKQECFLNLKMASLHYNHSAKFLKELTLSMDELEENFRGMLKS\n",
      "                     AATKVTTVLATKTAEYSEMVSLFETPRKTREPFILEENEIYGFDLASSHLDTVKLILN\n",
      "                     INIESPVVSIPRKPGSPELLVGHLGQIFIQNFVAGDDESRSDRLQVEIKDIKLYSLNC\n",
      "                     TQLAGREAVGSEGSRMFCPPSGSGSANSQEEAHFTRHDFFESLHRGQAFHILNNTTIQ\n",
      "                     FKLEKIPIERESELTFSLSPDDLGTSSIMKIEGKFVNPVQVVLAKHVYEQVLQTLDNL\n",
      "                     VYSEDLNKYPASATSSPCPDSPLPPLSTCGESSVERKENGLFSHSSLSNTSQKSLSVK\n",
      "                     EVKSFTQIQATFCISELQVQLSGDLTLGAQGLVSLKFQDFEVEFSKDHPQTLSIQIAL\n",
      "                     HSLLMEDLLEKNPDSKYKNLMVSRGAPKPSSLAQKEYLSQSCPSVSNVEYPDMPRSLP\n",
      "                     SHMEEAPNVFQLYQRPTSASRKKQKEVQDKDYPLTPPPSPTVDEPKILVGKSKFDDSL\n",
      "                     VHINIFLVDKKHPEFSSSYNRVNRSIDVDFNCLDVLITLQTWVVILDFFGIGSTADNH\n",
      "                     AMRLPPEGILHNVKLEPHASMESGLQDPVNTKLDLKVHSLSLVLNKTTSELAKANVSK\n",
      "                     LVAHLEMIEGDLALQGSIGSLSLSDLTCHGEFYRERFTTSGEEALIFQTFKYGRPDPL\n",
      "                     LRREHDIRVSLRMASVQYVHTQRFQAEVVAFIQHFTQLQDVLGRQRAAIEGQTVRDQA\n",
      "                     QRCSRVLLDIEAGAPVLLIPESSRSNNLIVANLGKLKVKNKFLFAGFPGTFSLQDKES\n",
      "                     VPSASPTGIPKHSLRKTTSTEEPRGTHSQGQFTMPLAGMSLGSLKSEFVPSTSTKQQG\n",
      "                     PQPTLSVGQESSSPEDHVCLLDCVVVDLQDMDIFAAERHPREYSKAPEDSSGDLIFPS\n",
      "                     YFVRQTGGSLLTEPCRLKLQVERNLDKEISHTVPDISIHGNLSSVHCSLDLYKYKLIR\n",
      "                     GLLENNLGEPIEEFMRPYDLQDPRIHTVLSGEVYTCMCFLIDMVNVSLELKDPKRKEG\n",
      "                     AGSLARFDFKKCKLLYESFSNQTKSINLVSHSMMAFDTRYAGQKTSPGMTNVFSCIFQ\n",
      "                     PAKNSSTTQGSIQIELHFRSTKDSSCFTVVLNNLRVFLIFDWLLLVHDFLHTPSDIKK\n",
      "                     QNHVTPSRHRNSSSESAIVPKTVKSGVVTKRSSLPVSNERHLEVKVNVTGTEFVVIED\n",
      "                     VSCFDTNAIILKGTTVLTYKPRFVDRPFSGSLFGIEVFSCRLGNEHDTALSIVDPVQI\n",
      "                     QMELVGNSSYQNSSGLMDAFNSEDFPPVLEIQLQALDIRLSYNDVQLFLAIAKSIPEQ\n",
      "                     ANAAVPDSVALESDSVGTYLPGASRVGEEIREGTRHTLDPVLELQLARLQELGFSMDD\n",
      "                     CRKALLACQGQLKKAASWLFKNAEPLKSLSLASTSRDSPGAVAAPLISGVEIKAESVC\n",
      "                     ICFIDDCMDCDVPLAELTFSRLNFLQRVRTSPEGYAHFTLSGDYYNRALSGWEPFIEP\n",
      "                     WPCSVSWQQQAASRLHPPRLKLEAKAKPRLDINITSVLIDQYVSTKESWMADYCKDDK\n",
      "                     DIESAKSEDWMGSSVDPPCFGQSLPLVYLRTRSTASLTNLEHQIYARAEVKTPKRRQP\n",
      "                     FVPFALRNHTGCTLWFATLTTTPTRAALSHSGSPGVVPEGNGTFLDDTHNVSEWREVL\n",
      "                     TGEEIPFEFEARGKLRHRHTHDLRIHQLQVRVNGWEQVSPVSVDKVGTFFRYAAPDKN\n",
      "                     SSSSTIGSPSSRTNIIHPQVYFSSLPPVRVVFAVTMEGSARKVITVRSALIVRNRLET\n",
      "                     PMELRLDSPSAPDKPVVLPAIMPGDSFAVPLHLTSWRLQARPKGLGVFFCKAPIHWTN\n",
      "                     VVKTAEISSSKRECHSMDTEKSRFFRFCVAIKKENYPDYMPSNIFSDSAKQIFRQPGH\n",
      "                     TIYLLPTVVICNLLPCELDFYVKGMPINGTLKPGKEAALHTADTSQNIELGVSLENFP\n",
      "                     LCKELLIPPGTQNYMVRMRLYDVNRRQLNLTIRIVCRAEGSLKIFISAPYWLINKTGL\n",
      "                     PLIFRQDNAKTDAAGQFEEHELARSLSPLLFCYADKEQPNLCTMRIGRGIHPEGMPGW\n",
      "                     CQGFSLDGGSGVRALKVIQQGNRPGLIYNIGIDVKKGRGRYIDTCMVIFAPRYLLDNK\n",
      "                     SSHKLAFAQREFARGQGTANPEGYISTLPGSSVVFHWPRNDYDQLLCVRLMDVPNCIW\n",
      "                     SGGFEVNKNNSFHINMRDTLGKCFFLRVEITLRGATYRISFSDTDQLPPPFRIDNFSK\n",
      "                     VPVVFTQHGVAEPRLRTEVKPMTSLDYAWDEPTLPPFITLTVKGAGSSEINCNMNDFQ\n",
      "                     DNRQLYYENFIYIAATYTFSGLQEGTGRPVASNKAITCAELVLDVSPKTQRVILKKKE\n",
      "                     PGKRSQLWRMTGTGMLAHEGSSVPHNPNKPSAARSTEGSAILDIAGLAAVTDNRYEPL\n",
      "                     MLRKPDRRRSTTQTWSFREGKLTCGLHGLVVQAKGGLSGLFDGAEVVLGPDTSMELLG\n",
      "                     PVPPEQQFINQKMRPGSGMLSIRVIPDGPTRALQITDFCHRKSSRSYEVDELPVTEQE\n",
      "                     LQKLKNPDTEQELEVLVRLEGGIGLSLINKVPEELVFASLTGINVHYTQLATSHMLEL\n",
      "                     SIQDVQVDNQLIGTTQPFMLYVTPLSNENEVIETGPAVQVNAVKFPSKSALTNIYKHL\n",
      "                     MITAQRFTVQIEEKLLLKLLSFFGYDQAESEVEKYDENLHEKTAEQGGTPIRYYFENL\n",
      "                     KISIPQIKLSVFTSNKLPLDLKALKSTLGFPLIRFEDAVINLDPFTRVHPYETKEFII\n",
      "                     NDILKHFQEELLSQAARILGSVDFLGNPMGLLNDVSEGVTGLIKYGNVGGLIRNVTHG\n",
      "                     VSNSAAKFAGTLSDGLGKTMDNRHQSEREYIRYHAATSGEHLVAGIHGLAHGIIGGLT\n",
      "                     SVITSTVEGVKTEGGVSGFISGLGKGLVGTVTKPVAGALDFASETAQAVRDTATLSGP\n",
      "                     RTQAQRVRKPRCCTGPQGLLPRYSESQAEGQEQLFKLTDNIQDEFFIAVENIDSYCVL\n",
      "                     ISSKAVYFLKSGDYVDREAIFLEVKYDDLYHCLVSKDHGKVYVQVTKKAVSTSSGVSI\n",
      "                     PGPSHQKPMVHVKSEVLAVKLSQEINYAKSLYYEQQLMLRLSENREQLELDS\"\n",
      "     misc_feature    2145..2147\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphoserine. {ECO:0000244|PubMed:23186163,\n",
      "                     ECO:0000244|PubMed:24275569}; propagated from\n",
      "                     UniProtKB/Swiss-Prot (Q5THJ4.2); phosphorylation site\"\n",
      "     misc_feature    3258..3260\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphoserine. {ECO:0000244|PubMed:23186163};\n",
      "                     propagated from UniProtKB/Swiss-Prot (Q5THJ4.2);\n",
      "                     phosphorylation site\"\n",
      "     misc_feature    3270..3272\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphoserine. {ECO:0000244|PubMed:23186163};\n",
      "                     propagated from UniProtKB/Swiss-Prot (Q5THJ4.2);\n",
      "                     phosphorylation site\"\n",
      "     misc_feature    3282..3284\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphoserine. {ECO:0000244|PubMed:20068231,\n",
      "                     ECO:0000244|PubMed:23186163}; propagated from\n",
      "                     UniProtKB/Swiss-Prot (Q5THJ4.2); phosphorylation site\"\n",
      "     misc_feature    3570..3572\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphoserine. {ECO:0000244|PubMed:23186163,\n",
      "                     ECO:0000244|PubMed:24275569}; propagated from\n",
      "                     UniProtKB/Swiss-Prot (Q5THJ4.2); phosphorylation site\"\n",
      "     misc_feature    4179..4181\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphoserine. {ECO:0000244|PubMed:24275569};\n",
      "                     propagated from UniProtKB/Swiss-Prot (Q5THJ4.2);\n",
      "                     phosphorylation site\"\n",
      "     misc_feature    4950..4952\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphoserine. {ECO:0000244|PubMed:23186163};\n",
      "                     propagated from UniProtKB/Swiss-Prot (Q5THJ4.2);\n",
      "                     phosphorylation site\"\n",
      "     misc_feature    4965..4967\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphoserine. {ECO:0000244|PubMed:23186163};\n",
      "                     propagated from UniProtKB/Swiss-Prot (Q5THJ4.2);\n",
      "                     phosphorylation site\"\n",
      "     misc_feature    5253..5255\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphoserine. {ECO:0000244|PubMed:23186163};\n",
      "                     propagated from UniProtKB/Swiss-Prot (Q5THJ4.2);\n",
      "                     phosphorylation site\"\n",
      "     misc_feature    5439..5441\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphothreonine. {ECO:0000244|PubMed:18669648,\n",
      "                     ECO:0000244|PubMed:19690332, ECO:0000244|PubMed:23186163,\n",
      "                     ECO:0000244|PubMed:24275569}; propagated from\n",
      "                     UniProtKB/Swiss-Prot (Q5THJ4.2); phosphorylation site\"\n",
      "     misc_feature    5451..5453\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphoserine. {ECO:0000244|PubMed:24275569};\n",
      "                     propagated from UniProtKB/Swiss-Prot (Q5THJ4.2);\n",
      "                     phosphorylation site\"\n",
      "     misc_feature    7461..7463\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphoserine. {ECO:0000244|PubMed:19690332,\n",
      "                     ECO:0000244|PubMed:21406692}; propagated from\n",
      "                     UniProtKB/Swiss-Prot (Q5THJ4.2); phosphorylation site\"\n",
      "     misc_feature    8169..8171\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphoserine. {ECO:0000244|PubMed:23186163,\n",
      "                     ECO:0000244|PubMed:24275569}; propagated from\n",
      "                     UniProtKB/Swiss-Prot (Q5THJ4.2); phosphorylation site\"\n",
      "     misc_feature    8739..8741\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphoserine. {ECO:0000244|PubMed:23186163};\n",
      "                     propagated from UniProtKB/Swiss-Prot (Q5THJ4.2);\n",
      "                     phosphorylation site\"\n",
      "     misc_feature    8748..8750\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphoserine. {ECO:0000244|PubMed:23186163};\n",
      "                     propagated from UniProtKB/Swiss-Prot (Q5THJ4.2);\n",
      "                     phosphorylation site\"\n",
      "     misc_feature    9105..9107\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphoserine. {ECO:0000244|PubMed:23186163};\n",
      "                     propagated from UniProtKB/Swiss-Prot (Q5THJ4.2);\n",
      "                     phosphorylation site\"\n",
      "     misc_feature    10728..10730\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"N6-acetyllysine. {ECO:0000244|PubMed:19608861};\n",
      "                     propagated from UniProtKB/Swiss-Prot (Q5THJ4.2);\n",
      "                     acetylation site\"\n",
      "     exon            256..333\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            334..524\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            525..605\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            606..722\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            723..827\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            828..998\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            999..1099\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            1100..1268\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            1269..1370\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            1371..1572\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            1573..1752\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            1753..1883\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            1884..1959\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            1960..2130\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            2131..2261\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            2262..2394\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            2395..4608\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     STS             3768..3954\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /standard_name=\"AL033990\"\n",
      "                     /db_xref=\"UniSTS:93406\"\n",
      "     exon            4609..4760\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            4761..5792\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            5793..5883\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            5884..6010\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            6011..6191\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            6192..6374\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            6375..6597\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            6598..6808\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            6809..6980\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            6981..7093\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            7094..7306\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            7307..7572\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            7573..7706\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            7707..7862\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            7863..8073\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            8074..8148\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            8149..8355\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            8356..8445\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            8446..8586\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            8587..8709\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            8710..8784\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            8785..8890\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            8891..9043\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            9044..9179\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            9180..9227\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            9228..9378\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            9379..9589\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            9590..9837\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            9838..10029\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            10030..10156\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            10157..10299\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            10300..10430\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            10431..10606\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            10607..10730\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            10731..10966\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            10967..11075\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            11076..11239\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            11240..11348\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            11349..11528\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            11529..11642\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            11643..11792\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            11793..11942\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            11943..12039\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            12040..12188\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            12189..12323\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            12324..12491\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            12492..12624\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            12625..12820\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            12821..12952\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            12953..13193\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            13194..16348\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     STS             13440..13681\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /standard_name=\"RH80215\"\n",
      "                     /db_xref=\"UniSTS:86366\"\n",
      "     STS             13808..14681\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /standard_name=\"VPS13D__5075\"\n",
      "                     /db_xref=\"UniSTS:466553\"\n",
      "     STS             14541..14631\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /standard_name=\"SHGC-24271\"\n",
      "                     /db_xref=\"UniSTS:76148\"\n",
      "     STS             16056..16246\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /standard_name=\"A001Y09\"\n",
      "                     /db_xref=\"UniSTS:59564\"\n",
      "     STS             16056..16246\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /standard_name=\"G19775\"\n",
      "                     /db_xref=\"UniSTS:59563\"\n",
      "     STS             16120..16280\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /standard_name=\"D1S1907E\"\n",
      "                     /db_xref=\"UniSTS:64139\"\n",
      "     STS             16122..16349\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /standard_name=\"RH11824\"\n",
      "                     /db_xref=\"UniSTS:83385\"\n",
      "     regulatory      16322..16327\n",
      "                     /regulatory_class=\"polyA_signal_sequence\"\n",
      "                     /gene=\"VPS13D\"\n",
      "     polyA_site      16348\n",
      "                     /gene=\"VPS13D\"\n",
      "ORIGIN      \n",
      "        1 gcgggcctgc gccattgagg agcggcgggg aggaaacgcc gcgcagcgcc gggctggggc\n",
      "       61 gggcggcccg ggacaccgac agatttttct gtgaccatga aagagagaaa taaagaatga\n",
      "      121 tccatgattt ctaaacacct tttcctgagg atatagtcat gttggaaggc cttgtagcct\n",
      "      181 gggttctcaa tacctatttg ggaaaatatg tcaataacct gaacactgac cagctctcag\n",
      "      241 ttgcacttct caaaggtgct gttgaattag aaaacttgcc attaaagaaa gatgccttga\n",
      "      301 aagaattgga attaccattt gaagtcaaag ctggcttcat tgggaaagta acccttcaga\n",
      "      361 ttccctttta tcgcccccat gtggaccctt gggtgatctc catctccagc cttcacttaa\n",
      "      421 ttggagcccc agagaaaata caggatttca atgatgaaaa ggagaagctg ttggaaaggg\n",
      "      481 aacgtaagaa agcactactt caagccctgg aggagaaatg gaagaatgac cgccagcaga\n",
      "      541 aaggggagtc ctattggtat tcagttaccg cctccgtagt tacaaggatt gtggagaata\n",
      "      601 ttgaattaaa aattcaagat gtccatttac gctttgaaga tggtgtcacc aatccctccc\n",
      "      661 atccttttgc ttttggcatc tgcattaaga atgtgtccat gcaaaatgct gtgaatgagc\n",
      "      721 ctgtacagaa actaatgcgg aaaaagcaat tagacgtagc agaatttagc atctattggg\n",
      "      781 atgtcgattg cactttactg ggggatttgc ctcagatgga gttacaggag gccatggcca\n",
      "      841 ggagcatgga gagtcgcagc catcactacg tcctggagcc tgtgtttgca tctgctcttt\n",
      "      901 tgaagagaaa ctgctccaag aagcccctgc ggtctcggca cagtccccgt attgattgtg\n",
      "      961 atattcagct ggagaccatt cccttgaaac tctctcagct gcaataccgg caaatcatgg\n",
      "     1021 aattcctcaa ggagctggaa cgaaaggaga ggcaggtgaa gttccgaagg tggaaaccca\n",
      "     1081 aggtggcgat atctaagaac tgccgagaat ggtggtattt tgctttgaat gctaacttgt\n",
      "     1141 atgagatcag agagcagagg aaacgttgca cctgggactt tatgttgcac cgcgctcgtg\n",
      "     1201 atgctgtatc ttacactgac aaatatttca acaagttaaa aggaggcctg ctgtccacag\n",
      "     1261 atgacaagga ggaaatgtgt cggattgaag aggaacagag ctttgaggaa ttgaagattt\n",
      "     1321 tgcgtgaact ggttcatgat cgatttcaca aacaggaaga actagcagag agtctgcggg\n",
      "     1381 agcctcagtt tgattctcca ggagcctgtc cgggagcccc agaacccggt ggaggcagtg\n",
      "     1441 ggatgctgca gtatctccag tcctggtttc ctggatgggg tggctggtac gggcagcaga\n",
      "     1501 ccccagaagg gaatgtggtt gagggactgt cagcagagca acaggagcag tggattcctg\n",
      "     1561 aagagatcct gggcactgag gagttttttg accccactgc agatgcctcg tgtatgaaca\n",
      "     1621 cgtatacaaa gcgagatcat gtctttgcca aactgaattt gcagttgcag cgaggtacag\n",
      "     1681 tgactctgtt acacaaggag caaggaactc ctcaaatgaa tgaaagtgct ttcatgcagc\n",
      "     1741 tcgagttttc agatgtaaaa cttctagcag agtctcttcc tcgaagaaat tcctcgttgc\n",
      "     1801 tttcagtccg gttgggtgga ctgtttcttc gagacctggc tacagaagga actatgtttc\n",
      "     1861 ctcttctagt cttccctaat ccacaaaaag aagttggcag agtctcacaa tcttttggtc\n",
      "     1921 tacaaactac atctgcagac agaagtgatc attacccagc tgcagatcca gatggccccg\n",
      "     1981 tttttgagat gctgtatgag agaaatccgg cgcacagcca ctttgagagg cggctcaatg\n",
      "     2041 tcagcacaag gcccttgaac atcatataca atccgcaggc cattaaaaaa gtagcagact\n",
      "     2101 ttttctacaa gggaaaggtt catacctcag gttttggtta tcagtctgaa cttgagctga\n",
      "     2161 gagtggctga agctgcccga agacaatata acaagctgaa gatgcagacc aaggcagaaa\n",
      "     2221 tccggcaaac tcttgatcgt ttgctagtgg gtgatttcat tgaggagagt aaacgatgga\n",
      "     2281 ccgtgcggct ggatatttct gcccctcagg tgatatttcc tgatgatttc aaattcaaga\n",
      "     2341 atcctgtgtt agttgtcgtg gatctaggaa gaatgctttt gacgaacacc caagataact\n",
      "     2401 ccaggaggaa aagtagggat gggtcagcat ctgaagagac ccagtttagt gatgatgaat\n",
      "     2461 ataagacccc cctggccaca cctcctaaca ccccacctcc cgagtcaagc agcagcaacg\n",
      "     2521 gagagaaaac acctcccttt tctggagttg agttcagtga agaacagctt caagcacatt\n",
      "     2581 taatgagcac aaagatgtat gagaggtact cgctgtcatt tatggacctc cagatcatgg\n",
      "     2641 ttggacgagt gaaagacaat tggaagcatg tccaggatat tgacgtggga ccaacacatg\n",
      "     2701 tggtagagaa gttcaacgtt cacctacagt tagagcgtcg attgatttat acttcagatc\n",
      "     2761 ccaaatatcc aggagccgtg ctctcaggca acttaccaga cttaaaaatc cacattaatg\n",
      "     2821 aagataaaat atctgcacta aagaattgct ttgctctcct caccacccca gaaatgaaaa\n",
      "     2881 cttctgacac tcagattaaa gaaaagattt ttccccagga ggagcagcgg ggaagtttgc\n",
      "     2941 aagactccgt aatgaattta acccagagca ttgtgttgtt ggagcagcat acccgcgagg\n",
      "     3001 ttctggtgga gtcgcagctc ctcctggcgg aatttaaagt gaactgtatg cagcttggtg\n",
      "     3061 ttgagagcaa tggccggtac atttctgtgc tcaaggtgtt tggtaccaat gctcactttg\n",
      "     3121 tgaagaggcc ttatgatgct gaagtctccc taactgttca tggtttgctc ctggtggata\n",
      "     3181 ccatgcagac atatggtgct gattttgacc ttttgatggc ttcacataag aacttgagct\n",
      "     3241 ttgatattcc aacgggaagc cttcgggata gcagggccca gtctcctgtc tctggaccga\n",
      "     3301 atgtggccca cttaactgat ggagctacac tgaacgaccg atcagctact agtgtttcac\n",
      "     3361 ttgacaaaat tcttaccaaa gagcaagagt cccttattaa gttggaatat cagtttgtga\n",
      "     3421 gttcagagtg cccatcgatg aatttagaca gtactcttca ggtgatttcc ctacaggtga\n",
      "     3481 ataatttaga tattatcctc aatccagaga cgattgtgga gctaattggt tttcttcaaa\n",
      "     3541 aatcctttcc caaggaaaaa gatgatttaa gtcctcaacc tttaatgact gattttgaaa\n",
      "     3601 gaagcttcag agaacaagga acttaccagt ctacatatga acaaaacact gaggttgcag\n",
      "     3661 tggaaatcca taggctgaac ttactgcttc ttcggacagt gggcatggca aatagagaga\n",
      "     3721 aatatggcag aaaaattgca actgcaagta taggtggcac caaagttaat gtctcaatgg\n",
      "     3781 gtagcacgtt tgacatgaat ggttctcttg gctgtttaca gcttatggat ttgacacaag\n",
      "     3841 ataacgttaa aaaccagtat gttgtcagca ttgggaattc tgtaggctat gaaaatatca\n",
      "     3901 tcagtgatat tggctacttt gaatctgtgt ttgtcagaat ggaagatgca gccctcactg\n",
      "     3961 aagctttgag tttcacgttt gttgagagat ctaaacagga gtgttttctc aacctgaaga\n",
      "     4021 tggcttcttt acattataac cactctgcta agtttttgaa ggagttgacg ttatccatgg\n",
      "     4081 atgaactgga agaaaatttt cgaggtatgc tgaaaagcgc agccaccaaa gtcaccacag\n",
      "     4141 tactagctac caagactgcc gagtatagcg agatggtatc gctctttgaa actccaagga\n",
      "     4201 agactcggga accctttatc ttagaggaaa atgaaatata tgggtttgac ctagcttcgt\n",
      "     4261 ctcatttgga cactgtaaag ctaatcttga acataaacat tgaatcacca gttgtttcta\n",
      "     4321 tccctcggaa gccggggagt cctgagttgt tggtgggaca cttgggacag atattcatcc\n",
      "     4381 agaattttgt ggcgggagat gatgaatcca gaagtgaccg tctgcaggtg gaaatcaagg\n",
      "     4441 acattaaact gtattctttg aattgcaccc agttggcagg tagagaagct gttgggtctg\n",
      "     4501 aaggaagccg gatgttttgc ccaccttccg ggtctggcag tgccaacagt caggaggaag\n",
      "     4561 ctcatttcac acgacatgat ttctttgaat ctttgcatag aggtcaagct tttcacatcc\n",
      "     4621 tgaacaacac caccattcag tttaaactgg agaagatccc tatagagaga gaatctgaat\n",
      "     4681 tgactttttc tcttagccca gatgacctgg gaacttctag catcatgaag attgaaggaa\n",
      "     4741 aatttgtcaa tccagttcag gtggtgttag caaagcatgt atatgagcag gttttacaaa\n",
      "     4801 ccctggacaa tctcgtgtac agtgaagatc tgaataagta tccagccagt gctacctcct\n",
      "     4861 ccccttgccc tgattctcct ctgcctcccc tcagtacctg tggagaatct tctgttgaaa\n",
      "     4921 ggaaggagaa tggattgttc agccactcca gcctttctaa cacctctcag aagtcattgt\n",
      "     4981 cagtgaagga agtcaaatcc tttactcaga ttcaagccac cttttgtata tcagagcttc\n",
      "     5041 aggttcagct aagtggagat ctgactttgg gggcccaagg tcttgtgagc ttaaagtttc\n",
      "     5101 aggactttga ggtggaattc agtaaagacc atccccagac tttatctatt cagattgccc\n",
      "     5161 tgcattctct gctgatggag gacttattgg agaagaatcc agattctaaa tataagaacc\n",
      "     5221 tgatggtgtc tcgaggagcc cctaagccat ctagtttagc acaaaaagaa tacctttctc\n",
      "     5281 agtcttgccc ctcagtgtcc aatgtggaat atcctgatat gcctcggtct ctcccttccc\n",
      "     5341 acatggaaga agctcctaat gtcttccagt tgtatcaaag gcccacctct gcgtcccgga\n",
      "     5401 aaaagcaaaa ggaagtccaa gacaaggact atcccttgac cccacctcct tctccaacag\n",
      "     5461 tggatgagcc caagatactt gttggaaaga gtaaatttga tgattcctta gtccacatca\n",
      "     5521 acatattctt ggtagataag aaacatccag aattctcttc cagttacaat cgagttaacc\n",
      "     5581 ggagcattga tgttgatttt aattgcttgg atgtgctgat cacactgcaa acctgggttg\n",
      "     5641 tgatattaga cttttttgga atcggctcca ctgcagacaa ccacgcaatg aggctgcctc\n",
      "     5701 ctgagggcat tctgcacaac gtgaagttgg agccacatgc ctccatggag tctggacttc\n",
      "     5761 aggatccagt gaacaccaaa ctggatctca aggttcattc actttctcta gtgctgaata\n",
      "     5821 agaccaccag tgagcttgcc aaagcaaatg tgtccaaatt agtagcacac ctggaaatga\n",
      "     5881 ttgagggaga cctggcctta cagggcagca ttgggagtct gtctctaagt gacctcacat\n",
      "     5941 gccatggaga gttctacaga gaacggttca ctaccagtgg tgaagaagca ctcatcttcc\n",
      "     6001 agacttttaa atacggacgg cctgaccctc tgctccggag agaacacgac attcgcgtga\n",
      "     6061 gcctccggat ggcctctgtg cagtatgtgc atactcagcg tttccaggca gaggtggtgg\n",
      "     6121 ccttcattca gcatttcact cagctgcagg atgtcttagg gcgccagcga gctgctattg\n",
      "     6181 aggggcagac ggtgagagat caagcccagc gctgttcacg ggttctcctg gatattgagg\n",
      "     6241 ctggtgctcc cgttctcttg atcccagaaa gttccagatc aaataatctg attgtagcaa\n",
      "     6301 atttggggaa gttgaaagtc aaaaataagt ttctgtttgc tggttttcct ggcacctttt\n",
      "     6361 ccctacaaga taaggaatct gtgccttcag cttccccaac gggtattccc aaacacagtc\n",
      "     6421 tgaggaaaac gacaagcacg gaggagccca ggggaaccca ttcccagggg cagttcacga\n",
      "     6481 tgcctcttgc tggaatgagc ctaggaagcc tgaagagtga gtttgtgccc agtacctcca\n",
      "     6541 ccaagcagca agggccgcaa cccacactgt ctgttggcca agagtccagt agtccagaag\n",
      "     6601 accatgtctg cctgctggat tgcgttgtcg tggatctcca ggacatggac atctttgctg\n",
      "     6661 cagagagaca tccgagagaa tactcgaagg caccagagga tagtagtgga gatctgatct\n",
      "     6721 tcccttccta ttttgtgcga cagacaggag gaagcctctt aaccgagcct tgtaggctga\n",
      "     6781 aattgcaggt ggaaaggaat ttggacaaag aaataagtca tactgtgcca gacatatcta\n",
      "     6841 tccatggcaa tctctcctca gtccactgct ctctggatct gtataaatac aagctgatcc\n",
      "     6901 gcggcttatt agagaacaac ctgggagaac ccatagagga atttatgcgg ccttatgatt\n",
      "     6961 tacaagatcc aagaattcat actgtcctga gtggagaagt gtacacctgt atgtgcttcc\n",
      "     7021 tcattgatat ggtgaatgta agtctggagc ttaaagatcc aaaaagaaaa gaaggtgctg\n",
      "     7081 ggtccctagc cagatttgac ttcaagaaat gcaaactgct ctatgaaagt ttttccaacc\n",
      "     7141 aaaccaagtc cattaacttg gtttcccatt ccatgatggc ttttgacacc cgttatgctg\n",
      "     7201 ggcagaagac cagccctggc atgacgaatg tgttcagctg tatctttcag cccgctaaga\n",
      "     7261 acagcagcac cacccaaggg tccattcaga ttgaactaca tttcagatct accaaggatt\n",
      "     7321 cctcctgctt tacagtagtt ctcaacaatc tccgtgtgtt tctcatattt gactggctac\n",
      "     7381 tgttagtcca tgattttctc cacactccca gtgatattaa gaaacaaaat catgttactc\n",
      "     7441 cttctcgcca ccgtaactct agcagcgaat ctgctatagt tcccaaaact gtgaagagtg\n",
      "     7501 gagtagttac caagcggtct tcccttcctg tgtccaatga aaggcacctg gaggtcaagg\n",
      "     7561 tcaatgtaac aggtacggag tttgtggtca ttgaagatgt gtcctgcttc gacaccaatg\n",
      "     7621 ccattattct gaaaggcacc acagtgctca cctataagcc ccggtttgtt gatcgcccct\n",
      "     7681 tttcaggaag tttgtttggc attgaggtgt tttcatgccg actagggaat gagcatgata\n",
      "     7741 cagctctttc aattgtggat cccgtacaaa ttcaaatgga gttggtgggg aattcttctt\n",
      "     7801 atcaaaatag ttcaggattg atggatgcat tcaatagtga agatttccca cctgtcctgg\n",
      "     7861 agattcagtt acaagccctg gatatcagac tctcctataa tgatgttcag ctgtttcttg\n",
      "     7921 ccattgcaaa atccatccca gagcaagcta atgctgcagt gccagactca gtggccctgg\n",
      "     7981 agtcagactc cgttggcact taccttccag gtgcatctcg cgttggagag gaaatcagag\n",
      "     8041 aagggacaag acacacctta gatcctgtct tggagttaca gctggctagg ctgcaggagc\n",
      "     8101 tgggattcag catggatgat tgtcgcaaag ctcttttggc gtgtcaaggc caattgaaaa\n",
      "     8161 aggcagcaag ttggttgttt aagaatgcgg aacctctgaa gtctctttcc ttggcctcca\n",
      "     8221 ccagccgaga tagcccaggg gctgtggcag cgccattgat ctctggcgtg gagatcaaag\n",
      "     8281 ctgagagtgt gtgcatctgt ttcatcgatg actgcatgga ttgtgatgtt cctctcgctg\n",
      "     8341 aactcacctt ttcccgtctg aattttcttc agcgtgtaag aactagccct gaaggctatg\n",
      "     8401 cccacttcac cctttctgga gattattata accgtgctct ttcaggctgg gagccattta\n",
      "     8461 ttgagccttg gccatgctct gtatcctggc aacagcaggc agctagtcgt ctccatcctc\n",
      "     8521 ctcgactgaa gctagaagcc aaggccaaac ctcgtttgga tatcaatatc acttctgtgc\n",
      "     8581 taattgacca gtatgtaagt accaaggaat cgtggatggc agattactgt aaagatgaca\n",
      "     8641 aggacataga gtcagctaaa tcagaagact ggatgggctc ttcggtggat cctccatgtt\n",
      "     8701 ttggacaaag cctccccctt gtctacctta gaactaggag tacagccagt ctgactaacc\n",
      "     8761 tagagcacca gatctatgct agagcagagg tgaaaacccc caagcgccgg cagccatttg\n",
      "     8821 tcccctttgc tctgaggaac cacacggggt gcactttgtg gtttgccacc ctgaccacca\n",
      "     8881 cacccaccag agctgcactc tctcacagtg ggagtccagg ggtagttcca gaagggaacg\n",
      "     8941 gaacatttct cgatgatact cacaatgtta gtgaatggcg agaagtcctt acaggtgaag\n",
      "     9001 agattccctt tgaatttgaa gcaagaggaa agttaagaca cagacacacc catgacctcc\n",
      "     9061 ggattcatca actgcaagtg agagtaaatg gctgggagca agtgagccca gtgtctgtgg\n",
      "     9121 acaaagtcgg gacctttttt cgatatgcag caccagataa aaattcatct tcctctacga\n",
      "     9181 ttggcagccc aagcagcaga acaaatatta tacatcccca ggtttatttc tcttcactcc\n",
      "     9241 caccagtgcg ggtggtcttt gcagtgacta tggaaggcag tgcacggaaa gtcatcactg\n",
      "     9301 tccggtcagc cctcattgtg aggaacagac ttgagacacc aatggaacta agactggata\n",
      "     9361 gcccatcagc tccagacaag ccagtggtgc ttcctgctat catgccaggg gattcgtttg\n",
      "     9421 ctgtgccttt acacctcact tcttggcggc tacaggcccg gcccaaagga ttgggtgtat\n",
      "     9481 ttttctgtaa ggctcccatt cattggacca atgtagtgaa gactgcagaa attagtagca\n",
      "     9541 gtaaacgaga gtgccactct atggacacag aaaaaagccg atttttcagg ttttgtgtgg\n",
      "     9601 ctataaagaa agagaattat ccagattata tgccctcaaa catattttct gacagtgcaa\n",
      "     9661 aacagatttt cagacagcct gggcatacca tatatctcct gccaactgtg gtaatctgca\n",
      "     9721 acttgctacc ctgtgaactt gatttttatg ttaaaggaat gccaattaat gggacgctga\n",
      "     9781 aacctggcaa ggaggcagct ctccatacag ctgatacatc ccagaacatt gagctggggg\n",
      "     9841 tatcactgga gaatttcccc ctctgtaaag aattgctcat tccacctgga acccaaaact\n",
      "     9901 atatggtgag aatgcgactc tatgacgtca accgtcggca gctgaacctc accatccgga\n",
      "     9961 ttgtgtgtcg agcagaagga tccttaaaga tcttcatttc tgctccatat tggctgatta\n",
      "    10021 acaaaacagg gttgccactg atcttcagac aggacaatgc caagacagat gctgcaggcc\n",
      "    10081 agtttgagga gcatgagctg gcccgtagcc tgagtcctct cttattctgc tatgctgaca\n",
      "    10141 aagagcagcc aaacctctgc acgatgagaa tcggaagggg gattcatcca gaaggcatgc\n",
      "    10201 cgggctggtg tcagggcttc tccctggatg gtggtagtgg tgtccgagct ttgaaagtca\n",
      "    10261 tccagcaagg aaaccgccca gggctgatct ataacattgg tattgatgtc aagaaaggcc\n",
      "    10321 gaggtcgata cattgatacc tgcatggtca tctttgcccc ccgttacctg ttagataata\n",
      "    10381 aatcatctca caagcttgca tttgcacaga gggaatttgc caggggacag ggaacagcca\n",
      "    10441 atcccgaagg ttacatttcc acccttcctg gttccagtgt ggtgttccac tggcctcgga\n",
      "    10501 atgactatga tcagctattg tgtgtcagac tgatggacgt tcccaattgt atttggtctg\n",
      "    10561 gaggctttga agtcaacaag aataattcct tccatatcaa catgagggat accttgggaa\n",
      "    10621 aatgcttctt cctacgagtg gaaattactc tccgaggagc tacgtatagg atctcattta\n",
      "    10681 gtgacacaga tcagttacct cctcctttcc gaattgacaa cttttctaag gtcccggttg\n",
      "    10741 tctttactca gcatggcgta gctgaaccca ggctccggac tgaagtgaag cccatgactt\n",
      "    10801 cattggatta tgcctgggac gaacccacct tgccaccttt tatcactctg actgttaaag\n",
      "    10861 gggcagggtc ctctgagatc aactgcaaca tgaatgattt ccaggataat cggcagcttt\n",
      "    10921 attatgaaaa tttcatttac attgctgcta catatacatt ctctggcttg caggagggaa\n",
      "    10981 caggcaggcc tgtggcttcc aacaaggcca ttacctgtgc ggagctcgtt ttggatgtct\n",
      "    11041 cacccaagac acaaagagtc attttaaaaa agaaggaacc aggaaagcgt tctcagctgt\n",
      "    11101 ggaggatgac aggaacagga atgctggccc atgagggctc ctcagttcct cacaatccca\n",
      "    11161 ataagccctc agccgcccgc tccaccgagg ggtctgccat cttagatatt gctggtctcg\n",
      "    11221 ctgcagtgac tgacaacaga tacgagccac tgatgctgag aaagcctgac cgcaggcgaa\n",
      "    11281 gcacaactca gacgtggagt ttccgagaag gaaaactgac ctgtgggtta catgggttgg\n",
      "    11341 tcgtccaggc caaaggagga ctttctggtt tgtttgatgg agctgaagtt gttcttggtc\n",
      "    11401 ctgacacttc catggagctt ttggggccag ttccacctga acaacaattt attaatcaaa\n",
      "    11461 aaatgagacc tggttctgga atgttatcca tcagagtcat cccagatgga ccaactagag\n",
      "    11521 cactccagat aacagatttc tgccaccgga aaagcagccg ttcatatgaa gtggatgaac\n",
      "    11581 ttcctgtcac cgaacaagag ctgcagaaat taaagaatcc agatacagag caggaattgg\n",
      "    11641 aagtgcttgt gaggttagaa ggtggaattg ggttgtcctt aattaataaa gtcccagaag\n",
      "    11701 aactggtctt tgcaagtctt acaggaatca atgtgcacta tacacagctg gcaaccagtc\n",
      "    11761 acatgcttga actcagcata caggatgtac aggtggacaa tcagctcatt ggtaccacgc\n",
      "    11821 agcccttcat gctctatgtg actcccctga gcaatgagaa tgaggtcatc gagaccggcc\n",
      "    11881 cagctgtgca agtcaacgca gtgaagttcc ccagtaagag tgcactgacc aacatctaca\n",
      "    11941 agcatctgat gatcacagct cagagattca cagtgcaaat tgaggagaaa ctgctcctca\n",
      "    12001 agctgctaag tttctttggc tacgatcaag cagaatcaga ggtggaaaaa tatgatgaaa\n",
      "    12061 acctccatga aaagacagct gagcaaggtg gaacaccaat tcgatactac tttgaaaatc\n",
      "    12121 tcaaaatcag cattcctcag atcaagctaa gtgtgttcac ctccaacaag ctcccattgg\n",
      "    12181 atcttaaggc cctaaaaagc accttggggt ttcctttgat acggtttgaa gacgctgtga\n",
      "    12241 ttaatctaga tccattcact cgggtacatc cctatgagac caaggagttc atcatcaatg\n",
      "    12301 atatcctcaa acatttccag gaggaactcc tcagccaggc agctcgaatc ctgggatcag\n",
      "    12361 tggattttct tggcaatcct atggggcttt tgaatgatgt ttctgaaggg gttactggac\n",
      "    12421 tgataaaata tggaaatgtc gggggcctca tcagaaatgt tacacacgga gtatcaaact\n",
      "    12481 ctgctgccaa gtttgctgga acattatcag atggcttagg gaagacgatg gacaatcggc\n",
      "    12541 atcagtcaga gcgggagtac atcaggtacc atgcagccac aagtggtgaa caccttgtag\n",
      "    12601 ccggcatcca tggcctggct catggtatca ttggtggact gaccagtgtt ataacttcga\n",
      "    12661 cagtggaagg tgtgaaaaca gaagggggtg tcagcggttt catatctggc cttggaaaag\n",
      "    12721 ggcttgttgg cactgtaacc aagccagtgg caggcgccct ggattttgca tcagaaacag\n",
      "    12781 cccaggcggt gagagacaca gccacactca gcggccccag gactcaagca cagagggttc\n",
      "    12841 ggaaaccgcg ttgctgcacg gggccccagg ggctgcttcc ccgatattct gagagccagg\n",
      "    12901 cggaaggaca ggagcagctc ttcaaactca cagacaacat acaggacgaa ttcttcatcg\n",
      "    12961 ctgtggagaa cattgacagc tactgcgtgc tcatctcctc caaagctgtt tacttcctga\n",
      "    13021 aaagtggaga ctacgtggat cgagaagcca ttttcctaga agtcaaatac gatgacctct\n",
      "    13081 accactgcct tgtctccaaa gaccatggga aggtgtatgt gcaggtgacc aagaaagccg\n",
      "    13141 tgagcacgag cagtggagtg tccatccccg gcccctccca ccagaagccc atggtccatg\n",
      "    13201 tgaaatctga ggtccttgct gtcaagttgt cacaagaaat aaactacgca aagagcctct\n",
      "    13261 actatgaaca gcagcttatg ttaagactca gcgaaaaccg agagcagctg gagctggact\n",
      "    13321 cctgaagccc cgctgctgag atgggcgctc ccgacacagc gcagacccac caggaggaaa\n",
      "    13381 gaggcccagc tctcagctga cgatggaggc agaaccggag tcgggtttgg ggaagttgtc\n",
      "    13441 aaggaatgag ggaaagtaaa tcctcatgag gaaaagtaca aatggaaatc gtattaattt\n",
      "    13501 gtgaggcagg gagttatttt agattatggg aaataatttt taaaggtatt ggttaaataa\n",
      "    13561 cgtttaaaaa catgtactga gatgaatcta atttttagat tgccctgtat tttgttaaca\n",
      "    13621 tgtatatatg tacaacagtg tgtttgtaaa tatataggaa cgtttctgaa cagggtctgt\n",
      "    13681 gctatgtgta aaggtttgtt aactgtaaag taatataaag ttatattgga tcttctattg\n",
      "    13741 cactaattct agatgtctaa ttcaggatac tgtctataga aaggcattct taaaagttaa\n",
      "    13801 agaatgttac gtcttagttt tggagactaa agtattccca gtaaagtggg ttgaggtgag\n",
      "    13861 ggctgtggtc ctgaaaggga cgcctttgac atcgtggctg tccagttggg ctgtgagctg\n",
      "    13921 tggcacccag gactggcgct ggcccttcag aaggatctag gagaggggct tgggagccca\n",
      "    13981 cttttaattt ctcaccccca ttttacaaag agtgcttaga ttcttacaaa ttatgatgta\n",
      "    14041 agttatccat ttggcttttt cctaactagt cttaccaaac ttagggggaa acctgtgctc\n",
      "    14101 cattaccaca tgggtgcaag tcagcattgt aagttttctc aggttattat tattagagag\n",
      "    14161 gttggaaaca ttggtaaact ctgttgattg agaaggaaaa aaaaagtccc attgaactgt\n",
      "    14221 tgcaacaaat cagaaatcca cataaaagtg ctctcctgcc tgggcagcaa caaccaagaa\n",
      "    14281 caaagccccg ggactgtttt ctttttaata aagccacagg caggcatcgt agctccacag\n",
      "    14341 cccgagggga cacaggatgg aaaccccagg atgagaaggg agcagggaga gttccagaaa\n",
      "    14401 gggggatgaa ataggagtat taaaaagctg cgttggtaag tttttcatgg aaccaagatt\n",
      "    14461 tgacaaaggc atctcttatc cttggtttta aattcctgct gggagcaagg cctggtatga\n",
      "    14521 gcgccctggg tcttgttttt ggtgtttcgc ttttctgtaa ggattaagca gatagggaga\n",
      "    14581 agggaaaagg ggcctcactt tagaatgaat gagtcacctt gtgattttta aatttttatt\n",
      "    14641 ttaataaagc taatcaattt ctacaacctt gtcacatgta gctgagtctg ggatgactca\n",
      "    14701 gtggatcagt ggatgcgtgg aaggttttgg tgtttataac tcatgaccca aatccttcca\n",
      "    14761 agacacaaat gaagctaatc ttcttcccct ccctttcctc tcccacttcc cctctgtgtc\n",
      "    14821 tgtgtctgtg tgtgtgtgtg tgtgtgtgtg tgtgtgcgcg cgcgcgcgtg catgcagaga\n",
      "    14881 ggaaggaagg gagctttagc acaagtagct acatagtgtt caaggatcag ctttcttgaa\n",
      "    14941 ctccgaggct tggttgcctt tgctgaactc caagttagtg gattgcagaa tggaaacttg\n",
      "    15001 gcttttgcgg cactgggtga gttttagttt gtgtgtgtct tgctgggggg tggtgatgat\n",
      "    15061 tgtctcagca ctcacgcact gcacaagatg gcagcaggat acagcactgc acaagatggc\n",
      "    15121 agctcctctg cagcttcctc ctcagcctcc ctccttgcac ccccacaggt ttggcttgtg\n",
      "    15181 gtttttgtca tcagtaacct actgcctgag atcatgatct cttaaaagat gagactctcg\n",
      "    15241 gaagggttga ttgtatgcgt cagtgagcct tctatcacct tctggaacaa agtcacttga\n",
      "    15301 aatctcttga tgagattaag gagtttagtg ttactaagaa aatctgcttt gggccgcagc\n",
      "    15361 agtgctgggt gttctcagac ctgactgagg aagttagctg cgggctgccc tgtgggctgg\n",
      "    15421 tgcttcagga ggaatccaga gaagtgttca gatgcccccc ttgggctcct ttctaatttt\n",
      "    15481 aatcagctct ttaaatagct gcccatctcc tgtgattgca caaccaagca ctttgacatt\n",
      "    15541 tgcaccttag gagaggcaga tgttaaaatg gaatccaaag accacctagg gcggggctgg\n",
      "    15601 gtgggagatg ggagggccaa ctgcgagctg ctccacttct cagctctccc ctgccctgca\n",
      "    15661 gccctgggcc agacaaggcc agaaggtttc aggggcattt gacatcccct cctggttctc\n",
      "    15721 accaggaaaa catccaaagc tttggaggaa acaggccctg cccctggctc cttaaatgcc\n",
      "    15781 ccgtctcttt gtaaactgat attcagccag caatgcctaa gactttgtta agatcatttc\n",
      "    15841 tactgctttt ctttctgctt caaacacaca gttcgtctct gaggaaagta aaataaatgg\n",
      "    15901 aataagagta aattgggtaa ggagatatcc aaagctaccc agtcccttga cccagcacag\n",
      "    15961 ttggccgacc cgtgtcactc cctggctgtc gctgcttctc tgtgctcact gaagggtgag\n",
      "    16021 ccaggccagt gcttccccag cccctgggcc tggtcactac acagtggaaa acagacaagc\n",
      "    16081 ggccccttcc ccaaatccca agagtgtctt gctgcttggt gggtgctcat cgcaatgttc\n",
      "    16141 tgaaggctcc agggccactt tgtttgtaag tatgatctgg gcctcaaaat accatagtag\n",
      "    16201 ctgcttgata aaattctaaa aatatctggt tctctattat gtaaacacta ttacagtcac\n",
      "    16261 cagtgtgtga agactcttga gtctggttct catatcagag tcatcatttt tcttcctgtg\n",
      "    16321 gaataaaatg ccttgtggac ttcccaaaaa aaaaaaaaaa\n",
      "//\n",
      "\n",
      "LOCUS       NM_018156              16285 bp    mRNA    linear   PRI 03-OCT-2017\n",
      "DEFINITION  Homo sapiens vacuolar protein sorting 13 homolog D (VPS13D),\n",
      "            transcript variant 2, mRNA.\n",
      "ACCESSION   NM_018156\n",
      "VERSION     NM_018156.3\n",
      "KEYWORDS    RefSeq.\n",
      "SOURCE      Homo sapiens (human)\n",
      "  ORGANISM  Homo sapiens\n",
      "            Eukaryota; Metazoa; Chordata; Craniata; Vertebrata; Euteleostomi;\n",
      "            Mammalia; Eutheria; Euarchontoglires; Primates; Haplorrhini;\n",
      "            Catarrhini; Hominidae; Homo.\n",
      "REFERENCE   1  (bases 1 to 16285)\n",
      "  AUTHORS   Nakada TA, Boyd JH, Russell JA, Aguirre-Hernandez R, Wilkinson MD,\n",
      "            Thair SA, Nakada E, McConechy MK, Fjell CD and Walley KR.\n",
      "  TITLE     VPS13D Gene Variant Is Associated with Altered IL-6 Production and\n",
      "            Mortality in Septic Shock\n",
      "  JOURNAL   J Innate Immun 7 (5), 545-553 (2015)\n",
      "   PUBMED   25896417\n",
      "  REMARK    GeneRIF: genetic polymorphism is associated with il-6 production\n",
      "            and mortality in patients with septic shock\n",
      "REFERENCE   2  (bases 1 to 16285)\n",
      "  AUTHORS   Koriyama H, Nakagami H, Katsuya T, Sugimoto K, Yamashita H, Takami\n",
      "            Y, Maeda S, Kubo M, Takahashi A, Nakamura Y, Ogihara T, Rakugi H,\n",
      "            Kaneda Y and Morishita R.\n",
      "  TITLE     Identification of evidence suggestive of an association with\n",
      "            peripheral arterial disease at the OSBPL10 locus by genome-wide\n",
      "            investigation in the Japanese population\n",
      "  JOURNAL   J. Atheroscler. Thromb. 17 (10), 1054-1062 (2010)\n",
      "   PUBMED   20610895\n",
      "  REMARK    GeneRIF: Observational study and genome-wide association study of\n",
      "            gene-disease association. (HuGE Navigator)\n",
      "REFERENCE   3  (bases 1 to 16285)\n",
      "  AUTHORS   Gonzales PA, Pisitkun T, Hoffert JD, Tchapyjnikov D, Star RA, Kleta\n",
      "            R, Wang NS and Knepper MA.\n",
      "  TITLE     Large-scale proteomics and phosphoproteomics of urinary exosomes\n",
      "  JOURNAL   J. Am. Soc. Nephrol. 20 (2), 363-379 (2009)\n",
      "   PUBMED   19056867\n",
      "REFERENCE   4  (bases 1 to 16285)\n",
      "  AUTHORS   Velayos-Baeza A, Vettori A, Copley RR, Dobson-Stone C and Monaco\n",
      "            AP.\n",
      "  TITLE     Analysis of the human VPS13 gene family\n",
      "  JOURNAL   Genomics 84 (3), 536-549 (2004)\n",
      "   PUBMED   15498460\n",
      "REFERENCE   5  (bases 1 to 16285)\n",
      "  AUTHORS   Nakayama M, Kikuno R and Ohara O.\n",
      "  TITLE     Protein-protein interactions between large proteins: two-hybrid\n",
      "            screening using a functionally classified library composed of long\n",
      "            cDNAs\n",
      "  JOURNAL   Genome Res. 12 (11), 1773-1784 (2002)\n",
      "   PUBMED   12421765\n",
      "COMMENT     REVIEWED REFSEQ: This record has been curated by NCBI staff. The\n",
      "            reference sequence was derived from HY121734.1, AJ608775.1,\n",
      "            BX682532.4 and BC051804.1.\n",
      "            On Dec 15, 2014 this sequence version replaced gi:54607140.\n",
      "            \n",
      "            Summary: This gene encodes a protein belonging to the\n",
      "            vacuolar-protein-sorting-13 gene family. In yeast,\n",
      "            vacuolar-protein-sorting-13 proteins are involved in trafficking of\n",
      "            membrane proteins between the trans-Golgi network and the\n",
      "            prevacuolar compartment. While several transcript variants may\n",
      "            exist for this gene, the full-length natures of only two have been\n",
      "            described to date. These two represent the major variants of this\n",
      "            gene and encode distinct isoforms. [provided by RefSeq, Jul 2008].\n",
      "            \n",
      "            Transcript Variant: This variant (2) lacks an alternate in-frame\n",
      "            exon, compared to variant 1, resulting in a shorter protein\n",
      "            (isoform 2), compared to isoform 1.\n",
      "            \n",
      "            ##Evidence-Data-START##\n",
      "            Transcript exon combination :: AJ608775.1 [ECO:0000332]\n",
      "            RNAseq introns              :: single sample supports all introns\n",
      "                                           SAMEA1965299, SAMEA1968540\n",
      "                                           [ECO:0000348]\n",
      "            ##Evidence-Data-END##\n",
      "            COMPLETENESS: complete on the 3' end.\n",
      "PRIMARY     REFSEQ_SPAN         PRIMARY_IDENTIFIER PRIMARY_SPAN        COMP\n",
      "            1-28                HY121734.1         1-28\n",
      "            29-8504             AJ608775.1         1-8476\n",
      "            8505-8505           BX682532.4         21312-21312\n",
      "            8506-8740           AJ608775.1         8478-8712\n",
      "            8741-8741           BX682532.4         29838-29838\n",
      "            8742-14587          AJ608775.1         8714-14559\n",
      "            14588-16285         BC051804.1         4065-5762\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..16285\n",
      "                     /organism=\"Homo sapiens\"\n",
      "                     /mol_type=\"mRNA\"\n",
      "                     /db_xref=\"taxon:9606\"\n",
      "                     /chromosome=\"1\"\n",
      "                     /map=\"1p36.22-p36.21\"\n",
      "     gene            1..16285\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /note=\"vacuolar protein sorting 13 homolog D\"\n",
      "                     /db_xref=\"GeneID:55187\"\n",
      "                     /db_xref=\"HGNC:HGNC:23595\"\n",
      "                     /db_xref=\"MIM:608877\"\n",
      "     exon            1..82\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            83..255\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     misc_feature    132..134\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /note=\"upstream in-frame stop codon\"\n",
      "     CDS             159..13250\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /note=\"isoform 2 is encoded by transcript variant 2;\n",
      "                     vacuolar protein sorting-associated protein 13D; vacuolar\n",
      "                     protein sorting 13D\"\n",
      "                     /codon_start=1\n",
      "                     /product=\"vacuolar protein sorting-associated protein 13D\n",
      "                     isoform 2\"\n",
      "                     /protein_id=\"NP_060626.2\"\n",
      "                     /db_xref=\"CCDS:CCDS30589.1\"\n",
      "                     /db_xref=\"GeneID:55187\"\n",
      "                     /db_xref=\"HGNC:HGNC:23595\"\n",
      "                     /db_xref=\"MIM:608877\"\n",
      "                     /translation=\"MLEGLVAWVLNTYLGKYVNNLNTDQLSVALLKGAVELENLPLKK\n",
      "                     DALKELELPFEVKAGFIGKVTLQIPFYRPHVDPWVISISSLHLIGAPEKIQDFNDEKE\n",
      "                     KLLERERKKALLQALEEKWKNDRQQKGESYWYSVTASVVTRIVENIELKIQDVHLRFE\n",
      "                     DGVTNPSHPFAFGICIKNVSMQNAVNEPVQKLMRKKQLDVAEFSIYWDVDCTLLGDLP\n",
      "                     QMELQEAMARSMESRSHHYVLEPVFASALLKRNCSKKPLRSRHSPRIDCDIQLETIPL\n",
      "                     KLSQLQYRQIMEFLKELERKERQVKFRRWKPKVAISKNCREWWYFALNANLYEIREQR\n",
      "                     KRCTWDFMLHRARDAVSYTDKYFNKLKGGLLSTDDKEEMCRIEEEQSFEELKILRELV\n",
      "                     HDRFHKQEELAESLREPQFDSPGACPGAPEPGGGSGMLQYLQSWFPGWGGWYGQQTPE\n",
      "                     GNVVEGLSAEQQEQWIPEEILGTEEFFDPTADASCMNTYTKRDHVFAKLNLQLQRGTV\n",
      "                     TLLHKEQGTPQMNESAFMQLEFSDVKLLAESLPRRNSSLLSVRLGGLFLRDLATEGTM\n",
      "                     FPLLVFPNPQKEVGRVSQSFGLQTTSADRSDHYPAADPDGPVFEMLYERNPAHSHFER\n",
      "                     RLNVSTRPLNIIYNPQAIKKVADFFYKGKVHTSGFGYQSELELRVAEAARRQYNKLKM\n",
      "                     QTKAEIRQTLDRLLVGDFIEESKRWTVRLDISAPQVIFPDDFKFKNPVLVVVDLGRML\n",
      "                     LTNTQDNSRRKSRDGSASEETQFSDDEYKTPLATPPNTPPPESSSSNGEKTPPFSGVE\n",
      "                     FSEEQLQAHLMSTKMYERYSLSFMDLQIMVGRVKDNWKHVQDIDVGPTHVVEKFNVHL\n",
      "                     QLERRLIYTSDPKYPGAVLSGNLPDLKIHINEDKISALKNCFALLTTPEMKTSDTQIK\n",
      "                     EKIFPQEEQRGSLQDSVMNLTQSIVLLEQHTREVLVESQLLLAEFKVNCMQLGVESNG\n",
      "                     RYISVLKVFGTNAHFVKRPYDAEVSLTVHGLLLVDTMQTYGADFDLLMASHKNLSFDI\n",
      "                     PTGSLRDSRAQSPVSGPNVAHLTDGATLNDRSATSVSLDKILTKEQESLIKLEYQFVS\n",
      "                     SECPSMNLDSTLQVISLQVNNLDIILNPETIVELIGFLQKSFPKEKDDLSPQPLMTDF\n",
      "                     ERSFREQGTYQSTYEQNTEVAVEIHRLNLLLLRTVGMANREKYGRKIATASIGGTKVN\n",
      "                     VSMGSTFDMNGSLGCLQLMDLTQDNVKNQYVVSIGNSVGYENIISDIGYFESVFVRME\n",
      "                     DAALTEALSFTFVERSKQECFLNLKMASLHYNHSAKFLKELTLSMDELEENFRGMLKS\n",
      "                     AATKVTTVLATKTAEYSEMVSLFETPRKTREPFILEENEIYGFDLASSHLDTVKLILN\n",
      "                     INIESPVVSIPRKPGSPELLVGHLGQIFIQNFVAGDDESRSDRLQVEIKDIKLYSLNC\n",
      "                     TQLAGREAVGSEGSRMFCPPSGSGSANSQEEAHFTRHDFFESLHRGQAFHILNNTTIQ\n",
      "                     FKLEKIPIERESELTFSLSPDDLGTSSIMKIEGKFVNPVQVVLAKHVYEQVLQTLDNL\n",
      "                     VYSEDLNKYPASATSSPCPDSPLPPLSTCGESSVERKENGLFSHSSLSNTSQKSLSVK\n",
      "                     EVKSFTQIQATFCISELQVQLSGDLTLGAQGLVSLKFQDFEVEFSKDHPQTLSIQIAL\n",
      "                     HSLLMEDLLEKNPDSKYKNLMVSRGAPKPSSLAQKEYLSQSCPSVSNVEYPDMPRSLP\n",
      "                     SHMEEAPNVFQLYQRPTSASRKKQKEVQDKDYPLTPPPSPTVDEPKILVGKSKFDDSL\n",
      "                     VHINIFLVDKKHPEFSSSYNRVNRSIDVDFNCLDVLITLQTWVVILDFFGIGSTADNH\n",
      "                     AMRLPPEGILHNVKLEPHASMESGLQDPVNTKLDLKVHSLSLVLNKTTSELAKANVSK\n",
      "                     LVAHLEMIEGDLALQGSIGSLSLSDLTCHGEFYRERFTTSGEEALIFQTFKYGRPDPL\n",
      "                     LRREHDIRVSLRMASVQYVHTQRFQAEVVAFIQHFTQLQDVLGRQRAAIEGQTVRDQA\n",
      "                     QRCSRVLLDIEAGAPVLLIPESSRSNNLIVANLGKLKVKNKFLFAGFPGTFSLQDKES\n",
      "                     VPSASPTGIPKHSLRKTTSTEEPRGTHSQGQFTMPLAGMSLGSLKSEFVPSTSTKQQG\n",
      "                     PQPTLSVGQESSSPEDHVCLLDCVVVDLQDMDIFAAERHPREYSKAPEDSSGDLIFPS\n",
      "                     YFVRQTGGSLLTEPCRLKLQVERNLDKEISHTVPDISIHGNLSSVHCSLDLYKYKLIR\n",
      "                     GLLENNLGEPIEEFMRPYDLQDPRIHTVLSGEVYTCMCFLIDMVNVSLELKDPKRKEG\n",
      "                     AGSLARFDFKKCKLLYESFSNQTKSINLVSHSMMAFDTRYAGQKTSPGMTNVFSCIFQ\n",
      "                     PAKNSSTTQGSIQIELHFRSTKDSSCFTVVLNNLRVFLIFDWLLLVHDFLHTPSDIKK\n",
      "                     QNHVTPSRHRNSSSESAIVPKTVKSGVVTKRSSLPVSNERHLEVKVNVTGTEFVVIED\n",
      "                     VSCFDTNAIILKGTTVLTYKPRFVDRPFSGSLFGIEVFSCRLGNEHDTALSIVDPVQI\n",
      "                     QMELVGNSSYQNSSGLMDAFNSEDFPPVLEIQLQALDIRLSYNDVQLFLAIAKSIPEQ\n",
      "                     ANAAVPDSVALESDSVGTYLPGASRVGEEIREGTRHTLDPVLELQLARLQELGFSMDD\n",
      "                     CRKALLACQGQLKKAASWLFKNAEPLKSLSLASTSRDSPGAVAAPLISGVEIKAESVC\n",
      "                     ICFIDDCMDCDVPLAELTFSRLNFLQRVRTSPEGYAHFTLSGDYYNRALSGWEPFIEP\n",
      "                     WPCSVSWQQQAASRLHPPRLKLEAKAKPRLDINITSVLIDQYVSTKESWMADYCKDDK\n",
      "                     DIESAKSEDWMGSSVDPPCFGQTEVKTPKRRQPFVPFALRNHTGCTLWFATLTTTPTR\n",
      "                     AALSHSGSPGVVPEGNGTFLDDTHNVSEWREVLTGEEIPFEFEARGKLRHRHTHDLRI\n",
      "                     HQLQVRVNGWEQVSPVSVDKVGTFFRYAAPDKNSSSSTIGSPSSRTNIIHPQVYFSSL\n",
      "                     PPVRVVFAVTMEGSARKVITVRSALIVRNRLETPMELRLDSPSAPDKPVVLPAIMPGD\n",
      "                     SFAVPLHLTSWRLQARPKGLGVFFCKAPIHWTNVVKTAEISSSKRECHSMDTEKSRFF\n",
      "                     RFCVAIKKENYPDYMPSNIFSDSAKQIFRQPGHTIYLLPTVVICNLLPCELDFYVKGM\n",
      "                     PINGTLKPGKEAALHTADTSQNIELGVSLENFPLCKELLIPPGTQNYMVRMRLYDVNR\n",
      "                     RQLNLTIRIVCRAEGSLKIFISAPYWLINKTGLPLIFRQDNAKTDAAGQFEEHELARS\n",
      "                     LSPLLFCYADKEQPNLCTMRIGRGIHPEGMPGWCQGFSLDGGSGVRALKVIQQGNRPG\n",
      "                     LIYNIGIDVKKGRGRYIDTCMVIFAPRYLLDNKSSHKLAFAQREFARGQGTANPEGYI\n",
      "                     STLPGSSVVFHWPRNDYDQLLCVRLMDVPNCIWSGGFEVNKNNSFHINMRDTLGKCFF\n",
      "                     LRVEITLRGATYRISFSDTDQLPPPFRIDNFSKVPVVFTQHGVAEPRLRTEVKPMTSL\n",
      "                     DYAWDEPTLPPFITLTVKGAGSSEINCNMNDFQDNRQLYYENFIYIAATYTFSGLQEG\n",
      "                     TGRPVASNKAITCAELVLDVSPKTQRVILKKKEPGKRSQLWRMTGTGMLAHEGSSVPH\n",
      "                     NPNKPSAARSTEGSAILDIAGLAAVTDNRYEPLMLRKPDRRRSTTQTWSFREGKLTCG\n",
      "                     LHGLVVQAKGGLSGLFDGAEVVLGPDTSMELLGPVPPEQQFINQKMRPGSGMLSIRVI\n",
      "                     PDGPTRALQITDFCHRKSSRSYEVDELPVTEQELQKLKNPDTEQELEVLVRLEGGIGL\n",
      "                     SLINKVPEELVFASLTGINVHYTQLATSHMLELSIQDVQVDNQLIGTTQPFMLYVTPL\n",
      "                     SNENEVIETGPAVQVNAVKFPSKSALTNIYKHLMITAQRFTVQIEEKLLLKLLSFFGY\n",
      "                     DQAESEVEKYDENLHEKTAEQGGTPIRYYFENLKISIPQIKLSVFTSNKLPLDLKALK\n",
      "                     STLGFPLIRFEDAVINLDPFTRVHPYETKEFIINDILKHFQEELLSQAARILGSVDFL\n",
      "                     GNPMGLLNDVSEGVTGLIKYGNVGGLIRNVTHGVSNSAAKFAGTLSDGLGKTMDNRHQ\n",
      "                     SEREYIRYHAATSGEHLVAGIHGLAHGIIGGLTSVITSTVEGVKTEGGVSGFISGLGK\n",
      "                     GLVGTVTKPVAGALDFASETAQAVRDTATLSGPRTQAQRVRKPRCCTGPQGLLPRYSE\n",
      "                     SQAEGQEQLFKLTDNIQDEFFIAVENIDSYCVLISSKAVYFLKSGDYVDREAIFLEVK\n",
      "                     YDDLYHCLVSKDHGKVYVQVTKKAVSTSSGVSIPGPSHQKPMVHVKSEVLAVKLSQEI\n",
      "                     NYAKSLYYEQQLMLRLSENREQLELDS\"\n",
      "     misc_feature    2145..2147\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphoserine. {ECO:0000244|PubMed:23186163,\n",
      "                     ECO:0000244|PubMed:24275569}; propagated from\n",
      "                     UniProtKB/Swiss-Prot (Q5THJ4.2); phosphorylation site\"\n",
      "     misc_feature    3258..3260\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphoserine. {ECO:0000244|PubMed:23186163};\n",
      "                     propagated from UniProtKB/Swiss-Prot (Q5THJ4.2);\n",
      "                     phosphorylation site\"\n",
      "     misc_feature    3270..3272\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphoserine. {ECO:0000244|PubMed:23186163};\n",
      "                     propagated from UniProtKB/Swiss-Prot (Q5THJ4.2);\n",
      "                     phosphorylation site\"\n",
      "     misc_feature    3282..3284\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphoserine. {ECO:0000244|PubMed:20068231,\n",
      "                     ECO:0000244|PubMed:23186163}; propagated from\n",
      "                     UniProtKB/Swiss-Prot (Q5THJ4.2); phosphorylation site\"\n",
      "     misc_feature    3570..3572\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphoserine. {ECO:0000244|PubMed:23186163,\n",
      "                     ECO:0000244|PubMed:24275569}; propagated from\n",
      "                     UniProtKB/Swiss-Prot (Q5THJ4.2); phosphorylation site\"\n",
      "     misc_feature    4179..4181\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphoserine. {ECO:0000244|PubMed:24275569};\n",
      "                     propagated from UniProtKB/Swiss-Prot (Q5THJ4.2);\n",
      "                     phosphorylation site\"\n",
      "     misc_feature    4950..4952\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphoserine. {ECO:0000244|PubMed:23186163};\n",
      "                     propagated from UniProtKB/Swiss-Prot (Q5THJ4.2);\n",
      "                     phosphorylation site\"\n",
      "     misc_feature    4965..4967\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphoserine. {ECO:0000244|PubMed:23186163};\n",
      "                     propagated from UniProtKB/Swiss-Prot (Q5THJ4.2);\n",
      "                     phosphorylation site\"\n",
      "     misc_feature    5253..5255\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphoserine. {ECO:0000244|PubMed:23186163};\n",
      "                     propagated from UniProtKB/Swiss-Prot (Q5THJ4.2);\n",
      "                     phosphorylation site\"\n",
      "     misc_feature    5439..5441\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphothreonine. {ECO:0000244|PubMed:18669648,\n",
      "                     ECO:0000244|PubMed:19690332, ECO:0000244|PubMed:23186163,\n",
      "                     ECO:0000244|PubMed:24275569}; propagated from\n",
      "                     UniProtKB/Swiss-Prot (Q5THJ4.2); phosphorylation site\"\n",
      "     misc_feature    5451..5453\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphoserine. {ECO:0000244|PubMed:24275569};\n",
      "                     propagated from UniProtKB/Swiss-Prot (Q5THJ4.2);\n",
      "                     phosphorylation site\"\n",
      "     misc_feature    7461..7463\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphoserine. {ECO:0000244|PubMed:19690332,\n",
      "                     ECO:0000244|PubMed:21406692}; propagated from\n",
      "                     UniProtKB/Swiss-Prot (Q5THJ4.2); phosphorylation site\"\n",
      "     misc_feature    8169..8171\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphoserine. {ECO:0000244|PubMed:23186163,\n",
      "                     ECO:0000244|PubMed:24275569}; propagated from\n",
      "                     UniProtKB/Swiss-Prot (Q5THJ4.2); phosphorylation site\"\n",
      "     misc_feature    9030..9032\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Phosphoserine. {ECO:0000244|PubMed:23186163};\n",
      "                     propagated from UniProtKB/Swiss-Prot (Q5THJ4.2);\n",
      "                     phosphorylation site\"\n",
      "     misc_feature    10653..10655\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"N6-acetyllysine. {ECO:0000244|PubMed:19608861};\n",
      "                     propagated from UniProtKB/Swiss-Prot (Q5THJ4.2);\n",
      "                     acetylation site\"\n",
      "     exon            256..333\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            334..524\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            525..605\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            606..722\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            723..827\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            828..998\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            999..1099\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            1100..1268\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            1269..1370\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            1371..1572\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            1573..1752\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            1753..1883\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            1884..1959\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            1960..2130\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            2131..2261\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            2262..2394\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            2395..4608\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     STS             3768..3954\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /standard_name=\"AL033990\"\n",
      "                     /db_xref=\"UniSTS:93406\"\n",
      "     exon            4609..4760\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            4761..5792\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            5793..5883\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            5884..6010\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            6011..6191\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            6192..6374\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            6375..6597\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            6598..6808\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            6809..6980\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            6981..7093\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            7094..7306\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            7307..7572\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            7573..7706\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            7707..7862\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            7863..8073\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            8074..8148\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            8149..8355\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            8356..8445\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            8446..8586\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            8587..8709\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            8710..8815\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            8816..8968\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            8969..9104\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            9105..9152\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            9153..9303\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            9304..9514\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            9515..9762\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            9763..9954\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            9955..10081\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            10082..10224\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            10225..10355\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            10356..10531\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            10532..10655\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            10656..10891\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            10892..11000\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            11001..11164\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            11165..11273\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            11274..11453\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            11454..11567\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            11568..11717\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            11718..11867\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            11868..11964\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            11965..12113\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            12114..12248\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            12249..12416\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            12417..12549\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            12550..12745\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            12746..12877\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            12878..13118\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     exon            13119..16273\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /inference=\"alignment:Splign:1.39.8\"\n",
      "     STS             13365..13606\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /standard_name=\"RH80215\"\n",
      "                     /db_xref=\"UniSTS:86366\"\n",
      "     STS             13733..14606\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /standard_name=\"VPS13D__5075\"\n",
      "                     /db_xref=\"UniSTS:466553\"\n",
      "     STS             14466..14556\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /standard_name=\"SHGC-24271\"\n",
      "                     /db_xref=\"UniSTS:76148\"\n",
      "     STS             15981..16171\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /standard_name=\"A001Y09\"\n",
      "                     /db_xref=\"UniSTS:59564\"\n",
      "     STS             15981..16171\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /standard_name=\"G19775\"\n",
      "                     /db_xref=\"UniSTS:59563\"\n",
      "     STS             16045..16205\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /standard_name=\"D1S1907E\"\n",
      "                     /db_xref=\"UniSTS:64139\"\n",
      "     STS             16047..16274\n",
      "                     /gene=\"VPS13D\"\n",
      "                     /standard_name=\"RH11824\"\n",
      "                     /db_xref=\"UniSTS:83385\"\n",
      "     regulatory      16247..16252\n",
      "                     /regulatory_class=\"polyA_signal_sequence\"\n",
      "                     /gene=\"VPS13D\"\n",
      "     polyA_site      16273\n",
      "                     /gene=\"VPS13D\"\n",
      "ORIGIN      \n",
      "        1 gcgggcctgc gccattgagg agcggcgggg aggaaacgcc gcgcagcgcc gggctggggc\n",
      "       61 gggcggcccg ggacaccgac agatttttct gtgaccatga aagagagaaa taaagaatga\n",
      "      121 tccatgattt ctaaacacct tttcctgagg atatagtcat gttggaaggc cttgtagcct\n",
      "      181 gggttctcaa tacctatttg ggaaaatatg tcaataacct gaacactgac cagctctcag\n",
      "      241 ttgcacttct caaaggtgct gttgaattag aaaacttgcc attaaagaaa gatgccttga\n",
      "      301 aagaattgga attaccattt gaagtcaaag ctggcttcat tgggaaagta acccttcaga\n",
      "      361 ttccctttta tcgcccccat gtggaccctt gggtgatctc catctccagc cttcacttaa\n",
      "      421 ttggagcccc agagaaaata caggatttca atgatgaaaa ggagaagctg ttggaaaggg\n",
      "      481 aacgtaagaa agcactactt caagccctgg aggagaaatg gaagaatgac cgccagcaga\n",
      "      541 aaggggagtc ctattggtat tcagttaccg cctccgtagt tacaaggatt gtggagaata\n",
      "      601 ttgaattaaa aattcaagat gtccatttac gctttgaaga tggtgtcacc aatccctccc\n",
      "      661 atccttttgc ttttggcatc tgcattaaga atgtgtccat gcaaaatgct gtgaatgagc\n",
      "      721 ctgtacagaa actaatgcgg aaaaagcaat tagacgtagc agaatttagc atctattggg\n",
      "      781 atgtcgattg cactttactg ggggatttgc ctcagatgga gttacaggag gccatggcca\n",
      "      841 ggagcatgga gagtcgcagc catcactacg tcctggagcc tgtgtttgca tctgctcttt\n",
      "      901 tgaagagaaa ctgctccaag aagcccctgc ggtctcggca cagtccccgt attgattgtg\n",
      "      961 atattcagct ggagaccatt cccttgaaac tctctcagct gcaataccgg caaatcatgg\n",
      "     1021 aattcctcaa ggagctggaa cgaaaggaga ggcaggtgaa gttccgaagg tggaaaccca\n",
      "     1081 aggtggcgat atctaagaac tgccgagaat ggtggtattt tgctttgaat gctaacttgt\n",
      "     1141 atgagatcag agagcagagg aaacgttgca cctgggactt tatgttgcac cgcgctcgtg\n",
      "     1201 atgctgtatc ttacactgac aaatatttca acaagttaaa aggaggcctg ctgtccacag\n",
      "     1261 atgacaagga ggaaatgtgt cggattgaag aggaacagag ctttgaggaa ttgaagattt\n",
      "     1321 tgcgtgaact ggttcatgat cgatttcaca aacaggaaga actagcagag agtctgcggg\n",
      "     1381 agcctcagtt tgattctcca ggagcctgtc cgggagcccc agaacccggt ggaggcagtg\n",
      "     1441 ggatgctgca gtatctccag tcctggtttc ctggatgggg tggctggtac gggcagcaga\n",
      "     1501 ccccagaagg gaatgtggtt gagggactgt cagcagagca acaggagcag tggattcctg\n",
      "     1561 aagagatcct gggcactgag gagttttttg accccactgc agatgcctcg tgtatgaaca\n",
      "     1621 cgtatacaaa gcgagatcat gtctttgcca aactgaattt gcagttgcag cgaggtacag\n",
      "     1681 tgactctgtt acacaaggag caaggaactc ctcaaatgaa tgaaagtgct ttcatgcagc\n",
      "     1741 tcgagttttc agatgtaaaa cttctagcag agtctcttcc tcgaagaaat tcctcgttgc\n",
      "     1801 tttcagtccg gttgggtgga ctgtttcttc gagacctggc tacagaagga actatgtttc\n",
      "     1861 ctcttctagt cttccctaat ccacaaaaag aagttggcag agtctcacaa tcttttggtc\n",
      "     1921 tacaaactac atctgcagac agaagtgatc attacccagc tgcagatcca gatggccccg\n",
      "     1981 tttttgagat gctgtatgag agaaatccgg cgcacagcca ctttgagagg cggctcaatg\n",
      "     2041 tcagcacaag gcccttgaac atcatataca atccgcaggc cattaaaaaa gtagcagact\n",
      "     2101 ttttctacaa gggaaaggtt catacctcag gttttggtta tcagtctgaa cttgagctga\n",
      "     2161 gagtggctga agctgcccga agacaatata acaagctgaa gatgcagacc aaggcagaaa\n",
      "     2221 tccggcaaac tcttgatcgt ttgctagtgg gtgatttcat tgaggagagt aaacgatgga\n",
      "     2281 ccgtgcggct ggatatttct gcccctcagg tgatatttcc tgatgatttc aaattcaaga\n",
      "     2341 atcctgtgtt agttgtcgtg gatctaggaa gaatgctttt gacgaacacc caagataact\n",
      "     2401 ccaggaggaa aagtagggat gggtcagcat ctgaagagac ccagtttagt gatgatgaat\n",
      "     2461 ataagacccc cctggccaca cctcctaaca ccccacctcc cgagtcaagc agcagcaacg\n",
      "     2521 gagagaaaac acctcccttt tctggagttg agttcagtga agaacagctt caagcacatt\n",
      "     2581 taatgagcac aaagatgtat gagaggtact cgctgtcatt tatggacctc cagatcatgg\n",
      "     2641 ttggacgagt gaaagacaat tggaagcatg tccaggatat tgacgtggga ccaacacatg\n",
      "     2701 tggtagagaa gttcaacgtt cacctacagt tagagcgtcg attgatttat acttcagatc\n",
      "     2761 ccaaatatcc aggagccgtg ctctcaggca acttaccaga cttaaaaatc cacattaatg\n",
      "     2821 aagataaaat atctgcacta aagaattgct ttgctctcct caccacccca gaaatgaaaa\n",
      "     2881 cttctgacac tcagattaaa gaaaagattt ttccccagga ggagcagcgg ggaagtttgc\n",
      "     2941 aagactccgt aatgaattta acccagagca ttgtgttgtt ggagcagcat acccgcgagg\n",
      "     3001 ttctggtgga gtcgcagctc ctcctggcgg aatttaaagt gaactgtatg cagcttggtg\n",
      "     3061 ttgagagcaa tggccggtac atttctgtgc tcaaggtgtt tggtaccaat gctcactttg\n",
      "     3121 tgaagaggcc ttatgatgct gaagtctccc taactgttca tggtttgctc ctggtggata\n",
      "     3181 ccatgcagac atatggtgct gattttgacc ttttgatggc ttcacataag aacttgagct\n",
      "     3241 ttgatattcc aacgggaagc cttcgggata gcagggccca gtctcctgtc tctggaccga\n",
      "     3301 atgtggccca cttaactgat ggagctacac tgaacgaccg atcagctact agtgtttcac\n",
      "     3361 ttgacaaaat tcttaccaaa gagcaagagt cccttattaa gttggaatat cagtttgtga\n",
      "     3421 gttcagagtg cccatcgatg aatttagaca gtactcttca ggtgatttcc ctacaggtga\n",
      "     3481 ataatttaga tattatcctc aatccagaga cgattgtgga gctaattggt tttcttcaaa\n",
      "     3541 aatcctttcc caaggaaaaa gatgatttaa gtcctcaacc tttaatgact gattttgaaa\n",
      "     3601 gaagcttcag agaacaagga acttaccagt ctacatatga acaaaacact gaggttgcag\n",
      "     3661 tggaaatcca taggctgaac ttactgcttc ttcggacagt gggcatggca aatagagaga\n",
      "     3721 aatatggcag aaaaattgca actgcaagta taggtggcac caaagttaat gtctcaatgg\n",
      "     3781 gtagcacgtt tgacatgaat ggttctcttg gctgtttaca gcttatggat ttgacacaag\n",
      "     3841 ataacgttaa aaaccagtat gttgtcagca ttgggaattc tgtaggctat gaaaatatca\n",
      "     3901 tcagtgatat tggctacttt gaatctgtgt ttgtcagaat ggaagatgca gccctcactg\n",
      "     3961 aagctttgag tttcacgttt gttgagagat ctaaacagga gtgttttctc aacctgaaga\n",
      "     4021 tggcttcttt acattataac cactctgcta agtttttgaa ggagttgacg ttatccatgg\n",
      "     4081 atgaactgga agaaaatttt cgaggtatgc tgaaaagcgc agccaccaaa gtcaccacag\n",
      "     4141 tactagctac caagactgcc gagtatagcg agatggtatc gctctttgaa actccaagga\n",
      "     4201 agactcggga accctttatc ttagaggaaa atgaaatata tgggtttgac ctagcttcgt\n",
      "     4261 ctcatttgga cactgtaaag ctaatcttga acataaacat tgaatcacca gttgtttcta\n",
      "     4321 tccctcggaa gccggggagt cctgagttgt tggtgggaca cttgggacag atattcatcc\n",
      "     4381 agaattttgt ggcgggagat gatgaatcca gaagtgaccg tctgcaggtg gaaatcaagg\n",
      "     4441 acattaaact gtattctttg aattgcaccc agttggcagg tagagaagct gttgggtctg\n",
      "     4501 aaggaagccg gatgttttgc ccaccttccg ggtctggcag tgccaacagt caggaggaag\n",
      "     4561 ctcatttcac acgacatgat ttctttgaat ctttgcatag aggtcaagct tttcacatcc\n",
      "     4621 tgaacaacac caccattcag tttaaactgg agaagatccc tatagagaga gaatctgaat\n",
      "     4681 tgactttttc tcttagccca gatgacctgg gaacttctag catcatgaag attgaaggaa\n",
      "     4741 aatttgtcaa tccagttcag gtggtgttag caaagcatgt atatgagcag gttttacaaa\n",
      "     4801 ccctggacaa tctcgtgtac agtgaagatc tgaataagta tccagccagt gctacctcct\n",
      "     4861 ccccttgccc tgattctcct ctgcctcccc tcagtacctg tggagaatct tctgttgaaa\n",
      "     4921 ggaaggagaa tggattgttc agccactcca gcctttctaa cacctctcag aagtcattgt\n",
      "     4981 cagtgaagga agtcaaatcc tttactcaga ttcaagccac cttttgtata tcagagcttc\n",
      "     5041 aggttcagct aagtggagat ctgactttgg gggcccaagg tcttgtgagc ttaaagtttc\n",
      "     5101 aggactttga ggtggaattc agtaaagacc atccccagac tttatctatt cagattgccc\n",
      "     5161 tgcattctct gctgatggag gacttattgg agaagaatcc agattctaaa tataagaacc\n",
      "     5221 tgatggtgtc tcgaggagcc cctaagccat ctagtttagc acaaaaagaa tacctttctc\n",
      "     5281 agtcttgccc ctcagtgtcc aatgtggaat atcctgatat gcctcggtct ctcccttccc\n",
      "     5341 acatggaaga agctcctaat gtcttccagt tgtatcaaag gcccacctct gcgtcccgga\n",
      "     5401 aaaagcaaaa ggaagtccaa gacaaggact atcccttgac cccacctcct tctccaacag\n",
      "     5461 tggatgagcc caagatactt gttggaaaga gtaaatttga tgattcctta gtccacatca\n",
      "     5521 acatattctt ggtagataag aaacatccag aattctcttc cagttacaat cgagttaacc\n",
      "     5581 ggagcattga tgttgatttt aattgcttgg atgtgctgat cacactgcaa acctgggttg\n",
      "     5641 tgatattaga cttttttgga atcggctcca ctgcagacaa ccacgcaatg aggctgcctc\n",
      "     5701 ctgagggcat tctgcacaac gtgaagttgg agccacatgc ctccatggag tctggacttc\n",
      "     5761 aggatccagt gaacaccaaa ctggatctca aggttcattc actttctcta gtgctgaata\n",
      "     5821 agaccaccag tgagcttgcc aaagcaaatg tgtccaaatt agtagcacac ctggaaatga\n",
      "     5881 ttgagggaga cctggcctta cagggcagca ttgggagtct gtctctaagt gacctcacat\n",
      "     5941 gccatggaga gttctacaga gaacggttca ctaccagtgg tgaagaagca ctcatcttcc\n",
      "     6001 agacttttaa atacggacgg cctgaccctc tgctccggag agaacacgac attcgcgtga\n",
      "     6061 gcctccggat ggcctctgtg cagtatgtgc atactcagcg tttccaggca gaggtggtgg\n",
      "     6121 ccttcattca gcatttcact cagctgcagg atgtcttagg gcgccagcga gctgctattg\n",
      "     6181 aggggcagac ggtgagagat caagcccagc gctgttcacg ggttctcctg gatattgagg\n",
      "     6241 ctggtgctcc cgttctcttg atcccagaaa gttccagatc aaataatctg attgtagcaa\n",
      "     6301 atttggggaa gttgaaagtc aaaaataagt ttctgtttgc tggttttcct ggcacctttt\n",
      "     6361 ccctacaaga taaggaatct gtgccttcag cttccccaac gggtattccc aaacacagtc\n",
      "     6421 tgaggaaaac gacaagcacg gaggagccca ggggaaccca ttcccagggg cagttcacga\n",
      "     6481 tgcctcttgc tggaatgagc ctaggaagcc tgaagagtga gtttgtgccc agtacctcca\n",
      "     6541 ccaagcagca agggccgcaa cccacactgt ctgttggcca agagtccagt agtccagaag\n",
      "     6601 accatgtctg cctgctggat tgcgttgtcg tggatctcca ggacatggac atctttgctg\n",
      "     6661 cagagagaca tccgagagaa tactcgaagg caccagagga tagtagtgga gatctgatct\n",
      "     6721 tcccttccta ttttgtgcga cagacaggag gaagcctctt aaccgagcct tgtaggctga\n",
      "     6781 aattgcaggt ggaaaggaat ttggacaaag aaataagtca tactgtgcca gacatatcta\n",
      "     6841 tccatggcaa tctctcctca gtccactgct ctctggatct gtataaatac aagctgatcc\n",
      "     6901 gcggcttatt agagaacaac ctgggagaac ccatagagga atttatgcgg ccttatgatt\n",
      "     6961 tacaagatcc aagaattcat actgtcctga gtggagaagt gtacacctgt atgtgcttcc\n",
      "     7021 tcattgatat ggtgaatgta agtctggagc ttaaagatcc aaaaagaaaa gaaggtgctg\n",
      "     7081 ggtccctagc cagatttgac ttcaagaaat gcaaactgct ctatgaaagt ttttccaacc\n",
      "     7141 aaaccaagtc cattaacttg gtttcccatt ccatgatggc ttttgacacc cgttatgctg\n",
      "     7201 ggcagaagac cagccctggc atgacgaatg tgttcagctg tatctttcag cccgctaaga\n",
      "     7261 acagcagcac cacccaaggg tccattcaga ttgaactaca tttcagatct accaaggatt\n",
      "     7321 cctcctgctt tacagtagtt ctcaacaatc tccgtgtgtt tctcatattt gactggctac\n",
      "     7381 tgttagtcca tgattttctc cacactccca gtgatattaa gaaacaaaat catgttactc\n",
      "     7441 cttctcgcca ccgtaactct agcagcgaat ctgctatagt tcccaaaact gtgaagagtg\n",
      "     7501 gagtagttac caagcggtct tcccttcctg tgtccaatga aaggcacctg gaggtcaagg\n",
      "     7561 tcaatgtaac aggtacggag tttgtggtca ttgaagatgt gtcctgcttc gacaccaatg\n",
      "     7621 ccattattct gaaaggcacc acagtgctca cctataagcc ccggtttgtt gatcgcccct\n",
      "     7681 tttcaggaag tttgtttggc attgaggtgt tttcatgccg actagggaat gagcatgata\n",
      "     7741 cagctctttc aattgtggat cccgtacaaa ttcaaatgga gttggtgggg aattcttctt\n",
      "     7801 atcaaaatag ttcaggattg atggatgcat tcaatagtga agatttccca cctgtcctgg\n",
      "     7861 agattcagtt acaagccctg gatatcagac tctcctataa tgatgttcag ctgtttcttg\n",
      "     7921 ccattgcaaa atccatccca gagcaagcta atgctgcagt gccagactca gtggccctgg\n",
      "     7981 agtcagactc cgttggcact taccttccag gtgcatctcg cgttggagag gaaatcagag\n",
      "     8041 aagggacaag acacacctta gatcctgtct tggagttaca gctggctagg ctgcaggagc\n",
      "     8101 tgggattcag catggatgat tgtcgcaaag ctcttttggc gtgtcaaggc caattgaaaa\n",
      "     8161 aggcagcaag ttggttgttt aagaatgcgg aacctctgaa gtctctttcc ttggcctcca\n",
      "     8221 ccagccgaga tagcccaggg gctgtggcag cgccattgat ctctggcgtg gagatcaaag\n",
      "     8281 ctgagagtgt gtgcatctgt ttcatcgatg actgcatgga ttgtgatgtt cctctcgctg\n",
      "     8341 aactcacctt ttcccgtctg aattttcttc agcgtgtaag aactagccct gaaggctatg\n",
      "     8401 cccacttcac cctttctgga gattattata accgtgctct ttcaggctgg gagccattta\n",
      "     8461 ttgagccttg gccatgctct gtatcctggc aacagcaggc agctagtcgt ctccatcctc\n",
      "     8521 ctcgactgaa gctagaagcc aaggccaaac ctcgtttgga tatcaatatc acttctgtgc\n",
      "     8581 taattgacca gtatgtaagt accaaggaat cgtggatggc agattactgt aaagatgaca\n",
      "     8641 aggacataga gtcagctaaa tcagaagact ggatgggctc ttcggtggat cctccatgtt\n",
      "     8701 ttggacaaac agaggtgaaa acccccaagc gccggcagcc atttgtcccc tttgctctga\n",
      "     8761 ggaaccacac ggggtgcact ttgtggtttg ccaccctgac caccacaccc accagagctg\n",
      "     8821 cactctctca cagtgggagt ccaggggtag ttccagaagg gaacggaaca tttctcgatg\n",
      "     8881 atactcacaa tgttagtgaa tggcgagaag tccttacagg tgaagagatt ccctttgaat\n",
      "     8941 ttgaagcaag aggaaagtta agacacagac acacccatga cctccggatt catcaactgc\n",
      "     9001 aagtgagagt aaatggctgg gagcaagtga gcccagtgtc tgtggacaaa gtcgggacct\n",
      "     9061 tttttcgata tgcagcacca gataaaaatt catcttcctc tacgattggc agcccaagca\n",
      "     9121 gcagaacaaa tattatacat ccccaggttt atttctcttc actcccacca gtgcgggtgg\n",
      "     9181 tctttgcagt gactatggaa ggcagtgcac ggaaagtcat cactgtccgg tcagccctca\n",
      "     9241 ttgtgaggaa cagacttgag acaccaatgg aactaagact ggatagccca tcagctccag\n",
      "     9301 acaagccagt ggtgcttcct gctatcatgc caggggattc gtttgctgtg cctttacacc\n",
      "     9361 tcacttcttg gcggctacag gcccggccca aaggattggg tgtatttttc tgtaaggctc\n",
      "     9421 ccattcattg gaccaatgta gtgaagactg cagaaattag tagcagtaaa cgagagtgcc\n",
      "     9481 actctatgga cacagaaaaa agccgatttt tcaggttttg tgtggctata aagaaagaga\n",
      "     9541 attatccaga ttatatgccc tcaaacatat tttctgacag tgcaaaacag attttcagac\n",
      "     9601 agcctgggca taccatatat ctcctgccaa ctgtggtaat ctgcaacttg ctaccctgtg\n",
      "     9661 aacttgattt ttatgttaaa ggaatgccaa ttaatgggac gctgaaacct ggcaaggagg\n",
      "     9721 cagctctcca tacagctgat acatcccaga acattgagct gggggtatca ctggagaatt\n",
      "     9781 tccccctctg taaagaattg ctcattccac ctggaaccca aaactatatg gtgagaatgc\n",
      "     9841 gactctatga cgtcaaccgt cggcagctga acctcaccat ccggattgtg tgtcgagcag\n",
      "     9901 aaggatcctt aaagatcttc atttctgctc catattggct gattaacaaa acagggttgc\n",
      "     9961 cactgatctt cagacaggac aatgccaaga cagatgctgc aggccagttt gaggagcatg\n",
      "    10021 agctggcccg tagcctgagt cctctcttat tctgctatgc tgacaaagag cagccaaacc\n",
      "    10081 tctgcacgat gagaatcgga agggggattc atccagaagg catgccgggc tggtgtcagg\n",
      "    10141 gcttctccct ggatggtggt agtggtgtcc gagctttgaa agtcatccag caaggaaacc\n",
      "    10201 gcccagggct gatctataac attggtattg atgtcaagaa aggccgaggt cgatacattg\n",
      "    10261 atacctgcat ggtcatcttt gccccccgtt acctgttaga taataaatca tctcacaagc\n",
      "    10321 ttgcatttgc acagagggaa tttgccaggg gacagggaac agccaatccc gaaggttaca\n",
      "    10381 tttccaccct tcctggttcc agtgtggtgt tccactggcc tcggaatgac tatgatcagc\n",
      "    10441 tattgtgtgt cagactgatg gacgttccca attgtatttg gtctggaggc tttgaagtca\n",
      "    10501 acaagaataa ttccttccat atcaacatga gggatacctt gggaaaatgc ttcttcctac\n",
      "    10561 gagtggaaat tactctccga ggagctacgt ataggatctc atttagtgac acagatcagt\n",
      "    10621 tacctcctcc tttccgaatt gacaactttt ctaaggtccc ggttgtcttt actcagcatg\n",
      "    10681 gcgtagctga acccaggctc cggactgaag tgaagcccat gacttcattg gattatgcct\n",
      "    10741 gggacgaacc caccttgcca ccttttatca ctctgactgt taaaggggca gggtcctctg\n",
      "    10801 agatcaactg caacatgaat gatttccagg ataatcggca gctttattat gaaaatttca\n",
      "    10861 tttacattgc tgctacatat acattctctg gcttgcagga gggaacaggc aggcctgtgg\n",
      "    10921 cttccaacaa ggccattacc tgtgcggagc tcgttttgga tgtctcaccc aagacacaaa\n",
      "    10981 gagtcatttt aaaaaagaag gaaccaggaa agcgttctca gctgtggagg atgacaggaa\n",
      "    11041 caggaatgct ggcccatgag ggctcctcag ttcctcacaa tcccaataag ccctcagccg\n",
      "    11101 cccgctccac cgaggggtct gccatcttag atattgctgg tctcgctgca gtgactgaca\n",
      "    11161 acagatacga gccactgatg ctgagaaagc ctgaccgcag gcgaagcaca actcagacgt\n",
      "    11221 ggagtttccg agaaggaaaa ctgacctgtg ggttacatgg gttggtcgtc caggccaaag\n",
      "    11281 gaggactttc tggtttgttt gatggagctg aagttgttct tggtcctgac acttccatgg\n",
      "    11341 agcttttggg gccagttcca cctgaacaac aatttattaa tcaaaaaatg agacctggtt\n",
      "    11401 ctggaatgtt atccatcaga gtcatcccag atggaccaac tagagcactc cagataacag\n",
      "    11461 atttctgcca ccggaaaagc agccgttcat atgaagtgga tgaacttcct gtcaccgaac\n",
      "    11521 aagagctgca gaaattaaag aatccagata cagagcagga attggaagtg cttgtgaggt\n",
      "    11581 tagaaggtgg aattgggttg tccttaatta ataaagtccc agaagaactg gtctttgcaa\n",
      "    11641 gtcttacagg aatcaatgtg cactatacac agctggcaac cagtcacatg cttgaactca\n",
      "    11701 gcatacagga tgtacaggtg gacaatcagc tcattggtac cacgcagccc ttcatgctct\n",
      "    11761 atgtgactcc cctgagcaat gagaatgagg tcatcgagac cggcccagct gtgcaagtca\n",
      "    11821 acgcagtgaa gttccccagt aagagtgcac tgaccaacat ctacaagcat ctgatgatca\n",
      "    11881 cagctcagag attcacagtg caaattgagg agaaactgct cctcaagctg ctaagtttct\n",
      "    11941 ttggctacga tcaagcagaa tcagaggtgg aaaaatatga tgaaaacctc catgaaaaga\n",
      "    12001 cagctgagca aggtggaaca ccaattcgat actactttga aaatctcaaa atcagcattc\n",
      "    12061 ctcagatcaa gctaagtgtg ttcacctcca acaagctccc attggatctt aaggccctaa\n",
      "    12121 aaagcacctt ggggtttcct ttgatacggt ttgaagacgc tgtgattaat ctagatccat\n",
      "    12181 tcactcgggt acatccctat gagaccaagg agttcatcat caatgatatc ctcaaacatt\n",
      "    12241 tccaggagga actcctcagc caggcagctc gaatcctggg atcagtggat tttcttggca\n",
      "    12301 atcctatggg gcttttgaat gatgtttctg aaggggttac tggactgata aaatatggaa\n",
      "    12361 atgtcggggg cctcatcaga aatgttacac acggagtatc aaactctgct gccaagtttg\n",
      "    12421 ctggaacatt atcagatggc ttagggaaga cgatggacaa tcggcatcag tcagagcggg\n",
      "    12481 agtacatcag gtaccatgca gccacaagtg gtgaacacct tgtagccggc atccatggcc\n",
      "    12541 tggctcatgg tatcattggt ggactgacca gtgttataac ttcgacagtg gaaggtgtga\n",
      "    12601 aaacagaagg gggtgtcagc ggtttcatat ctggccttgg aaaagggctt gttggcactg\n",
      "    12661 taaccaagcc agtggcaggc gccctggatt ttgcatcaga aacagcccag gcggtgagag\n",
      "    12721 acacagccac actcagcggc cccaggactc aagcacagag ggttcggaaa ccgcgttgct\n",
      "    12781 gcacggggcc ccaggggctg cttccccgat attctgagag ccaggcggaa ggacaggagc\n",
      "    12841 agctcttcaa actcacagac aacatacagg acgaattctt catcgctgtg gagaacattg\n",
      "    12901 acagctactg cgtgctcatc tcctccaaag ctgtttactt cctgaaaagt ggagactacg\n",
      "    12961 tggatcgaga agccattttc ctagaagtca aatacgatga cctctaccac tgccttgtct\n",
      "    13021 ccaaagacca tgggaaggtg tatgtgcagg tgaccaagaa agccgtgagc acgagcagtg\n",
      "    13081 gagtgtccat ccccggcccc tcccaccaga agcccatggt ccatgtgaaa tctgaggtcc\n",
      "    13141 ttgctgtcaa gttgtcacaa gaaataaact acgcaaagag cctctactat gaacagcagc\n",
      "    13201 ttatgttaag actcagcgaa aaccgagagc agctggagct ggactcctga agccccgctg\n",
      "    13261 ctgagatggg cgctcccgac acagcgcaga cccaccagga ggaaagaggc ccagctctca\n",
      "    13321 gctgacgatg gaggcagaac cggagtcggg tttggggaag ttgtcaagga atgagggaaa\n",
      "    13381 gtaaatcctc atgaggaaaa gtacaaatgg aaatcgtatt aatttgtgag gcagggagtt\n",
      "    13441 attttagatt atgggaaata atttttaaag gtattggtta aataacgttt aaaaacatgt\n",
      "    13501 actgagatga atctaatttt tagattgccc tgtattttgt taacatgtat atatgtacaa\n",
      "    13561 cagtgtgttt gtaaatatat aggaacgttt ctgaacaggg tctgtgctat gtgtaaaggt\n",
      "    13621 ttgttaactg taaagtaata taaagttata ttggatcttc tattgcacta attctagatg\n",
      "    13681 tctaattcag gatactgtct atagaaaggc attcttaaaa gttaaagaat gttacgtctt\n",
      "    13741 agttttggag actaaagtat tcccagtaaa gtgggttgag gtgagggctg tggtcctgaa\n",
      "    13801 agggacgcct ttgacatcgt ggctgtccag ttgggctgtg agctgtggca cccaggactg\n",
      "    13861 gcgctggccc ttcagaagga tctaggagag gggcttggga gcccactttt aatttctcac\n",
      "    13921 ccccatttta caaagagtgc ttagattctt acaaattatg atgtaagtta tccatttggc\n",
      "    13981 tttttcctaa ctagtcttac caaacttagg gggaaacctg tgctccatta ccacatgggt\n",
      "    14041 gcaagtcagc attgtaagtt ttctcaggtt attattatta gagaggttgg aaacattggt\n",
      "    14101 aaactctgtt gattgagaag gaaaaaaaaa gtcccattga actgttgcaa caaatcagaa\n",
      "    14161 atccacataa aagtgctctc ctgcctgggc agcaacaacc aagaacaaag ccccgggact\n",
      "    14221 gttttctttt taataaagcc acaggcaggc atcgtagctc cacagcccga ggggacacag\n",
      "    14281 gatggaaacc ccaggatgag aagggagcag ggagagttcc agaaaggggg atgaaatagg\n",
      "    14341 agtattaaaa agctgcgttg gtaagttttt catggaacca agatttgaca aaggcatctc\n",
      "    14401 ttatccttgg ttttaaattc ctgctgggag caaggcctgg tatgagcgcc ctgggtcttg\n",
      "    14461 tttttggtgt ttcgcttttc tgtaaggatt aagcagatag ggagaaggga aaaggggcct\n",
      "    14521 cactttagaa tgaatgagtc accttgtgat ttttaaattt ttattttaat aaagctaatc\n",
      "    14581 aatttctaca accttgtcac atgtagctga gtctgggatg actcagtgga tcagtggatg\n",
      "    14641 cgtggaaggt tttggtgttt ataactcatg acccaaatcc ttccaagaca caaatgaagc\n",
      "    14701 taatcttctt cccctccctt tcctctccca cttcccctct gtgtctgtgt ctgtgtgtgt\n",
      "    14761 gtgtgtgtgt gtgtgtgtgt gcgcgcgcgc gcgtgcatgc agagaggaag gaagggagct\n",
      "    14821 ttagcacaag tagctacata gtgttcaagg atcagctttc ttgaactccg aggcttggtt\n",
      "    14881 gcctttgctg aactccaagt tagtggattg cagaatggaa acttggcttt tgcggcactg\n",
      "    14941 ggtgagtttt agtttgtgtg tgtcttgctg gggggtggtg atgattgtct cagcactcac\n",
      "    15001 gcactgcaca agatggcagc aggatacagc actgcacaag atggcagctc ctctgcagct\n",
      "    15061 tcctcctcag cctccctcct tgcaccccca caggtttggc ttgtggtttt tgtcatcagt\n",
      "    15121 aacctactgc ctgagatcat gatctcttaa aagatgagac tctcggaagg gttgattgta\n",
      "    15181 tgcgtcagtg agccttctat caccttctgg aacaaagtca cttgaaatct cttgatgaga\n",
      "    15241 ttaaggagtt tagtgttact aagaaaatct gctttgggcc gcagcagtgc tgggtgttct\n",
      "    15301 cagacctgac tgaggaagtt agctgcgggc tgccctgtgg gctggtgctt caggaggaat\n",
      "    15361 ccagagaagt gttcagatgc cccccttggg ctcctttcta attttaatca gctctttaaa\n",
      "    15421 tagctgccca tctcctgtga ttgcacaacc aagcactttg acatttgcac cttaggagag\n",
      "    15481 gcagatgtta aaatggaatc caaagaccac ctagggcggg gctgggtggg agatgggagg\n",
      "    15541 gccaactgcg agctgctcca cttctcagct ctcccctgcc ctgcagccct gggccagaca\n",
      "    15601 aggccagaag gtttcagggg catttgacat cccctcctgg ttctcaccag gaaaacatcc\n",
      "    15661 aaagctttgg aggaaacagg ccctgcccct ggctccttaa atgccccgtc tctttgtaaa\n",
      "    15721 ctgatattca gccagcaatg cctaagactt tgttaagatc atttctactg cttttctttc\n",
      "    15781 tgcttcaaac acacagttcg tctctgagga aagtaaaata aatggaataa gagtaaattg\n",
      "    15841 ggtaaggaga tatccaaagc tacccagtcc cttgacccag cacagttggc cgacccgtgt\n",
      "    15901 cactccctgg ctgtcgctgc ttctctgtgc tcactgaagg gtgagccagg ccagtgcttc\n",
      "    15961 cccagcccct gggcctggtc actacacagt ggaaaacaga caagcggccc cttccccaaa\n",
      "    16021 tcccaagagt gtcttgctgc ttggtgggtg ctcatcgcaa tgttctgaag gctccagggc\n",
      "    16081 cactttgttt gtaagtatga tctgggcctc aaaataccat agtagctgct tgataaaatt\n",
      "    16141 ctaaaaatat ctggttctct attatgtaaa cactattaca gtcaccagtg tgtgaagact\n",
      "    16201 cttgagtctg gttctcatat cagagtcatc atttttcttc ctgtggaata aaatgccttg\n",
      "    16261 tggacttccc aaaaaaaaaa aaaaa\n",
      "//\n",
      "\n",
      "LOCUS       LT669800                 495 bp    DNA     linear   PLN 09-JAN-2017\n",
      "DEFINITION  Elymus wawawaiensis Gsp-1 gene for grain softness protein.\n",
      "ACCESSION   LT669800\n",
      "VERSION     LT669800.1\n",
      "KEYWORDS    .\n",
      "SOURCE      Elymus wawawaiensis\n",
      "  ORGANISM  Elymus wawawaiensis\n",
      "            Eukaryota; Viridiplantae; Streptophyta; Embryophyta; Tracheophyta;\n",
      "            Spermatophyta; Magnoliophyta; Liliopsida; Poales; Poaceae; BOP\n",
      "            clade; Pooideae; Triticodae; Triticeae; Hordeinae; Elymus.\n",
      "REFERENCE   1\n",
      "  AUTHORS   Wilkinson,M.D., Grimaldi,R. and Shewry,P.R.S.\n",
      "  TITLE     The puroindoline (Pin) genes consist of a diverse multigene family\n",
      "            with identification of important variants in Pooideae species\n",
      "  JOURNAL   Unpublished\n",
      "REFERENCE   2  (bases 1 to 495)\n",
      "  AUTHORS   Wilkinson,M.\n",
      "  TITLE     Direct Submission\n",
      "  JOURNAL   Submitted (08-NOV-2016) Lab122, ROTHAMSTED RESEARCH, Rothamsted\n",
      "            Research, Plant Science Centre, Harpenden, Herts. AL5 2JQ, UNITED\n",
      "            KINGDOM\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..495\n",
      "                     /organism=\"Elymus wawawaiensis\"\n",
      "                     /mol_type=\"genomic DNA\"\n",
      "                     /db_xref=\"taxon:129760\"\n",
      "                     /country=\"USA:Washington\"\n",
      "     gene            1..495\n",
      "                     /gene=\"Gsp-1\"\n",
      "     CDS             1..495\n",
      "                     /gene=\"Gsp-1\"\n",
      "                     /codon_start=1\n",
      "                     /product=\"grain softness protein\"\n",
      "                     /protein_id=\"SHD75395.1\"\n",
      "                     /translation=\"MKAFFLLAFLALVASTANAQYAEVPSPASEVPSMDDFGEWVAMT\n",
      "                     PGTSGSEHCEQEQPKLNSCSDYVMDRCVTKDMPLSWIFPRTWGERSCEEVQNQCCQQL\n",
      "                     RQTPPHCRCKAIWTSIQGDLSGFMSFQQGLKARKVQTAKSLPSKCNIDPKYCNIPITS\n",
      "                     GYYL\"\n",
      "ORIGIN      \n",
      "        1 atgaaggcct tcttcctcct agccttcctt gctctggtag cgagcactgc taatgcgcaa\n",
      "       61 tacgcggaag ttcccagccc agcctcagaa gttccctcaa tggatgattt tggggaatgg\n",
      "      121 gttgccatga cgcctggtac aagtggttcc gagcattgcg agcaagagca gccaaagcta\n",
      "      181 aactcttgta gcgattatgt tatggaccga tgcgtgacaa aggatatgcc gctctcttgg\n",
      "      241 atctttcccc ggacatgggg ggagagaagt tgcgaggagg tccaaaacca gtgctgccaa\n",
      "      301 cagttgaggc aaacgccgcc gcattgccgt tgcaaggcta tctggacatc aatccaaggc\n",
      "      361 gatctaagtg gcttcatgag ctttcaacaa ggtctgaaag ctagaaaggt gcagacggcc\n",
      "      421 aagagccttc cctccaagtg caacattgat ccaaaatatt gtaacatccc catcactagc\n",
      "      481 ggatattact tgtga\n",
      "//\n",
      "\n",
      "LOCUS       LT669799                 495 bp    DNA     linear   PLN 09-JAN-2017\n",
      "DEFINITION  Elymus nutans Gsp-1 gene for grain softness protein.\n",
      "ACCESSION   LT669799\n",
      "VERSION     LT669799.1\n",
      "KEYWORDS    .\n",
      "SOURCE      Elymus nutans\n",
      "  ORGANISM  Elymus nutans\n",
      "            Eukaryota; Viridiplantae; Streptophyta; Embryophyta; Tracheophyta;\n",
      "            Spermatophyta; Magnoliophyta; Liliopsida; Poales; Poaceae; BOP\n",
      "            clade; Pooideae; Triticodae; Triticeae; Hordeinae; Elymus.\n",
      "REFERENCE   1\n",
      "  AUTHORS   Wilkinson,M.D., Grimaldi,R. and Shewry,P.R.S.\n",
      "  TITLE     The puroindoline (Pin) genes consist of a diverse multigene family\n",
      "            with identification of important variants in Pooideae species\n",
      "  JOURNAL   Unpublished\n",
      "REFERENCE   2  (bases 1 to 495)\n",
      "  AUTHORS   Wilkinson,M.\n",
      "  TITLE     Direct Submission\n",
      "  JOURNAL   Submitted (08-NOV-2016) Lab122, ROTHAMSTED RESEARCH, Rothamsted\n",
      "            Research, Plant Science Centre, Harpenden, Herts. AL5 2JQ, UNITED\n",
      "            KINGDOM\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..495\n",
      "                     /organism=\"Elymus nutans\"\n",
      "                     /mol_type=\"genomic DNA\"\n",
      "                     /db_xref=\"taxon:400237\"\n",
      "                     /country=\"China:Xizang\"\n",
      "     gene            1..495\n",
      "                     /gene=\"Gsp-1\"\n",
      "     CDS             1..495\n",
      "                     /gene=\"Gsp-1\"\n",
      "                     /codon_start=1\n",
      "                     /product=\"grain softness protein\"\n",
      "                     /protein_id=\"SHD75394.1\"\n",
      "                     /translation=\"MKTFFLLAFLALIVSTAIAQFVEVPSPDAQAPTGDGFGEWVAIA\n",
      "                     PSASGSENCEEEQPKLDSCSDYVMDRCVMKDMPLYWVFPRTWGKRSCEEVRNQCCQQL\n",
      "                     RQTTSRCRCKAIWTSIQGDLSGFRGFQQGLKARTVQTAKSLPSKCNMDPKDCNIPITS\n",
      "                     GYYL\"\n",
      "ORIGIN      \n",
      "        1 atgaagacct tcttcctcct agctttcctt gctctgatag tgagcactgc tattgcacaa\n",
      "       61 tttgtggaag ttcccagccc agatgcacaa gctcccaccg gggatggttt tggggaatgg\n",
      "      121 gttgcgatag cgcctagtgc gagtggttct gagaattgcg aggaagagca gccaaagcta\n",
      "      181 gactcttgta gcgattatgt tatggaccgg tgtgtgatga aggatatgcc gctatattgg\n",
      "      241 gtctttcctc ggacttgggg gaagagaagc tgtgaggagg tccgaaacca gtgctgccag\n",
      "      301 caattgaggc aaacgacgtc gcgttgccgt tgcaaggcta tctggacatc aatccaaggc\n",
      "      361 gatctaagtg gcttcagggg ctttcaacaa ggtttgaaag cccgaacagt gcagacagcc\n",
      "      421 aagagccttc cctccaagtg caacatggat ccaaaagact gcaacatccc catcactagc\n",
      "      481 ggatattact tgtga\n",
      "//\n",
      "\n",
      "LOCUS       LT669798                 447 bp    DNA     linear   PLN 09-JAN-2017\n",
      "DEFINITION  Elymus repens Pinb-2v gene for puroindoline b.\n",
      "ACCESSION   LT669798\n",
      "VERSION     LT669798.1\n",
      "KEYWORDS    .\n",
      "SOURCE      Elymus repens\n",
      "  ORGANISM  Elymus repens\n",
      "            Eukaryota; Viridiplantae; Streptophyta; Embryophyta; Tracheophyta;\n",
      "            Spermatophyta; Magnoliophyta; Liliopsida; Poales; Poaceae; BOP\n",
      "            clade; Pooideae; Triticodae; Triticeae; Hordeinae; Elymus.\n",
      "REFERENCE   1\n",
      "  AUTHORS   Wilkinson,M.D., Grimaldi,R. and Shewry,P.R.S.\n",
      "  TITLE     The puroindoline (Pin) genes consist of a diverse multigene family\n",
      "            with identification of important variants in Pooideae species\n",
      "  JOURNAL   Unpublished\n",
      "REFERENCE   2  (bases 1 to 447)\n",
      "  AUTHORS   Wilkinson,M.\n",
      "  TITLE     Direct Submission\n",
      "  JOURNAL   Submitted (08-NOV-2016) Lab122, ROTHAMSTED RESEARCH, Rothamsted\n",
      "            Research, Plant Science Centre, Harpenden, Herts. AL5 2JQ, UNITED\n",
      "            KINGDOM\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..447\n",
      "                     /organism=\"Elymus repens\"\n",
      "                     /mol_type=\"genomic DNA\"\n",
      "                     /db_xref=\"taxon:52152\"\n",
      "                     /country=\"United Kingdom\"\n",
      "     gene            1..447\n",
      "                     /gene=\"Pinb-2v\"\n",
      "     CDS             1..447\n",
      "                     /gene=\"Pinb-2v\"\n",
      "                     /codon_start=1\n",
      "                     /product=\"puroindoline b\"\n",
      "                     /protein_id=\"SHD75393.1\"\n",
      "                     /translation=\"MKTLFLLALLSLVVSTTFAQYKEVGGSYENGGGGFGSQNCPPEK\n",
      "                     TKLDSCQDYVMERCLAVKGFPIARLLKWWKGACEQEVLDQCCQQLRPVAKKCRCEAIW\n",
      "                     RAVQGNLGGLFGFQQGQITNLGAMMLPSKCKMDSSCKFVHTNGYYY\"\n",
      "ORIGIN      \n",
      "        1 atgaagacct tattcctcct agctctcctt tctcttgtag tgagcacaac ctttgcgcaa\n",
      "       61 tacaaagagg ttggtggtag ctatgagaac ggtggcggag ggtttggttc tcaaaactgc\n",
      "      121 ccaccggaga agacgaagct ggactcttgc caggattacg tgatggagcg gtgcttggca\n",
      "      181 gtgaagggtt ttccgatcgc caggcttttg aaatggtgga agggcgcctg tgagcaagag\n",
      "      241 gtcctagacc agtgttgcca gcaactacgc ccggtagcaa aaaagtgccg atgtgaggcc\n",
      "      301 atttggagag ccgtccaagg caaccttggt ggcttatttg gctttcagca aggccagata\n",
      "      361 accaaccttg gggccatgat gttgccctcc aaatgcaaaa tggattccag ctgcaagttc\n",
      "      421 gtccatacta atggctatta ctactga\n",
      "//\n"
     ]
    }
   ],
   "source": [
    "require 'bio'\n",
    "\n",
    "ncbi = Bio::NCBI::REST.new\n",
    "max = 5\n",
    "\n",
    "# note that you can now use spaces in your search terms - BioRuby will format the URL correctly\n",
    "ids = ncbi.esearch('\"Wilkinson MD\"[AUTH]', {\"db\"=>\"nucleotide\", \"rettype\"=>\"gb\", \"retmax\" => max})\n",
    "\n",
    "idlist = ids.join(\",\")\n",
    "puts idlist\n",
    "puts \"\\n\\nRecords are\\n\\n\"\n",
    "records = ncbi.efetch(idlist, {\"db\"=>\"nucleotide\", \"rettype\"=>\"gb\", \"retmode\"=>\"text\", \"retmax\"=> max})\n",
    "puts records\n",
    "\n",
    "# and its just that easy! \n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Better use of PubMed\n",
    "\n",
    "Most reference managers (EndNote, Mendeley, and even LaTex) are able to work with files in BibTex format.  Let's do one last BioRuby call to PubMed to retrieve articles in that format.\n",
    "\n",
    "Our task is to get the abstracts for the nucleotide sequence files that were discovered in the search above.  This is called a \"Link\".  Earlier in this Lesson, we looked at the output from the EInfo tool - it told you what FIELDs you could query for each database, but also what LINKS existed between one database and another.  Let's look at that again - the EInfo about the nuccore database:\n",
    "\n",
    "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/einfo.fcgi?db=nuccore\n",
    "\n",
    "Near the end of that page you will find:\n",
    "\n",
    "    <Link>\n",
    "        <Name>nuccore_pubmed</Name>\n",
    "        <Menu>PubMed Links</Menu>\n",
    "        <Description>PubMed articles cited by Nucleotide sequence record</Description>\n",
    "        <DbTo>pubmed</DbTo>\n",
    "    </Link>\n",
    "\n",
    "This describes that there is a link between nuccore (the sequence database) and pubmed (the publications database).  The tool that we need to find the links between the list of IDs for Database A and the list of IDs for Database B is the ELink tool\n",
    "\n",
    "http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi\n",
    "\n",
    "It works like all other EUtils; however, there is no BioRuby wrapper for this one, so we must do the hard work ourselves.  To see some example code in Perl go to:  https://www.ncbi.nlm.nih.gov/books/NBK25498/#chapter3.Application_4_Finding_unique_se.  We will now do the same thing in Ruby.\n",
    "\n",
    "\n",
    "### ELink parameters are:  \n",
    "* dbfrom\n",
    "* db  (i.e. which db are you linking TO)\n",
    "* id = ...list of ids...\n",
    "\n",
    "So, like all other EUtils, we can construct a URL that calls the ELink service:\n",
    "\n",
    "http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=nuccore;db=pubmed;id=732693005,732693505,1129728243,1129728241,1129728239\n",
    "\n",
    "If you click on that, you will see the XML document that contains the PubMed IDs that correspond to the nucleotide sequence IDs we sent to the **elink** tool.  They are separated into \"Link Sets\", corresponding to different subsections of pubmed (e.g. **nuccore_pubmed_refseq**).  \n",
    "\n",
    "HOWEVER:  If you read a little bit further in the documentation (for all EUtilities!), the instructions say \"use POST when you have many IDs (>200)\" \n",
    "\n",
    ">**Required Parameter  Used only when input is from a UID list**\n",
    "\n",
    ">**id**\n",
    "\n",
    ">UID list. Either a single UID or a comma-delimited list of UIDs may be provided. All of the UIDs must be from the database specified by dbfrom. There is no set maximum for the number of UIDs that can be passed to ELink, but **if more than about 200 UIDs are to be provided, the request should be made using the HTTP POST method.**\n",
    "\n",
    "\n",
    "What does \"HTTP POST\" mean?   We will discuss it in detail in a later lecture, but for now, you only need to understand that it is another way of sending information to a Web server.  Using HTTP GET, you put the information at the end of the URL (<code>scriptname?a=b;c=d;...</code>).   With HTTP POST you send it in the \"body\" of a message.  \n",
    "\n",
    "It is important to learn how to create an HTTP POST request, so even though we _can_ use the URL above (because we only have 5 ID numbers), we should look at and understand the other solution.  An example of an HTTP POST is below, where we are sending a request to the ELink service of NCBI:\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "732693005,732693505,1129728243,1129728241,1129728239\n",
      "<?xml version=\"1.0\" encoding=\"UTF-8\" ?>\n",
      "<!DOCTYPE eLinkResult PUBLIC \"-//NLM//DTD elink 20101123//EN\" \"https://eutils.ncbi.nlm.nih.gov/eutils/dtd/20101123/elink.dtd\">\n",
      "<eLinkResult>\n",
      "\n",
      "  <LinkSet>\n",
      "    <DbFrom>nuccore</DbFrom>\n",
      "    <IdList>\n",
      "      <Id>732693005</Id>\n",
      "      <Id>732693505</Id>\n",
      "      <Id>1129728243</Id>\n",
      "      <Id>1129728241</Id>\n",
      "      <Id>1129728239</Id>\n",
      "    </IdList>\n",
      "    <LinkSetDb>\n",
      "      <DbTo>pubmed</DbTo>\n",
      "      <LinkName>nuccore_pubmed</LinkName>\n",
      "      \n",
      "        <Link>\n",
      "\t\t\t\t<Id>25896417</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>20610895</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>19056867</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>15498460</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>12421765</Id>\n",
      "\t\t\t</Link>\n",
      "      \n",
      "    </LinkSetDb>\n",
      "    \n",
      "    <LinkSetDb>\n",
      "      <DbTo>pubmed</DbTo>\n",
      "      <LinkName>nuccore_pubmed_refseq</LinkName>\n",
      "      \n",
      "        <Link>\n",
      "\t\t\t\t<Id>26344197</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>25896417</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>21182205</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>20610895</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>20379614</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>19056867</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>16710414</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>15498460</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>15489334</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>14702039</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>12477932</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>12421765</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>9455484</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>9110174</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>8619474</Id>\n",
      "\t\t\t</Link>\n",
      "      \n",
      "    </LinkSetDb>\n",
      "    <LinkSetDb>\n",
      "      <DbTo>pubmed</DbTo>\n",
      "      <LinkName>nuccore_pubmed_weighted</LinkName>\n",
      "      \n",
      "        <Link>\n",
      "\t\t\t\t<Id>26344197</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>25896417</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>21182205</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>20610895</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>20379614</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>19056867</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>16710414</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>15498460</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>15489334</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>14702039</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>12477932</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>12421765</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>9455484</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>9110174</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>8619474</Id>\n",
      "\t\t\t</Link>\n",
      "      \n",
      "    </LinkSetDb>\n",
      "  </LinkSet>\n",
      "</eLinkResult>\n",
      "\n"
     ]
    }
   ],
   "source": [
    "require 'net/http'\n",
    "require 'bio'\n",
    "\n",
    "ncbi = Bio::NCBI::REST.new\n",
    "max = 5\n",
    "\n",
    "# note that you can now use spaces in your search terms - BioRuby will format the URL correctly\n",
    "ids = ncbi.esearch('\"Wilkinson MD\"[AUTH]', {\"db\"=>\"nucleotide\", \"rettype\"=>\"gb\", \"retmax\" => max})\n",
    "\n",
    "idlist = ids.join(\",\")\n",
    "puts idlist\n",
    "\n",
    "# prepare for HTTP POST - this is very similar to what is in the \"fetch\" routine\n",
    "# from your other scripts.\n",
    "# Note:  some of what we are doing here is a bit hard to understand, because\n",
    "# NCBI uses \"https\" for its requests (encrypted messages) instead of simple http\n",
    "# HTTPS runs on port 443, and has to be explicitly activated\n",
    "elink = URI(\"https://eutils.ncbi.nlm.nih.gov:443/entrez/eutils/elink.fcgi\")\n",
    "request = Net::HTTP::Post.new(elink)\n",
    "https = Net::HTTP.new(elink.host,elink.port)\n",
    "https.use_ssl = true  # tell the HTTP connection to use ecryption (SSL = Secure Socket Layer)\n",
    "\n",
    "\n",
    "# here is where we create the content of the body - called \"form data\".  We will see what this\n",
    "# looks like in a later lecture.  For now, it is just treated like a Hash in Ruby.\n",
    "request.set_form_data(\n",
    "  'email' => 'markw@illuminae.com',\n",
    "  'dbfrom' => 'nuccore', \n",
    "  'db' => 'pubmed',\n",
    "  'retmode' => 'xml',\n",
    "  'id' => idlist)\n",
    "\n",
    "\n",
    "\n",
    "output = https.request(request)   # make the request (we really should check if it is successful)\n",
    "\n",
    "puts output.body\n",
    "\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "\n",
    "#  Parsing XML\n",
    "\n",
    "In an earlier lesson we learned how to extract information from JSON.  In the example above the output of the EUtils service is in XML format, so we now need to learn how to extract information from XML.\n",
    "\n",
    "The most popular Ruby library for XML parsing is called 'nokogiri' (this translates to English as \"saw\" and Spanish as \"sierra\" como \"sierra circular\").  So that's what we will use.  The XML output from EUtils is quite simple (we will see more complex examples later in the course!) so let's use it as our learning example\n",
    "\n",
    "\n",
    "    <eLinkResult>\n",
    "        <LinkSet>\n",
    "            <LinkSetDb>\n",
    "                  <DbTo>pubmed</DbTo>\n",
    "                  <LinkName>nuccore_pubmed</LinkName>\n",
    "\n",
    "                  <Link>\n",
    "                        <Id>25896417</Id>\n",
    "                  </Link>\n",
    "                  <Link>\n",
    "                        <Id>20610895</Id>\n",
    "                  </Link>\n",
    "                  <Link>\n",
    "                        <Id>19056867</Id>\n",
    "                  </Link>\n",
    "                  <Link>\n",
    "                        <Id>15498460</Id>\n",
    "                  </Link>\n",
    "                  <Link>\n",
    "                        <Id>12421765</Id>\n",
    "                  </Link>\n",
    "\n",
    "            </LinkSetDb>\n",
    "        </LinkSet>\n",
    "     <eLinkResut>\n",
    "\n",
    "This is the relevant section we want to extract IDs from.  XML documents are \"hierarchical\", sub-sections are enclosed in parent sections in the pattern:\n",
    "        \n",
    "        <A> <B> data </B>  </A>\n",
    "\n",
    "Given this structure, it is normal to talk about the \"path\" to the data.  In this case, the data is A-->B-->data.  \n",
    "\n",
    "In the output from ELink, the path is:  LinkSetDb-->Link-->Id-->data\n",
    "\n",
    "These \"paths\" are used in Ruby to define the piece of data we want to extract.  For example:\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PubMed = 25896417\n",
      "PubMed = 20610895\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "0"
      ]
     },
     "execution_count": 32,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "require 'nokogiri'\n",
    "\n",
    "xml = <<EOF\n",
    "\n",
    "    <eLinkResult>\n",
    "        <LinkSet>\n",
    "\n",
    "            <LinkSetDb>\n",
    "                <Link>\n",
    "                      <Id>25896417</Id>\n",
    "                </Link>\n",
    "                <Link>\n",
    "                      <Id>20610895</Id>\n",
    "                </Link>\n",
    "            </LinkSetDb>\n",
    "        </LinkSet>\n",
    "    <eLinkResut>\n",
    "\n",
    "EOF\n",
    "\n",
    "\n",
    "doc = Nokogiri::XML(xml)\n",
    "# puts doc.class  #  You should read the documentation for yourself one day!  \n",
    "                  # Also read about \"XPath\" for more complex XML documents\n",
    "\n",
    "doc.xpath('//LinkSet/LinkSetDb/Link').each do |node|    # the \"path\" (\"xpath\")to the data, starts with '//'\n",
    "  # puts node.class\n",
    "  puts \"PubMed = \" + node.at_xpath('Id').content\n",
    "end\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#  Putting it all together!\n",
    "\n",
    "Finally, we will create a script that:\n",
    "\n",
    "* retrieves the nucleotide database ids\n",
    "* retrieves the cross-reference links to PubMed\n",
    "* extracts the PubMed IDs\n",
    "* retrieves the PubMed Abstracts for those IDs\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 55,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PUBMED IDs are: [\"25896417\", \"20610895\", \"19056867\", \"15498460\", \"12421765\", \"26344197\", \"25896417\", \"21182205\", \"20610895\", \"20379614\", \"19056867\", \"16710414\", \"15498460\", \"15489334\", \"14702039\", \"12477932\", \"12421765\", \"9455484\", \"9110174\", \"8619474\", \"26344197\", \"25896417\", \"21182205\", \"20610895\", \"20379614\", \"19056867\", \"16710414\", \"15498460\", \"15489334\", \"14702039\", \"12477932\", \"12421765\", \"9455484\", \"9110174\", \"8619474\"]\n",
      "1. J Innate Immun. 2015;7(5):545-53. doi: 10.1159/000381265. Epub 2015 Apr 17.\n",
      "\n",
      "VPS13D Gene Variant Is Associated with Altered IL-6 Production and Mortality in\n",
      "Septic Shock.\n",
      "\n",
      "Nakada TA(1), Boyd JH, Russell JA, Aguirre-Hernndez R, Wilkinson MD, Thair SA,\n",
      "Nakada E, McConechy MK, Fjell CD, Walley KR.\n",
      "\n",
      "Author information: \n",
      "(1)Critical Care Research Laboratories, Centre for Heart Lung Innovation, St.\n",
      "Paul's Hospital, University of British Columbia, Vancouver, B.C., Canada.\n",
      "\n",
      "BACKGROUND: Genetic variations contribute to septic shock mortality. To discover \n",
      "a novel locus, we performed in vitro genome-wide association studies (GWAS) and\n",
      "further tested the result in a cohort of septic shock patients.\n",
      "METHODS: Two in vitro GWAS using a quantitative trait locus analysis of\n",
      "stimulated IL-6 production in lymphoblastoid cells from 60 individuals of\n",
      "European ancestry were performed. VPS13D rs6685273 was genotyped in European\n",
      "ancestry patients (n = 498). The VPS13D gene was silenced in vitro.\n",
      "RESULTS: Two GWAS using lymphoblastoid cells identified the locus of VPS13D\n",
      "rs6685273 that was significant in the same direction in both GWAS. The VPS13D\n",
      "rs6685273 C allele was associated with increased IL-6 production. Patients with\n",
      "septic shock who had the VPS13D rs6685273 CC genotype had an increased 28-day\n",
      "mortality (p = 0.023) and more organ failure (p < 0.05) compared to the CT/TT\n",
      "genotypes. VPS13D in vitro gene silencing in the HeLa cell line increased IL-6\n",
      "production. Furthermore, the rs6685273 genotype was associated with differential \n",
      "VPS13D splice variant expression.\n",
      "CONCLUSIONS: The VPS13D rs6685273 C allele was associated with increased IL-6\n",
      "production in vitro. The patients with the VPS13D rs6685273 CC genotype had\n",
      "increased 28-day mortality and increased organ failure. VPS13D appears to\n",
      "regulate IL-6 production.\n",
      "\n",
      " 2015 S. Karger AG, Basel.\n",
      "\n",
      "DOI: 10.1159/000381265 \n",
      "PMID: 25896417  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "2. J Atheroscler Thromb. 2010 Oct 27;17(10):1054-62. Epub 2010 Jul 2.\n",
      "\n",
      "Identification of evidence suggestive of an association with peripheral arterial \n",
      "disease at the OSBPL10 locus by genome-wide investigation in the Japanese\n",
      "population.\n",
      "\n",
      "Koriyama H(1), Nakagami H, Katsuya T, Sugimoto K, Yamashita H, Takami Y, Maeda S,\n",
      "Kubo M, Takahashi A, Nakamura Y, Ogihara T, Rakugi H, Kaneda Y, Morishita R.\n",
      "\n",
      "Author information: \n",
      "(1)Department of Geriatric Medicine and Nephrology, Osaka University Graduate\n",
      "School of Medicine, 2-2 Yamada-oka, Suita, Japan.\n",
      "\n",
      "AIM: Peripheral arterial disease (PAD) is a common cause of cardiovascular\n",
      "morbidity and an independent predictor of cardiovascular mortality. However,\n",
      "little is known about the genetic basis of PAD. To elucidate this, we performed a\n",
      "two-staged genome-wide association study in Japanese individuals.\n",
      "METHODS: We initially tested 222,285 single-nucleotide polymorphisms (SNPs).\n",
      "After the first screening in a panel of 195 PAD cases and 1,358 controls, 2,696\n",
      "SNPs (1.2%) were further genotyped in the second screening using another panel of\n",
      "699 PAD cases and 1540 controls. In both screenings, controls were subjects\n",
      "affected with some diseases other than PAD.\n",
      "RESULTS: When analyzed in the combined panel, the strongest signal of PAD\n",
      "association was observed at rs1902341 in the intron of OSBPL10 (p=4.7E-7 for\n",
      "trend test; OR=1.31, 95% CI 1.18-1.46). Also, PAD was modestly associated at\n",
      "several other loci such as rs2554503 in CSMD1 (p=5.7E-5; OR=1.32, 95% CI\n",
      "1.15-1.51) or rs235243 in VSP13D (p=0.04; OR=1.18, 95% CI 1.01-1.37).\n",
      "CONCLUSION: Our genome-wide exploration identified suggestive evidence of PAD\n",
      "association at the OSBPL10 locus. Because the association has not reached a\n",
      "genome-wide significant level, further replication study is warranted for\n",
      "verification in the Japanese population.\n",
      "\n",
      "\n",
      "PMID: 20610895  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "3. J Am Soc Nephrol. 2009 Feb;20(2):363-79. doi: 10.1681/ASN.2008040406. Epub 2008\n",
      "Dec 3.\n",
      "\n",
      "Large-scale proteomics and phosphoproteomics of urinary exosomes.\n",
      "\n",
      "Gonzales PA(1), Pisitkun T, Hoffert JD, Tchapyjnikov D, Star RA, Kleta R, Wang\n",
      "NS, Knepper MA.\n",
      "\n",
      "Author information: \n",
      "(1)Laboratory of Kidney and Electrolyte Metabolism, National Heart, Lung, and\n",
      "Blood Institute, National Institutes of Health, Bethesda, MD 20892, USA.\n",
      "\n",
      "Normal human urine contains large numbers of exosomes, which are 40- to 100-nm\n",
      "vesicles that originate as the internal vesicles in multivesicular bodies from\n",
      "every renal epithelial cell type facing the urinary space. Here, we used LC-MS/MS\n",
      "to profile the proteome of human urinary exosomes. Overall, the analysis\n",
      "identified 1132 proteins unambiguously, including 177 that are represented on the\n",
      "Online Mendelian Inheritance in Man database of disease-related genes, suggesting\n",
      "that exosome analysis is a potential approach to discover urinary biomarkers. We \n",
      "extended the proteomic analysis to phosphoproteomic profiling using neutral loss \n",
      "scanning, and this yielded multiple novel phosphorylation sites, including\n",
      "serine-811 in the thiazide-sensitive Na-Cl co-transporter, NCC. To demonstrate\n",
      "the potential use of exosome analysis to identify a genetic renal disease, we\n",
      "carried out immunoblotting of exosomes from urine samples of patients with a\n",
      "clinical diagnosis of Bartter syndrome type I, showing an absence of the\n",
      "sodium-potassium-chloride co-transporter 2, NKCC2. The proteomic data are\n",
      "publicly accessible at http://dir.nhlbi.nih.gov/papers/lkem/exosome/.\n",
      "\n",
      "DOI: 10.1681/ASN.2008040406 \n",
      "PMCID: PMC2637050\n",
      "PMID: 19056867  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "4. Genomics. 2004 Sep;84(3):536-49.\n",
      "\n",
      "Analysis of the human VPS13 gene family.\n",
      "\n",
      "Velayos-Baeza A(1), Vettori A, Copley RR, Dobson-Stone C, Monaco AP.\n",
      "\n",
      "Author information: \n",
      "(1)Wellcome Trust Centre for Human Genetics, University of Oxford, Headington,\n",
      "OX3 7BN Oxford, UK.\n",
      "\n",
      "The gene mutated in chorea-acanthocytosis (CHAC; approved gene symbol VPS13A)\n",
      "encodes chorein, a protein similar to yeast Vps13p. We detected several similar\n",
      "putative human proteins by BLAST analysis of chorein. We characterized the\n",
      "structure of three new genes encoding these CHAC-similar proteins, located on\n",
      "chromosomes 1p36, 8q22, and 15q21. The most similar gene in yeast to all four\n",
      "human genes is Vps13, and therefore the human genes were named VPS13A (CHAC,\n",
      "9q21), VPS13B (8q22), VPS13C (15q21), and VPS13D (1p36). VPS13B has recently been\n",
      "reported as COH1, altered in Cohen syndrome. For each gene, we describe several\n",
      "alternative splicing variants; at least two transcripts per gene are major forms.\n",
      "The expression pattern of these genes is ubiquitous, with some tissue-specific\n",
      "differences between several transcript variants. Protein sequence comparisons\n",
      "suggest that intramolecular duplications have played an important role in the\n",
      "evolution of this gene family.\n",
      "\n",
      "DOI: 10.1016/j.ygeno.2004.04.012 \n",
      "PMID: 15498460  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "5. Genome Res. 2002 Nov;12(11):1773-84.\n",
      "\n",
      "Protein-protein interactions between large proteins: two-hybrid screening using a\n",
      "functionally classified library composed of long cDNAs.\n",
      "\n",
      "Nakayama M(1), Kikuno R, Ohara O.\n",
      "\n",
      "Author information: \n",
      "(1)Department of Human Gene Research, Kazusa DNA Research Institute, Kisarazu,\n",
      "Chiba 292-0818, Japan. nmanabu@kazusa.or.jp\n",
      "\n",
      "Large proteins have multiple domains that are potentially capable of binding many\n",
      "kinds of partners. It is conceivable, therefore, that such proteins could\n",
      "function as an intricate framework of assembly protein complexes. To\n",
      "comprehensively study protein-protein interactions between large KIAA proteins,\n",
      "we have constructed a library composed of 1087 KIAA cDNA clones based on prior\n",
      "functional classifications done in silico. We were guided by two principles that \n",
      "raise the success rate for detecting interactions per tested combination: we\n",
      "avoided testing low-probability combinations, and reduced the number of potential\n",
      "false negatives that arise from the fact that large proteins cannot reliably be\n",
      "expressed in yeast. The latter was addressed by constructing a cDNA library\n",
      "comprised of random fragments encoding large proteins. Cytoplasmic domains of\n",
      "KIAA transmembrane proteins (>1000 amino acids) were used as bait for yeast\n",
      "two-hybrid screening. Our analyses reveal that several KIAA proteins bearing a\n",
      "transmembrane region have the capability of binding to other KIAA proteins\n",
      "containing domains (e.g., PDZ, SH3, rhoGEF, and spectrin) known to be localized\n",
      "to highly specialized submembranous sites, indicating that they participate in\n",
      "cellular junction formation, receptor or channel clustering, and intracellular\n",
      "signaling events. Our representative library should be a very useful resource for\n",
      "detecting previously unidentified interactions because it complements\n",
      "conventional expression libraries, which seldom contain large cDNAs.\n",
      "\n",
      "DOI: 10.1101/gr.406902 \n",
      "PMCID: PMC187542\n",
      "PMID: 12421765  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "6. Nature. 2015 Sep 17;525(7569):339-44. doi: 10.1038/nature14877. Epub 2015 Sep 7.\n",
      "\n",
      "Panorama of ancient metazoan macromolecular complexes.\n",
      "\n",
      "Wan C(1)(2), Borgeson B(2), Phanse S(1), Tu F(2), Drew K(2), Clark G(3), Xiong\n",
      "X(4)(5), Kagan O(1), Kwan J(1)(4), Bezginov A(3), Chessman K(4)(5), Pal S(5),\n",
      "Cromar G(4)(5), Papoulas O(2), Ni Z(1), Boutz DR(2), Stoilova S(1), Havugimana\n",
      "PC(1), Guo X(1), Malty RH(6), Sarov M(7), Greenblatt J(1)(4), Babu M(6), Derry\n",
      "WB(4)(5), Tillier ER(3), Wallingford JB(2)(8), Parkinson J(4)(5), Marcotte\n",
      "EM(2)(8), Emili A(1)(4).\n",
      "\n",
      "Author information: \n",
      "(1)Donnelly Centre for Cellular and Biomolecular Research, University of Toronto,\n",
      "Toronto, Ontario M5S 3E1, Canada.\n",
      "(2)Center for Systems and Synthetic Biology, Institute for Cellular and Molecular\n",
      "Biology, University of Texas at Austin, Austin, Texas 78712, USA.\n",
      "(3)Department of Medical Biophysics, Toronto, Ontario M5G 1L7, Canada.\n",
      "(4)Department of Molecular Genetics, University of Toronto, Toronto, Ontario M5S \n",
      "1A8, Canada.\n",
      "(5)Hospital for Sick Children, Toronto, Ontario M5G 1X8, Canada.\n",
      "(6)Department of Biochemistry, University of Regina, Regina, Saskatchewan S4S\n",
      "0A2, Canada.\n",
      "(7)Max Planck Institute of Molecular Cell Biology and Genetics, 01307 Dresden,\n",
      "Germany.\n",
      "(8)Department of Molecular Biosciences, University of Texas at Austin, Austin,\n",
      "Texas 78712, USA.\n",
      "\n",
      "Comment in\n",
      "    Nat Methods. 2015 Nov;12(11):1011.\n",
      "    Trends Biochem Sci. 2016 Jan;41(1):4-6.\n",
      "\n",
      "Macromolecular complexes are essential to conserved biological processes, but\n",
      "their prevalence across animals is unclear. By combining extensive biochemical\n",
      "fractionation with quantitative mass spectrometry, here we directly examined the \n",
      "composition of soluble multiprotein complexes among diverse metazoan models.\n",
      "Using an integrative approach, we generated a draft conservation map consisting\n",
      "of more than one million putative high-confidence co-complex interactions for\n",
      "species with fully sequenced genomes that encompasses functional modules present \n",
      "broadly across all extant animals. Clustering reveals a spectrum of conservation,\n",
      "ranging from ancient eukaryotic assemblies that have probably served cellular\n",
      "housekeeping roles for at least one billion years, ancestral complexes that have \n",
      "accrued contemporary components, and rarer metazoan innovations linked to\n",
      "multicellularity. We validated these projections by independent co-fractionation \n",
      "experiments in evolutionarily distant species, affinity purification and\n",
      "functional analyses. The comprehensiveness, centrality and modularity of these\n",
      "reconstructed interactomes reflect their fundamental mechanistic importance and\n",
      "adaptive value to animal cell systems.\n",
      "\n",
      "DOI: 10.1038/nature14877 \n",
      "PMCID: PMC5036527\n",
      "PMID: 26344197  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "7. J Innate Immun. 2015;7(5):545-53. doi: 10.1159/000381265. Epub 2015 Apr 17.\n",
      "\n",
      "VPS13D Gene Variant Is Associated with Altered IL-6 Production and Mortality in\n",
      "Septic Shock.\n",
      "\n",
      "Nakada TA(1), Boyd JH, Russell JA, Aguirre-Hernndez R, Wilkinson MD, Thair SA,\n",
      "Nakada E, McConechy MK, Fjell CD, Walley KR.\n",
      "\n",
      "Author information: \n",
      "(1)Critical Care Research Laboratories, Centre for Heart Lung Innovation, St.\n",
      "Paul's Hospital, University of British Columbia, Vancouver, B.C., Canada.\n",
      "\n",
      "BACKGROUND: Genetic variations contribute to septic shock mortality. To discover \n",
      "a novel locus, we performed in vitro genome-wide association studies (GWAS) and\n",
      "further tested the result in a cohort of septic shock patients.\n",
      "METHODS: Two in vitro GWAS using a quantitative trait locus analysis of\n",
      "stimulated IL-6 production in lymphoblastoid cells from 60 individuals of\n",
      "European ancestry were performed. VPS13D rs6685273 was genotyped in European\n",
      "ancestry patients (n = 498). The VPS13D gene was silenced in vitro.\n",
      "RESULTS: Two GWAS using lymphoblastoid cells identified the locus of VPS13D\n",
      "rs6685273 that was significant in the same direction in both GWAS. The VPS13D\n",
      "rs6685273 C allele was associated with increased IL-6 production. Patients with\n",
      "septic shock who had the VPS13D rs6685273 CC genotype had an increased 28-day\n",
      "mortality (p = 0.023) and more organ failure (p < 0.05) compared to the CT/TT\n",
      "genotypes. VPS13D in vitro gene silencing in the HeLa cell line increased IL-6\n",
      "production. Furthermore, the rs6685273 genotype was associated with differential \n",
      "VPS13D splice variant expression.\n",
      "CONCLUSIONS: The VPS13D rs6685273 C allele was associated with increased IL-6\n",
      "production in vitro. The patients with the VPS13D rs6685273 CC genotype had\n",
      "increased 28-day mortality and increased organ failure. VPS13D appears to\n",
      "regulate IL-6 production.\n",
      "\n",
      " 2015 S. Karger AG, Basel.\n",
      "\n",
      "DOI: 10.1159/000381265 \n",
      "PMID: 25896417  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "8. Proteomics. 2011 Jan;11(1):172-9. doi: 10.1002/pmic.201000217. Epub 2010 Dec 1.\n",
      "\n",
      "Identification of proteins associated with ligand-activated estrogen receptor \n",
      "in human breast cancer cell nuclei by tandem affinity purification and nano\n",
      "LC-MS/MS.\n",
      "\n",
      "Tarallo R(1), Bamundo A, Nassa G, Nola E, Paris O, Ambrosino C, Facchiano A,\n",
      "Baumann M, Nyman TA, Weisz A.\n",
      "\n",
      "Author information: \n",
      "(1)Department of General Pathology, Second University of Naples, Napoli, Italy.\n",
      "\n",
      "Estrogen receptor  (ER-) is a key mediator of estrogen actions in breast cancer\n",
      "(BC) cells. Understanding the effects of ligand-activated ER- in target cells\n",
      "requires identification of the molecular partners acting in concert with this\n",
      "nuclear receptor to transduce the hormonal signal. We applied tandem affinity\n",
      "purification (TAP), glycerol gradient centrifugation and MS analysis to isolate\n",
      "and identify proteins interacting with ligand-activated ER- in MCF-7 cell\n",
      "nuclei. This led to the identification of 264 ER-associated proteins, whose\n",
      "functions highlight the hinge role of ER- in the coordination of multiple\n",
      "hormone-regulated nuclear processes in BC cells.\n",
      "\n",
      "Copyright  2011 WILEY-VCH Verlag GmbH & Co. KGaA, Weinheim.\n",
      "\n",
      "DOI: 10.1002/pmic.201000217 \n",
      "PMID: 21182205  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "9. J Atheroscler Thromb. 2010 Oct 27;17(10):1054-62. Epub 2010 Jul 2.\n",
      "\n",
      "Identification of evidence suggestive of an association with peripheral arterial \n",
      "disease at the OSBPL10 locus by genome-wide investigation in the Japanese\n",
      "population.\n",
      "\n",
      "Koriyama H(1), Nakagami H, Katsuya T, Sugimoto K, Yamashita H, Takami Y, Maeda S,\n",
      "Kubo M, Takahashi A, Nakamura Y, Ogihara T, Rakugi H, Kaneda Y, Morishita R.\n",
      "\n",
      "Author information: \n",
      "(1)Department of Geriatric Medicine and Nephrology, Osaka University Graduate\n",
      "School of Medicine, 2-2 Yamada-oka, Suita, Japan.\n",
      "\n",
      "AIM: Peripheral arterial disease (PAD) is a common cause of cardiovascular\n",
      "morbidity and an independent predictor of cardiovascular mortality. However,\n",
      "little is known about the genetic basis of PAD. To elucidate this, we performed a\n",
      "two-staged genome-wide association study in Japanese individuals.\n",
      "METHODS: We initially tested 222,285 single-nucleotide polymorphisms (SNPs).\n",
      "After the first screening in a panel of 195 PAD cases and 1,358 controls, 2,696\n",
      "SNPs (1.2%) were further genotyped in the second screening using another panel of\n",
      "699 PAD cases and 1540 controls. In both screenings, controls were subjects\n",
      "affected with some diseases other than PAD.\n",
      "RESULTS: When analyzed in the combined panel, the strongest signal of PAD\n",
      "association was observed at rs1902341 in the intron of OSBPL10 (p=4.7E-7 for\n",
      "trend test; OR=1.31, 95% CI 1.18-1.46). Also, PAD was modestly associated at\n",
      "several other loci such as rs2554503 in CSMD1 (p=5.7E-5; OR=1.32, 95% CI\n",
      "1.15-1.51) or rs235243 in VSP13D (p=0.04; OR=1.18, 95% CI 1.01-1.37).\n",
      "CONCLUSION: Our genome-wide exploration identified suggestive evidence of PAD\n",
      "association at the OSBPL10 locus. Because the association has not reached a\n",
      "genome-wide significant level, further replication study is warranted for\n",
      "verification in the Japanese population.\n",
      "\n",
      "\n",
      "PMID: 20610895  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "10. Mol Med. 2010 Jul-Aug;16(7-8):247-53. doi: 10.2119/molmed.2009.00159. Epub 2010\n",
      "Mar 17.\n",
      "\n",
      "Personalized smoking cessation: interactions between nicotine dose, dependence\n",
      "and quit-success genotype score.\n",
      "\n",
      "Rose JE(1), Behm FM, Drgon T, Johnson C, Uhl GR.\n",
      "\n",
      "Author information: \n",
      "(1)Department of Psychiatry and Center for Nicotine and Smoking Cessation\n",
      "Research, Duke University, Durham, North Carolina 27705, USA.\n",
      "rose0003@mc.duke.edu\n",
      "\n",
      "Erratum in\n",
      "    Mol Med. 2012;18(1):729.\n",
      "\n",
      "Improving and targeting nicotine replacement therapy (NRT) are cost-effective\n",
      "strategies for reducing adverse health consequences for smokers. Treatment\n",
      "studies document the efficacy of precessation NRT and support important roles for\n",
      "level of nicotine dependence and precessation smoking reduction in successful\n",
      "quitting. However, prior work has not identified the optimal precessation dose or\n",
      "means for personalizing NRT. Genome-wide association has identified groups of\n",
      "genomic markers associated with successful quitting, allowing us to develop a\n",
      "v1.0 \"quit-success\" genotype score. We now report influences of v1.0 quit-success\n",
      "genotype score, level of dependence and precessation smoking reduction in a\n",
      "smoking cessation trial that examined effects of 21 versus 42 mg/24 h\n",
      "precessation NRT. Four hundred seventy-nine smokers were randomized to 21 or 42\n",
      "mg NRT, initiated 2 wks prior to target quit dates. We monitored self-reported\n",
      "abstinence and end-expired air carbon monoxide (CO). Genotyping used Affymetrix\n",
      "arrays (Santa Clara, CA, USA). The primary outcome was 10-wk continuous smoking\n",
      "abstinence. NRT dose, level of nicotine dependence and genotype scores displayed \n",
      "significant interactive effects on successful quitting. Successful abstinence\n",
      "also was predicted by CO reductions during precessation NRT. These results\n",
      "document ways in which smoking cessation strategies can be personalized based on \n",
      "levels of nicotine dependence, genotype scores and CO monitoring. These\n",
      "assessments, taken together, can help match most smokers with optimal NRT doses\n",
      "and help rapidly identify some who may be better treated using other methods.\n",
      "\n",
      "DOI: 10.2119/molmed.2009.00159 \n",
      "PMCID: PMC2896464\n",
      "PMID: 20379614  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "11. J Am Soc Nephrol. 2009 Feb;20(2):363-79. doi: 10.1681/ASN.2008040406. Epub 2008\n",
      "Dec 3.\n",
      "\n",
      "Large-scale proteomics and phosphoproteomics of urinary exosomes.\n",
      "\n",
      "Gonzales PA(1), Pisitkun T, Hoffert JD, Tchapyjnikov D, Star RA, Kleta R, Wang\n",
      "NS, Knepper MA.\n",
      "\n",
      "Author information: \n",
      "(1)Laboratory of Kidney and Electrolyte Metabolism, National Heart, Lung, and\n",
      "Blood Institute, National Institutes of Health, Bethesda, MD 20892, USA.\n",
      "\n",
      "Normal human urine contains large numbers of exosomes, which are 40- to 100-nm\n",
      "vesicles that originate as the internal vesicles in multivesicular bodies from\n",
      "every renal epithelial cell type facing the urinary space. Here, we used LC-MS/MS\n",
      "to profile the proteome of human urinary exosomes. Overall, the analysis\n",
      "identified 1132 proteins unambiguously, including 177 that are represented on the\n",
      "Online Mendelian Inheritance in Man database of disease-related genes, suggesting\n",
      "that exosome analysis is a potential approach to discover urinary biomarkers. We \n",
      "extended the proteomic analysis to phosphoproteomic profiling using neutral loss \n",
      "scanning, and this yielded multiple novel phosphorylation sites, including\n",
      "serine-811 in the thiazide-sensitive Na-Cl co-transporter, NCC. To demonstrate\n",
      "the potential use of exosome analysis to identify a genetic renal disease, we\n",
      "carried out immunoblotting of exosomes from urine samples of patients with a\n",
      "clinical diagnosis of Bartter syndrome type I, showing an absence of the\n",
      "sodium-potassium-chloride co-transporter 2, NKCC2. The proteomic data are\n",
      "publicly accessible at http://dir.nhlbi.nih.gov/papers/lkem/exosome/.\n",
      "\n",
      "DOI: 10.1681/ASN.2008040406 \n",
      "PMCID: PMC2637050\n",
      "PMID: 19056867  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "12. Nature. 2006 May 18;441(7091):315-21.\n",
      "\n",
      "The DNA sequence and biological annotation of human chromosome 1.\n",
      "\n",
      "Gregory SG(1), Barlow KF, McLay KE, Kaul R, Swarbreck D, Dunham A, Scott CE, Howe\n",
      "KL, Woodfine K, Spencer CC, Jones MC, Gillson C, Searle S, Zhou Y, Kokocinski F, \n",
      "McDonald L, Evans R, Phillips K, Atkinson A, Cooper R, Jones C, Hall RE, Andrews \n",
      "TD, Lloyd C, Ainscough R, Almeida JP, Ambrose KD, Anderson F, Andrew RW, Ashwell \n",
      "RI, Aubin K, Babbage AK, Bagguley CL, Bailey J, Beasley H, Bethel G, Bird CP,\n",
      "Bray-Allen S, Brown JY, Brown AJ, Buckley D, Burton J, Bye J, Carder C, Chapman\n",
      "JC, Clark SY, Clarke G, Clee C, Cobley V, Collier RE, Corby N, Coville GJ, Davies\n",
      "J, Deadman R, Dunn M, Earthrowl M, Ellington AG, Errington H, Frankish A,\n",
      "Frankland J, French L, Garner P, Garnett J, Gay L, Ghori MR, Gibson R, Gilby LM, \n",
      "Gillett W, Glithero RJ, Grafham DV, Griffiths C, Griffiths-Jones S, Grocock R,\n",
      "Hammond S, Harrison ES, Hart E, Haugen E, Heath PD, Holmes S, Holt K, Howden PJ, \n",
      "Hunt AR, Hunt SE, Hunter G, Isherwood J, James R, Johnson C, Johnson D, Joy A,\n",
      "Kay M, Kershaw JK, Kibukawa M, Kimberley AM, King A, Knights AJ, Lad H, Laird G, \n",
      "Lawlor S, Leongamornlert DA, Lloyd DM, Loveland J, Lovell J, Lush MJ, Lyne R,\n",
      "Martin S, Mashreghi-Mohammadi M, Matthews L, Matthews NS, McLaren S, Milne S,\n",
      "Mistry S, Moore MJ, Nickerson T, O'Dell CN, Oliver K, Palmeiri A, Palmer SA,\n",
      "Parker A, Patel D, Pearce AV, Peck AI, Pelan S, Phelps K, Phillimore BJ, Plumb R,\n",
      "Rajan J, Raymond C, Rouse G, Saenphimmachak C, Sehra HK, Sheridan E, Shownkeen R,\n",
      "Sims S, Skuce CD, Smith M, Steward C, Subramanian S, Sycamore N, Tracey A,\n",
      "Tromans A, Van Helmond Z, Wall M, Wallis JM, White S, Whitehead SL, Wilkinson JE,\n",
      "Willey DL, Williams H, Wilming L, Wray PW, Wu Z, Coulson A, Vaudin M, Sulston JE,\n",
      "Durbin R, Hubbard T, Wooster R, Dunham I, Carter NP, McVean G, Ross MT, Harrow J,\n",
      "Olson MV, Beck S, Rogers J, Bentley DR, Banerjee R, Bryant SP, Burford DC,\n",
      "Burrill WD, Clegg SM, Dhami P, Dovey O, Faulkner LM, Gribble SM, Langford CF,\n",
      "Pandian RD, Porter KM, Prigmore E.\n",
      "\n",
      "Author information: \n",
      "(1)The Wellcome Trust Sanger Institute, The Wellcome Trust Genome Campus,\n",
      "Hinxton, Cambridgeshire CB10 1SA, UK. sgregory@chg.duhs.duke.edu\n",
      "\n",
      "Erratum in\n",
      "    Nature. 2006 Oct 26;443(7114):1013. Banerjee, R [added]; Bryant, SP [added];\n",
      "Burford, DC [added]; Burrill, WDH [added]; Clegg, SM [added]; Dhami, P [added];\n",
      "Dovey, O [added]; Faulkner, LM [added]; Gribble, SM [added]; Langford, CF\n",
      "[added]; Pandian, RD [added]; Porter, KM [added]; Prigmore, E [added].\n",
      "\n",
      "The reference sequence for each human chromosome provides the framework for\n",
      "understanding genome function, variation and evolution. Here we report the\n",
      "finished sequence and biological annotation of human chromosome 1. Chromosome 1\n",
      "is gene-dense, with 3,141 genes and 991 pseudogenes, and many coding sequences\n",
      "overlap. Rearrangements and mutations of chromosome 1 are prevalent in cancer and\n",
      "many other diseases. Patterns of sequence variation reveal signals of recent\n",
      "selection in specific genes that may contribute to human fitness, and also in\n",
      "regions where no function is evident. Fine-scale recombination occurs in hotspots\n",
      "of varying intensity along the sequence, and is enriched near genes. These and\n",
      "other studies of human biology and disease encoded within chromosome 1 are made\n",
      "possible with the highly accurate annotated sequence, as part of the completed\n",
      "set of chromosome sequences that comprise the reference human genome.\n",
      "\n",
      "DOI: 10.1038/nature04727 \n",
      "PMID: 16710414  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "13. Genomics. 2004 Sep;84(3):536-49.\n",
      "\n",
      "Analysis of the human VPS13 gene family.\n",
      "\n",
      "Velayos-Baeza A(1), Vettori A, Copley RR, Dobson-Stone C, Monaco AP.\n",
      "\n",
      "Author information: \n",
      "(1)Wellcome Trust Centre for Human Genetics, University of Oxford, Headington,\n",
      "OX3 7BN Oxford, UK.\n",
      "\n",
      "The gene mutated in chorea-acanthocytosis (CHAC; approved gene symbol VPS13A)\n",
      "encodes chorein, a protein similar to yeast Vps13p. We detected several similar\n",
      "putative human proteins by BLAST analysis of chorein. We characterized the\n",
      "structure of three new genes encoding these CHAC-similar proteins, located on\n",
      "chromosomes 1p36, 8q22, and 15q21. The most similar gene in yeast to all four\n",
      "human genes is Vps13, and therefore the human genes were named VPS13A (CHAC,\n",
      "9q21), VPS13B (8q22), VPS13C (15q21), and VPS13D (1p36). VPS13B has recently been\n",
      "reported as COH1, altered in Cohen syndrome. For each gene, we describe several\n",
      "alternative splicing variants; at least two transcripts per gene are major forms.\n",
      "The expression pattern of these genes is ubiquitous, with some tissue-specific\n",
      "differences between several transcript variants. Protein sequence comparisons\n",
      "suggest that intramolecular duplications have played an important role in the\n",
      "evolution of this gene family.\n",
      "\n",
      "DOI: 10.1016/j.ygeno.2004.04.012 \n",
      "PMID: 15498460  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "14. Genome Res. 2004 Oct;14(10B):2121-7.\n",
      "\n",
      "The status, quality, and expansion of the NIH full-length cDNA project: the\n",
      "Mammalian Gene Collection (MGC).\n",
      "\n",
      "Gerhard DS, Wagner L, Feingold EA, Shenmen CM, Grouse LH, Schuler G, Klein SL,\n",
      "Old S, Rasooly R, Good P, Guyer M, Peck AM, Derge JG, Lipman D, Collins FS, Jang \n",
      "W, Sherry S, Feolo M, Misquitta L, Lee E, Rotmistrovsky K, Greenhut SF, Schaefer \n",
      "CF, Buetow K, Bonner TI, Haussler D, Kent J, Kiekhaus M, Furey T, Brent M, Prange\n",
      "C, Schreiber K, Shapiro N, Bhat NK, Hopkins RF, Hsie F, Driscoll T, Soares MB,\n",
      "Casavant TL, Scheetz TE, Brown-stein MJ, Usdin TB, Toshiyuki S, Carninci P, Piao \n",
      "Y, Dudekula DB, Ko MS, Kawakami K, Suzuki Y, Sugano S, Gruber CE, Smith MR,\n",
      "Simmons B, Moore T, Waterman R, Johnson SL, Ruan Y, Wei CL, Mathavan S, Gunaratne\n",
      "PH, Wu J, Garcia AM, Hulyk SW, Fuh E, Yuan Y, Sneed A, Kowis C, Hodgson A, Muzny \n",
      "DM, McPherson J, Gibbs RA, Fahey J, Helton E, Ketteman M, Madan A, Rodrigues S,\n",
      "Sanchez A, Whiting M, Madari A, Young AC, Wetherby KD, Granite SJ, Kwong PN,\n",
      "Brinkley CP, Pearson RL, Bouffard GG, Blakesly RW, Green ED, Dickson MC,\n",
      "Rodriguez AC, Grimwood J, Schmutz J, Myers RM, Butterfield YS, Griffith M,\n",
      "Griffith OL, Krzywinski MI, Liao N, Morin R, Palmquist D, Petrescu AS, Skalska U,\n",
      "Smailus DE, Stott JM, Schnerch A, Schein JE, Jones SJ, Holt RA, Baross A, Marra\n",
      "MA, Clifton S, Makowski KA, Bosak S, Malek J; MGC Project Team.\n",
      "\n",
      "Erratum in\n",
      "    Genome Res. 2006 Jun;16(6):804. Morrin, Ryan [corrected to Morin, Ryan].\n",
      "\n",
      "The National Institutes of Health's Mammalian Gene Collection (MGC) project was\n",
      "designed to generate and sequence a publicly accessible cDNA resource containing \n",
      "a complete open reading frame (ORF) for every human and mouse gene. The project\n",
      "initially used a random strategy to select clones from a large number of cDNA\n",
      "libraries from diverse tissues. Candidate clones were chosen based on 5'-EST\n",
      "sequences, and then fully sequenced to high accuracy and analyzed by algorithms\n",
      "developed for this project. Currently, more than 11,000 human and 10,000 mouse\n",
      "genes are represented in MGC by at least one clone with a full ORF. The random\n",
      "selection approach is now reaching a saturation point, and a transition to\n",
      "protocols targeted at the missing transcripts is now required to complete the\n",
      "mouse and human collections. Comparison of the sequence of the MGC clones to\n",
      "reference genome sequences reveals that most cDNA clones are of very high\n",
      "sequence quality, although it is likely that some cDNAs may carry missense\n",
      "variants as a consequence of experimental artifact, such as PCR, cloning, or\n",
      "reverse transcriptase errors. Recently, a rat cDNA component was added to the\n",
      "project, and ongoing frog (Xenopus) and zebrafish (Danio) cDNA projects were\n",
      "expanded to take advantage of the high-throughput MGC pipeline.\n",
      "\n",
      "DOI: 10.1101/gr.2596504 \n",
      "PMCID: PMC528928\n",
      "PMID: 15489334  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "15. Nat Genet. 2004 Jan;36(1):40-5. Epub 2003 Dec 21.\n",
      "\n",
      "Complete sequencing and characterization of 21,243 full-length human cDNAs.\n",
      "\n",
      "Ota T(1), Suzuki Y, Nishikawa T, Otsuki T, Sugiyama T, Irie R, Wakamatsu A,\n",
      "Hayashi K, Sato H, Nagai K, Kimura K, Makita H, Sekine M, Obayashi M, Nishi T,\n",
      "Shibahara T, Tanaka T, Ishii S, Yamamoto J, Saito K, Kawai Y, Isono Y, Nakamura\n",
      "Y, Nagahari K, Murakami K, Yasuda T, Iwayanagi T, Wagatsuma M, Shiratori A, Sudo \n",
      "H, Hosoiri T, Kaku Y, Kodaira H, Kondo H, Sugawara M, Takahashi M, Kanda K, Yokoi\n",
      "T, Furuya T, Kikkawa E, Omura Y, Abe K, Kamihara K, Katsuta N, Sato K, Tanikawa\n",
      "M, Yamazaki M, Ninomiya K, Ishibashi T, Yamashita H, Murakawa K, Fujimori K,\n",
      "Tanai H, Kimata M, Watanabe M, Hiraoka S, Chiba Y, Ishida S, Ono Y, Takiguchi S, \n",
      "Watanabe S, Yosida M, Hotuta T, Kusano J, Kanehori K, Takahashi-Fujii A, Hara H, \n",
      "Tanase TO, Nomura Y, Togiya S, Komai F, Hara R, Takeuchi K, Arita M, Imose N,\n",
      "Musashino K, Yuuki H, Oshima A, Sasaki N, Aotsuka S, Yoshikawa Y, Matsunawa H,\n",
      "Ichihara T, Shiohata N, Sano S, Moriya S, Momiyama H, Satoh N, Takami S,\n",
      "Terashima Y, Suzuki O, Nakagawa S, Senoh A, Mizoguchi H, Goto Y, Shimizu F,\n",
      "Wakebe H, Hishigaki H, Watanabe T, Sugiyama A, Takemoto M, Kawakami B, Yamazaki\n",
      "M, Watanabe K, Kumagai A, Itakura S, Fukuzumi Y, Fujimori Y, Komiyama M, Tashiro \n",
      "H, Tanigami A, Fujiwara T, Ono T, Yamada K, Fujii Y, Ozaki K, Hirao M, Ohmori Y, \n",
      "Kawabata A, Hikiji T, Kobatake N, Inagaki H, Ikema Y, Okamoto S, Okitani R,\n",
      "Kawakami T, Noguchi S, Itoh T, Shigeta K, Senba T, Matsumura K, Nakajima Y,\n",
      "Mizuno T, Morinaga M, Sasaki M, Togashi T, Oyama M, Hata H, Watanabe M, Komatsu\n",
      "T, Mizushima-Sugano J, Satoh T, Shirai Y, Takahashi Y, Nakagawa K, Okumura K,\n",
      "Nagase T, Nomura N, Kikuchi H, Masuho Y, Yamashita R, Nakai K, Yada T, Nakamura\n",
      "Y, Ohara O, Isogai T, Sugano S.\n",
      "\n",
      "Author information: \n",
      "(1)Helix Research Institute, 1532-3 Yana, Kisarazu, Chiba 292-0812, Japan.\n",
      "\n",
      "As a base for human transcriptome and functional genomics, we created the\n",
      "\"full-length long Japan\" (FLJ) collection of sequenced human cDNAs. We determined\n",
      "the entire sequence of 21,243 selected clones and found that 14,490 cDNAs (10,897\n",
      "clusters) were unique to the FLJ collection. About half of them (5,416) seemed to\n",
      "be protein-coding. Of those, 1,999 clusters had not been predicted by\n",
      "computational methods. The distribution of GC content of nonpredicted cDNAs had a\n",
      "peak at approximately 58% compared with a peak at approximately 42%for predicted \n",
      "cDNAs. Thus, there seems to be a slight bias against GC-rich transcripts in\n",
      "current gene prediction procedures. The rest of the cDNAs unique to the FLJ\n",
      "collection (5,481) contained no obvious open reading frames (ORFs) and thus are\n",
      "candidate noncoding RNAs. About one-fourth of them (1,378) showed a clear pattern\n",
      "of splicing. The distribution of GC content of noncoding cDNAs was narrow and had\n",
      "a peak at approximately 42%, relatively low compared with that of protein-coding \n",
      "cDNAs.\n",
      "\n",
      "DOI: 10.1038/ng1285 \n",
      "PMID: 14702039  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "16. Proc Natl Acad Sci U S A. 2002 Dec 24;99(26):16899-903. Epub 2002 Dec 11.\n",
      "\n",
      "Generation and initial analysis of more than 15,000 full-length human and mouse\n",
      "cDNA sequences.\n",
      "\n",
      "Strausberg RL(1), Feingold EA, Grouse LH, Derge JG, Klausner RD, Collins FS,\n",
      "Wagner L, Shenmen CM, Schuler GD, Altschul SF, Zeeberg B, Buetow KH, Schaefer CF,\n",
      "Bhat NK, Hopkins RF, Jordan H, Moore T, Max SI, Wang J, Hsieh F, Diatchenko L,\n",
      "Marusina K, Farmer AA, Rubin GM, Hong L, Stapleton M, Soares MB, Bonaldo MF,\n",
      "Casavant TL, Scheetz TE, Brownstein MJ, Usdin TB, Toshiyuki S, Carninci P, Prange\n",
      "C, Raha SS, Loquellano NA, Peters GJ, Abramson RD, Mullahy SJ, Bosak SA, McEwan\n",
      "PJ, McKernan KJ, Malek JA, Gunaratne PH, Richards S, Worley KC, Hale S, Garcia\n",
      "AM, Gay LJ, Hulyk SW, Villalon DK, Muzny DM, Sodergren EJ, Lu X, Gibbs RA, Fahey \n",
      "J, Helton E, Ketteman M, Madan A, Rodrigues S, Sanchez A, Whiting M, Madan A,\n",
      "Young AC, Shevchenko Y, Bouffard GG, Blakesley RW, Touchman JW, Green ED, Dickson\n",
      "MC, Rodriguez AC, Grimwood J, Schmutz J, Myers RM, Butterfield YS, Krzywinski MI,\n",
      "Skalska U, Smailus DE, Schnerch A, Schein JE, Jones SJ, Marra MA; Mammalian Gene \n",
      "Collection Program Team.\n",
      "\n",
      "Author information: \n",
      "(1)National Cancer Institute, Bethessda, MD 20892-2580, USA. rls@nih.gov\n",
      "\n",
      "The National Institutes of Health Mammalian Gene Collection (MGC) Program is a\n",
      "multiinstitutional effort to identify and sequence a cDNA clone containing a\n",
      "complete ORF for each human and mouse gene. ESTs were generated from libraries\n",
      "enriched for full-length cDNAs and analyzed to identify candidate full-ORF\n",
      "clones, which then were sequenced to high accuracy. The MGC has currently\n",
      "sequenced and verified the full ORF for a nonredundant set of >9,000 human and\n",
      ">6,000 mouse genes. Candidate full-ORF clones for an additional 7,800 human and\n",
      "3,500 mouse genes also have been identified. All MGC sequences and clones are\n",
      "available without restriction through public databases and clone distribution\n",
      "networks (see http:mgc.nci.nih.gov).\n",
      "\n",
      "DOI: 10.1073/pnas.242603899 \n",
      "PMCID: PMC139241\n",
      "PMID: 12477932  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "17. Genome Res. 2002 Nov;12(11):1773-84.\n",
      "\n",
      "Protein-protein interactions between large proteins: two-hybrid screening using a\n",
      "functionally classified library composed of long cDNAs.\n",
      "\n",
      "Nakayama M(1), Kikuno R, Ohara O.\n",
      "\n",
      "Author information: \n",
      "(1)Department of Human Gene Research, Kazusa DNA Research Institute, Kisarazu,\n",
      "Chiba 292-0818, Japan. nmanabu@kazusa.or.jp\n",
      "\n",
      "Large proteins have multiple domains that are potentially capable of binding many\n",
      "kinds of partners. It is conceivable, therefore, that such proteins could\n",
      "function as an intricate framework of assembly protein complexes. To\n",
      "comprehensively study protein-protein interactions between large KIAA proteins,\n",
      "we have constructed a library composed of 1087 KIAA cDNA clones based on prior\n",
      "functional classifications done in silico. We were guided by two principles that \n",
      "raise the success rate for detecting interactions per tested combination: we\n",
      "avoided testing low-probability combinations, and reduced the number of potential\n",
      "false negatives that arise from the fact that large proteins cannot reliably be\n",
      "expressed in yeast. The latter was addressed by constructing a cDNA library\n",
      "comprised of random fragments encoding large proteins. Cytoplasmic domains of\n",
      "KIAA transmembrane proteins (>1000 amino acids) were used as bait for yeast\n",
      "two-hybrid screening. Our analyses reveal that several KIAA proteins bearing a\n",
      "transmembrane region have the capability of binding to other KIAA proteins\n",
      "containing domains (e.g., PDZ, SH3, rhoGEF, and spectrin) known to be localized\n",
      "to highly specialized submembranous sites, indicating that they participate in\n",
      "cellular junction formation, receptor or channel clustering, and intracellular\n",
      "signaling events. Our representative library should be a very useful resource for\n",
      "detecting previously unidentified interactions because it complements\n",
      "conventional expression libraries, which seldom contain large cDNAs.\n",
      "\n",
      "DOI: 10.1101/gr.406902 \n",
      "PMCID: PMC187542\n",
      "PMID: 12421765  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "18. DNA Res. 1997 Oct 31;4(5):345-9.\n",
      "\n",
      "Characterization of cDNA clones in size-fractionated cDNA libraries from human\n",
      "brain.\n",
      "\n",
      "Seki N(1), Ohira M, Nagase T, Ishikawa K, Miyajima N, Nakajima D, Nomura N, Ohara\n",
      "O.\n",
      "\n",
      "Author information: \n",
      "(1)Kazusa DNA Research Institute, Chiba, Japan. nseki@kazusa.or.jp\n",
      "\n",
      "To evaluate the size-fractionated cDNA libraries of human brain previously\n",
      "constructed (O. O'hara et al. DNA Research, 4, 53-59, 1997), the occurrence of\n",
      "chimeric clones and the content of clones with coding potentiality were analyzed \n",
      "using the randomly sampled clones with insert sizes of 5 to 7 kb. When the\n",
      "chromosomal location of 30 clones was determined by the radiation-hybrid mapping \n",
      "method, the map positions assigned from the 3'- and 5'-end sequences separately\n",
      "were coincident for 29 clones, suggesting that the occurrence of chimeric clones \n",
      "is at most 1/30. Using 91 clones mapped to chromosome 1, the content of clones\n",
      "that have the potentiality coding for proteins larger than 100 amino acid\n",
      "residues was estimated to be approximately 50% (46 out of 91 clones) on the basis\n",
      "of nucleotide sequence analysis and coding potentiality assay in vitro. No\n",
      "significant open reading frames were detected in the remaining clones. Although\n",
      "the clones coding for short peptides may not have been included in the above\n",
      "estimation, the libraries constructed from the whole brain mRNA fraction appear\n",
      "to contain a considerable amount of clones corresponding to the 5'-truncated\n",
      "transcripts in an unprocessed form and/or those with long 3'-untranslated\n",
      "regions.\n",
      "\n",
      "\n",
      "PMID: 9455484  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "19. Genome Res. 1997 Apr;7(4):353-8.\n",
      "\n",
      "Large-scale concatenation cDNA sequencing.\n",
      "\n",
      "Yu W, Andersson B, Worley KC, Muzny DM, Ding Y, Liu W, Ricafrente JY, Wentland\n",
      "MA, Lennon G, Gibbs RA.\n",
      "\n",
      "A total of 100 kb of DNA derived from 69 individual human brain cDNA clones of\n",
      "0.7-2.0 kb were sequenced by concatenated cDNA sequencing (CCS), whereby multiple\n",
      "individual DNA fragments are sequenced simultaneously in a single shotgun\n",
      "library. The method yielded accurate sequences and a similar efficiency compared \n",
      "with other shotgun libraries constructed from single DNA fragments (> 20 kb).\n",
      "Computer analyses were carried out on 65 cDNA clone sequences and their\n",
      "corresponding end sequences to examine both nucleic acid and amino acid sequence \n",
      "similarities in the databases. Thirty-seven clones revealed no DNA database\n",
      "matches, 12 clones generated exact matches (> or = 98% identity), and 16 clones\n",
      "generated nonexact matches (57%-97% identity) to either known human or other\n",
      "species genes. Of those 28 matched clones, 8 had corresponding end sequences that\n",
      "failed to identify similarities. In a protein similarity search, 27 clone\n",
      "sequences displayed significant matches, whereas only 20 of the end sequences had\n",
      "matches to known protein sequences. Our data indicate that full-length cDNA\n",
      "insert sequences provide significantly more nucleic acid and protein sequence\n",
      "similarity matches than expressed sequence tags (ESTs) for database searching.\n",
      "\n",
      "\n",
      "PMCID: PMC139146\n",
      "PMID: 9110174  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "20. Anal Biochem. 1996 Apr 5;236(1):107-13.\n",
      "\n",
      "A \"double adaptor\" method for improved shotgun library construction.\n",
      "\n",
      "Andersson B(1), Wentland MA, Ricafrente JY, Liu W, Gibbs RA.\n",
      "\n",
      "Author information: \n",
      "(1)Department of Molecular and Human Genetics, Baylor College of Medicine, One\n",
      "Baylor Plaza, Houston, Texas, 77030, USA.\n",
      "\n",
      "The efficiency of shotgun DNA sequencing depends to a great extent on the quality\n",
      "of the random-subclone libraries used. We here describe a novel \"double adaptor\" \n",
      "strategy for efficient construction of high-quality shotgun libraries. In this\n",
      "method, randomly sheared and end-repaired fragments are ligated to\n",
      "oligonucleotide adaptors creating 12-base overhangs. Nonphosphorylated\n",
      "oligonucleotides are used, which prevents formation of adaptor dimers and ensures\n",
      "efficient ligation of insert to adaptor. The vector is prepared from a modified\n",
      "M13 vector, by KpnI/PstI digestion followed by ligation to oligonucleotides with \n",
      "ends complementary to the overhangs created in the digest. These adaptors create \n",
      "5'-overhangs complementary to those on the inserts. Following annealing of insert\n",
      "to vector, the DNA is directly used for transformation without a ligation step.\n",
      "This protocol is robust and shows three- to fivefold higher yield of clones\n",
      "compared to previous protocols. No chimeric clones can be detected and the\n",
      "background of clones without an insert is <1%. The procedure is rapid and shows\n",
      "potential for automation.\n",
      "\n",
      "DOI: 10.1006/abio.1996.0138 \n",
      "PMID: 8619474  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "21. Nature. 2015 Sep 17;525(7569):339-44. doi: 10.1038/nature14877. Epub 2015 Sep 7.\n",
      "\n",
      "Panorama of ancient metazoan macromolecular complexes.\n",
      "\n",
      "Wan C(1)(2), Borgeson B(2), Phanse S(1), Tu F(2), Drew K(2), Clark G(3), Xiong\n",
      "X(4)(5), Kagan O(1), Kwan J(1)(4), Bezginov A(3), Chessman K(4)(5), Pal S(5),\n",
      "Cromar G(4)(5), Papoulas O(2), Ni Z(1), Boutz DR(2), Stoilova S(1), Havugimana\n",
      "PC(1), Guo X(1), Malty RH(6), Sarov M(7), Greenblatt J(1)(4), Babu M(6), Derry\n",
      "WB(4)(5), Tillier ER(3), Wallingford JB(2)(8), Parkinson J(4)(5), Marcotte\n",
      "EM(2)(8), Emili A(1)(4).\n",
      "\n",
      "Author information: \n",
      "(1)Donnelly Centre for Cellular and Biomolecular Research, University of Toronto,\n",
      "Toronto, Ontario M5S 3E1, Canada.\n",
      "(2)Center for Systems and Synthetic Biology, Institute for Cellular and Molecular\n",
      "Biology, University of Texas at Austin, Austin, Texas 78712, USA.\n",
      "(3)Department of Medical Biophysics, Toronto, Ontario M5G 1L7, Canada.\n",
      "(4)Department of Molecular Genetics, University of Toronto, Toronto, Ontario M5S \n",
      "1A8, Canada.\n",
      "(5)Hospital for Sick Children, Toronto, Ontario M5G 1X8, Canada.\n",
      "(6)Department of Biochemistry, University of Regina, Regina, Saskatchewan S4S\n",
      "0A2, Canada.\n",
      "(7)Max Planck Institute of Molecular Cell Biology and Genetics, 01307 Dresden,\n",
      "Germany.\n",
      "(8)Department of Molecular Biosciences, University of Texas at Austin, Austin,\n",
      "Texas 78712, USA.\n",
      "\n",
      "Comment in\n",
      "    Nat Methods. 2015 Nov;12(11):1011.\n",
      "    Trends Biochem Sci. 2016 Jan;41(1):4-6.\n",
      "\n",
      "Macromolecular complexes are essential to conserved biological processes, but\n",
      "their prevalence across animals is unclear. By combining extensive biochemical\n",
      "fractionation with quantitative mass spectrometry, here we directly examined the \n",
      "composition of soluble multiprotein complexes among diverse metazoan models.\n",
      "Using an integrative approach, we generated a draft conservation map consisting\n",
      "of more than one million putative high-confidence co-complex interactions for\n",
      "species with fully sequenced genomes that encompasses functional modules present \n",
      "broadly across all extant animals. Clustering reveals a spectrum of conservation,\n",
      "ranging from ancient eukaryotic assemblies that have probably served cellular\n",
      "housekeeping roles for at least one billion years, ancestral complexes that have \n",
      "accrued contemporary components, and rarer metazoan innovations linked to\n",
      "multicellularity. We validated these projections by independent co-fractionation \n",
      "experiments in evolutionarily distant species, affinity purification and\n",
      "functional analyses. The comprehensiveness, centrality and modularity of these\n",
      "reconstructed interactomes reflect their fundamental mechanistic importance and\n",
      "adaptive value to animal cell systems.\n",
      "\n",
      "DOI: 10.1038/nature14877 \n",
      "PMCID: PMC5036527\n",
      "PMID: 26344197  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "22. J Innate Immun. 2015;7(5):545-53. doi: 10.1159/000381265. Epub 2015 Apr 17.\n",
      "\n",
      "VPS13D Gene Variant Is Associated with Altered IL-6 Production and Mortality in\n",
      "Septic Shock.\n",
      "\n",
      "Nakada TA(1), Boyd JH, Russell JA, Aguirre-Hernndez R, Wilkinson MD, Thair SA,\n",
      "Nakada E, McConechy MK, Fjell CD, Walley KR.\n",
      "\n",
      "Author information: \n",
      "(1)Critical Care Research Laboratories, Centre for Heart Lung Innovation, St.\n",
      "Paul's Hospital, University of British Columbia, Vancouver, B.C., Canada.\n",
      "\n",
      "BACKGROUND: Genetic variations contribute to septic shock mortality. To discover \n",
      "a novel locus, we performed in vitro genome-wide association studies (GWAS) and\n",
      "further tested the result in a cohort of septic shock patients.\n",
      "METHODS: Two in vitro GWAS using a quantitative trait locus analysis of\n",
      "stimulated IL-6 production in lymphoblastoid cells from 60 individuals of\n",
      "European ancestry were performed. VPS13D rs6685273 was genotyped in European\n",
      "ancestry patients (n = 498). The VPS13D gene was silenced in vitro.\n",
      "RESULTS: Two GWAS using lymphoblastoid cells identified the locus of VPS13D\n",
      "rs6685273 that was significant in the same direction in both GWAS. The VPS13D\n",
      "rs6685273 C allele was associated with increased IL-6 production. Patients with\n",
      "septic shock who had the VPS13D rs6685273 CC genotype had an increased 28-day\n",
      "mortality (p = 0.023) and more organ failure (p < 0.05) compared to the CT/TT\n",
      "genotypes. VPS13D in vitro gene silencing in the HeLa cell line increased IL-6\n",
      "production. Furthermore, the rs6685273 genotype was associated with differential \n",
      "VPS13D splice variant expression.\n",
      "CONCLUSIONS: The VPS13D rs6685273 C allele was associated with increased IL-6\n",
      "production in vitro. The patients with the VPS13D rs6685273 CC genotype had\n",
      "increased 28-day mortality and increased organ failure. VPS13D appears to\n",
      "regulate IL-6 production.\n",
      "\n",
      " 2015 S. Karger AG, Basel.\n",
      "\n",
      "DOI: 10.1159/000381265 \n",
      "PMID: 25896417  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "23. Proteomics. 2011 Jan;11(1):172-9. doi: 10.1002/pmic.201000217. Epub 2010 Dec 1.\n",
      "\n",
      "Identification of proteins associated with ligand-activated estrogen receptor \n",
      "in human breast cancer cell nuclei by tandem affinity purification and nano\n",
      "LC-MS/MS.\n",
      "\n",
      "Tarallo R(1), Bamundo A, Nassa G, Nola E, Paris O, Ambrosino C, Facchiano A,\n",
      "Baumann M, Nyman TA, Weisz A.\n",
      "\n",
      "Author information: \n",
      "(1)Department of General Pathology, Second University of Naples, Napoli, Italy.\n",
      "\n",
      "Estrogen receptor  (ER-) is a key mediator of estrogen actions in breast cancer\n",
      "(BC) cells. Understanding the effects of ligand-activated ER- in target cells\n",
      "requires identification of the molecular partners acting in concert with this\n",
      "nuclear receptor to transduce the hormonal signal. We applied tandem affinity\n",
      "purification (TAP), glycerol gradient centrifugation and MS analysis to isolate\n",
      "and identify proteins interacting with ligand-activated ER- in MCF-7 cell\n",
      "nuclei. This led to the identification of 264 ER-associated proteins, whose\n",
      "functions highlight the hinge role of ER- in the coordination of multiple\n",
      "hormone-regulated nuclear processes in BC cells.\n",
      "\n",
      "Copyright  2011 WILEY-VCH Verlag GmbH & Co. KGaA, Weinheim.\n",
      "\n",
      "DOI: 10.1002/pmic.201000217 \n",
      "PMID: 21182205  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "24. J Atheroscler Thromb. 2010 Oct 27;17(10):1054-62. Epub 2010 Jul 2.\n",
      "\n",
      "Identification of evidence suggestive of an association with peripheral arterial \n",
      "disease at the OSBPL10 locus by genome-wide investigation in the Japanese\n",
      "population.\n",
      "\n",
      "Koriyama H(1), Nakagami H, Katsuya T, Sugimoto K, Yamashita H, Takami Y, Maeda S,\n",
      "Kubo M, Takahashi A, Nakamura Y, Ogihara T, Rakugi H, Kaneda Y, Morishita R.\n",
      "\n",
      "Author information: \n",
      "(1)Department of Geriatric Medicine and Nephrology, Osaka University Graduate\n",
      "School of Medicine, 2-2 Yamada-oka, Suita, Japan.\n",
      "\n",
      "AIM: Peripheral arterial disease (PAD) is a common cause of cardiovascular\n",
      "morbidity and an independent predictor of cardiovascular mortality. However,\n",
      "little is known about the genetic basis of PAD. To elucidate this, we performed a\n",
      "two-staged genome-wide association study in Japanese individuals.\n",
      "METHODS: We initially tested 222,285 single-nucleotide polymorphisms (SNPs).\n",
      "After the first screening in a panel of 195 PAD cases and 1,358 controls, 2,696\n",
      "SNPs (1.2%) were further genotyped in the second screening using another panel of\n",
      "699 PAD cases and 1540 controls. In both screenings, controls were subjects\n",
      "affected with some diseases other than PAD.\n",
      "RESULTS: When analyzed in the combined panel, the strongest signal of PAD\n",
      "association was observed at rs1902341 in the intron of OSBPL10 (p=4.7E-7 for\n",
      "trend test; OR=1.31, 95% CI 1.18-1.46). Also, PAD was modestly associated at\n",
      "several other loci such as rs2554503 in CSMD1 (p=5.7E-5; OR=1.32, 95% CI\n",
      "1.15-1.51) or rs235243 in VSP13D (p=0.04; OR=1.18, 95% CI 1.01-1.37).\n",
      "CONCLUSION: Our genome-wide exploration identified suggestive evidence of PAD\n",
      "association at the OSBPL10 locus. Because the association has not reached a\n",
      "genome-wide significant level, further replication study is warranted for\n",
      "verification in the Japanese population.\n",
      "\n",
      "\n",
      "PMID: 20610895  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "25. Mol Med. 2010 Jul-Aug;16(7-8):247-53. doi: 10.2119/molmed.2009.00159. Epub 2010\n",
      "Mar 17.\n",
      "\n",
      "Personalized smoking cessation: interactions between nicotine dose, dependence\n",
      "and quit-success genotype score.\n",
      "\n",
      "Rose JE(1), Behm FM, Drgon T, Johnson C, Uhl GR.\n",
      "\n",
      "Author information: \n",
      "(1)Department of Psychiatry and Center for Nicotine and Smoking Cessation\n",
      "Research, Duke University, Durham, North Carolina 27705, USA.\n",
      "rose0003@mc.duke.edu\n",
      "\n",
      "Erratum in\n",
      "    Mol Med. 2012;18(1):729.\n",
      "\n",
      "Improving and targeting nicotine replacement therapy (NRT) are cost-effective\n",
      "strategies for reducing adverse health consequences for smokers. Treatment\n",
      "studies document the efficacy of precessation NRT and support important roles for\n",
      "level of nicotine dependence and precessation smoking reduction in successful\n",
      "quitting. However, prior work has not identified the optimal precessation dose or\n",
      "means for personalizing NRT. Genome-wide association has identified groups of\n",
      "genomic markers associated with successful quitting, allowing us to develop a\n",
      "v1.0 \"quit-success\" genotype score. We now report influences of v1.0 quit-success\n",
      "genotype score, level of dependence and precessation smoking reduction in a\n",
      "smoking cessation trial that examined effects of 21 versus 42 mg/24 h\n",
      "precessation NRT. Four hundred seventy-nine smokers were randomized to 21 or 42\n",
      "mg NRT, initiated 2 wks prior to target quit dates. We monitored self-reported\n",
      "abstinence and end-expired air carbon monoxide (CO). Genotyping used Affymetrix\n",
      "arrays (Santa Clara, CA, USA). The primary outcome was 10-wk continuous smoking\n",
      "abstinence. NRT dose, level of nicotine dependence and genotype scores displayed \n",
      "significant interactive effects on successful quitting. Successful abstinence\n",
      "also was predicted by CO reductions during precessation NRT. These results\n",
      "document ways in which smoking cessation strategies can be personalized based on \n",
      "levels of nicotine dependence, genotype scores and CO monitoring. These\n",
      "assessments, taken together, can help match most smokers with optimal NRT doses\n",
      "and help rapidly identify some who may be better treated using other methods.\n",
      "\n",
      "DOI: 10.2119/molmed.2009.00159 \n",
      "PMCID: PMC2896464\n",
      "PMID: 20379614  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "26. J Am Soc Nephrol. 2009 Feb;20(2):363-79. doi: 10.1681/ASN.2008040406. Epub 2008\n",
      "Dec 3.\n",
      "\n",
      "Large-scale proteomics and phosphoproteomics of urinary exosomes.\n",
      "\n",
      "Gonzales PA(1), Pisitkun T, Hoffert JD, Tchapyjnikov D, Star RA, Kleta R, Wang\n",
      "NS, Knepper MA.\n",
      "\n",
      "Author information: \n",
      "(1)Laboratory of Kidney and Electrolyte Metabolism, National Heart, Lung, and\n",
      "Blood Institute, National Institutes of Health, Bethesda, MD 20892, USA.\n",
      "\n",
      "Normal human urine contains large numbers of exosomes, which are 40- to 100-nm\n",
      "vesicles that originate as the internal vesicles in multivesicular bodies from\n",
      "every renal epithelial cell type facing the urinary space. Here, we used LC-MS/MS\n",
      "to profile the proteome of human urinary exosomes. Overall, the analysis\n",
      "identified 1132 proteins unambiguously, including 177 that are represented on the\n",
      "Online Mendelian Inheritance in Man database of disease-related genes, suggesting\n",
      "that exosome analysis is a potential approach to discover urinary biomarkers. We \n",
      "extended the proteomic analysis to phosphoproteomic profiling using neutral loss \n",
      "scanning, and this yielded multiple novel phosphorylation sites, including\n",
      "serine-811 in the thiazide-sensitive Na-Cl co-transporter, NCC. To demonstrate\n",
      "the potential use of exosome analysis to identify a genetic renal disease, we\n",
      "carried out immunoblotting of exosomes from urine samples of patients with a\n",
      "clinical diagnosis of Bartter syndrome type I, showing an absence of the\n",
      "sodium-potassium-chloride co-transporter 2, NKCC2. The proteomic data are\n",
      "publicly accessible at http://dir.nhlbi.nih.gov/papers/lkem/exosome/.\n",
      "\n",
      "DOI: 10.1681/ASN.2008040406 \n",
      "PMCID: PMC2637050\n",
      "PMID: 19056867  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "27. Nature. 2006 May 18;441(7091):315-21.\n",
      "\n",
      "The DNA sequence and biological annotation of human chromosome 1.\n",
      "\n",
      "Gregory SG(1), Barlow KF, McLay KE, Kaul R, Swarbreck D, Dunham A, Scott CE, Howe\n",
      "KL, Woodfine K, Spencer CC, Jones MC, Gillson C, Searle S, Zhou Y, Kokocinski F, \n",
      "McDonald L, Evans R, Phillips K, Atkinson A, Cooper R, Jones C, Hall RE, Andrews \n",
      "TD, Lloyd C, Ainscough R, Almeida JP, Ambrose KD, Anderson F, Andrew RW, Ashwell \n",
      "RI, Aubin K, Babbage AK, Bagguley CL, Bailey J, Beasley H, Bethel G, Bird CP,\n",
      "Bray-Allen S, Brown JY, Brown AJ, Buckley D, Burton J, Bye J, Carder C, Chapman\n",
      "JC, Clark SY, Clarke G, Clee C, Cobley V, Collier RE, Corby N, Coville GJ, Davies\n",
      "J, Deadman R, Dunn M, Earthrowl M, Ellington AG, Errington H, Frankish A,\n",
      "Frankland J, French L, Garner P, Garnett J, Gay L, Ghori MR, Gibson R, Gilby LM, \n",
      "Gillett W, Glithero RJ, Grafham DV, Griffiths C, Griffiths-Jones S, Grocock R,\n",
      "Hammond S, Harrison ES, Hart E, Haugen E, Heath PD, Holmes S, Holt K, Howden PJ, \n",
      "Hunt AR, Hunt SE, Hunter G, Isherwood J, James R, Johnson C, Johnson D, Joy A,\n",
      "Kay M, Kershaw JK, Kibukawa M, Kimberley AM, King A, Knights AJ, Lad H, Laird G, \n",
      "Lawlor S, Leongamornlert DA, Lloyd DM, Loveland J, Lovell J, Lush MJ, Lyne R,\n",
      "Martin S, Mashreghi-Mohammadi M, Matthews L, Matthews NS, McLaren S, Milne S,\n",
      "Mistry S, Moore MJ, Nickerson T, O'Dell CN, Oliver K, Palmeiri A, Palmer SA,\n",
      "Parker A, Patel D, Pearce AV, Peck AI, Pelan S, Phelps K, Phillimore BJ, Plumb R,\n",
      "Rajan J, Raymond C, Rouse G, Saenphimmachak C, Sehra HK, Sheridan E, Shownkeen R,\n",
      "Sims S, Skuce CD, Smith M, Steward C, Subramanian S, Sycamore N, Tracey A,\n",
      "Tromans A, Van Helmond Z, Wall M, Wallis JM, White S, Whitehead SL, Wilkinson JE,\n",
      "Willey DL, Williams H, Wilming L, Wray PW, Wu Z, Coulson A, Vaudin M, Sulston JE,\n",
      "Durbin R, Hubbard T, Wooster R, Dunham I, Carter NP, McVean G, Ross MT, Harrow J,\n",
      "Olson MV, Beck S, Rogers J, Bentley DR, Banerjee R, Bryant SP, Burford DC,\n",
      "Burrill WD, Clegg SM, Dhami P, Dovey O, Faulkner LM, Gribble SM, Langford CF,\n",
      "Pandian RD, Porter KM, Prigmore E.\n",
      "\n",
      "Author information: \n",
      "(1)The Wellcome Trust Sanger Institute, The Wellcome Trust Genome Campus,\n",
      "Hinxton, Cambridgeshire CB10 1SA, UK. sgregory@chg.duhs.duke.edu\n",
      "\n",
      "Erratum in\n",
      "    Nature. 2006 Oct 26;443(7114):1013. Banerjee, R [added]; Bryant, SP [added];\n",
      "Burford, DC [added]; Burrill, WDH [added]; Clegg, SM [added]; Dhami, P [added];\n",
      "Dovey, O [added]; Faulkner, LM [added]; Gribble, SM [added]; Langford, CF\n",
      "[added]; Pandian, RD [added]; Porter, KM [added]; Prigmore, E [added].\n",
      "\n",
      "The reference sequence for each human chromosome provides the framework for\n",
      "understanding genome function, variation and evolution. Here we report the\n",
      "finished sequence and biological annotation of human chromosome 1. Chromosome 1\n",
      "is gene-dense, with 3,141 genes and 991 pseudogenes, and many coding sequences\n",
      "overlap. Rearrangements and mutations of chromosome 1 are prevalent in cancer and\n",
      "many other diseases. Patterns of sequence variation reveal signals of recent\n",
      "selection in specific genes that may contribute to human fitness, and also in\n",
      "regions where no function is evident. Fine-scale recombination occurs in hotspots\n",
      "of varying intensity along the sequence, and is enriched near genes. These and\n",
      "other studies of human biology and disease encoded within chromosome 1 are made\n",
      "possible with the highly accurate annotated sequence, as part of the completed\n",
      "set of chromosome sequences that comprise the reference human genome.\n",
      "\n",
      "DOI: 10.1038/nature04727 \n",
      "PMID: 16710414  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "28. Genomics. 2004 Sep;84(3):536-49.\n",
      "\n",
      "Analysis of the human VPS13 gene family.\n",
      "\n",
      "Velayos-Baeza A(1), Vettori A, Copley RR, Dobson-Stone C, Monaco AP.\n",
      "\n",
      "Author information: \n",
      "(1)Wellcome Trust Centre for Human Genetics, University of Oxford, Headington,\n",
      "OX3 7BN Oxford, UK.\n",
      "\n",
      "The gene mutated in chorea-acanthocytosis (CHAC; approved gene symbol VPS13A)\n",
      "encodes chorein, a protein similar to yeast Vps13p. We detected several similar\n",
      "putative human proteins by BLAST analysis of chorein. We characterized the\n",
      "structure of three new genes encoding these CHAC-similar proteins, located on\n",
      "chromosomes 1p36, 8q22, and 15q21. The most similar gene in yeast to all four\n",
      "human genes is Vps13, and therefore the human genes were named VPS13A (CHAC,\n",
      "9q21), VPS13B (8q22), VPS13C (15q21), and VPS13D (1p36). VPS13B has recently been\n",
      "reported as COH1, altered in Cohen syndrome. For each gene, we describe several\n",
      "alternative splicing variants; at least two transcripts per gene are major forms.\n",
      "The expression pattern of these genes is ubiquitous, with some tissue-specific\n",
      "differences between several transcript variants. Protein sequence comparisons\n",
      "suggest that intramolecular duplications have played an important role in the\n",
      "evolution of this gene family.\n",
      "\n",
      "DOI: 10.1016/j.ygeno.2004.04.012 \n",
      "PMID: 15498460  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "29. Genome Res. 2004 Oct;14(10B):2121-7.\n",
      "\n",
      "The status, quality, and expansion of the NIH full-length cDNA project: the\n",
      "Mammalian Gene Collection (MGC).\n",
      "\n",
      "Gerhard DS, Wagner L, Feingold EA, Shenmen CM, Grouse LH, Schuler G, Klein SL,\n",
      "Old S, Rasooly R, Good P, Guyer M, Peck AM, Derge JG, Lipman D, Collins FS, Jang \n",
      "W, Sherry S, Feolo M, Misquitta L, Lee E, Rotmistrovsky K, Greenhut SF, Schaefer \n",
      "CF, Buetow K, Bonner TI, Haussler D, Kent J, Kiekhaus M, Furey T, Brent M, Prange\n",
      "C, Schreiber K, Shapiro N, Bhat NK, Hopkins RF, Hsie F, Driscoll T, Soares MB,\n",
      "Casavant TL, Scheetz TE, Brown-stein MJ, Usdin TB, Toshiyuki S, Carninci P, Piao \n",
      "Y, Dudekula DB, Ko MS, Kawakami K, Suzuki Y, Sugano S, Gruber CE, Smith MR,\n",
      "Simmons B, Moore T, Waterman R, Johnson SL, Ruan Y, Wei CL, Mathavan S, Gunaratne\n",
      "PH, Wu J, Garcia AM, Hulyk SW, Fuh E, Yuan Y, Sneed A, Kowis C, Hodgson A, Muzny \n",
      "DM, McPherson J, Gibbs RA, Fahey J, Helton E, Ketteman M, Madan A, Rodrigues S,\n",
      "Sanchez A, Whiting M, Madari A, Young AC, Wetherby KD, Granite SJ, Kwong PN,\n",
      "Brinkley CP, Pearson RL, Bouffard GG, Blakesly RW, Green ED, Dickson MC,\n",
      "Rodriguez AC, Grimwood J, Schmutz J, Myers RM, Butterfield YS, Griffith M,\n",
      "Griffith OL, Krzywinski MI, Liao N, Morin R, Palmquist D, Petrescu AS, Skalska U,\n",
      "Smailus DE, Stott JM, Schnerch A, Schein JE, Jones SJ, Holt RA, Baross A, Marra\n",
      "MA, Clifton S, Makowski KA, Bosak S, Malek J; MGC Project Team.\n",
      "\n",
      "Erratum in\n",
      "    Genome Res. 2006 Jun;16(6):804. Morrin, Ryan [corrected to Morin, Ryan].\n",
      "\n",
      "The National Institutes of Health's Mammalian Gene Collection (MGC) project was\n",
      "designed to generate and sequence a publicly accessible cDNA resource containing \n",
      "a complete open reading frame (ORF) for every human and mouse gene. The project\n",
      "initially used a random strategy to select clones from a large number of cDNA\n",
      "libraries from diverse tissues. Candidate clones were chosen based on 5'-EST\n",
      "sequences, and then fully sequenced to high accuracy and analyzed by algorithms\n",
      "developed for this project. Currently, more than 11,000 human and 10,000 mouse\n",
      "genes are represented in MGC by at least one clone with a full ORF. The random\n",
      "selection approach is now reaching a saturation point, and a transition to\n",
      "protocols targeted at the missing transcripts is now required to complete the\n",
      "mouse and human collections. Comparison of the sequence of the MGC clones to\n",
      "reference genome sequences reveals that most cDNA clones are of very high\n",
      "sequence quality, although it is likely that some cDNAs may carry missense\n",
      "variants as a consequence of experimental artifact, such as PCR, cloning, or\n",
      "reverse transcriptase errors. Recently, a rat cDNA component was added to the\n",
      "project, and ongoing frog (Xenopus) and zebrafish (Danio) cDNA projects were\n",
      "expanded to take advantage of the high-throughput MGC pipeline.\n",
      "\n",
      "DOI: 10.1101/gr.2596504 \n",
      "PMCID: PMC528928\n",
      "PMID: 15489334  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "30. Nat Genet. 2004 Jan;36(1):40-5. Epub 2003 Dec 21.\n",
      "\n",
      "Complete sequencing and characterization of 21,243 full-length human cDNAs.\n",
      "\n",
      "Ota T(1), Suzuki Y, Nishikawa T, Otsuki T, Sugiyama T, Irie R, Wakamatsu A,\n",
      "Hayashi K, Sato H, Nagai K, Kimura K, Makita H, Sekine M, Obayashi M, Nishi T,\n",
      "Shibahara T, Tanaka T, Ishii S, Yamamoto J, Saito K, Kawai Y, Isono Y, Nakamura\n",
      "Y, Nagahari K, Murakami K, Yasuda T, Iwayanagi T, Wagatsuma M, Shiratori A, Sudo \n",
      "H, Hosoiri T, Kaku Y, Kodaira H, Kondo H, Sugawara M, Takahashi M, Kanda K, Yokoi\n",
      "T, Furuya T, Kikkawa E, Omura Y, Abe K, Kamihara K, Katsuta N, Sato K, Tanikawa\n",
      "M, Yamazaki M, Ninomiya K, Ishibashi T, Yamashita H, Murakawa K, Fujimori K,\n",
      "Tanai H, Kimata M, Watanabe M, Hiraoka S, Chiba Y, Ishida S, Ono Y, Takiguchi S, \n",
      "Watanabe S, Yosida M, Hotuta T, Kusano J, Kanehori K, Takahashi-Fujii A, Hara H, \n",
      "Tanase TO, Nomura Y, Togiya S, Komai F, Hara R, Takeuchi K, Arita M, Imose N,\n",
      "Musashino K, Yuuki H, Oshima A, Sasaki N, Aotsuka S, Yoshikawa Y, Matsunawa H,\n",
      "Ichihara T, Shiohata N, Sano S, Moriya S, Momiyama H, Satoh N, Takami S,\n",
      "Terashima Y, Suzuki O, Nakagawa S, Senoh A, Mizoguchi H, Goto Y, Shimizu F,\n",
      "Wakebe H, Hishigaki H, Watanabe T, Sugiyama A, Takemoto M, Kawakami B, Yamazaki\n",
      "M, Watanabe K, Kumagai A, Itakura S, Fukuzumi Y, Fujimori Y, Komiyama M, Tashiro \n",
      "H, Tanigami A, Fujiwara T, Ono T, Yamada K, Fujii Y, Ozaki K, Hirao M, Ohmori Y, \n",
      "Kawabata A, Hikiji T, Kobatake N, Inagaki H, Ikema Y, Okamoto S, Okitani R,\n",
      "Kawakami T, Noguchi S, Itoh T, Shigeta K, Senba T, Matsumura K, Nakajima Y,\n",
      "Mizuno T, Morinaga M, Sasaki M, Togashi T, Oyama M, Hata H, Watanabe M, Komatsu\n",
      "T, Mizushima-Sugano J, Satoh T, Shirai Y, Takahashi Y, Nakagawa K, Okumura K,\n",
      "Nagase T, Nomura N, Kikuchi H, Masuho Y, Yamashita R, Nakai K, Yada T, Nakamura\n",
      "Y, Ohara O, Isogai T, Sugano S.\n",
      "\n",
      "Author information: \n",
      "(1)Helix Research Institute, 1532-3 Yana, Kisarazu, Chiba 292-0812, Japan.\n",
      "\n",
      "As a base for human transcriptome and functional genomics, we created the\n",
      "\"full-length long Japan\" (FLJ) collection of sequenced human cDNAs. We determined\n",
      "the entire sequence of 21,243 selected clones and found that 14,490 cDNAs (10,897\n",
      "clusters) were unique to the FLJ collection. About half of them (5,416) seemed to\n",
      "be protein-coding. Of those, 1,999 clusters had not been predicted by\n",
      "computational methods. The distribution of GC content of nonpredicted cDNAs had a\n",
      "peak at approximately 58% compared with a peak at approximately 42%for predicted \n",
      "cDNAs. Thus, there seems to be a slight bias against GC-rich transcripts in\n",
      "current gene prediction procedures. The rest of the cDNAs unique to the FLJ\n",
      "collection (5,481) contained no obvious open reading frames (ORFs) and thus are\n",
      "candidate noncoding RNAs. About one-fourth of them (1,378) showed a clear pattern\n",
      "of splicing. The distribution of GC content of noncoding cDNAs was narrow and had\n",
      "a peak at approximately 42%, relatively low compared with that of protein-coding \n",
      "cDNAs.\n",
      "\n",
      "DOI: 10.1038/ng1285 \n",
      "PMID: 14702039  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "31. Proc Natl Acad Sci U S A. 2002 Dec 24;99(26):16899-903. Epub 2002 Dec 11.\n",
      "\n",
      "Generation and initial analysis of more than 15,000 full-length human and mouse\n",
      "cDNA sequences.\n",
      "\n",
      "Strausberg RL(1), Feingold EA, Grouse LH, Derge JG, Klausner RD, Collins FS,\n",
      "Wagner L, Shenmen CM, Schuler GD, Altschul SF, Zeeberg B, Buetow KH, Schaefer CF,\n",
      "Bhat NK, Hopkins RF, Jordan H, Moore T, Max SI, Wang J, Hsieh F, Diatchenko L,\n",
      "Marusina K, Farmer AA, Rubin GM, Hong L, Stapleton M, Soares MB, Bonaldo MF,\n",
      "Casavant TL, Scheetz TE, Brownstein MJ, Usdin TB, Toshiyuki S, Carninci P, Prange\n",
      "C, Raha SS, Loquellano NA, Peters GJ, Abramson RD, Mullahy SJ, Bosak SA, McEwan\n",
      "PJ, McKernan KJ, Malek JA, Gunaratne PH, Richards S, Worley KC, Hale S, Garcia\n",
      "AM, Gay LJ, Hulyk SW, Villalon DK, Muzny DM, Sodergren EJ, Lu X, Gibbs RA, Fahey \n",
      "J, Helton E, Ketteman M, Madan A, Rodrigues S, Sanchez A, Whiting M, Madan A,\n",
      "Young AC, Shevchenko Y, Bouffard GG, Blakesley RW, Touchman JW, Green ED, Dickson\n",
      "MC, Rodriguez AC, Grimwood J, Schmutz J, Myers RM, Butterfield YS, Krzywinski MI,\n",
      "Skalska U, Smailus DE, Schnerch A, Schein JE, Jones SJ, Marra MA; Mammalian Gene \n",
      "Collection Program Team.\n",
      "\n",
      "Author information: \n",
      "(1)National Cancer Institute, Bethessda, MD 20892-2580, USA. rls@nih.gov\n",
      "\n",
      "The National Institutes of Health Mammalian Gene Collection (MGC) Program is a\n",
      "multiinstitutional effort to identify and sequence a cDNA clone containing a\n",
      "complete ORF for each human and mouse gene. ESTs were generated from libraries\n",
      "enriched for full-length cDNAs and analyzed to identify candidate full-ORF\n",
      "clones, which then were sequenced to high accuracy. The MGC has currently\n",
      "sequenced and verified the full ORF for a nonredundant set of >9,000 human and\n",
      ">6,000 mouse genes. Candidate full-ORF clones for an additional 7,800 human and\n",
      "3,500 mouse genes also have been identified. All MGC sequences and clones are\n",
      "available without restriction through public databases and clone distribution\n",
      "networks (see http:mgc.nci.nih.gov).\n",
      "\n",
      "DOI: 10.1073/pnas.242603899 \n",
      "PMCID: PMC139241\n",
      "PMID: 12477932  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "32. Genome Res. 2002 Nov;12(11):1773-84.\n",
      "\n",
      "Protein-protein interactions between large proteins: two-hybrid screening using a\n",
      "functionally classified library composed of long cDNAs.\n",
      "\n",
      "Nakayama M(1), Kikuno R, Ohara O.\n",
      "\n",
      "Author information: \n",
      "(1)Department of Human Gene Research, Kazusa DNA Research Institute, Kisarazu,\n",
      "Chiba 292-0818, Japan. nmanabu@kazusa.or.jp\n",
      "\n",
      "Large proteins have multiple domains that are potentially capable of binding many\n",
      "kinds of partners. It is conceivable, therefore, that such proteins could\n",
      "function as an intricate framework of assembly protein complexes. To\n",
      "comprehensively study protein-protein interactions between large KIAA proteins,\n",
      "we have constructed a library composed of 1087 KIAA cDNA clones based on prior\n",
      "functional classifications done in silico. We were guided by two principles that \n",
      "raise the success rate for detecting interactions per tested combination: we\n",
      "avoided testing low-probability combinations, and reduced the number of potential\n",
      "false negatives that arise from the fact that large proteins cannot reliably be\n",
      "expressed in yeast. The latter was addressed by constructing a cDNA library\n",
      "comprised of random fragments encoding large proteins. Cytoplasmic domains of\n",
      "KIAA transmembrane proteins (>1000 amino acids) were used as bait for yeast\n",
      "two-hybrid screening. Our analyses reveal that several KIAA proteins bearing a\n",
      "transmembrane region have the capability of binding to other KIAA proteins\n",
      "containing domains (e.g., PDZ, SH3, rhoGEF, and spectrin) known to be localized\n",
      "to highly specialized submembranous sites, indicating that they participate in\n",
      "cellular junction formation, receptor or channel clustering, and intracellular\n",
      "signaling events. Our representative library should be a very useful resource for\n",
      "detecting previously unidentified interactions because it complements\n",
      "conventional expression libraries, which seldom contain large cDNAs.\n",
      "\n",
      "DOI: 10.1101/gr.406902 \n",
      "PMCID: PMC187542\n",
      "PMID: 12421765  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "33. DNA Res. 1997 Oct 31;4(5):345-9.\n",
      "\n",
      "Characterization of cDNA clones in size-fractionated cDNA libraries from human\n",
      "brain.\n",
      "\n",
      "Seki N(1), Ohira M, Nagase T, Ishikawa K, Miyajima N, Nakajima D, Nomura N, Ohara\n",
      "O.\n",
      "\n",
      "Author information: \n",
      "(1)Kazusa DNA Research Institute, Chiba, Japan. nseki@kazusa.or.jp\n",
      "\n",
      "To evaluate the size-fractionated cDNA libraries of human brain previously\n",
      "constructed (O. O'hara et al. DNA Research, 4, 53-59, 1997), the occurrence of\n",
      "chimeric clones and the content of clones with coding potentiality were analyzed \n",
      "using the randomly sampled clones with insert sizes of 5 to 7 kb. When the\n",
      "chromosomal location of 30 clones was determined by the radiation-hybrid mapping \n",
      "method, the map positions assigned from the 3'- and 5'-end sequences separately\n",
      "were coincident for 29 clones, suggesting that the occurrence of chimeric clones \n",
      "is at most 1/30. Using 91 clones mapped to chromosome 1, the content of clones\n",
      "that have the potentiality coding for proteins larger than 100 amino acid\n",
      "residues was estimated to be approximately 50% (46 out of 91 clones) on the basis\n",
      "of nucleotide sequence analysis and coding potentiality assay in vitro. No\n",
      "significant open reading frames were detected in the remaining clones. Although\n",
      "the clones coding for short peptides may not have been included in the above\n",
      "estimation, the libraries constructed from the whole brain mRNA fraction appear\n",
      "to contain a considerable amount of clones corresponding to the 5'-truncated\n",
      "transcripts in an unprocessed form and/or those with long 3'-untranslated\n",
      "regions.\n",
      "\n",
      "\n",
      "PMID: 9455484  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "34. Genome Res. 1997 Apr;7(4):353-8.\n",
      "\n",
      "Large-scale concatenation cDNA sequencing.\n",
      "\n",
      "Yu W, Andersson B, Worley KC, Muzny DM, Ding Y, Liu W, Ricafrente JY, Wentland\n",
      "MA, Lennon G, Gibbs RA.\n",
      "\n",
      "A total of 100 kb of DNA derived from 69 individual human brain cDNA clones of\n",
      "0.7-2.0 kb were sequenced by concatenated cDNA sequencing (CCS), whereby multiple\n",
      "individual DNA fragments are sequenced simultaneously in a single shotgun\n",
      "library. The method yielded accurate sequences and a similar efficiency compared \n",
      "with other shotgun libraries constructed from single DNA fragments (> 20 kb).\n",
      "Computer analyses were carried out on 65 cDNA clone sequences and their\n",
      "corresponding end sequences to examine both nucleic acid and amino acid sequence \n",
      "similarities in the databases. Thirty-seven clones revealed no DNA database\n",
      "matches, 12 clones generated exact matches (> or = 98% identity), and 16 clones\n",
      "generated nonexact matches (57%-97% identity) to either known human or other\n",
      "species genes. Of those 28 matched clones, 8 had corresponding end sequences that\n",
      "failed to identify similarities. In a protein similarity search, 27 clone\n",
      "sequences displayed significant matches, whereas only 20 of the end sequences had\n",
      "matches to known protein sequences. Our data indicate that full-length cDNA\n",
      "insert sequences provide significantly more nucleic acid and protein sequence\n",
      "similarity matches than expressed sequence tags (ESTs) for database searching.\n",
      "\n",
      "\n",
      "PMCID: PMC139146\n",
      "PMID: 9110174  [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "35. Anal Biochem. 1996 Apr 5;236(1):107-13.\n",
      "\n",
      "A \"double adaptor\" method for improved shotgun library construction.\n",
      "\n",
      "Andersson B(1), Wentland MA, Ricafrente JY, Liu W, Gibbs RA.\n",
      "\n",
      "Author information: \n",
      "(1)Department of Molecular and Human Genetics, Baylor College of Medicine, One\n",
      "Baylor Plaza, Houston, Texas, 77030, USA.\n",
      "\n",
      "The efficiency of shotgun DNA sequencing depends to a great extent on the quality\n",
      "of the random-subclone libraries used. We here describe a novel \"double adaptor\" \n",
      "strategy for efficient construction of high-quality shotgun libraries. In this\n",
      "method, randomly sheared and end-repaired fragments are ligated to\n",
      "oligonucleotide adaptors creating 12-base overhangs. Nonphosphorylated\n",
      "oligonucleotides are used, which prevents formation of adaptor dimers and ensures\n",
      "efficient ligation of insert to adaptor. The vector is prepared from a modified\n",
      "M13 vector, by KpnI/PstI digestion followed by ligation to oligonucleotides with \n",
      "ends complementary to the overhangs created in the digest. These adaptors create \n",
      "5'-overhangs complementary to those on the inserts. Following annealing of insert\n",
      "to vector, the DNA is directly used for transformation without a ligation step.\n",
      "This protocol is robust and shows three- to fivefold higher yield of clones\n",
      "compared to previous protocols. No chimeric clones can be detected and the\n",
      "background of clones without an insert is <1%. The procedure is rapid and shows\n",
      "potential for automation.\n",
      "\n",
      "DOI: 10.1006/abio.1996.0138 \n",
      "PMID: 8619474  [Indexed for MEDLINE]\n"
     ]
    }
   ],
   "source": [
    "require 'net/http'\n",
    "require 'bio'\n",
    "require 'nokogiri'\n",
    "\n",
    "\n",
    "ncbi = Bio::NCBI::REST.new\n",
    "max = 5\n",
    "\n",
    "# note that you can now use spaces in your search terms - BioRuby will format the URL correctly\n",
    "ids = ncbi.esearch('\"Wilkinson MD\"[AUTH]', {\"db\"=>\"nucleotide\", \"rettype\"=>\"gb\", \"retmax\" => max})\n",
    "\n",
    "idlist = ids.join(\",\")\n",
    "\n",
    "elink = URI(\"https://eutils.ncbi.nlm.nih.gov:443/entrez/eutils/elink.fcgi\")\n",
    "request = Net::HTTP::Post.new(elink)\n",
    "https = Net::HTTP.new(elink.host,elink.port)\n",
    "https.use_ssl = true  # tell the HTTP connection to use ecryption (SSL = Secure Socket Layer)\n",
    "\n",
    "request.set_form_data(\n",
    "  'email' => 'markw@illuminae.com',\n",
    "  'dbfrom' => 'nuccore', \n",
    "  'db' => 'pubmed',\n",
    "  'retmode' => 'xml',\n",
    "  'id' => idlist)\n",
    "\n",
    "output = https.request(request)   # make the request (we really should check if it is successful)\n",
    "\n",
    "linkset = output.body\n",
    "\n",
    "\n",
    "doc = Nokogiri::XML(linkset)\n",
    "\n",
    "pubmedids = []  # force it to be a string so that we can use '<<' operator\n",
    "\n",
    "doc.xpath('//LinkSet/LinkSetDb/Link').each do |node|    # the \"path\" (\"xpath\")to the data, starts with '//'\n",
    "   pubmedids << node.at_xpath('Id').content.to_s\n",
    "end\n",
    "puts \"PUBMED IDs are: #{pubmedids}\"\n",
    "\n",
    "pubmedrecords = ncbi.efetch(pubmedids, {\"db\"=>\"pubmed\", \"rettype\"=>\"abstract\", \"retmode\"=>\"text\"})\n",
    "\n",
    "puts pubmedrecords.force_encoding(\"utf-8\")  # NOTE - this is a useful trick for text that has international characters!\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "\n",
    "# Bio::PubMed , Bio::MEDLINE and bibtex format\n",
    "\n",
    "The output above is great for a human, but not very useful for a machine.  Most reference managers can accept new records in bibtex format.  Can we get bibtex format for these records?  Sure!\n",
    "\n",
    "Bio::PubMed is a shortcut to the ESearch and EFetch of utilities for the pubmed database.  It is used like this:\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#===========\n",
    "#  This is only for your information\n",
    "# we already have a list of PubMed IDs from our LinkList above\n",
    "# but you can use Bio::PubMed.esearch to get them \n",
    "# from a search against the pubmed database\n",
    "# ===========\n",
    "#keywords = \"apoptosis death\"\n",
    "#options = {\n",
    "#  'maxdate' => '2016/05/31',\n",
    "#  'retmax' => 2,\n",
    "#}\n",
    "#entries = Bio::PubMed.esearch(keywords, options)  # entries is a list of pubmed ids\n",
    "# ===========\n",
    "\n",
    "\n",
    "pubmedids = [\"25896417\", \"20610895\", \"19056867\"]\n",
    "\n",
    "Bio::PubMed.efetch(pubmedids).each do |pubmedrecord|\n",
    "  # pubmedrecord is a string of the record, in pubmed format\n",
    "  # we can use that to create a new Bio::MEDLINE object\n",
    "  medline = Bio::MEDLINE.new(pubmedrecord)\n",
    "  reference = medline.reference  # reference is a Bio::Reference object\n",
    "  puts reference.bibtex   # the .bibtex method creates the bibtex string of the Bio::Reference\n",
    "end"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<pre>\n",
    "  \n",
    "\n",
    "</pre>\n",
    "  \n",
    "# The Final Code\n",
    "\n",
    "So finally we put the BibTex code at the end of our script, and...\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 95,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "PUBMED IDs are: [\"25896417\", \"20610895\", \"19056867\", \"15498460\", \"12421765\", \"26344197\", \"25896417\", \"21182205\", \"20610895\", \"20379614\", \"19056867\", \"16710414\", \"15498460\", \"15489334\", \"14702039\", \"12477932\", \"12421765\", \"9455484\", \"9110174\", \"8619474\", \"26344197\", \"25896417\", \"21182205\", \"20610895\", \"20379614\", \"19056867\", \"16710414\", \"15498460\", \"15489334\", \"14702039\", \"12477932\", \"12421765\", \"9455484\", \"9110174\", \"8619474\"]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "@article{PMID:25896417,\n",
      "  author       = {Nakada, T. A. and Boyd, J. H. and Russell, J. A. and Aguirre-Hernandez, R. and Wilkinson, M. D. and Thair, S. A. and Nakada, E. and McConechy, M. K. and Fjell, C. D. and Walley, K. R.},\n",
      "  title        = {VPS13D Gene Variant Is Associated with Altered IL-6 Production and Mortality in Septic Shock.},\n",
      "  journal      = {J Innate Immun},\n",
      "  year         = {2015},\n",
      "  volume       = {7},\n",
      "  number       = {5},\n",
      "  pages        = {545--553},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/25896417},\n",
      "}\n",
      "\n",
      "@article{PMID:20610895,\n",
      "  author       = {Koriyama, H. and Nakagami, H. and Katsuya, T. and Sugimoto, K. and Yamashita, H. and Takami, Y. and Maeda, S. and Kubo, M. and Takahashi, A. and Nakamura, Y. and Ogihara, T. and Rakugi, H. and Kaneda, Y. and Morishita, R.},\n",
      "  title        = {Identification of evidence suggestive of an association with peripheral arterial disease at the OSBPL10 locus by genome-wide investigation in the Japanese population.},\n",
      "  journal      = {J Atheroscler Thromb},\n",
      "  year         = {2010},\n",
      "  volume       = {17},\n",
      "  number       = {10},\n",
      "  pages        = {1054--1062},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/20610895},\n",
      "}\n",
      "\n",
      "@article{PMID:19056867,\n",
      "  author       = {Gonzales, P. A. and Pisitkun, T. and Hoffert, J. D. and Tchapyjnikov, D. and Star, R. A. and Kleta, R. and Wang, N. S. and Knepper, M. A.},\n",
      "  title        = {Large-scale proteomics and phosphoproteomics of urinary exosomes.},\n",
      "  journal      = {J Am Soc Nephrol},\n",
      "  year         = {2009},\n",
      "  volume       = {20},\n",
      "  number       = {2},\n",
      "  pages        = {363--379},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/19056867},\n",
      "}\n",
      "\n",
      "@article{PMID:15498460,\n",
      "  author       = {Velayos-Baeza, A. and Vettori, A. and Copley, R. R. and Dobson-Stone, C. and Monaco, A. P.},\n",
      "  title        = {Analysis of the human VPS13 gene family.},\n",
      "  journal      = {Genomics},\n",
      "  year         = {2004},\n",
      "  volume       = {84},\n",
      "  number       = {3},\n",
      "  pages        = {536--549},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/15498460},\n",
      "}\n",
      "\n",
      "@article{PMID:12421765,\n",
      "  author       = {Nakayama, M. and Kikuno, R. and Ohara, O.},\n",
      "  title        = {Protein-protein interactions between large proteins: two-hybrid screening using a functionally classified library composed of long cDNAs.},\n",
      "  journal      = {Genome Res},\n",
      "  year         = {2002},\n",
      "  volume       = {12},\n",
      "  number       = {11},\n",
      "  pages        = {1773--1784},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/12421765},\n",
      "}\n",
      "\n",
      "@article{PMID:26344197,\n",
      "  author       = {Wan, C. and Borgeson, B. and Phanse, S. and Tu, F. and Drew, K. and Clark, G. and Xiong, X. and Kagan, O. and Kwan, J. and Bezginov, A. and Chessman, K. and Pal, S. and Cromar, G. and Papoulas, O. and Ni, Z. and Boutz, D. R. and Stoilova, S. and Havugimana, P. C. and Guo, X. and Malty, R. H. and Sarov, M. and Greenblatt, J. and Babu, M. and Derry, W. B. and Tillier, E. R. and Wallingford, J. B. and Parkinson, J. and Marcotte, E. M. and Emili, A.},\n",
      "  title        = {Panorama of ancient metazoan macromolecular complexes.},\n",
      "  journal      = {Nature},\n",
      "  year         = {2015},\n",
      "  volume       = {525},\n",
      "  number       = {7569},\n",
      "  pages        = {339--344},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/26344197},\n",
      "}\n",
      "\n",
      "@article{PMID:25896417,\n",
      "  author       = {Nakada, T. A. and Boyd, J. H. and Russell, J. A. and Aguirre-Hernandez, R. and Wilkinson, M. D. and Thair, S. A. and Nakada, E. and McConechy, M. K. and Fjell, C. D. and Walley, K. R.},\n",
      "  title        = {VPS13D Gene Variant Is Associated with Altered IL-6 Production and Mortality in Septic Shock.},\n",
      "  journal      = {J Innate Immun},\n",
      "  year         = {2015},\n",
      "  volume       = {7},\n",
      "  number       = {5},\n",
      "  pages        = {545--553},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/25896417},\n",
      "}\n",
      "\n",
      "@article{PMID:21182205,\n",
      "  author       = {Tarallo, R. and Bamundo, A. and Nassa, G. and Nola, E. and Paris, O. and Ambrosino, C. and Facchiano, A. and Baumann, M. and Nyman, T. A. and Weisz, A.},\n",
      "  title        = {Identification of proteins associated with ligand-activated estrogen receptor alpha in human breast cancer cell nuclei by tandem affinity purification and nano LC-MS/MS.},\n",
      "  journal      = {Proteomics},\n",
      "  year         = {2011},\n",
      "  volume       = {11},\n",
      "  number       = {1},\n",
      "  pages        = {172--179},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/21182205},\n",
      "}\n",
      "\n",
      "@article{PMID:20610895,\n",
      "  author       = {Koriyama, H. and Nakagami, H. and Katsuya, T. and Sugimoto, K. and Yamashita, H. and Takami, Y. and Maeda, S. and Kubo, M. and Takahashi, A. and Nakamura, Y. and Ogihara, T. and Rakugi, H. and Kaneda, Y. and Morishita, R.},\n",
      "  title        = {Identification of evidence suggestive of an association with peripheral arterial disease at the OSBPL10 locus by genome-wide investigation in the Japanese population.},\n",
      "  journal      = {J Atheroscler Thromb},\n",
      "  year         = {2010},\n",
      "  volume       = {17},\n",
      "  number       = {10},\n",
      "  pages        = {1054--1062},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/20610895},\n",
      "}\n",
      "\n",
      "@article{PMID:20379614,\n",
      "  author       = {Rose, J. E. and Behm, F. M. and Drgon, T. and Johnson, C. and Uhl, G. R.},\n",
      "  title        = {Personalized smoking cessation: interactions between nicotine dose, dependence and quit-success genotype score.},\n",
      "  journal      = {Mol Med},\n",
      "  year         = {2010},\n",
      "  volume       = {16},\n",
      "  number       = {7-8},\n",
      "  pages        = {247--253},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/20379614},\n",
      "}\n",
      "\n",
      "@article{PMID:19056867,\n",
      "  author       = {Gonzales, P. A. and Pisitkun, T. and Hoffert, J. D. and Tchapyjnikov, D. and Star, R. A. and Kleta, R. and Wang, N. S. and Knepper, M. A.},\n",
      "  title        = {Large-scale proteomics and phosphoproteomics of urinary exosomes.},\n",
      "  journal      = {J Am Soc Nephrol},\n",
      "  year         = {2009},\n",
      "  volume       = {20},\n",
      "  number       = {2},\n",
      "  pages        = {363--379},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/19056867},\n",
      "}\n",
      "\n",
      "@article{PMID:16710414,\n",
      "  author       = {Gregory, S. G. and Barlow, K. F. and McLay, K. E. and Kaul, R. and Swarbreck, D. and Dunham, A. and Scott, C. E. and Howe, K. L. and Woodfine, K. and Spencer, C. C. and Jones, M. C. and Gillson, C. and Searle, S. and Zhou, Y. and Kokocinski, F. and McDonald, L. and Evans, R. and Phillips, K. and Atkinson, A. and Cooper, R. and Jones, C. and Hall, R. E. and Andrews, T. D. and Lloyd, C. and Ainscough, R. and Almeida, J. P. and Ambrose, K. D. and Anderson, F. and Andrew, R. W. and Ashwell, R. I. and Aubin, K. and Babbage, A. K. and Bagguley, C. L. and Bailey, J. and Beasley, H. and Bethel, G. and Bird, C. P. and Bray-Allen, S. and Brown, J. Y. and Brown, A. J. and Buckley, D. and Burton, J. and Bye, J. and Carder, C. and Chapman, J. C. and Clark, S. Y. and Clarke, G. and Clee, C. and Cobley, V. and Collier, R. E. and Corby, N. and Coville, G. J. and Davies, J. and Deadman, R. and Dunn, M. and Earthrowl, M. and Ellington, A. G. and Errington, H. and Frankish, A. and Frankland, J. and French, L. and Garner, P. and Garnett, J. and Gay, L. and Ghori, M. R. and Gibson, R. and Gilby, L. M. and Gillett, W. and Glithero, R. J. and Grafham, D. V. and Griffiths, C. and Griffiths-Jones, S. and Grocock, R. and Hammond, S. and Harrison, E. S. and Hart, E. and Haugen, E. and Heath, P. D. and Holmes, S. and Holt, K. and Howden, P. J. and Hunt, A. R. and Hunt, S. E. and Hunter, G. and Isherwood, J. and James, R. and Johnson, C. and Johnson, D. and Joy, A. and Kay, M. and Kershaw, J. K. and Kibukawa, M. and Kimberley, A. M. and King, A. and Knights, A. J. and Lad, H. and Laird, G. and Lawlor, S. and Leongamornlert, D. A. and Lloyd, D. M. and Loveland, J. and Lovell, J. and Lush, M. J. and Lyne, R. and Martin, S. and Mashreghi-Mohammadi, M. and Matthews, L. and Matthews, N. S. and McLaren, S. and Milne, S. and Mistry, S. and Moore, M. J. and Nickerson, T. and O'Dell, C. N. and Oliver, K. and Palmeiri, A. and Palmer, S. A. and Parker, A. and Patel, D. and Pearce, A. V. and Peck, A. I. and Pelan, S. and Phelps, K. and Phillimore, B. J. and Plumb, R. and Rajan, J. and Raymond, C. and Rouse, G. and Saenphimmachak, C. and Sehra, H. K. and Sheridan, E. and Shownkeen, R. and Sims, S. and Skuce, C. D. and Smith, M. and Steward, C. and Subramanian, S. and Sycamore, N. and Tracey, A. and Tromans, A. and Van Helmond, Z. and Wall, M. and Wallis, J. M. and White, S. and Whitehead, S. L. and Wilkinson, J. E. and Willey, D. L. and Williams, H. and Wilming, L. and Wray, P. W. and Wu, Z. and Coulson, A. and Vaudin, M. and Sulston, J. E. and Durbin, R. and Hubbard, T. and Wooster, R. and Dunham, I. and Carter, N. P. and McVean, G. and Ross, M. T. and Harrow, J. and Olson, M. V. and Beck, S. and Rogers, J. and Bentley, D. R. and Banerjee, R. and Bryant, S. P. and Burford, D. C. and Burrill, W. D. and Clegg, S. M. and Dhami, P. and Dovey, O. and Faulkner, L. M. and Gribble, S. M. and Langford, C. F. and Pandian, R. D. and Porter, K. M. and Prigmore, E.},\n",
      "  title        = {The DNA sequence and biological annotation of human chromosome 1.},\n",
      "  journal      = {Nature},\n",
      "  year         = {2006},\n",
      "  volume       = {441},\n",
      "  number       = {7091},\n",
      "  pages        = {315--321},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/16710414},\n",
      "}\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "@article{PMID:15498460,\n",
      "  author       = {Velayos-Baeza, A. and Vettori, A. and Copley, R. R. and Dobson-Stone, C. and Monaco, A. P.},\n",
      "  title        = {Analysis of the human VPS13 gene family.},\n",
      "  journal      = {Genomics},\n",
      "  year         = {2004},\n",
      "  volume       = {84},\n",
      "  number       = {3},\n",
      "  pages        = {536--549},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/15498460},\n",
      "}\n",
      "\n",
      "@article{PMID:15489334,\n",
      "  author       = {Gerhard, D. S. and Wagner, L. and Feingold, E. A. and Shenmen, C. M. and Grouse, L. H. and Schuler, G. and Klein, S. L. and Old, S. and Rasooly, R. and Good, P. and Guyer, M. and Peck, A. M. and Derge, J. G. and Lipman, D. and Collins, F. S. and Jang, W. and Sherry, S. and Feolo, M. and Misquitta, L. and Lee, E. and Rotmistrovsky, K. and Greenhut, S. F. and Schaefer, C. F. and Buetow, K. and Bonner, T. I. and Haussler, D. and Kent, J. and Kiekhaus, M. and Furey, T. and Brent, M. and Prange, C. and Schreiber, K. and Shapiro, N. and Bhat, N. K. and Hopkins, R. F. and Hsie, F. and Driscoll, T. and Soares, M. B. and Casavant, T. L. and Scheetz, T. E. and Brown-stein, M. J. and Usdin, T. B. and Toshiyuki, S. and Carninci, P. and Piao, Y. and Dudekula, D. B. and Ko, M. S. and Kawakami, K. and Suzuki, Y. and Sugano, S. and Gruber, C. E. and Smith, M. R. and Simmons, B. and Moore, T. and Waterman, R. and Johnson, S. L. and Ruan, Y. and Wei, C. L. and Mathavan, S. and Gunaratne, P. H. and Wu, J. and Garcia, A. M. and Hulyk, S. W. and Fuh, E. and Yuan, Y. and Sneed, A. and Kowis, C. and Hodgson, A. and Muzny, D. M. and McPherson, J. and Gibbs, R. A. and Fahey, J. and Helton, E. and Ketteman, M. and Madan, A. and Rodrigues, S. and Sanchez, A. and Whiting, M. and Madari, A. and Young, A. C. and Wetherby, K. D. and Granite, S. J. and Kwong, P. N. and Brinkley, C. P. and Pearson, R. L. and Bouffard, G. G. and Blakesly, R. W. and Green, E. D. and Dickson, M. C. and Rodriguez, A. C. and Grimwood, J. and Schmutz, J. and Myers, R. M. and Butterfield, Y. S. and Griffith, M. and Griffith, O. L. and Krzywinski, M. I. and Liao, N. and Morin, R. and Palmquist, D. and Petrescu, A. S. and Skalska, U. and Smailus, D. E. and Stott, J. M. and Schnerch, A. and Schein, J. E. and Jones, S. J. and Holt, R. A. and Baross, A. and Marra, M. A. and Clifton, S. and Makowski, K. A. and Bosak, S. and Malek, J.},\n",
      "  title        = {The status, quality, and expansion of the NIH full-length cDNA project: the Mammalian Gene Collection (MGC).},\n",
      "  journal      = {Genome Res},\n",
      "  year         = {2004},\n",
      "  volume       = {14},\n",
      "  number       = {10B},\n",
      "  pages        = {2121--2127},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/15489334},\n",
      "}\n",
      "\n",
      "@article{PMID:14702039,\n",
      "  author       = {Ota, T. and Suzuki, Y. and Nishikawa, T. and Otsuki, T. and Sugiyama, T. and Irie, R. and Wakamatsu, A. and Hayashi, K. and Sato, H. and Nagai, K. and Kimura, K. and Makita, H. and Sekine, M. and Obayashi, M. and Nishi, T. and Shibahara, T. and Tanaka, T. and Ishii, S. and Yamamoto, J. and Saito, K. and Kawai, Y. and Isono, Y. and Nakamura, Y. and Nagahari, K. and Murakami, K. and Yasuda, T. and Iwayanagi, T. and Wagatsuma, M. and Shiratori, A. and Sudo, H. and Hosoiri, T. and Kaku, Y. and Kodaira, H. and Kondo, H. and Sugawara, M. and Takahashi, M. and Kanda, K. and Yokoi, T. and Furuya, T. and Kikkawa, E. and Omura, Y. and Abe, K. and Kamihara, K. and Katsuta, N. and Sato, K. and Tanikawa, M. and Yamazaki, M. and Ninomiya, K. and Ishibashi, T. and Yamashita, H. and Murakawa, K. and Fujimori, K. and Tanai, H. and Kimata, M. and Watanabe, M. and Hiraoka, S. and Chiba, Y. and Ishida, S. and Ono, Y. and Takiguchi, S. and Watanabe, S. and Yosida, M. and Hotuta, T. and Kusano, J. and Kanehori, K. and Takahashi-Fujii, A. and Hara, H. and Tanase, T. O. and Nomura, Y. and Togiya, S. and Komai, F. and Hara, R. and Takeuchi, K. and Arita, M. and Imose, N. and Musashino, K. and Yuuki, H. and Oshima, A. and Sasaki, N. and Aotsuka, S. and Yoshikawa, Y. and Matsunawa, H. and Ichihara, T. and Shiohata, N. and Sano, S. and Moriya, S. and Momiyama, H. and Satoh, N. and Takami, S. and Terashima, Y. and Suzuki, O. and Nakagawa, S. and Senoh, A. and Mizoguchi, H. and Goto, Y. and Shimizu, F. and Wakebe, H. and Hishigaki, H. and Watanabe, T. and Sugiyama, A. and Takemoto, M. and Kawakami, B. and Yamazaki, M. and Watanabe, K. and Kumagai, A. and Itakura, S. and Fukuzumi, Y. and Fujimori, Y. and Komiyama, M. and Tashiro, H. and Tanigami, A. and Fujiwara, T. and Ono, T. and Yamada, K. and Fujii, Y. and Ozaki, K. and Hirao, M. and Ohmori, Y. and Kawabata, A. and Hikiji, T. and Kobatake, N. and Inagaki, H. and Ikema, Y. and Okamoto, S. and Okitani, R. and Kawakami, T. and Noguchi, S. and Itoh, T. and Shigeta, K. and Senba, T. and Matsumura, K. and Nakajima, Y. and Mizuno, T. and Morinaga, M. and Sasaki, M. and Togashi, T. and Oyama, M. and Hata, H. and Watanabe, M. and Komatsu, T. and Mizushima-Sugano, J. and Satoh, T. and Shirai, Y. and Takahashi, Y. and Nakagawa, K. and Okumura, K. and Nagase, T. and Nomura, N. and Kikuchi, H. and Masuho, Y. and Yamashita, R. and Nakai, K. and Yada, T. and Nakamura, Y. and Ohara, O. and Isogai, T. and Sugano, S.},\n",
      "  title        = {Complete sequencing and characterization of 21,243 full-length human cDNAs.},\n",
      "  journal      = {Nat Genet},\n",
      "  year         = {2004},\n",
      "  volume       = {36},\n",
      "  number       = {1},\n",
      "  pages        = {40--45},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/14702039},\n",
      "}\n",
      "\n",
      "@article{PMID:12477932,\n",
      "  author       = {Strausberg, R. L. and Feingold, E. A. and Grouse, L. H. and Derge, J. G. and Klausner, R. D. and Collins, F. S. and Wagner, L. and Shenmen, C. M. and Schuler, G. D. and Altschul, S. F. and Zeeberg, B. and Buetow, K. H. and Schaefer, C. F. and Bhat, N. K. and Hopkins, R. F. and Jordan, H. and Moore, T. and Max, S. I. and Wang, J. and Hsieh, F. and Diatchenko, L. and Marusina, K. and Farmer, A. A. and Rubin, G. M. and Hong, L. and Stapleton, M. and Soares, M. B. and Bonaldo, M. F. and Casavant, T. L. and Scheetz, T. E. and Brownstein, M. J. and Usdin, T. B. and Toshiyuki, S. and Carninci, P. and Prange, C. and Raha, S. S. and Loquellano, N. A. and Peters, G. J. and Abramson, R. D. and Mullahy, S. J. and Bosak, S. A. and McEwan, P. J. and McKernan, K. J. and Malek, J. A. and Gunaratne, P. H. and Richards, S. and Worley, K. C. and Hale, S. and Garcia, A. M. and Gay, L. J. and Hulyk, S. W. and Villalon, D. K. and Muzny, D. M. and Sodergren, E. J. and Lu, X. and Gibbs, R. A. and Fahey, J. and Helton, E. and Ketteman, M. and Madan, A. and Rodrigues, S. and Sanchez, A. and Whiting, M. and Madan, A. and Young, A. C. and Shevchenko, Y. and Bouffard, G. G. and Blakesley, R. W. and Touchman, J. W. and Green, E. D. and Dickson, M. C. and Rodriguez, A. C. and Grimwood, J. and Schmutz, J. and Myers, R. M. and Butterfield, Y. S. and Krzywinski, M. I. and Skalska, U. and Smailus, D. E. and Schnerch, A. and Schein, J. E. and Jones, S. J. and Marra, M. A.},\n",
      "  title        = {Generation and initial analysis of more than 15,000 full-length human and mouse cDNA sequences.},\n",
      "  journal      = {Proc Natl Acad Sci U S A},\n",
      "  year         = {2002},\n",
      "  volume       = {99},\n",
      "  number       = {26},\n",
      "  pages        = {16899--16903},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/12477932},\n",
      "}\n",
      "\n",
      "@article{PMID:12421765,\n",
      "  author       = {Nakayama, M. and Kikuno, R. and Ohara, O.},\n",
      "  title        = {Protein-protein interactions between large proteins: two-hybrid screening using a functionally classified library composed of long cDNAs.},\n",
      "  journal      = {Genome Res},\n",
      "  year         = {2002},\n",
      "  volume       = {12},\n",
      "  number       = {11},\n",
      "  pages        = {1773--1784},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/12421765},\n",
      "}\n",
      "\n",
      "@article{PMID:9455484,\n",
      "  author       = {Seki, N. and Ohira, M. and Nagase, T. and Ishikawa, K. and Miyajima, N. and Nakajima, D. and Nomura, N. and Ohara, O.},\n",
      "  title        = {Characterization of cDNA clones in size-fractionated cDNA libraries from human brain.},\n",
      "  journal      = {DNA Res},\n",
      "  year         = {1997},\n",
      "  volume       = {4},\n",
      "  number       = {5},\n",
      "  pages        = {345--349},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/9455484},\n",
      "}\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "@article{PMID:9110174,\n",
      "  author       = {Yu, W. and Andersson, B. and Worley, K. C. and Muzny, D. M. and Ding, Y. and Liu, W. and Ricafrente, J. Y. and Wentland, M. A. and Lennon, G. and Gibbs, R. A.},\n",
      "  title        = {Large-scale concatenation cDNA sequencing.},\n",
      "  journal      = {Genome Res},\n",
      "  year         = {1997},\n",
      "  volume       = {7},\n",
      "  number       = {4},\n",
      "  pages        = {353--358},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/9110174},\n",
      "}\n",
      "\n",
      "@article{PMID:8619474,\n",
      "  author       = {Andersson, B. and Wentland, M. A. and Ricafrente, J. Y. and Liu, W. and Gibbs, R. A.},\n",
      "  title        = {A \"double adaptor\" method for improved shotgun library construction.},\n",
      "  journal      = {Anal Biochem},\n",
      "  year         = {1996},\n",
      "  volume       = {236},\n",
      "  number       = {1},\n",
      "  pages        = {107--113},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/8619474},\n",
      "}\n",
      "\n",
      "@article{PMID:26344197,\n",
      "  author       = {Wan, C. and Borgeson, B. and Phanse, S. and Tu, F. and Drew, K. and Clark, G. and Xiong, X. and Kagan, O. and Kwan, J. and Bezginov, A. and Chessman, K. and Pal, S. and Cromar, G. and Papoulas, O. and Ni, Z. and Boutz, D. R. and Stoilova, S. and Havugimana, P. C. and Guo, X. and Malty, R. H. and Sarov, M. and Greenblatt, J. and Babu, M. and Derry, W. B. and Tillier, E. R. and Wallingford, J. B. and Parkinson, J. and Marcotte, E. M. and Emili, A.},\n",
      "  title        = {Panorama of ancient metazoan macromolecular complexes.},\n",
      "  journal      = {Nature},\n",
      "  year         = {2015},\n",
      "  volume       = {525},\n",
      "  number       = {7569},\n",
      "  pages        = {339--344},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/26344197},\n",
      "}\n",
      "\n",
      "@article{PMID:25896417,\n",
      "  author       = {Nakada, T. A. and Boyd, J. H. and Russell, J. A. and Aguirre-Hernandez, R. and Wilkinson, M. D. and Thair, S. A. and Nakada, E. and McConechy, M. K. and Fjell, C. D. and Walley, K. R.},\n",
      "  title        = {VPS13D Gene Variant Is Associated with Altered IL-6 Production and Mortality in Septic Shock.},\n",
      "  journal      = {J Innate Immun},\n",
      "  year         = {2015},\n",
      "  volume       = {7},\n",
      "  number       = {5},\n",
      "  pages        = {545--553},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/25896417},\n",
      "}\n",
      "\n",
      "@article{PMID:21182205,\n",
      "  author       = {Tarallo, R. and Bamundo, A. and Nassa, G. and Nola, E. and Paris, O. and Ambrosino, C. and Facchiano, A. and Baumann, M. and Nyman, T. A. and Weisz, A.},\n",
      "  title        = {Identification of proteins associated with ligand-activated estrogen receptor alpha in human breast cancer cell nuclei by tandem affinity purification and nano LC-MS/MS.},\n",
      "  journal      = {Proteomics},\n",
      "  year         = {2011},\n",
      "  volume       = {11},\n",
      "  number       = {1},\n",
      "  pages        = {172--179},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/21182205},\n",
      "}\n",
      "\n",
      "@article{PMID:20610895,\n",
      "  author       = {Koriyama, H. and Nakagami, H. and Katsuya, T. and Sugimoto, K. and Yamashita, H. and Takami, Y. and Maeda, S. and Kubo, M. and Takahashi, A. and Nakamura, Y. and Ogihara, T. and Rakugi, H. and Kaneda, Y. and Morishita, R.},\n",
      "  title        = {Identification of evidence suggestive of an association with peripheral arterial disease at the OSBPL10 locus by genome-wide investigation in the Japanese population.},\n",
      "  journal      = {J Atheroscler Thromb},\n",
      "  year         = {2010},\n",
      "  volume       = {17},\n",
      "  number       = {10},\n",
      "  pages        = {1054--1062},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/20610895},\n",
      "}\n",
      "\n",
      "@article{PMID:20379614,\n",
      "  author       = {Rose, J. E. and Behm, F. M. and Drgon, T. and Johnson, C. and Uhl, G. R.},\n",
      "  title        = {Personalized smoking cessation: interactions between nicotine dose, dependence and quit-success genotype score.},\n",
      "  journal      = {Mol Med},\n",
      "  year         = {2010},\n",
      "  volume       = {16},\n",
      "  number       = {7-8},\n",
      "  pages        = {247--253},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/20379614},\n",
      "}\n",
      "\n",
      "@article{PMID:19056867,\n",
      "  author       = {Gonzales, P. A. and Pisitkun, T. and Hoffert, J. D. and Tchapyjnikov, D. and Star, R. A. and Kleta, R. and Wang, N. S. and Knepper, M. A.},\n",
      "  title        = {Large-scale proteomics and phosphoproteomics of urinary exosomes.},\n",
      "  journal      = {J Am Soc Nephrol},\n",
      "  year         = {2009},\n",
      "  volume       = {20},\n",
      "  number       = {2},\n",
      "  pages        = {363--379},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/19056867},\n",
      "}\n",
      "\n",
      "@article{PMID:16710414,\n",
      "  author       = {Gregory, S. G. and Barlow, K. F. and McLay, K. E. and Kaul, R. and Swarbreck, D. and Dunham, A. and Scott, C. E. and Howe, K. L. and Woodfine, K. and Spencer, C. C. and Jones, M. C. and Gillson, C. and Searle, S. and Zhou, Y. and Kokocinski, F. and McDonald, L. and Evans, R. and Phillips, K. and Atkinson, A. and Cooper, R. and Jones, C. and Hall, R. E. and Andrews, T. D. and Lloyd, C. and Ainscough, R. and Almeida, J. P. and Ambrose, K. D. and Anderson, F. and Andrew, R. W. and Ashwell, R. I. and Aubin, K. and Babbage, A. K. and Bagguley, C. L. and Bailey, J. and Beasley, H. and Bethel, G. and Bird, C. P. and Bray-Allen, S. and Brown, J. Y. and Brown, A. J. and Buckley, D. and Burton, J. and Bye, J. and Carder, C. and Chapman, J. C. and Clark, S. Y. and Clarke, G. and Clee, C. and Cobley, V. and Collier, R. E. and Corby, N. and Coville, G. J. and Davies, J. and Deadman, R. and Dunn, M. and Earthrowl, M. and Ellington, A. G. and Errington, H. and Frankish, A. and Frankland, J. and French, L. and Garner, P. and Garnett, J. and Gay, L. and Ghori, M. R. and Gibson, R. and Gilby, L. M. and Gillett, W. and Glithero, R. J. and Grafham, D. V. and Griffiths, C. and Griffiths-Jones, S. and Grocock, R. and Hammond, S. and Harrison, E. S. and Hart, E. and Haugen, E. and Heath, P. D. and Holmes, S. and Holt, K. and Howden, P. J. and Hunt, A. R. and Hunt, S. E. and Hunter, G. and Isherwood, J. and James, R. and Johnson, C. and Johnson, D. and Joy, A. and Kay, M. and Kershaw, J. K. and Kibukawa, M. and Kimberley, A. M. and King, A. and Knights, A. J. and Lad, H. and Laird, G. and Lawlor, S. and Leongamornlert, D. A. and Lloyd, D. M. and Loveland, J. and Lovell, J. and Lush, M. J. and Lyne, R. and Martin, S. and Mashreghi-Mohammadi, M. and Matthews, L. and Matthews, N. S. and McLaren, S. and Milne, S. and Mistry, S. and Moore, M. J. and Nickerson, T. and O'Dell, C. N. and Oliver, K. and Palmeiri, A. and Palmer, S. A. and Parker, A. and Patel, D. and Pearce, A. V. and Peck, A. I. and Pelan, S. and Phelps, K. and Phillimore, B. J. and Plumb, R. and Rajan, J. and Raymond, C. and Rouse, G. and Saenphimmachak, C. and Sehra, H. K. and Sheridan, E. and Shownkeen, R. and Sims, S. and Skuce, C. D. and Smith, M. and Steward, C. and Subramanian, S. and Sycamore, N. and Tracey, A. and Tromans, A. and Van Helmond, Z. and Wall, M. and Wallis, J. M. and White, S. and Whitehead, S. L. and Wilkinson, J. E. and Willey, D. L. and Williams, H. and Wilming, L. and Wray, P. W. and Wu, Z. and Coulson, A. and Vaudin, M. and Sulston, J. E. and Durbin, R. and Hubbard, T. and Wooster, R. and Dunham, I. and Carter, N. P. and McVean, G. and Ross, M. T. and Harrow, J. and Olson, M. V. and Beck, S. and Rogers, J. and Bentley, D. R. and Banerjee, R. and Bryant, S. P. and Burford, D. C. and Burrill, W. D. and Clegg, S. M. and Dhami, P. and Dovey, O. and Faulkner, L. M. and Gribble, S. M. and Langford, C. F. and Pandian, R. D. and Porter, K. M. and Prigmore, E.},\n",
      "  title        = {The DNA sequence and biological annotation of human chromosome 1.},\n",
      "  journal      = {Nature},\n",
      "  year         = {2006},\n",
      "  volume       = {441},\n",
      "  number       = {7091},\n",
      "  pages        = {315--321},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/16710414},\n",
      "}\n",
      "\n",
      "@article{PMID:15498460,\n",
      "  author       = {Velayos-Baeza, A. and Vettori, A. and Copley, R. R. and Dobson-Stone, C. and Monaco, A. P.},\n",
      "  title        = {Analysis of the human VPS13 gene family.},\n",
      "  journal      = {Genomics},\n",
      "  year         = {2004},\n",
      "  volume       = {84},\n",
      "  number       = {3},\n",
      "  pages        = {536--549},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/15498460},\n",
      "}\n",
      "\n",
      "@article{PMID:15489334,\n",
      "  author       = {Gerhard, D. S. and Wagner, L. and Feingold, E. A. and Shenmen, C. M. and Grouse, L. H. and Schuler, G. and Klein, S. L. and Old, S. and Rasooly, R. and Good, P. and Guyer, M. and Peck, A. M. and Derge, J. G. and Lipman, D. and Collins, F. S. and Jang, W. and Sherry, S. and Feolo, M. and Misquitta, L. and Lee, E. and Rotmistrovsky, K. and Greenhut, S. F. and Schaefer, C. F. and Buetow, K. and Bonner, T. I. and Haussler, D. and Kent, J. and Kiekhaus, M. and Furey, T. and Brent, M. and Prange, C. and Schreiber, K. and Shapiro, N. and Bhat, N. K. and Hopkins, R. F. and Hsie, F. and Driscoll, T. and Soares, M. B. and Casavant, T. L. and Scheetz, T. E. and Brown-stein, M. J. and Usdin, T. B. and Toshiyuki, S. and Carninci, P. and Piao, Y. and Dudekula, D. B. and Ko, M. S. and Kawakami, K. and Suzuki, Y. and Sugano, S. and Gruber, C. E. and Smith, M. R. and Simmons, B. and Moore, T. and Waterman, R. and Johnson, S. L. and Ruan, Y. and Wei, C. L. and Mathavan, S. and Gunaratne, P. H. and Wu, J. and Garcia, A. M. and Hulyk, S. W. and Fuh, E. and Yuan, Y. and Sneed, A. and Kowis, C. and Hodgson, A. and Muzny, D. M. and McPherson, J. and Gibbs, R. A. and Fahey, J. and Helton, E. and Ketteman, M. and Madan, A. and Rodrigues, S. and Sanchez, A. and Whiting, M. and Madari, A. and Young, A. C. and Wetherby, K. D. and Granite, S. J. and Kwong, P. N. and Brinkley, C. P. and Pearson, R. L. and Bouffard, G. G. and Blakesly, R. W. and Green, E. D. and Dickson, M. C. and Rodriguez, A. C. and Grimwood, J. and Schmutz, J. and Myers, R. M. and Butterfield, Y. S. and Griffith, M. and Griffith, O. L. and Krzywinski, M. I. and Liao, N. and Morin, R. and Palmquist, D. and Petrescu, A. S. and Skalska, U. and Smailus, D. E. and Stott, J. M. and Schnerch, A. and Schein, J. E. and Jones, S. J. and Holt, R. A. and Baross, A. and Marra, M. A. and Clifton, S. and Makowski, K. A. and Bosak, S. and Malek, J.},\n",
      "  title        = {The status, quality, and expansion of the NIH full-length cDNA project: the Mammalian Gene Collection (MGC).},\n",
      "  journal      = {Genome Res},\n",
      "  year         = {2004},\n",
      "  volume       = {14},\n",
      "  number       = {10B},\n",
      "  pages        = {2121--2127},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/15489334},\n",
      "}\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "@article{PMID:14702039,\n",
      "  author       = {Ota, T. and Suzuki, Y. and Nishikawa, T. and Otsuki, T. and Sugiyama, T. and Irie, R. and Wakamatsu, A. and Hayashi, K. and Sato, H. and Nagai, K. and Kimura, K. and Makita, H. and Sekine, M. and Obayashi, M. and Nishi, T. and Shibahara, T. and Tanaka, T. and Ishii, S. and Yamamoto, J. and Saito, K. and Kawai, Y. and Isono, Y. and Nakamura, Y. and Nagahari, K. and Murakami, K. and Yasuda, T. and Iwayanagi, T. and Wagatsuma, M. and Shiratori, A. and Sudo, H. and Hosoiri, T. and Kaku, Y. and Kodaira, H. and Kondo, H. and Sugawara, M. and Takahashi, M. and Kanda, K. and Yokoi, T. and Furuya, T. and Kikkawa, E. and Omura, Y. and Abe, K. and Kamihara, K. and Katsuta, N. and Sato, K. and Tanikawa, M. and Yamazaki, M. and Ninomiya, K. and Ishibashi, T. and Yamashita, H. and Murakawa, K. and Fujimori, K. and Tanai, H. and Kimata, M. and Watanabe, M. and Hiraoka, S. and Chiba, Y. and Ishida, S. and Ono, Y. and Takiguchi, S. and Watanabe, S. and Yosida, M. and Hotuta, T. and Kusano, J. and Kanehori, K. and Takahashi-Fujii, A. and Hara, H. and Tanase, T. O. and Nomura, Y. and Togiya, S. and Komai, F. and Hara, R. and Takeuchi, K. and Arita, M. and Imose, N. and Musashino, K. and Yuuki, H. and Oshima, A. and Sasaki, N. and Aotsuka, S. and Yoshikawa, Y. and Matsunawa, H. and Ichihara, T. and Shiohata, N. and Sano, S. and Moriya, S. and Momiyama, H. and Satoh, N. and Takami, S. and Terashima, Y. and Suzuki, O. and Nakagawa, S. and Senoh, A. and Mizoguchi, H. and Goto, Y. and Shimizu, F. and Wakebe, H. and Hishigaki, H. and Watanabe, T. and Sugiyama, A. and Takemoto, M. and Kawakami, B. and Yamazaki, M. and Watanabe, K. and Kumagai, A. and Itakura, S. and Fukuzumi, Y. and Fujimori, Y. and Komiyama, M. and Tashiro, H. and Tanigami, A. and Fujiwara, T. and Ono, T. and Yamada, K. and Fujii, Y. and Ozaki, K. and Hirao, M. and Ohmori, Y. and Kawabata, A. and Hikiji, T. and Kobatake, N. and Inagaki, H. and Ikema, Y. and Okamoto, S. and Okitani, R. and Kawakami, T. and Noguchi, S. and Itoh, T. and Shigeta, K. and Senba, T. and Matsumura, K. and Nakajima, Y. and Mizuno, T. and Morinaga, M. and Sasaki, M. and Togashi, T. and Oyama, M. and Hata, H. and Watanabe, M. and Komatsu, T. and Mizushima-Sugano, J. and Satoh, T. and Shirai, Y. and Takahashi, Y. and Nakagawa, K. and Okumura, K. and Nagase, T. and Nomura, N. and Kikuchi, H. and Masuho, Y. and Yamashita, R. and Nakai, K. and Yada, T. and Nakamura, Y. and Ohara, O. and Isogai, T. and Sugano, S.},\n",
      "  title        = {Complete sequencing and characterization of 21,243 full-length human cDNAs.},\n",
      "  journal      = {Nat Genet},\n",
      "  year         = {2004},\n",
      "  volume       = {36},\n",
      "  number       = {1},\n",
      "  pages        = {40--45},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/14702039},\n",
      "}\n",
      "\n",
      "@article{PMID:12477932,\n",
      "  author       = {Strausberg, R. L. and Feingold, E. A. and Grouse, L. H. and Derge, J. G. and Klausner, R. D. and Collins, F. S. and Wagner, L. and Shenmen, C. M. and Schuler, G. D. and Altschul, S. F. and Zeeberg, B. and Buetow, K. H. and Schaefer, C. F. and Bhat, N. K. and Hopkins, R. F. and Jordan, H. and Moore, T. and Max, S. I. and Wang, J. and Hsieh, F. and Diatchenko, L. and Marusina, K. and Farmer, A. A. and Rubin, G. M. and Hong, L. and Stapleton, M. and Soares, M. B. and Bonaldo, M. F. and Casavant, T. L. and Scheetz, T. E. and Brownstein, M. J. and Usdin, T. B. and Toshiyuki, S. and Carninci, P. and Prange, C. and Raha, S. S. and Loquellano, N. A. and Peters, G. J. and Abramson, R. D. and Mullahy, S. J. and Bosak, S. A. and McEwan, P. J. and McKernan, K. J. and Malek, J. A. and Gunaratne, P. H. and Richards, S. and Worley, K. C. and Hale, S. and Garcia, A. M. and Gay, L. J. and Hulyk, S. W. and Villalon, D. K. and Muzny, D. M. and Sodergren, E. J. and Lu, X. and Gibbs, R. A. and Fahey, J. and Helton, E. and Ketteman, M. and Madan, A. and Rodrigues, S. and Sanchez, A. and Whiting, M. and Madan, A. and Young, A. C. and Shevchenko, Y. and Bouffard, G. G. and Blakesley, R. W. and Touchman, J. W. and Green, E. D. and Dickson, M. C. and Rodriguez, A. C. and Grimwood, J. and Schmutz, J. and Myers, R. M. and Butterfield, Y. S. and Krzywinski, M. I. and Skalska, U. and Smailus, D. E. and Schnerch, A. and Schein, J. E. and Jones, S. J. and Marra, M. A.},\n",
      "  title        = {Generation and initial analysis of more than 15,000 full-length human and mouse cDNA sequences.},\n",
      "  journal      = {Proc Natl Acad Sci U S A},\n",
      "  year         = {2002},\n",
      "  volume       = {99},\n",
      "  number       = {26},\n",
      "  pages        = {16899--16903},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/12477932},\n",
      "}\n",
      "\n",
      "@article{PMID:12421765,\n",
      "  author       = {Nakayama, M. and Kikuno, R. and Ohara, O.},\n",
      "  title        = {Protein-protein interactions between large proteins: two-hybrid screening using a functionally classified library composed of long cDNAs.},\n",
      "  journal      = {Genome Res},\n",
      "  year         = {2002},\n",
      "  volume       = {12},\n",
      "  number       = {11},\n",
      "  pages        = {1773--1784},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/12421765},\n",
      "}\n",
      "\n",
      "@article{PMID:9455484,\n",
      "  author       = {Seki, N. and Ohira, M. and Nagase, T. and Ishikawa, K. and Miyajima, N. and Nakajima, D. and Nomura, N. and Ohara, O.},\n",
      "  title        = {Characterization of cDNA clones in size-fractionated cDNA libraries from human brain.},\n",
      "  journal      = {DNA Res},\n",
      "  year         = {1997},\n",
      "  volume       = {4},\n",
      "  number       = {5},\n",
      "  pages        = {345--349},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/9455484},\n",
      "}\n",
      "\n",
      "@article{PMID:9110174,\n",
      "  author       = {Yu, W. and Andersson, B. and Worley, K. C. and Muzny, D. M. and Ding, Y. and Liu, W. and Ricafrente, J. Y. and Wentland, M. A. and Lennon, G. and Gibbs, R. A.},\n",
      "  title        = {Large-scale concatenation cDNA sequencing.},\n",
      "  journal      = {Genome Res},\n",
      "  year         = {1997},\n",
      "  volume       = {7},\n",
      "  number       = {4},\n",
      "  pages        = {353--358},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/9110174},\n",
      "}\n",
      "\n",
      "@article{PMID:8619474,\n",
      "  author       = {Andersson, B. and Wentland, M. A. and Ricafrente, J. Y. and Liu, W. and Gibbs, R. A.},\n",
      "  title        = {A \"double adaptor\" method for improved shotgun library construction.},\n",
      "  journal      = {Anal Biochem},\n",
      "  year         = {1996},\n",
      "  volume       = {236},\n",
      "  number       = {1},\n",
      "  pages        = {107--113},\n",
      "  url          = {http://www.ncbi.nlm.nih.gov/pubmed/8619474},\n",
      "}\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "[\"PMID- 25896417\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20160620\\nLR  - 20150828\\nIS  - 1662-8128 (Electronic)\\nIS  - 1662-811X (Linking)\\nVI  - 7\\nIP  - 5\\nDP  - 2015\\nTI  - VPS13D Gene Variant Is Associated with Altered IL-6 Production and Mortality in\\n      Septic Shock.\\nPG  - 545-53\\nLID - 10.1159/000381265 [doi]\\nAB  - BACKGROUND: Genetic variations contribute to septic shock mortality. To discover \\n      a novel locus, we performed in vitro genome-wide association studies (GWAS) and\\n      further tested the result in a cohort of septic shock patients. METHODS: Two in\\n      vitro GWAS using a quantitative trait locus analysis of stimulated IL-6\\n      production in lymphoblastoid cells from 60 individuals of European ancestry were \\n      performed. VPS13D rs6685273 was genotyped in European ancestry patients (n =\\n      498). The VPS13D gene was silenced in vitro. RESULTS: Two GWAS using\\n      lymphoblastoid cells identified the locus of VPS13D rs6685273 that was\\n      significant in the same direction in both GWAS. The VPS13D rs6685273 C allele was\\n      associated with increased IL-6 production. Patients with septic shock who had the\\n      VPS13D rs6685273 CC genotype had an increased 28-day mortality (p = 0.023) and\\n      more organ failure (p < 0.05) compared to the CT/TT genotypes. VPS13D in vitro\\n      gene silencing in the HeLa cell line increased IL-6 production. Furthermore, the \\n      rs6685273 genotype was associated with differential VPS13D splice variant\\n      expression. CONCLUSIONS: The VPS13D rs6685273 C allele was associated with\\n      increased IL-6 production in vitro. The patients with the VPS13D rs6685273 CC\\n      genotype had increased 28-day mortality and increased organ failure. VPS13D\\n      appears to regulate IL-6 production.\\nCI  - (c) 2015 S. Karger AG, Basel.\\nFAU - Nakada, Taka-aki\\nAU  - Nakada TA\\nAD  - Critical Care Research Laboratories, Centre for Heart Lung Innovation, St. Paul's\\n      Hospital, University of British Columbia, Vancouver, B.C., Canada.\\nFAU - Boyd, John H\\nAU  - Boyd JH\\nFAU - Russell, James A\\nAU  - Russell JA\\nFAU - Aguirre-Hernandez, Rosalia\\nAU  - Aguirre-Hernandez R\\nFAU - Wilkinson, Mark D\\nAU  - Wilkinson MD\\nFAU - Thair, Simone A\\nAU  - Thair SA\\nFAU - Nakada, Emiri\\nAU  - Nakada E\\nFAU - McConechy, Melissa K\\nAU  - McConechy MK\\nFAU - Fjell, Christopher D\\nAU  - Fjell CD\\nFAU - Walley, Keith R\\nAU  - Walley KR\\nLA  - eng\\nGR  - Canadian Institutes of Health Research/Canada\\nPT  - Journal Article\\nPT  - Multicenter Study\\nPT  - Randomized Controlled Trial\\nPT  - Research Support, Non-U.S. Gov't\\nDEP - 20150417\\nPL  - Switzerland\\nTA  - J Innate Immun\\nJT  - Journal of innate immunity\\nJID - 101469471\\nRN  - 0 (Interleukin-6)\\nRN  - 0 (Proteins)\\nRN  - 0 (VPS13D protein, human)\\nSB  - IM\\nMH  - Aged\\nMH  - Cohort Studies\\nMH  - European Continental Ancestry Group/genetics\\nMH  - Female\\nMH  - Gene Frequency\\nMH  - Genetic Predisposition to Disease\\nMH  - Genome-Wide Association Study\\nMH  - Genotype\\nMH  - HeLa Cells\\nMH  - Humans\\nMH  - Interleukin-6/genetics/*metabolism\\nMH  - Lymphocyte Activation/genetics\\nMH  - Lymphocytes/*physiology\\nMH  - Male\\nMH  - Middle Aged\\nMH  - Polymorphism, Single Nucleotide\\nMH  - Proteins/*genetics\\nMH  - Shock, Septic/*genetics/immunology/*mortality\\nMH  - Spain\\nMH  - Survival Analysis\\nEDAT- 2015/04/22 06:00\\nMHDA- 2016/06/21 06:00\\nCRDT- 2015/04/22 06:00\\nPHST- 2014/12/17 00:00 [received]\\nPHST- 2015/02/26 00:00 [accepted]\\nPHST- 2015/04/22 06:00 [entrez]\\nPHST- 2015/04/22 06:00 [pubmed]\\nPHST- 2016/06/21 06:00 [medline]\\nAID - 000381265 [pii]\\nAID - 10.1159/000381265 [doi]\\nPST - ppublish\\nSO  - J Innate Immun. 2015;7(5):545-53. doi: 10.1159/000381265. Epub 2015 Apr 17.\", \"PMID- 20610895\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20110308\\nLR  - 20120216\\nIS  - 1880-3873 (Electronic)\\nIS  - 1340-3478 (Linking)\\nVI  - 17\\nIP  - 10\\nDP  - 2010 Oct 27\\nTI  - Identification of evidence suggestive of an association with peripheral arterial \\n      disease at the OSBPL10 locus by genome-wide investigation in the Japanese\\n      population.\\nPG  - 1054-62\\nAB  - AIM: Peripheral arterial disease (PAD) is a common cause of cardiovascular\\n      morbidity and an independent predictor of cardiovascular mortality. However,\\n      little is known about the genetic basis of PAD. To elucidate this, we performed a\\n      two-staged genome-wide association study in Japanese individuals. METHODS: We\\n      initially tested 222,285 single-nucleotide polymorphisms (SNPs). After the first \\n      screening in a panel of 195 PAD cases and 1,358 controls, 2,696 SNPs (1.2%) were \\n      further genotyped in the second screening using another panel of 699 PAD cases\\n      and 1540 controls. In both screenings, controls were subjects affected with some \\n      diseases other than PAD. RESULTS: When analyzed in the combined panel, the\\n      strongest signal of PAD association was observed at rs1902341 in the intron of\\n      OSBPL10 (p=4.7E-7 for trend test; OR=1.31, 95% CI 1.18-1.46). Also, PAD was\\n      modestly associated at several other loci such as rs2554503 in CSMD1 (p=5.7E-5;\\n      OR=1.32, 95% CI 1.15-1.51) or rs235243 in VSP13D (p=0.04; OR=1.18, 95% CI\\n      1.01-1.37). CONCLUSION: Our genome-wide exploration identified suggestive\\n      evidence of PAD association at the OSBPL10 locus. Because the association has not\\n      reached a genome-wide significant level, further replication study is warranted\\n      for verification in the Japanese population.\\nFAU - Koriyama, Hiroshi\\nAU  - Koriyama H\\nAD  - Department of Geriatric Medicine and Nephrology, Osaka University Graduate School\\n      of Medicine, 2-2 Yamada-oka, Suita, Japan.\\nFAU - Nakagami, Hironori\\nAU  - Nakagami H\\nFAU - Katsuya, Tomohiro\\nAU  - Katsuya T\\nFAU - Sugimoto, Ken\\nAU  - Sugimoto K\\nFAU - Yamashita, Hidetoshi\\nAU  - Yamashita H\\nFAU - Takami, Yoichi\\nAU  - Takami Y\\nFAU - Maeda, Shiro\\nAU  - Maeda S\\nFAU - Kubo, Michiaki\\nAU  - Kubo M\\nFAU - Takahashi, Atsushi\\nAU  - Takahashi A\\nFAU - Nakamura, Yusuke\\nAU  - Nakamura Y\\nFAU - Ogihara, Toshio\\nAU  - Ogihara T\\nFAU - Rakugi, Hiromi\\nAU  - Rakugi H\\nFAU - Kaneda, Yasufumi\\nAU  - Kaneda Y\\nFAU - Morishita, Ryuichi\\nAU  - Morishita R\\nLA  - eng\\nPT  - Comparative Study\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov't\\nDEP - 20100702\\nPL  - Japan\\nTA  - J Atheroscler Thromb\\nJT  - Journal of atherosclerosis and thrombosis\\nJID - 9506298\\nRN  - 0 (Receptors, Steroid)\\nRN  - 0 (oxysterol binding protein)\\nSB  - IM\\nMH  - Aged\\nMH  - Asian Continental Ancestry Group/*genetics\\nMH  - Case-Control Studies\\nMH  - Female\\nMH  - Genetic Predisposition to Disease\\nMH  - Genome, Human\\nMH  - *Genome-Wide Association Study\\nMH  - Genotype\\nMH  - Humans\\nMH  - Japan/epidemiology\\nMH  - Linkage Disequilibrium\\nMH  - Male\\nMH  - Middle Aged\\nMH  - Peripheral Vascular Diseases/epidemiology/*genetics\\nMH  - Polymorphism, Single Nucleotide/*genetics\\nMH  - Receptors, Steroid/*genetics\\nEDAT- 2010/07/09 06:00\\nMHDA- 2011/03/09 06:00\\nCRDT- 2010/07/09 06:00\\nPHST- 2010/07/09 06:00 [entrez]\\nPHST- 2010/07/09 06:00 [pubmed]\\nPHST- 2011/03/09 06:00 [medline]\\nAID - JST.JSTAGE/jat/4291 [pii]\\nPST - ppublish\\nSO  - J Atheroscler Thromb. 2010 Oct 27;17(10):1054-62. Epub 2010 Jul 2.\", \"PMID- 19056867\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20090226\\nLR  - 20170730\\nIS  - 1533-3450 (Electronic)\\nIS  - 1046-6673 (Linking)\\nVI  - 20\\nIP  - 2\\nDP  - 2009 Feb\\nTI  - Large-scale proteomics and phosphoproteomics of urinary exosomes.\\nPG  - 363-79\\nLID - 10.1681/ASN.2008040406 [doi]\\nAB  - Normal human urine contains large numbers of exosomes, which are 40- to 100-nm\\n      vesicles that originate as the internal vesicles in multivesicular bodies from\\n      every renal epithelial cell type facing the urinary space. Here, we used LC-MS/MS\\n      to profile the proteome of human urinary exosomes. Overall, the analysis\\n      identified 1132 proteins unambiguously, including 177 that are represented on the\\n      Online Mendelian Inheritance in Man database of disease-related genes, suggesting\\n      that exosome analysis is a potential approach to discover urinary biomarkers. We \\n      extended the proteomic analysis to phosphoproteomic profiling using neutral loss \\n      scanning, and this yielded multiple novel phosphorylation sites, including\\n      serine-811 in the thiazide-sensitive Na-Cl co-transporter, NCC. To demonstrate\\n      the potential use of exosome analysis to identify a genetic renal disease, we\\n      carried out immunoblotting of exosomes from urine samples of patients with a\\n      clinical diagnosis of Bartter syndrome type I, showing an absence of the\\n      sodium-potassium-chloride co-transporter 2, NKCC2. The proteomic data are\\n      publicly accessible at http://dir.nhlbi.nih.gov/papers/lkem/exosome/.\\nFAU - Gonzales, Patricia A\\nAU  - Gonzales PA\\nAD  - Laboratory of Kidney and Electrolyte Metabolism, National Heart, Lung, and Blood \\n      Institute, National Institutes of Health, Bethesda, MD 20892, USA.\\nFAU - Pisitkun, Trairak\\nAU  - Pisitkun T\\nFAU - Hoffert, Jason D\\nAU  - Hoffert JD\\nFAU - Tchapyjnikov, Dmitry\\nAU  - Tchapyjnikov D\\nFAU - Star, Robert A\\nAU  - Star RA\\nFAU - Kleta, Robert\\nAU  - Kleta R\\nFAU - Wang, Nam Sun\\nAU  - Wang NS\\nFAU - Knepper, Mark A\\nAU  - Knepper MA\\nLA  - eng\\nGR  - Z01 HL001285-21/Intramural NIH HHS/United States\\nGR  - Z99 HL999999/Intramural NIH HHS/United States\\nPT  - Journal Article\\nDEP - 20081203\\nPL  - United States\\nTA  - J Am Soc Nephrol\\nJT  - Journal of the American Society of Nephrology : JASN\\nJID - 9013836\\nRN  - 0 (Phosphoproteins)\\nRN  - 0 (Proteome)\\nRN  - 0 (Thiazides)\\nRN  - EC 3.6.1.- (Adenosine Triphosphatases)\\nSB  - IM\\nMH  - Adenosine Triphosphatases/chemistry\\nMH  - Adult\\nMH  - Bartter Syndrome/urine\\nMH  - Chromatography, Liquid/methods\\nMH  - Exosomes/metabolism\\nMH  - Female\\nMH  - Humans\\nMH  - Male\\nMH  - Mass Spectrometry/methods\\nMH  - Phosphoproteins/chemistry\\nMH  - Phosphorylation\\nMH  - Proteome\\nMH  - Proteomics/*methods\\nMH  - Thiazides/chemistry\\nMH  - *Urine\\nPMC - PMC2637050\\nOID - NLM: PMC2637050\\nEDAT- 2008/12/06 09:00\\nMHDA- 2009/02/27 09:00\\nCRDT- 2008/12/06 09:00\\nPHST- 2008/12/06 09:00 [pubmed]\\nPHST- 2009/02/27 09:00 [medline]\\nPHST- 2008/12/06 09:00 [entrez]\\nAID - ASN.2008040406 [pii]\\nAID - 10.1681/ASN.2008040406 [doi]\\nPST - ppublish\\nSO  - J Am Soc Nephrol. 2009 Feb;20(2):363-79. doi: 10.1681/ASN.2008040406. Epub 2008\\n      Dec 3.\", \"PMID- 15498460\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20050124\\nLR  - 20070516\\nIS  - 0888-7543 (Print)\\nIS  - 0888-7543 (Linking)\\nVI  - 84\\nIP  - 3\\nDP  - 2004 Sep\\nTI  - Analysis of the human VPS13 gene family.\\nPG  - 536-49\\nAB  - The gene mutated in chorea-acanthocytosis (CHAC; approved gene symbol VPS13A)\\n      encodes chorein, a protein similar to yeast Vps13p. We detected several similar\\n      putative human proteins by BLAST analysis of chorein. We characterized the\\n      structure of three new genes encoding these CHAC-similar proteins, located on\\n      chromosomes 1p36, 8q22, and 15q21. The most similar gene in yeast to all four\\n      human genes is Vps13, and therefore the human genes were named VPS13A (CHAC,\\n      9q21), VPS13B (8q22), VPS13C (15q21), and VPS13D (1p36). VPS13B has recently been\\n      reported as COH1, altered in Cohen syndrome. For each gene, we describe several\\n      alternative splicing variants; at least two transcripts per gene are major forms.\\n      The expression pattern of these genes is ubiquitous, with some tissue-specific\\n      differences between several transcript variants. Protein sequence comparisons\\n      suggest that intramolecular duplications have played an important role in the\\n      evolution of this gene family.\\nFAU - Velayos-Baeza, Antonio\\nAU  - Velayos-Baeza A\\nAD  - Wellcome Trust Centre for Human Genetics, University of Oxford, Headington, OX3\\n      7BN Oxford, UK.\\nFAU - Vettori, Andrea\\nAU  - Vettori A\\nFAU - Copley, Richard R\\nAU  - Copley RR\\nFAU - Dobson-Stone, Carol\\nAU  - Dobson-Stone C\\nFAU - Monaco, A P\\nAU  - Monaco AP\\nLA  - eng\\nSI  - GENBANK/AJ608769\\nSI  - GENBANK/AJ608770\\nSI  - GENBANK/AJ608771\\nSI  - GENBANK/AJ608772\\nSI  - GENBANK/AJ608773\\nSI  - GENBANK/AJ608774\\nSI  - GENBANK/AJ608775\\nSI  - GENBANK/AJ626859\\nSI  - GENBANK/AJ626860\\nSI  - GENBANK/AJ626861\\nPT  - Comparative Study\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov't\\nPL  - United States\\nTA  - Genomics\\nJT  - Genomics\\nJID - 8800135\\nRN  - 0 (Membrane Proteins)\\nRN  - 0 (Proteins)\\nRN  - 0 (VPS13A protein, human)\\nRN  - 0 (VPS13B protein, human)\\nRN  - 0 (VPS13C protein, human)\\nRN  - 0 (VPS13D protein, human)\\nRN  - 0 (Vesicular Transport Proteins)\\nSB  - IM\\nMH  - Alternative Splicing/*genetics\\nMH  - Amino Acid Sequence\\nMH  - Base Sequence\\nMH  - Blotting, Northern\\nMH  - Evolution, Molecular\\nMH  - Gene Components\\nMH  - Gene Duplication\\nMH  - Gene Expression Profiling\\nMH  - Humans\\nMH  - Membrane Proteins/*genetics\\nMH  - Molecular Sequence Data\\nMH  - Multigene Family/*genetics\\nMH  - Phylogeny\\nMH  - Proteins/*genetics\\nMH  - Reverse Transcriptase Polymerase Chain Reaction\\nMH  - Sequence Alignment\\nMH  - Sequence Analysis, DNA\\nMH  - Sequence Homology\\nMH  - Vesicular Transport Proteins\\nEDAT- 2004/10/23 09:00\\nMHDA- 2005/01/26 09:00\\nCRDT- 2004/10/23 09:00\\nPHST- 2004/02/13 00:00 [received]\\nPHST- 2004/04/19 00:00 [accepted]\\nPHST- 2004/10/23 09:00 [pubmed]\\nPHST- 2005/01/26 09:00 [medline]\\nPHST- 2004/10/23 09:00 [entrez]\\nAID - S0888754304001156 [pii]\\nAID - 10.1016/j.ygeno.2004.04.012 [doi]\\nPST - ppublish\\nSO  - Genomics. 2004 Sep;84(3):536-49. doi: 10.1016/j.ygeno.2004.04.012.\", \"PMID- 12421765\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20030114\\nLR  - 20140611\\nIS  - 1088-9051 (Print)\\nIS  - 1088-9051 (Linking)\\nVI  - 12\\nIP  - 11\\nDP  - 2002 Nov\\nTI  - Protein-protein interactions between large proteins: two-hybrid screening using a\\n      functionally classified library composed of long cDNAs.\\nPG  - 1773-84\\nAB  - Large proteins have multiple domains that are potentially capable of binding many\\n      kinds of partners. It is conceivable, therefore, that such proteins could\\n      function as an intricate framework of assembly protein complexes. To\\n      comprehensively study protein-protein interactions between large KIAA proteins,\\n      we have constructed a library composed of 1087 KIAA cDNA clones based on prior\\n      functional classifications done in silico. We were guided by two principles that \\n      raise the success rate for detecting interactions per tested combination: we\\n      avoided testing low-probability combinations, and reduced the number of potential\\n      false negatives that arise from the fact that large proteins cannot reliably be\\n      expressed in yeast. The latter was addressed by constructing a cDNA library\\n      comprised of random fragments encoding large proteins. Cytoplasmic domains of\\n      KIAA transmembrane proteins (>1000 amino acids) were used as bait for yeast\\n      two-hybrid screening. Our analyses reveal that several KIAA proteins bearing a\\n      transmembrane region have the capability of binding to other KIAA proteins\\n      containing domains (e.g., PDZ, SH3, rhoGEF, and spectrin) known to be localized\\n      to highly specialized submembranous sites, indicating that they participate in\\n      cellular junction formation, receptor or channel clustering, and intracellular\\n      signaling events. Our representative library should be a very useful resource for\\n      detecting previously unidentified interactions because it complements\\n      conventional expression libraries, which seldom contain large cDNAs.\\nFAU - Nakayama, Manabu\\nAU  - Nakayama M\\nAD  - Department of Human Gene Research, Kazusa DNA Research Institute, Kisarazu, Chiba\\n      292-0818, Japan. nmanabu@kazusa.or.jp\\nFAU - Kikuno, Reiko\\nAU  - Kikuno R\\nFAU - Ohara, Osamu\\nAU  - Ohara O\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov't\\nPL  - United States\\nTA  - Genome Res\\nJT  - Genome research\\nJID - 9518021\\nRN  - 0 (DNA, Complementary)\\nRN  - 0 (Peptides)\\nRN  - 0 (Proteins)\\nSB  - IM\\nMH  - Brain Chemistry/genetics\\nMH  - Cloning, Molecular\\nMH  - Computational Biology/instrumentation/methods\\nMH  - Cytoplasm/chemistry\\nMH  - DNA, Complementary/chemistry/*genetics\\nMH  - *Gene Library\\nMH  - Humans\\nMH  - Leukemia, Myeloid/genetics/pathology\\nMH  - Peptide Mapping/methods\\nMH  - Peptides/genetics/physiology\\nMH  - Protein Interaction Mapping/*methods\\nMH  - Protein Structure, Tertiary/physiology\\nMH  - Proteins/chemistry/genetics/physiology\\nMH  - Software\\nMH  - Tumor Cells, Cultured\\nMH  - *Two-Hybrid System Techniques\\nMH  - src Homology Domains/genetics/physiology\\nPMC - PMC187542\\nOID - NLM: PMC187542\\nEDAT- 2002/11/08 04:00\\nMHDA- 2003/01/15 04:00\\nCRDT- 2002/11/08 04:00\\nPHST- 2002/11/08 04:00 [pubmed]\\nPHST- 2003/01/15 04:00 [medline]\\nPHST- 2002/11/08 04:00 [entrez]\\nAID - 10.1101/gr.406902 [doi]\\nPST - ppublish\\nSO  - Genome Res. 2002 Nov;12(11):1773-84. doi: 10.1101/gr.406902.\", \"PMID- 26344197\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20151014\\nLR  - 20170220\\nIS  - 1476-4687 (Electronic)\\nIS  - 0028-0836 (Linking)\\nVI  - 525\\nIP  - 7569\\nDP  - 2015 Sep 17\\nTI  - Panorama of ancient metazoan macromolecular complexes.\\nPG  - 339-44\\nLID - 10.1038/nature14877 [doi]\\nAB  - Macromolecular complexes are essential to conserved biological processes, but\\n      their prevalence across animals is unclear. By combining extensive biochemical\\n      fractionation with quantitative mass spectrometry, here we directly examined the \\n      composition of soluble multiprotein complexes among diverse metazoan models.\\n      Using an integrative approach, we generated a draft conservation map consisting\\n      of more than one million putative high-confidence co-complex interactions for\\n      species with fully sequenced genomes that encompasses functional modules present \\n      broadly across all extant animals. Clustering reveals a spectrum of conservation,\\n      ranging from ancient eukaryotic assemblies that have probably served cellular\\n      housekeeping roles for at least one billion years, ancestral complexes that have \\n      accrued contemporary components, and rarer metazoan innovations linked to\\n      multicellularity. We validated these projections by independent co-fractionation \\n      experiments in evolutionarily distant species, affinity purification and\\n      functional analyses. The comprehensiveness, centrality and modularity of these\\n      reconstructed interactomes reflect their fundamental mechanistic importance and\\n      adaptive value to animal cell systems.\\nFAU - Wan, Cuihong\\nAU  - Wan C\\nAD  - Donnelly Centre for Cellular and Biomolecular Research, University of Toronto,\\n      Toronto, Ontario M5S 3E1, Canada.\\nAD  - Center for Systems and Synthetic Biology, Institute for Cellular and Molecular\\n      Biology, University of Texas at Austin, Austin, Texas 78712, USA.\\nFAU - Borgeson, Blake\\nAU  - Borgeson B\\nAD  - Center for Systems and Synthetic Biology, Institute for Cellular and Molecular\\n      Biology, University of Texas at Austin, Austin, Texas 78712, USA.\\nFAU - Phanse, Sadhna\\nAU  - Phanse S\\nAD  - Donnelly Centre for Cellular and Biomolecular Research, University of Toronto,\\n      Toronto, Ontario M5S 3E1, Canada.\\nFAU - Tu, Fan\\nAU  - Tu F\\nAD  - Center for Systems and Synthetic Biology, Institute for Cellular and Molecular\\n      Biology, University of Texas at Austin, Austin, Texas 78712, USA.\\nFAU - Drew, Kevin\\nAU  - Drew K\\nAD  - Center for Systems and Synthetic Biology, Institute for Cellular and Molecular\\n      Biology, University of Texas at Austin, Austin, Texas 78712, USA.\\nFAU - Clark, Greg\\nAU  - Clark G\\nAD  - Department of Medical Biophysics, Toronto, Ontario M5G 1L7, Canada.\\nFAU - Xiong, Xuejian\\nAU  - Xiong X\\nAD  - Department of Molecular Genetics, University of Toronto, Toronto, Ontario M5S\\n      1A8, Canada.\\nAD  - Hospital for Sick Children, Toronto, Ontario M5G 1X8, Canada.\\nFAU - Kagan, Olga\\nAU  - Kagan O\\nAD  - Donnelly Centre for Cellular and Biomolecular Research, University of Toronto,\\n      Toronto, Ontario M5S 3E1, Canada.\\nFAU - Kwan, Julian\\nAU  - Kwan J\\nAD  - Donnelly Centre for Cellular and Biomolecular Research, University of Toronto,\\n      Toronto, Ontario M5S 3E1, Canada.\\nAD  - Department of Molecular Genetics, University of Toronto, Toronto, Ontario M5S\\n      1A8, Canada.\\nFAU - Bezginov, Alexandr\\nAU  - Bezginov A\\nAD  - Department of Medical Biophysics, Toronto, Ontario M5G 1L7, Canada.\\nFAU - Chessman, Kyle\\nAU  - Chessman K\\nAD  - Department of Molecular Genetics, University of Toronto, Toronto, Ontario M5S\\n      1A8, Canada.\\nAD  - Hospital for Sick Children, Toronto, Ontario M5G 1X8, Canada.\\nFAU - Pal, Swati\\nAU  - Pal S\\nAD  - Hospital for Sick Children, Toronto, Ontario M5G 1X8, Canada.\\nFAU - Cromar, Graham\\nAU  - Cromar G\\nAD  - Department of Molecular Genetics, University of Toronto, Toronto, Ontario M5S\\n      1A8, Canada.\\nAD  - Hospital for Sick Children, Toronto, Ontario M5G 1X8, Canada.\\nFAU - Papoulas, Ophelia\\nAU  - Papoulas O\\nAD  - Center for Systems and Synthetic Biology, Institute for Cellular and Molecular\\n      Biology, University of Texas at Austin, Austin, Texas 78712, USA.\\nFAU - Ni, Zuyao\\nAU  - Ni Z\\nAD  - Donnelly Centre for Cellular and Biomolecular Research, University of Toronto,\\n      Toronto, Ontario M5S 3E1, Canada.\\nFAU - Boutz, Daniel R\\nAU  - Boutz DR\\nAD  - Center for Systems and Synthetic Biology, Institute for Cellular and Molecular\\n      Biology, University of Texas at Austin, Austin, Texas 78712, USA.\\nFAU - Stoilova, Snejana\\nAU  - Stoilova S\\nAD  - Donnelly Centre for Cellular and Biomolecular Research, University of Toronto,\\n      Toronto, Ontario M5S 3E1, Canada.\\nFAU - Havugimana, Pierre C\\nAU  - Havugimana PC\\nAD  - Donnelly Centre for Cellular and Biomolecular Research, University of Toronto,\\n      Toronto, Ontario M5S 3E1, Canada.\\nFAU - Guo, Xinghua\\nAU  - Guo X\\nAD  - Donnelly Centre for Cellular and Biomolecular Research, University of Toronto,\\n      Toronto, Ontario M5S 3E1, Canada.\\nFAU - Malty, Ramy H\\nAU  - Malty RH\\nAD  - Department of Biochemistry, University of Regina, Regina, Saskatchewan S4S 0A2,\\n      Canada.\\nFAU - Sarov, Mihail\\nAU  - Sarov M\\nAD  - Max Planck Institute of Molecular Cell Biology and Genetics, 01307 Dresden,\\n      Germany.\\nFAU - Greenblatt, Jack\\nAU  - Greenblatt J\\nAD  - Donnelly Centre for Cellular and Biomolecular Research, University of Toronto,\\n      Toronto, Ontario M5S 3E1, Canada.\\nAD  - Department of Molecular Genetics, University of Toronto, Toronto, Ontario M5S\\n      1A8, Canada.\\nFAU - Babu, Mohan\\nAU  - Babu M\\nAD  - Department of Biochemistry, University of Regina, Regina, Saskatchewan S4S 0A2,\\n      Canada.\\nFAU - Derry, W Brent\\nAU  - Derry WB\\nAD  - Department of Molecular Genetics, University of Toronto, Toronto, Ontario M5S\\n      1A8, Canada.\\nAD  - Hospital for Sick Children, Toronto, Ontario M5G 1X8, Canada.\\nFAU - Tillier, Elisabeth R\\nAU  - Tillier ER\\nAD  - Department of Medical Biophysics, Toronto, Ontario M5G 1L7, Canada.\\nFAU - Wallingford, John B\\nAU  - Wallingford JB\\nAD  - Center for Systems and Synthetic Biology, Institute for Cellular and Molecular\\n      Biology, University of Texas at Austin, Austin, Texas 78712, USA.\\nAD  - Department of Molecular Biosciences, University of Texas at Austin, Austin, Texas\\n      78712, USA.\\nFAU - Parkinson, John\\nAU  - Parkinson J\\nAD  - Department of Molecular Genetics, University of Toronto, Toronto, Ontario M5S\\n      1A8, Canada.\\nAD  - Hospital for Sick Children, Toronto, Ontario M5G 1X8, Canada.\\nFAU - Marcotte, Edward M\\nAU  - Marcotte EM\\nAD  - Center for Systems and Synthetic Biology, Institute for Cellular and Molecular\\n      Biology, University of Texas at Austin, Austin, Texas 78712, USA.\\nAD  - Department of Molecular Biosciences, University of Texas at Austin, Austin, Texas\\n      78712, USA.\\nFAU - Emili, Andrew\\nAU  - Emili A\\nAD  - Donnelly Centre for Cellular and Biomolecular Research, University of Toronto,\\n      Toronto, Ontario M5S 3E1, Canada.\\nAD  - Department of Molecular Genetics, University of Toronto, Toronto, Ontario M5S\\n      1A8, Canada.\\nLA  - eng\\nGR  - F32GM112495/GM/NIGMS NIH HHS/United States\\nGR  - R01 GM076536/GM/NIGMS NIH HHS/United States\\nGR  - DP1 GM106408/GM/NIGMS NIH HHS/United States\\nGR  - R01 GM067779/GM/NIGMS NIH HHS/United States\\nGR  - Canadian Institutes of Health Research/Canada\\nGR  - F32 GM112495/GM/NIGMS NIH HHS/United States\\nPT  - Journal Article\\nPT  - Research Support, N.I.H., Extramural\\nPT  - Research Support, Non-U.S. Gov't\\nPT  - Research Support, U.S. Gov't, Non-P.H.S.\\nDEP - 20150907\\nPL  - England\\nTA  - Nature\\nJT  - Nature\\nJID - 0410462\\nRN  - 0 (Multiprotein Complexes)\\nSB  - IM\\nCIN - Nat Methods. 2015 Nov;12(11):1011. PMID: 26824107\\nCIN - Trends Biochem Sci. 2016 Jan;41(1):4-6. PMID: 26682499\\nMH  - Animals\\nMH  - Datasets as Topic\\nMH  - *Evolution, Molecular\\nMH  - Humans\\nMH  - Multiprotein Complexes/*chemistry/*metabolism\\nMH  - Protein Interaction Mapping\\nMH  - *Protein Interaction Maps\\nMH  - Reproducibility of Results\\nMH  - Systems Biology\\nMH  - Tandem Mass Spectrometry\\nPMC - PMC5036527\\nMID - NIHMS704400\\nOID - NLM: NIHMS704400\\nOID - NLM: PMC5036527\\nEDAT- 2015/09/08 06:00\\nMHDA- 2015/10/16 06:00\\nCRDT- 2015/09/08 06:00\\nPHST- 2014/12/15 00:00 [received]\\nPHST- 2015/06/30 00:00 [accepted]\\nPHST- 2015/09/08 06:00 [entrez]\\nPHST- 2015/09/08 06:00 [pubmed]\\nPHST- 2015/10/16 06:00 [medline]\\nAID - nature14877 [pii]\\nAID - 10.1038/nature14877 [doi]\\nPST - ppublish\\nSO  - Nature. 2015 Sep 17;525(7569):339-44. doi: 10.1038/nature14877. Epub 2015 Sep 7.\", \"PMID- 25896417\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20160620\\nLR  - 20150828\\nIS  - 1662-8128 (Electronic)\\nIS  - 1662-811X (Linking)\\nVI  - 7\\nIP  - 5\\nDP  - 2015\\nTI  - VPS13D Gene Variant Is Associated with Altered IL-6 Production and Mortality in\\n      Septic Shock.\\nPG  - 545-53\\nLID - 10.1159/000381265 [doi]\\nAB  - BACKGROUND: Genetic variations contribute to septic shock mortality. To discover \\n      a novel locus, we performed in vitro genome-wide association studies (GWAS) and\\n      further tested the result in a cohort of septic shock patients. METHODS: Two in\\n      vitro GWAS using a quantitative trait locus analysis of stimulated IL-6\\n      production in lymphoblastoid cells from 60 individuals of European ancestry were \\n      performed. VPS13D rs6685273 was genotyped in European ancestry patients (n =\\n      498). The VPS13D gene was silenced in vitro. RESULTS: Two GWAS using\\n      lymphoblastoid cells identified the locus of VPS13D rs6685273 that was\\n      significant in the same direction in both GWAS. The VPS13D rs6685273 C allele was\\n      associated with increased IL-6 production. Patients with septic shock who had the\\n      VPS13D rs6685273 CC genotype had an increased 28-day mortality (p = 0.023) and\\n      more organ failure (p < 0.05) compared to the CT/TT genotypes. VPS13D in vitro\\n      gene silencing in the HeLa cell line increased IL-6 production. Furthermore, the \\n      rs6685273 genotype was associated with differential VPS13D splice variant\\n      expression. CONCLUSIONS: The VPS13D rs6685273 C allele was associated with\\n      increased IL-6 production in vitro. The patients with the VPS13D rs6685273 CC\\n      genotype had increased 28-day mortality and increased organ failure. VPS13D\\n      appears to regulate IL-6 production.\\nCI  - (c) 2015 S. Karger AG, Basel.\\nFAU - Nakada, Taka-aki\\nAU  - Nakada TA\\nAD  - Critical Care Research Laboratories, Centre for Heart Lung Innovation, St. Paul's\\n      Hospital, University of British Columbia, Vancouver, B.C., Canada.\\nFAU - Boyd, John H\\nAU  - Boyd JH\\nFAU - Russell, James A\\nAU  - Russell JA\\nFAU - Aguirre-Hernandez, Rosalia\\nAU  - Aguirre-Hernandez R\\nFAU - Wilkinson, Mark D\\nAU  - Wilkinson MD\\nFAU - Thair, Simone A\\nAU  - Thair SA\\nFAU - Nakada, Emiri\\nAU  - Nakada E\\nFAU - McConechy, Melissa K\\nAU  - McConechy MK\\nFAU - Fjell, Christopher D\\nAU  - Fjell CD\\nFAU - Walley, Keith R\\nAU  - Walley KR\\nLA  - eng\\nGR  - Canadian Institutes of Health Research/Canada\\nPT  - Journal Article\\nPT  - Multicenter Study\\nPT  - Randomized Controlled Trial\\nPT  - Research Support, Non-U.S. Gov't\\nDEP - 20150417\\nPL  - Switzerland\\nTA  - J Innate Immun\\nJT  - Journal of innate immunity\\nJID - 101469471\\nRN  - 0 (Interleukin-6)\\nRN  - 0 (Proteins)\\nRN  - 0 (VPS13D protein, human)\\nSB  - IM\\nMH  - Aged\\nMH  - Cohort Studies\\nMH  - European Continental Ancestry Group/genetics\\nMH  - Female\\nMH  - Gene Frequency\\nMH  - Genetic Predisposition to Disease\\nMH  - Genome-Wide Association Study\\nMH  - Genotype\\nMH  - HeLa Cells\\nMH  - Humans\\nMH  - Interleukin-6/genetics/*metabolism\\nMH  - Lymphocyte Activation/genetics\\nMH  - Lymphocytes/*physiology\\nMH  - Male\\nMH  - Middle Aged\\nMH  - Polymorphism, Single Nucleotide\\nMH  - Proteins/*genetics\\nMH  - Shock, Septic/*genetics/immunology/*mortality\\nMH  - Spain\\nMH  - Survival Analysis\\nEDAT- 2015/04/22 06:00\\nMHDA- 2016/06/21 06:00\\nCRDT- 2015/04/22 06:00\\nPHST- 2014/12/17 00:00 [received]\\nPHST- 2015/02/26 00:00 [accepted]\\nPHST- 2015/04/22 06:00 [entrez]\\nPHST- 2015/04/22 06:00 [pubmed]\\nPHST- 2016/06/21 06:00 [medline]\\nAID - 000381265 [pii]\\nAID - 10.1159/000381265 [doi]\\nPST - ppublish\\nSO  - J Innate Immun. 2015;7(5):545-53. doi: 10.1159/000381265. Epub 2015 Apr 17.\", \"PMID- 21182205\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20110407\\nLR  - 20101224\\nIS  - 1615-9861 (Electronic)\\nIS  - 1615-9853 (Linking)\\nVI  - 11\\nIP  - 1\\nDP  - 2011 Jan\\nTI  - Identification of proteins associated with ligand-activated estrogen receptor\\n      alpha in human breast cancer cell nuclei by tandem affinity purification and nano\\n      LC-MS/MS.\\nPG  - 172-9\\nLID - 10.1002/pmic.201000217 [doi]\\nAB  - Estrogen receptor alpha (ER-alpha) is a key mediator of estrogen actions in\\n      breast cancer (BC) cells. Understanding the effects of ligand-activated ER-alpha \\n      in target cells requires identification of the molecular partners acting in\\n      concert with this nuclear receptor to transduce the hormonal signal. We applied\\n      tandem affinity purification (TAP), glycerol gradient centrifugation and MS\\n      analysis to isolate and identify proteins interacting with ligand-activated\\n      ER-alpha in MCF-7 cell nuclei. This led to the identification of 264\\n      ER-associated proteins, whose functions highlight the hinge role of ER-alpha in\\n      the coordination of multiple hormone-regulated nuclear processes in BC cells.\\nCI  - Copyright (c) 2011 WILEY-VCH Verlag GmbH & Co. KGaA, Weinheim.\\nFAU - Tarallo, Roberta\\nAU  - Tarallo R\\nAD  - Department of General Pathology, Second University of Naples, Napoli, Italy.\\nFAU - Bamundo, Angela\\nAU  - Bamundo A\\nFAU - Nassa, Giovanni\\nAU  - Nassa G\\nFAU - Nola, Ernesto\\nAU  - Nola E\\nFAU - Paris, Ornella\\nAU  - Paris O\\nFAU - Ambrosino, Concetta\\nAU  - Ambrosino C\\nFAU - Facchiano, Angelo\\nAU  - Facchiano A\\nFAU - Baumann, Marc\\nAU  - Baumann M\\nFAU - Nyman, Tuula A\\nAU  - Nyman TA\\nFAU - Weisz, Alessandro\\nAU  - Weisz A\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov't\\nDEP - 20101201\\nPL  - Germany\\nTA  - Proteomics\\nJT  - Proteomics\\nJID - 101092707\\nRN  - 0 (Estrogen Receptor alpha)\\nSB  - IM\\nMH  - Breast Neoplasms/*metabolism\\nMH  - Cell Line, Tumor\\nMH  - Chromatography, Affinity\\nMH  - Chromatography, Liquid/*methods\\nMH  - Estrogen Receptor alpha/*metabolism\\nMH  - Female\\nMH  - Humans\\nMH  - Tandem Mass Spectrometry/*methods\\nEDAT- 2010/12/25 06:00\\nMHDA- 2011/04/08 06:00\\nCRDT- 2010/12/25 06:00\\nPHST- 2010/04/02 00:00 [received]\\nPHST- 2010/08/03 00:00 [revised]\\nPHST- 2010/10/04 00:00 [accepted]\\nPHST- 2010/12/25 06:00 [entrez]\\nPHST- 2010/12/25 06:00 [pubmed]\\nPHST- 2011/04/08 06:00 [medline]\\nAID - 10.1002/pmic.201000217 [doi]\\nPST - ppublish\\nSO  - Proteomics. 2011 Jan;11(1):172-9. doi: 10.1002/pmic.201000217. Epub 2010 Dec 1.\", \"PMID- 20610895\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20110308\\nLR  - 20120216\\nIS  - 1880-3873 (Electronic)\\nIS  - 1340-3478 (Linking)\\nVI  - 17\\nIP  - 10\\nDP  - 2010 Oct 27\\nTI  - Identification of evidence suggestive of an association with peripheral arterial \\n      disease at the OSBPL10 locus by genome-wide investigation in the Japanese\\n      population.\\nPG  - 1054-62\\nAB  - AIM: Peripheral arterial disease (PAD) is a common cause of cardiovascular\\n      morbidity and an independent predictor of cardiovascular mortality. However,\\n      little is known about the genetic basis of PAD. To elucidate this, we performed a\\n      two-staged genome-wide association study in Japanese individuals. METHODS: We\\n      initially tested 222,285 single-nucleotide polymorphisms (SNPs). After the first \\n      screening in a panel of 195 PAD cases and 1,358 controls, 2,696 SNPs (1.2%) were \\n      further genotyped in the second screening using another panel of 699 PAD cases\\n      and 1540 controls. In both screenings, controls were subjects affected with some \\n      diseases other than PAD. RESULTS: When analyzed in the combined panel, the\\n      strongest signal of PAD association was observed at rs1902341 in the intron of\\n      OSBPL10 (p=4.7E-7 for trend test; OR=1.31, 95% CI 1.18-1.46). Also, PAD was\\n      modestly associated at several other loci such as rs2554503 in CSMD1 (p=5.7E-5;\\n      OR=1.32, 95% CI 1.15-1.51) or rs235243 in VSP13D (p=0.04; OR=1.18, 95% CI\\n      1.01-1.37). CONCLUSION: Our genome-wide exploration identified suggestive\\n      evidence of PAD association at the OSBPL10 locus. Because the association has not\\n      reached a genome-wide significant level, further replication study is warranted\\n      for verification in the Japanese population.\\nFAU - Koriyama, Hiroshi\\nAU  - Koriyama H\\nAD  - Department of Geriatric Medicine and Nephrology, Osaka University Graduate School\\n      of Medicine, 2-2 Yamada-oka, Suita, Japan.\\nFAU - Nakagami, Hironori\\nAU  - Nakagami H\\nFAU - Katsuya, Tomohiro\\nAU  - Katsuya T\\nFAU - Sugimoto, Ken\\nAU  - Sugimoto K\\nFAU - Yamashita, Hidetoshi\\nAU  - Yamashita H\\nFAU - Takami, Yoichi\\nAU  - Takami Y\\nFAU - Maeda, Shiro\\nAU  - Maeda S\\nFAU - Kubo, Michiaki\\nAU  - Kubo M\\nFAU - Takahashi, Atsushi\\nAU  - Takahashi A\\nFAU - Nakamura, Yusuke\\nAU  - Nakamura Y\\nFAU - Ogihara, Toshio\\nAU  - Ogihara T\\nFAU - Rakugi, Hiromi\\nAU  - Rakugi H\\nFAU - Kaneda, Yasufumi\\nAU  - Kaneda Y\\nFAU - Morishita, Ryuichi\\nAU  - Morishita R\\nLA  - eng\\nPT  - Comparative Study\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov't\\nDEP - 20100702\\nPL  - Japan\\nTA  - J Atheroscler Thromb\\nJT  - Journal of atherosclerosis and thrombosis\\nJID - 9506298\\nRN  - 0 (Receptors, Steroid)\\nRN  - 0 (oxysterol binding protein)\\nSB  - IM\\nMH  - Aged\\nMH  - Asian Continental Ancestry Group/*genetics\\nMH  - Case-Control Studies\\nMH  - Female\\nMH  - Genetic Predisposition to Disease\\nMH  - Genome, Human\\nMH  - *Genome-Wide Association Study\\nMH  - Genotype\\nMH  - Humans\\nMH  - Japan/epidemiology\\nMH  - Linkage Disequilibrium\\nMH  - Male\\nMH  - Middle Aged\\nMH  - Peripheral Vascular Diseases/epidemiology/*genetics\\nMH  - Polymorphism, Single Nucleotide/*genetics\\nMH  - Receptors, Steroid/*genetics\\nEDAT- 2010/07/09 06:00\\nMHDA- 2011/03/09 06:00\\nCRDT- 2010/07/09 06:00\\nPHST- 2010/07/09 06:00 [entrez]\\nPHST- 2010/07/09 06:00 [pubmed]\\nPHST- 2011/03/09 06:00 [medline]\\nAID - JST.JSTAGE/jat/4291 [pii]\\nPST - ppublish\\nSO  - J Atheroscler Thromb. 2010 Oct 27;17(10):1054-62. Epub 2010 Jul 2.\", \"PMID- 20379614\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20101108\\nLR  - 20170318\\nIS  - 1528-3658 (Electronic)\\nIS  - 1076-1551 (Linking)\\nVI  - 16\\nIP  - 7-8\\nDP  - 2010 Jul-Aug\\nTI  - Personalized smoking cessation: interactions between nicotine dose, dependence\\n      and quit-success genotype score.\\nPG  - 247-53\\nLID - 10.2119/molmed.2009.00159 [doi]\\nAB  - Improving and targeting nicotine replacement therapy (NRT) are cost-effective\\n      strategies for reducing adverse health consequences for smokers. Treatment\\n      studies document the efficacy of precessation NRT and support important roles for\\n      level of nicotine dependence and precessation smoking reduction in successful\\n      quitting. However, prior work has not identified the optimal precessation dose or\\n      means for personalizing NRT. Genome-wide association has identified groups of\\n      genomic markers associated with successful quitting, allowing us to develop a\\n      v1.0 \\\"quit-success\\\" genotype score. We now report influences of v1.0 quit-success\\n      genotype score, level of dependence and precessation smoking reduction in a\\n      smoking cessation trial that examined effects of 21 versus 42 mg/24 h\\n      precessation NRT. Four hundred seventy-nine smokers were randomized to 21 or 42\\n      mg NRT, initiated 2 wks prior to target quit dates. We monitored self-reported\\n      abstinence and end-expired air carbon monoxide (CO). Genotyping used Affymetrix\\n      arrays (Santa Clara, CA, USA). The primary outcome was 10-wk continuous smoking\\n      abstinence. NRT dose, level of nicotine dependence and genotype scores displayed \\n      significant interactive effects on successful quitting. Successful abstinence\\n      also was predicted by CO reductions during precessation NRT. These results\\n      document ways in which smoking cessation strategies can be personalized based on \\n      levels of nicotine dependence, genotype scores and CO monitoring. These\\n      assessments, taken together, can help match most smokers with optimal NRT doses\\n      and help rapidly identify some who may be better treated using other methods.\\nFAU - Rose, Jed E\\nAU  - Rose JE\\nAD  - Department of Psychiatry and Center for Nicotine and Smoking Cessation Research, \\n      Duke University, Durham, North Carolina 27705, USA. rose0003@mc.duke.edu\\nFAU - Behm, Frederique M\\nAU  - Behm FM\\nFAU - Drgon, Tomas\\nAU  - Drgon T\\nFAU - Johnson, Catherine\\nAU  - Johnson C\\nFAU - Uhl, George R\\nAU  - Uhl GR\\nLA  - eng\\nSI  - ClinicalTrials.gov/NCT00734617\\nGR  - P50 DA027840/DA/NIDA NIH HHS/United States\\nGR  - Intramural NIH HHS/United States\\nPT  - Journal Article\\nPT  - Randomized Controlled Trial\\nPT  - Research Support, N.I.H., Intramural\\nPT  - Research Support, Non-U.S. Gov't\\nDEP - 20100317\\nPL  - United States\\nTA  - Mol Med\\nJT  - Molecular medicine (Cambridge, Mass.)\\nJID - 9501023\\nRN  - 0 (Delayed-Action Preparations)\\nRN  - 6M3C89ZY6R (Nicotine)\\nRN  - 7U1EE4V452 (Carbon Monoxide)\\nSB  - IM\\nEIN - Mol Med. 2012;18(1):729\\nMH  - Administration, Cutaneous\\nMH  - Adult\\nMH  - Carbon Monoxide/analysis\\nMH  - Delayed-Action Preparations\\nMH  - Double-Blind Method\\nMH  - Female\\nMH  - Genome-Wide Association Study\\nMH  - Genotype\\nMH  - Humans\\nMH  - Logistic Models\\nMH  - Male\\nMH  - Middle Aged\\nMH  - Nicotine/*administration & dosage\\nMH  - Polymorphism, Single Nucleotide\\nMH  - Precision Medicine/*methods\\nMH  - Smoking Cessation/*methods\\nMH  - Tobacco Use Disorder/*drug therapy/*genetics\\nPMC - PMC2896464\\nOID - NLM: PMC2896464\\nEDAT- 2010/04/10 06:00\\nMHDA- 2010/11/09 06:00\\nCRDT- 2010/04/10 06:00\\nPHST- 2009/10/31 00:00 [received]\\nPHST- 2010/03/16 00:00 [accepted]\\nPHST- 2010/04/10 06:00 [entrez]\\nPHST- 2010/04/10 06:00 [pubmed]\\nPHST- 2010/11/09 06:00 [medline]\\nAID - molmed.2009.00159 [pii]\\nAID - 10.2119/molmed.2009.00159 [doi]\\nPST - ppublish\\nSO  - Mol Med. 2010 Jul-Aug;16(7-8):247-53. doi: 10.2119/molmed.2009.00159. Epub 2010\\n      Mar 17.\", \"PMID- 19056867\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20090226\\nLR  - 20170730\\nIS  - 1533-3450 (Electronic)\\nIS  - 1046-6673 (Linking)\\nVI  - 20\\nIP  - 2\\nDP  - 2009 Feb\\nTI  - Large-scale proteomics and phosphoproteomics of urinary exosomes.\\nPG  - 363-79\\nLID - 10.1681/ASN.2008040406 [doi]\\nAB  - Normal human urine contains large numbers of exosomes, which are 40- to 100-nm\\n      vesicles that originate as the internal vesicles in multivesicular bodies from\\n      every renal epithelial cell type facing the urinary space. Here, we used LC-MS/MS\\n      to profile the proteome of human urinary exosomes. Overall, the analysis\\n      identified 1132 proteins unambiguously, including 177 that are represented on the\\n      Online Mendelian Inheritance in Man database of disease-related genes, suggesting\\n      that exosome analysis is a potential approach to discover urinary biomarkers. We \\n      extended the proteomic analysis to phosphoproteomic profiling using neutral loss \\n      scanning, and this yielded multiple novel phosphorylation sites, including\\n      serine-811 in the thiazide-sensitive Na-Cl co-transporter, NCC. To demonstrate\\n      the potential use of exosome analysis to identify a genetic renal disease, we\\n      carried out immunoblotting of exosomes from urine samples of patients with a\\n      clinical diagnosis of Bartter syndrome type I, showing an absence of the\\n      sodium-potassium-chloride co-transporter 2, NKCC2. The proteomic data are\\n      publicly accessible at http://dir.nhlbi.nih.gov/papers/lkem/exosome/.\\nFAU - Gonzales, Patricia A\\nAU  - Gonzales PA\\nAD  - Laboratory of Kidney and Electrolyte Metabolism, National Heart, Lung, and Blood \\n      Institute, National Institutes of Health, Bethesda, MD 20892, USA.\\nFAU - Pisitkun, Trairak\\nAU  - Pisitkun T\\nFAU - Hoffert, Jason D\\nAU  - Hoffert JD\\nFAU - Tchapyjnikov, Dmitry\\nAU  - Tchapyjnikov D\\nFAU - Star, Robert A\\nAU  - Star RA\\nFAU - Kleta, Robert\\nAU  - Kleta R\\nFAU - Wang, Nam Sun\\nAU  - Wang NS\\nFAU - Knepper, Mark A\\nAU  - Knepper MA\\nLA  - eng\\nGR  - Z01 HL001285-21/Intramural NIH HHS/United States\\nGR  - Z99 HL999999/Intramural NIH HHS/United States\\nPT  - Journal Article\\nDEP - 20081203\\nPL  - United States\\nTA  - J Am Soc Nephrol\\nJT  - Journal of the American Society of Nephrology : JASN\\nJID - 9013836\\nRN  - 0 (Phosphoproteins)\\nRN  - 0 (Proteome)\\nRN  - 0 (Thiazides)\\nRN  - EC 3.6.1.- (Adenosine Triphosphatases)\\nSB  - IM\\nMH  - Adenosine Triphosphatases/chemistry\\nMH  - Adult\\nMH  - Bartter Syndrome/urine\\nMH  - Chromatography, Liquid/methods\\nMH  - Exosomes/metabolism\\nMH  - Female\\nMH  - Humans\\nMH  - Male\\nMH  - Mass Spectrometry/methods\\nMH  - Phosphoproteins/chemistry\\nMH  - Phosphorylation\\nMH  - Proteome\\nMH  - Proteomics/*methods\\nMH  - Thiazides/chemistry\\nMH  - *Urine\\nPMC - PMC2637050\\nOID - NLM: PMC2637050\\nEDAT- 2008/12/06 09:00\\nMHDA- 2009/02/27 09:00\\nCRDT- 2008/12/06 09:00\\nPHST- 2008/12/06 09:00 [pubmed]\\nPHST- 2009/02/27 09:00 [medline]\\nPHST- 2008/12/06 09:00 [entrez]\\nAID - ASN.2008040406 [pii]\\nAID - 10.1681/ASN.2008040406 [doi]\\nPST - ppublish\\nSO  - J Am Soc Nephrol. 2009 Feb;20(2):363-79. doi: 10.1681/ASN.2008040406. Epub 2008\\n      Dec 3.\", \"PMID- 16710414\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20060614\\nLR  - 20170922\\nIS  - 1476-4687 (Electronic)\\nIS  - 0028-0836 (Linking)\\nVI  - 441\\nIP  - 7091\\nDP  - 2006 May 18\\nTI  - The DNA sequence and biological annotation of human chromosome 1.\\nPG  - 315-21\\nAB  - The reference sequence for each human chromosome provides the framework for\\n      understanding genome function, variation and evolution. Here we report the\\n      finished sequence and biological annotation of human chromosome 1. Chromosome 1\\n      is gene-dense, with 3,141 genes and 991 pseudogenes, and many coding sequences\\n      overlap. Rearrangements and mutations of chromosome 1 are prevalent in cancer and\\n      many other diseases. Patterns of sequence variation reveal signals of recent\\n      selection in specific genes that may contribute to human fitness, and also in\\n      regions where no function is evident. Fine-scale recombination occurs in hotspots\\n      of varying intensity along the sequence, and is enriched near genes. These and\\n      other studies of human biology and disease encoded within chromosome 1 are made\\n      possible with the highly accurate annotated sequence, as part of the completed\\n      set of chromosome sequences that comprise the reference human genome.\\nFAU - Gregory, S G\\nAU  - Gregory SG\\nAD  - The Wellcome Trust Sanger Institute, The Wellcome Trust Genome Campus, Hinxton,\\n      Cambridgeshire CB10 1SA, UK. sgregory@chg.duhs.duke.edu\\nFAU - Barlow, K F\\nAU  - Barlow KF\\nFAU - McLay, K E\\nAU  - McLay KE\\nFAU - Kaul, R\\nAU  - Kaul R\\nFAU - Swarbreck, D\\nAU  - Swarbreck D\\nFAU - Dunham, A\\nAU  - Dunham A\\nFAU - Scott, C E\\nAU  - Scott CE\\nFAU - Howe, K L\\nAU  - Howe KL\\nFAU - Woodfine, K\\nAU  - Woodfine K\\nFAU - Spencer, C C A\\nAU  - Spencer CC\\nFAU - Jones, M C\\nAU  - Jones MC\\nFAU - Gillson, C\\nAU  - Gillson C\\nFAU - Searle, S\\nAU  - Searle S\\nFAU - Zhou, Y\\nAU  - Zhou Y\\nFAU - Kokocinski, F\\nAU  - Kokocinski F\\nFAU - McDonald, L\\nAU  - McDonald L\\nFAU - Evans, R\\nAU  - Evans R\\nFAU - Phillips, K\\nAU  - Phillips K\\nFAU - Atkinson, A\\nAU  - Atkinson A\\nFAU - Cooper, R\\nAU  - Cooper R\\nFAU - Jones, C\\nAU  - Jones C\\nFAU - Hall, R E\\nAU  - Hall RE\\nFAU - Andrews, T D\\nAU  - Andrews TD\\nFAU - Lloyd, C\\nAU  - Lloyd C\\nFAU - Ainscough, R\\nAU  - Ainscough R\\nFAU - Almeida, J P\\nAU  - Almeida JP\\nFAU - Ambrose, K D\\nAU  - Ambrose KD\\nFAU - Anderson, F\\nAU  - Anderson F\\nFAU - Andrew, R W\\nAU  - Andrew RW\\nFAU - Ashwell, R I S\\nAU  - Ashwell RI\\nFAU - Aubin, K\\nAU  - Aubin K\\nFAU - Babbage, A K\\nAU  - Babbage AK\\nFAU - Bagguley, C L\\nAU  - Bagguley CL\\nFAU - Bailey, J\\nAU  - Bailey J\\nFAU - Beasley, H\\nAU  - Beasley H\\nFAU - Bethel, G\\nAU  - Bethel G\\nFAU - Bird, C P\\nAU  - Bird CP\\nFAU - Bray-Allen, S\\nAU  - Bray-Allen S\\nFAU - Brown, J Y\\nAU  - Brown JY\\nFAU - Brown, A J\\nAU  - Brown AJ\\nFAU - Buckley, D\\nAU  - Buckley D\\nFAU - Burton, J\\nAU  - Burton J\\nFAU - Bye, J\\nAU  - Bye J\\nFAU - Carder, C\\nAU  - Carder C\\nFAU - Chapman, J C\\nAU  - Chapman JC\\nFAU - Clark, S Y\\nAU  - Clark SY\\nFAU - Clarke, G\\nAU  - Clarke G\\nFAU - Clee, C\\nAU  - Clee C\\nFAU - Cobley, V\\nAU  - Cobley V\\nFAU - Collier, R E\\nAU  - Collier RE\\nFAU - Corby, N\\nAU  - Corby N\\nFAU - Coville, G J\\nAU  - Coville GJ\\nFAU - Davies, J\\nAU  - Davies J\\nFAU - Deadman, R\\nAU  - Deadman R\\nFAU - Dunn, M\\nAU  - Dunn M\\nFAU - Earthrowl, M\\nAU  - Earthrowl M\\nFAU - Ellington, A G\\nAU  - Ellington AG\\nFAU - Errington, H\\nAU  - Errington H\\nFAU - Frankish, A\\nAU  - Frankish A\\nFAU - Frankland, J\\nAU  - Frankland J\\nFAU - French, L\\nAU  - French L\\nFAU - Garner, P\\nAU  - Garner P\\nFAU - Garnett, J\\nAU  - Garnett J\\nFAU - Gay, L\\nAU  - Gay L\\nFAU - Ghori, M R J\\nAU  - Ghori MR\\nFAU - Gibson, R\\nAU  - Gibson R\\nFAU - Gilby, L M\\nAU  - Gilby LM\\nFAU - Gillett, W\\nAU  - Gillett W\\nFAU - Glithero, R J\\nAU  - Glithero RJ\\nFAU - Grafham, D V\\nAU  - Grafham DV\\nFAU - Griffiths, C\\nAU  - Griffiths C\\nFAU - Griffiths-Jones, S\\nAU  - Griffiths-Jones S\\nFAU - Grocock, R\\nAU  - Grocock R\\nFAU - Hammond, S\\nAU  - Hammond S\\nFAU - Harrison, E S I\\nAU  - Harrison ES\\nFAU - Hart, E\\nAU  - Hart E\\nFAU - Haugen, E\\nAU  - Haugen E\\nFAU - Heath, P D\\nAU  - Heath PD\\nFAU - Holmes, S\\nAU  - Holmes S\\nFAU - Holt, K\\nAU  - Holt K\\nFAU - Howden, P J\\nAU  - Howden PJ\\nFAU - Hunt, A R\\nAU  - Hunt AR\\nFAU - Hunt, S E\\nAU  - Hunt SE\\nFAU - Hunter, G\\nAU  - Hunter G\\nFAU - Isherwood, J\\nAU  - Isherwood J\\nFAU - James, R\\nAU  - James R\\nFAU - Johnson, C\\nAU  - Johnson C\\nFAU - Johnson, D\\nAU  - Johnson D\\nFAU - Joy, A\\nAU  - Joy A\\nFAU - Kay, M\\nAU  - Kay M\\nFAU - Kershaw, J K\\nAU  - Kershaw JK\\nFAU - Kibukawa, M\\nAU  - Kibukawa M\\nFAU - Kimberley, A M\\nAU  - Kimberley AM\\nFAU - King, A\\nAU  - King A\\nFAU - Knights, A J\\nAU  - Knights AJ\\nFAU - Lad, H\\nAU  - Lad H\\nFAU - Laird, G\\nAU  - Laird G\\nFAU - Lawlor, S\\nAU  - Lawlor S\\nFAU - Leongamornlert, D A\\nAU  - Leongamornlert DA\\nFAU - Lloyd, D M\\nAU  - Lloyd DM\\nFAU - Loveland, J\\nAU  - Loveland J\\nFAU - Lovell, J\\nAU  - Lovell J\\nFAU - Lush, M J\\nAU  - Lush MJ\\nFAU - Lyne, R\\nAU  - Lyne R\\nFAU - Martin, S\\nAU  - Martin S\\nFAU - Mashreghi-Mohammadi, M\\nAU  - Mashreghi-Mohammadi M\\nFAU - Matthews, L\\nAU  - Matthews L\\nFAU - Matthews, N S W\\nAU  - Matthews NS\\nFAU - McLaren, S\\nAU  - McLaren S\\nFAU - Milne, S\\nAU  - Milne S\\nFAU - Mistry, S\\nAU  - Mistry S\\nFAU - Moore, M J F\\nAU  - Moore MJ\\nFAU - Nickerson, T\\nAU  - Nickerson T\\nFAU - O'Dell, C N\\nAU  - O'Dell CN\\nFAU - Oliver, K\\nAU  - Oliver K\\nFAU - Palmeiri, A\\nAU  - Palmeiri A\\nFAU - Palmer, S A\\nAU  - Palmer SA\\nFAU - Parker, A\\nAU  - Parker A\\nFAU - Patel, D\\nAU  - Patel D\\nFAU - Pearce, A V\\nAU  - Pearce AV\\nFAU - Peck, A I\\nAU  - Peck AI\\nFAU - Pelan, S\\nAU  - Pelan S\\nFAU - Phelps, K\\nAU  - Phelps K\\nFAU - Phillimore, B J\\nAU  - Phillimore BJ\\nFAU - Plumb, R\\nAU  - Plumb R\\nFAU - Rajan, J\\nAU  - Rajan J\\nFAU - Raymond, C\\nAU  - Raymond C\\nFAU - Rouse, G\\nAU  - Rouse G\\nFAU - Saenphimmachak, C\\nAU  - Saenphimmachak C\\nFAU - Sehra, H K\\nAU  - Sehra HK\\nFAU - Sheridan, E\\nAU  - Sheridan E\\nFAU - Shownkeen, R\\nAU  - Shownkeen R\\nFAU - Sims, S\\nAU  - Sims S\\nFAU - Skuce, C D\\nAU  - Skuce CD\\nFAU - Smith, M\\nAU  - Smith M\\nFAU - Steward, C\\nAU  - Steward C\\nFAU - Subramanian, S\\nAU  - Subramanian S\\nFAU - Sycamore, N\\nAU  - Sycamore N\\nFAU - Tracey, A\\nAU  - Tracey A\\nFAU - Tromans, A\\nAU  - Tromans A\\nFAU - Van Helmond, Z\\nAU  - Van Helmond Z\\nFAU - Wall, M\\nAU  - Wall M\\nFAU - Wallis, J M\\nAU  - Wallis JM\\nFAU - White, S\\nAU  - White S\\nFAU - Whitehead, S L\\nAU  - Whitehead SL\\nFAU - Wilkinson, J E\\nAU  - Wilkinson JE\\nFAU - Willey, D L\\nAU  - Willey DL\\nFAU - Williams, H\\nAU  - Williams H\\nFAU - Wilming, L\\nAU  - Wilming L\\nFAU - Wray, P W\\nAU  - Wray PW\\nFAU - Wu, Z\\nAU  - Wu Z\\nFAU - Coulson, A\\nAU  - Coulson A\\nFAU - Vaudin, M\\nAU  - Vaudin M\\nFAU - Sulston, J E\\nAU  - Sulston JE\\nFAU - Durbin, R\\nAU  - Durbin R\\nFAU - Hubbard, T\\nAU  - Hubbard T\\nFAU - Wooster, R\\nAU  - Wooster R\\nFAU - Dunham, I\\nAU  - Dunham I\\nFAU - Carter, N P\\nAU  - Carter NP\\nFAU - McVean, G\\nAU  - McVean G\\nFAU - Ross, M T\\nAU  - Ross MT\\nFAU - Harrow, J\\nAU  - Harrow J\\nFAU - Olson, M V\\nAU  - Olson MV\\nFAU - Beck, S\\nAU  - Beck S\\nFAU - Rogers, J\\nAU  - Rogers J\\nFAU - Bentley, D R\\nAU  - Bentley DR\\nFAU - Banerjee, R\\nAU  - Banerjee R\\nFAU - Bryant, S P\\nAU  - Bryant SP\\nFAU - Burford, D C\\nAU  - Burford DC\\nFAU - Burrill, W D H\\nAU  - Burrill WD\\nFAU - Clegg, S M\\nAU  - Clegg SM\\nFAU - Dhami, P\\nAU  - Dhami P\\nFAU - Dovey, O\\nAU  - Dovey O\\nFAU - Faulkner, L M\\nAU  - Faulkner LM\\nFAU - Gribble, S M\\nAU  - Gribble SM\\nFAU - Langford, C F\\nAU  - Langford CF\\nFAU - Pandian, R D\\nAU  - Pandian RD\\nFAU - Porter, K M\\nAU  - Porter KM\\nFAU - Prigmore, E\\nAU  - Prigmore E\\nLA  - eng\\nSI  - RefSeq/NC_000001\\nGR  - G0000107/Medical Research Council/United Kingdom\\nGR  - Wellcome Trust/United Kingdom\\nPT  - Journal Article\\nPT  - Research Support, N.I.H., Extramural\\nPT  - Research Support, Non-U.S. Gov't\\nPL  - England\\nTA  - Nature\\nJT  - Nature\\nJID - 0410462\\nSB  - IM\\nEIN - Nature. 2006 Oct 26;443(7114):1013. Banerjee, R [added]; Bryant, SP [added];\\n      Burford, DC [added]; Burrill, WDH [added]; Clegg, SM [added]; Dhami, P [added];\\n      Dovey, O [added]; Faulkner, LM [added]; Gribble, SM [added]; Langford, CF\\n      [added]; Pandian, RD [added]; Porter, KM [added]; Prigmore, E [added]\\nMH  - Base Sequence\\nMH  - Chromosomes, Human, Pair 1/*genetics\\nMH  - DNA Replication Timing\\nMH  - Disease\\nMH  - Gene Duplication\\nMH  - Genes/genetics\\nMH  - Genetic Variation/genetics\\nMH  - Genomics\\nMH  - Humans\\nMH  - Molecular Sequence Data\\nMH  - Open Reading Frames/genetics\\nMH  - Pseudogenes/genetics\\nMH  - Recombination, Genetic/genetics\\nMH  - Selection, Genetic\\nMH  - Sequence Analysis, DNA\\nEDAT- 2006/05/20 09:00\\nMHDA- 2006/06/15 09:00\\nCRDT- 2006/05/20 09:00\\nPHST- 2005/12/24 00:00 [received]\\nPHST- 2006/03/13 00:00 [accepted]\\nPHST- 2006/05/20 09:00 [pubmed]\\nPHST- 2006/06/15 09:00 [medline]\\nPHST- 2006/05/20 09:00 [entrez]\\nAID - nature04727 [pii]\\nAID - 10.1038/nature04727 [doi]\\nPST - ppublish\\nSO  - Nature. 2006 May 18;441(7091):315-21. doi: 10.1038/nature04727.\", \"PMID- 15498460\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20050124\\nLR  - 20070516\\nIS  - 0888-7543 (Print)\\nIS  - 0888-7543 (Linking)\\nVI  - 84\\nIP  - 3\\nDP  - 2004 Sep\\nTI  - Analysis of the human VPS13 gene family.\\nPG  - 536-49\\nAB  - The gene mutated in chorea-acanthocytosis (CHAC; approved gene symbol VPS13A)\\n      encodes chorein, a protein similar to yeast Vps13p. We detected several similar\\n      putative human proteins by BLAST analysis of chorein. We characterized the\\n      structure of three new genes encoding these CHAC-similar proteins, located on\\n      chromosomes 1p36, 8q22, and 15q21. The most similar gene in yeast to all four\\n      human genes is Vps13, and therefore the human genes were named VPS13A (CHAC,\\n      9q21), VPS13B (8q22), VPS13C (15q21), and VPS13D (1p36). VPS13B has recently been\\n      reported as COH1, altered in Cohen syndrome. For each gene, we describe several\\n      alternative splicing variants; at least two transcripts per gene are major forms.\\n      The expression pattern of these genes is ubiquitous, with some tissue-specific\\n      differences between several transcript variants. Protein sequence comparisons\\n      suggest that intramolecular duplications have played an important role in the\\n      evolution of this gene family.\\nFAU - Velayos-Baeza, Antonio\\nAU  - Velayos-Baeza A\\nAD  - Wellcome Trust Centre for Human Genetics, University of Oxford, Headington, OX3\\n      7BN Oxford, UK.\\nFAU - Vettori, Andrea\\nAU  - Vettori A\\nFAU - Copley, Richard R\\nAU  - Copley RR\\nFAU - Dobson-Stone, Carol\\nAU  - Dobson-Stone C\\nFAU - Monaco, A P\\nAU  - Monaco AP\\nLA  - eng\\nSI  - GENBANK/AJ608769\\nSI  - GENBANK/AJ608770\\nSI  - GENBANK/AJ608771\\nSI  - GENBANK/AJ608772\\nSI  - GENBANK/AJ608773\\nSI  - GENBANK/AJ608774\\nSI  - GENBANK/AJ608775\\nSI  - GENBANK/AJ626859\\nSI  - GENBANK/AJ626860\\nSI  - GENBANK/AJ626861\\nPT  - Comparative Study\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov't\\nPL  - United States\\nTA  - Genomics\\nJT  - Genomics\\nJID - 8800135\\nRN  - 0 (Membrane Proteins)\\nRN  - 0 (Proteins)\\nRN  - 0 (VPS13A protein, human)\\nRN  - 0 (VPS13B protein, human)\\nRN  - 0 (VPS13C protein, human)\\nRN  - 0 (VPS13D protein, human)\\nRN  - 0 (Vesicular Transport Proteins)\\nSB  - IM\\nMH  - Alternative Splicing/*genetics\\nMH  - Amino Acid Sequence\\nMH  - Base Sequence\\nMH  - Blotting, Northern\\nMH  - Evolution, Molecular\\nMH  - Gene Components\\nMH  - Gene Duplication\\nMH  - Gene Expression Profiling\\nMH  - Humans\\nMH  - Membrane Proteins/*genetics\\nMH  - Molecular Sequence Data\\nMH  - Multigene Family/*genetics\\nMH  - Phylogeny\\nMH  - Proteins/*genetics\\nMH  - Reverse Transcriptase Polymerase Chain Reaction\\nMH  - Sequence Alignment\\nMH  - Sequence Analysis, DNA\\nMH  - Sequence Homology\\nMH  - Vesicular Transport Proteins\\nEDAT- 2004/10/23 09:00\\nMHDA- 2005/01/26 09:00\\nCRDT- 2004/10/23 09:00\\nPHST- 2004/02/13 00:00 [received]\\nPHST- 2004/04/19 00:00 [accepted]\\nPHST- 2004/10/23 09:00 [pubmed]\\nPHST- 2005/01/26 09:00 [medline]\\nPHST- 2004/10/23 09:00 [entrez]\\nAID - S0888754304001156 [pii]\\nAID - 10.1016/j.ygeno.2004.04.012 [doi]\\nPST - ppublish\\nSO  - Genomics. 2004 Sep;84(3):536-49. doi: 10.1016/j.ygeno.2004.04.012.\", \"PMID- 15489334\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20041116\\nLR  - 20170219\\nIS  - 1088-9051 (Print)\\nIS  - 1088-9051 (Linking)\\nVI  - 14\\nIP  - 10B\\nDP  - 2004 Oct\\nTI  - The status, quality, and expansion of the NIH full-length cDNA project: the\\n      Mammalian Gene Collection (MGC).\\nPG  - 2121-7\\nAB  - The National Institutes of Health's Mammalian Gene Collection (MGC) project was\\n      designed to generate and sequence a publicly accessible cDNA resource containing \\n      a complete open reading frame (ORF) for every human and mouse gene. The project\\n      initially used a random strategy to select clones from a large number of cDNA\\n      libraries from diverse tissues. Candidate clones were chosen based on 5'-EST\\n      sequences, and then fully sequenced to high accuracy and analyzed by algorithms\\n      developed for this project. Currently, more than 11,000 human and 10,000 mouse\\n      genes are represented in MGC by at least one clone with a full ORF. The random\\n      selection approach is now reaching a saturation point, and a transition to\\n      protocols targeted at the missing transcripts is now required to complete the\\n      mouse and human collections. Comparison of the sequence of the MGC clones to\\n      reference genome sequences reveals that most cDNA clones are of very high\\n      sequence quality, although it is likely that some cDNAs may carry missense\\n      variants as a consequence of experimental artifact, such as PCR, cloning, or\\n      reverse transcriptase errors. Recently, a rat cDNA component was added to the\\n      project, and ongoing frog (Xenopus) and zebrafish (Danio) cDNA projects were\\n      expanded to take advantage of the high-throughput MGC pipeline.\\nFAU - Gerhard, Daniela S\\nAU  - Gerhard DS\\nFAU - Wagner, Lukas\\nAU  - Wagner L\\nFAU - Feingold, Elise A\\nAU  - Feingold EA\\nFAU - Shenmen, Carolyn M\\nAU  - Shenmen CM\\nFAU - Grouse, Lynette H\\nAU  - Grouse LH\\nFAU - Schuler, Greg\\nAU  - Schuler G\\nFAU - Klein, Steven L\\nAU  - Klein SL\\nFAU - Old, Susan\\nAU  - Old S\\nFAU - Rasooly, Rebekah\\nAU  - Rasooly R\\nFAU - Good, Peter\\nAU  - Good P\\nFAU - Guyer, Mark\\nAU  - Guyer M\\nFAU - Peck, Allison M\\nAU  - Peck AM\\nFAU - Derge, Jeffery G\\nAU  - Derge JG\\nFAU - Lipman, David\\nAU  - Lipman D\\nFAU - Collins, Francis S\\nAU  - Collins FS\\nFAU - Jang, Wonhee\\nAU  - Jang W\\nFAU - Sherry, Steven\\nAU  - Sherry S\\nFAU - Feolo, Mike\\nAU  - Feolo M\\nFAU - Misquitta, Leonie\\nAU  - Misquitta L\\nFAU - Lee, Eduardo\\nAU  - Lee E\\nFAU - Rotmistrovsky, Kirill\\nAU  - Rotmistrovsky K\\nFAU - Greenhut, Susan F\\nAU  - Greenhut SF\\nFAU - Schaefer, Carl F\\nAU  - Schaefer CF\\nFAU - Buetow, Kenneth\\nAU  - Buetow K\\nFAU - Bonner, Tom I\\nAU  - Bonner TI\\nFAU - Haussler, David\\nAU  - Haussler D\\nFAU - Kent, Jim\\nAU  - Kent J\\nFAU - Kiekhaus, Mark\\nAU  - Kiekhaus M\\nFAU - Furey, Terry\\nAU  - Furey T\\nFAU - Brent, Michael\\nAU  - Brent M\\nFAU - Prange, Christa\\nAU  - Prange C\\nFAU - Schreiber, Kirsten\\nAU  - Schreiber K\\nFAU - Shapiro, Nicole\\nAU  - Shapiro N\\nFAU - Bhat, Narayan K\\nAU  - Bhat NK\\nFAU - Hopkins, Ralph F\\nAU  - Hopkins RF\\nFAU - Hsie, Florence\\nAU  - Hsie F\\nFAU - Driscoll, Tom\\nAU  - Driscoll T\\nFAU - Soares, M Bento\\nAU  - Soares MB\\nFAU - Casavant, Tom L\\nAU  - Casavant TL\\nFAU - Scheetz, Todd E\\nAU  - Scheetz TE\\nFAU - Brown-stein, Michael J\\nAU  - Brown-stein MJ\\nFAU - Usdin, Ted B\\nAU  - Usdin TB\\nFAU - Toshiyuki, Shiraki\\nAU  - Toshiyuki S\\nFAU - Carninci, Piero\\nAU  - Carninci P\\nFAU - Piao, Yulan\\nAU  - Piao Y\\nFAU - Dudekula, Dawood B\\nAU  - Dudekula DB\\nFAU - Ko, Minoru S H\\nAU  - Ko MS\\nFAU - Kawakami, Koichi\\nAU  - Kawakami K\\nFAU - Suzuki, Yutaka\\nAU  - Suzuki Y\\nFAU - Sugano, Sumio\\nAU  - Sugano S\\nFAU - Gruber, C E\\nAU  - Gruber CE\\nFAU - Smith, M R\\nAU  - Smith MR\\nFAU - Simmons, Blake\\nAU  - Simmons B\\nFAU - Moore, Troy\\nAU  - Moore T\\nFAU - Waterman, Richard\\nAU  - Waterman R\\nFAU - Johnson, Stephen L\\nAU  - Johnson SL\\nFAU - Ruan, Yijun\\nAU  - Ruan Y\\nFAU - Wei, Chia Lin\\nAU  - Wei CL\\nFAU - Mathavan, S\\nAU  - Mathavan S\\nFAU - Gunaratne, Preethi H\\nAU  - Gunaratne PH\\nFAU - Wu, Jiaqian\\nAU  - Wu J\\nFAU - Garcia, Angela M\\nAU  - Garcia AM\\nFAU - Hulyk, Stephen W\\nAU  - Hulyk SW\\nFAU - Fuh, Edwin\\nAU  - Fuh E\\nFAU - Yuan, Ye\\nAU  - Yuan Y\\nFAU - Sneed, Anna\\nAU  - Sneed A\\nFAU - Kowis, Carla\\nAU  - Kowis C\\nFAU - Hodgson, Anne\\nAU  - Hodgson A\\nFAU - Muzny, Donna M\\nAU  - Muzny DM\\nFAU - McPherson, John\\nAU  - McPherson J\\nFAU - Gibbs, Richard A\\nAU  - Gibbs RA\\nFAU - Fahey, Jessica\\nAU  - Fahey J\\nFAU - Helton, Erin\\nAU  - Helton E\\nFAU - Ketteman, Mark\\nAU  - Ketteman M\\nFAU - Madan, Anuradha\\nAU  - Madan A\\nFAU - Rodrigues, Stephanie\\nAU  - Rodrigues S\\nFAU - Sanchez, Amy\\nAU  - Sanchez A\\nFAU - Whiting, Michelle\\nAU  - Whiting M\\nFAU - Madari, Anup\\nAU  - Madari A\\nFAU - Young, Alice C\\nAU  - Young AC\\nFAU - Wetherby, Keith D\\nAU  - Wetherby KD\\nFAU - Granite, Steven J\\nAU  - Granite SJ\\nFAU - Kwong, Peggy N\\nAU  - Kwong PN\\nFAU - Brinkley, Charles P\\nAU  - Brinkley CP\\nFAU - Pearson, Russell L\\nAU  - Pearson RL\\nFAU - Bouffard, Gerard G\\nAU  - Bouffard GG\\nFAU - Blakesly, Robert W\\nAU  - Blakesly RW\\nFAU - Green, Eric D\\nAU  - Green ED\\nFAU - Dickson, Mark C\\nAU  - Dickson MC\\nFAU - Rodriguez, Alex C\\nAU  - Rodriguez AC\\nFAU - Grimwood, Jane\\nAU  - Grimwood J\\nFAU - Schmutz, Jeremy\\nAU  - Schmutz J\\nFAU - Myers, Richard M\\nAU  - Myers RM\\nFAU - Butterfield, Yaron S N\\nAU  - Butterfield YS\\nFAU - Griffith, Malachi\\nAU  - Griffith M\\nFAU - Griffith, Obi L\\nAU  - Griffith OL\\nFAU - Krzywinski, Martin I\\nAU  - Krzywinski MI\\nFAU - Liao, Nancy\\nAU  - Liao N\\nFAU - Morin, Ryan\\nAU  - Morin R\\nFAU - Palmquist, Diana\\nAU  - Palmquist D\\nFAU - Petrescu, Anca S\\nAU  - Petrescu AS\\nFAU - Skalska, Ursula\\nAU  - Skalska U\\nFAU - Smailus, Duane E\\nAU  - Smailus DE\\nFAU - Stott, Jeff M\\nAU  - Stott JM\\nFAU - Schnerch, Angelique\\nAU  - Schnerch A\\nFAU - Schein, Jacqueline E\\nAU  - Schein JE\\nFAU - Jones, Steven J M\\nAU  - Jones SJ\\nFAU - Holt, Robert A\\nAU  - Holt RA\\nFAU - Baross, Agnes\\nAU  - Baross A\\nFAU - Marra, Marco A\\nAU  - Marra MA\\nFAU - Clifton, Sandra\\nAU  - Clifton S\\nFAU - Makowski, Kathryn A\\nAU  - Makowski KA\\nFAU - Bosak, Stephanie\\nAU  - Bosak S\\nFAU - Malek, Joel\\nAU  - Malek J\\nCN  - MGC Project Team\\nLA  - eng\\nGR  - N01-C0-12400/PHS HHS/United States\\nPT  - Comparative Study\\nPT  - Journal Article\\nPT  - Research Support, U.S. Gov't, P.H.S.\\nPL  - United States\\nTA  - Genome Res\\nJT  - Genome research\\nJID - 9518021\\nRN  - 0 (DNA Primers)\\nRN  - 0 (DNA, Complementary)\\nSB  - IM\\nEIN - Genome Res. 2006 Jun;16(6):804. Morrin, Ryan [corrected to Morin, Ryan]\\nMH  - Animals\\nMH  - Cloning, Molecular/*methods\\nMH  - Computational Biology\\nMH  - DNA Primers\\nMH  - *DNA, Complementary/genetics/metabolism\\nMH  - *Gene Library\\nMH  - Humans\\nMH  - Mice\\nMH  - National Institutes of Health (U.S.)\\nMH  - Open Reading Frames/*physiology\\nMH  - Rats\\nMH  - United States\\nMH  - Xenopus laevis/genetics\\nMH  - Zebrafish/genetics\\nPMC - PMC528928\\nOID - NLM: PMC528928\\nEDAT- 2004/10/19 09:00\\nMHDA- 2004/11/17 09:00\\nCRDT- 2004/10/19 09:00\\nPHST- 2004/10/19 09:00 [pubmed]\\nPHST- 2004/11/17 09:00 [medline]\\nPHST- 2004/10/19 09:00 [entrez]\\nAID - 14/10b/2121 [pii]\\nAID - 10.1101/gr.2596504 [doi]\\nPST - ppublish\\nSO  - Genome Res. 2004 Oct;14(10B):2121-7. doi: 10.1101/gr.2596504.\", \"PMID- 14702039\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20040202\\nLR  - 20061115\\nIS  - 1061-4036 (Print)\\nIS  - 1061-4036 (Linking)\\nVI  - 36\\nIP  - 1\\nDP  - 2004 Jan\\nTI  - Complete sequencing and characterization of 21,243 full-length human cDNAs.\\nPG  - 40-5\\nAB  - As a base for human transcriptome and functional genomics, we created the\\n      \\\"full-length long Japan\\\" (FLJ) collection of sequenced human cDNAs. We determined\\n      the entire sequence of 21,243 selected clones and found that 14,490 cDNAs (10,897\\n      clusters) were unique to the FLJ collection. About half of them (5,416) seemed to\\n      be protein-coding. Of those, 1,999 clusters had not been predicted by\\n      computational methods. The distribution of GC content of nonpredicted cDNAs had a\\n      peak at approximately 58% compared with a peak at approximately 42%for predicted \\n      cDNAs. Thus, there seems to be a slight bias against GC-rich transcripts in\\n      current gene prediction procedures. The rest of the cDNAs unique to the FLJ\\n      collection (5,481) contained no obvious open reading frames (ORFs) and thus are\\n      candidate noncoding RNAs. About one-fourth of them (1,378) showed a clear pattern\\n      of splicing. The distribution of GC content of noncoding cDNAs was narrow and had\\n      a peak at approximately 42%, relatively low compared with that of protein-coding \\n      cDNAs.\\nFAU - Ota, Toshio\\nAU  - Ota T\\nAD  - Helix Research Institute, 1532-3 Yana, Kisarazu, Chiba 292-0812, Japan.\\nFAU - Suzuki, Yutaka\\nAU  - Suzuki Y\\nFAU - Nishikawa, Tetsuo\\nAU  - Nishikawa T\\nFAU - Otsuki, Tetsuji\\nAU  - Otsuki T\\nFAU - Sugiyama, Tomoyasu\\nAU  - Sugiyama T\\nFAU - Irie, Ryotaro\\nAU  - Irie R\\nFAU - Wakamatsu, Ai\\nAU  - Wakamatsu A\\nFAU - Hayashi, Koji\\nAU  - Hayashi K\\nFAU - Sato, Hiroyuki\\nAU  - Sato H\\nFAU - Nagai, Keiichi\\nAU  - Nagai K\\nFAU - Kimura, Kouichi\\nAU  - Kimura K\\nFAU - Makita, Hiroshi\\nAU  - Makita H\\nFAU - Sekine, Mitsuo\\nAU  - Sekine M\\nFAU - Obayashi, Masaya\\nAU  - Obayashi M\\nFAU - Nishi, Tatsunari\\nAU  - Nishi T\\nFAU - Shibahara, Toshikazu\\nAU  - Shibahara T\\nFAU - Tanaka, Toshihiro\\nAU  - Tanaka T\\nFAU - Ishii, Shizuko\\nAU  - Ishii S\\nFAU - Yamamoto, Jun-ichi\\nAU  - Yamamoto J\\nFAU - Saito, Kaoru\\nAU  - Saito K\\nFAU - Kawai, Yuri\\nAU  - Kawai Y\\nFAU - Isono, Yuko\\nAU  - Isono Y\\nFAU - Nakamura, Yoshitaka\\nAU  - Nakamura Y\\nFAU - Nagahari, Kenji\\nAU  - Nagahari K\\nFAU - Murakami, Katsuhiko\\nAU  - Murakami K\\nFAU - Yasuda, Tomohiro\\nAU  - Yasuda T\\nFAU - Iwayanagi, Takao\\nAU  - Iwayanagi T\\nFAU - Wagatsuma, Masako\\nAU  - Wagatsuma M\\nFAU - Shiratori, Akiko\\nAU  - Shiratori A\\nFAU - Sudo, Hiroaki\\nAU  - Sudo H\\nFAU - Hosoiri, Takehiko\\nAU  - Hosoiri T\\nFAU - Kaku, Yoshiko\\nAU  - Kaku Y\\nFAU - Kodaira, Hiroyo\\nAU  - Kodaira H\\nFAU - Kondo, Hiroshi\\nAU  - Kondo H\\nFAU - Sugawara, Masanori\\nAU  - Sugawara M\\nFAU - Takahashi, Makiko\\nAU  - Takahashi M\\nFAU - Kanda, Katsuhiro\\nAU  - Kanda K\\nFAU - Yokoi, Takahide\\nAU  - Yokoi T\\nFAU - Furuya, Takako\\nAU  - Furuya T\\nFAU - Kikkawa, Emiko\\nAU  - Kikkawa E\\nFAU - Omura, Yuhi\\nAU  - Omura Y\\nFAU - Abe, Kumi\\nAU  - Abe K\\nFAU - Kamihara, Kumiko\\nAU  - Kamihara K\\nFAU - Katsuta, Naoko\\nAU  - Katsuta N\\nFAU - Sato, Kazuomi\\nAU  - Sato K\\nFAU - Tanikawa, Machiko\\nAU  - Tanikawa M\\nFAU - Yamazaki, Makoto\\nAU  - Yamazaki M\\nFAU - Ninomiya, Ken\\nAU  - Ninomiya K\\nFAU - Ishibashi, Tadashi\\nAU  - Ishibashi T\\nFAU - Yamashita, Hiromichi\\nAU  - Yamashita H\\nFAU - Murakawa, Katsuji\\nAU  - Murakawa K\\nFAU - Fujimori, Kiyoshi\\nAU  - Fujimori K\\nFAU - Tanai, Hiroyuki\\nAU  - Tanai H\\nFAU - Kimata, Manabu\\nAU  - Kimata M\\nFAU - Watanabe, Motoji\\nAU  - Watanabe M\\nFAU - Hiraoka, Susumu\\nAU  - Hiraoka S\\nFAU - Chiba, Yoshiyuki\\nAU  - Chiba Y\\nFAU - Ishida, Shinichi\\nAU  - Ishida S\\nFAU - Ono, Yukio\\nAU  - Ono Y\\nFAU - Takiguchi, Sumiyo\\nAU  - Takiguchi S\\nFAU - Watanabe, Susumu\\nAU  - Watanabe S\\nFAU - Yosida, Makoto\\nAU  - Yosida M\\nFAU - Hotuta, Tomoko\\nAU  - Hotuta T\\nFAU - Kusano, Junko\\nAU  - Kusano J\\nFAU - Kanehori, Keiichi\\nAU  - Kanehori K\\nFAU - Takahashi-Fujii, Asako\\nAU  - Takahashi-Fujii A\\nFAU - Hara, Hiroto\\nAU  - Hara H\\nFAU - Tanase, Tomo-o\\nAU  - Tanase TO\\nFAU - Nomura, Yoshiko\\nAU  - Nomura Y\\nFAU - Togiya, Sakae\\nAU  - Togiya S\\nFAU - Komai, Fukuyo\\nAU  - Komai F\\nFAU - Hara, Reiko\\nAU  - Hara R\\nFAU - Takeuchi, Kazuha\\nAU  - Takeuchi K\\nFAU - Arita, Miho\\nAU  - Arita M\\nFAU - Imose, Nobuyuki\\nAU  - Imose N\\nFAU - Musashino, Kaoru\\nAU  - Musashino K\\nFAU - Yuuki, Hisatsugu\\nAU  - Yuuki H\\nFAU - Oshima, Atsushi\\nAU  - Oshima A\\nFAU - Sasaki, Naokazu\\nAU  - Sasaki N\\nFAU - Aotsuka, Satoshi\\nAU  - Aotsuka S\\nFAU - Yoshikawa, Yoko\\nAU  - Yoshikawa Y\\nFAU - Matsunawa, Hiroshi\\nAU  - Matsunawa H\\nFAU - Ichihara, Tatsuo\\nAU  - Ichihara T\\nFAU - Shiohata, Namiko\\nAU  - Shiohata N\\nFAU - Sano, Sanae\\nAU  - Sano S\\nFAU - Moriya, Shogo\\nAU  - Moriya S\\nFAU - Momiyama, Hiroko\\nAU  - Momiyama H\\nFAU - Satoh, Noriko\\nAU  - Satoh N\\nFAU - Takami, Sachiko\\nAU  - Takami S\\nFAU - Terashima, Yuko\\nAU  - Terashima Y\\nFAU - Suzuki, Osamu\\nAU  - Suzuki O\\nFAU - Nakagawa, Satoshi\\nAU  - Nakagawa S\\nFAU - Senoh, Akihiro\\nAU  - Senoh A\\nFAU - Mizoguchi, Hiroshi\\nAU  - Mizoguchi H\\nFAU - Goto, Yoshihiro\\nAU  - Goto Y\\nFAU - Shimizu, Fumio\\nAU  - Shimizu F\\nFAU - Wakebe, Hirokazu\\nAU  - Wakebe H\\nFAU - Hishigaki, Haretsugu\\nAU  - Hishigaki H\\nFAU - Watanabe, Takeshi\\nAU  - Watanabe T\\nFAU - Sugiyama, Akio\\nAU  - Sugiyama A\\nFAU - Takemoto, Makoto\\nAU  - Takemoto M\\nFAU - Kawakami, Bunsei\\nAU  - Kawakami B\\nFAU - Yamazaki, Masaaki\\nAU  - Yamazaki M\\nFAU - Watanabe, Koji\\nAU  - Watanabe K\\nFAU - Kumagai, Ayako\\nAU  - Kumagai A\\nFAU - Itakura, Shoko\\nAU  - Itakura S\\nFAU - Fukuzumi, Yasuhito\\nAU  - Fukuzumi Y\\nFAU - Fujimori, Yoshifumi\\nAU  - Fujimori Y\\nFAU - Komiyama, Megumi\\nAU  - Komiyama M\\nFAU - Tashiro, Hiroyuki\\nAU  - Tashiro H\\nFAU - Tanigami, Akira\\nAU  - Tanigami A\\nFAU - Fujiwara, Tsutomu\\nAU  - Fujiwara T\\nFAU - Ono, Toshihide\\nAU  - Ono T\\nFAU - Yamada, Katsue\\nAU  - Yamada K\\nFAU - Fujii, Yuka\\nAU  - Fujii Y\\nFAU - Ozaki, Kouichi\\nAU  - Ozaki K\\nFAU - Hirao, Maasa\\nAU  - Hirao M\\nFAU - Ohmori, Yoshihiro\\nAU  - Ohmori Y\\nFAU - Kawabata, Ayako\\nAU  - Kawabata A\\nFAU - Hikiji, Takeshi\\nAU  - Hikiji T\\nFAU - Kobatake, Naoko\\nAU  - Kobatake N\\nFAU - Inagaki, Hiromi\\nAU  - Inagaki H\\nFAU - Ikema, Yasuko\\nAU  - Ikema Y\\nFAU - Okamoto, Sachiko\\nAU  - Okamoto S\\nFAU - Okitani, Rie\\nAU  - Okitani R\\nFAU - Kawakami, Takuma\\nAU  - Kawakami T\\nFAU - Noguchi, Saori\\nAU  - Noguchi S\\nFAU - Itoh, Tomoko\\nAU  - Itoh T\\nFAU - Shigeta, Keiko\\nAU  - Shigeta K\\nFAU - Senba, Tadashi\\nAU  - Senba T\\nFAU - Matsumura, Kyoka\\nAU  - Matsumura K\\nFAU - Nakajima, Yoshie\\nAU  - Nakajima Y\\nFAU - Mizuno, Takae\\nAU  - Mizuno T\\nFAU - Morinaga, Misato\\nAU  - Morinaga M\\nFAU - Sasaki, Masahide\\nAU  - Sasaki M\\nFAU - Togashi, Takushi\\nAU  - Togashi T\\nFAU - Oyama, Masaaki\\nAU  - Oyama M\\nFAU - Hata, Hiroko\\nAU  - Hata H\\nFAU - Watanabe, Manabu\\nAU  - Watanabe M\\nFAU - Komatsu, Takami\\nAU  - Komatsu T\\nFAU - Mizushima-Sugano, Junko\\nAU  - Mizushima-Sugano J\\nFAU - Satoh, Tadashi\\nAU  - Satoh T\\nFAU - Shirai, Yuko\\nAU  - Shirai Y\\nFAU - Takahashi, Yukiko\\nAU  - Takahashi Y\\nFAU - Nakagawa, Kiyomi\\nAU  - Nakagawa K\\nFAU - Okumura, Koji\\nAU  - Okumura K\\nFAU - Nagase, Takahiro\\nAU  - Nagase T\\nFAU - Nomura, Nobuo\\nAU  - Nomura N\\nFAU - Kikuchi, Hisashi\\nAU  - Kikuchi H\\nFAU - Masuho, Yasuhiko\\nAU  - Masuho Y\\nFAU - Yamashita, Riu\\nAU  - Yamashita R\\nFAU - Nakai, Kenta\\nAU  - Nakai K\\nFAU - Yada, Tetsushi\\nAU  - Yada T\\nFAU - Nakamura, Yusuke\\nAU  - Nakamura Y\\nFAU - Ohara, Osamu\\nAU  - Ohara O\\nFAU - Isogai, Takao\\nAU  - Isogai T\\nFAU - Sugano, Sumio\\nAU  - Sugano S\\nLA  - eng\\nPT  - Comparative Study\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov't\\nDEP - 20031221\\nPL  - United States\\nTA  - Nat Genet\\nJT  - Nature genetics\\nJID - 9216904\\nRN  - 0 (DNA, Complementary)\\nRN  - 0 (RNA, Messenger)\\nSB  - IM\\nMH  - Chromosomes, Human, 21-22 and Y\\nMH  - Chromosomes, Human, Pair 20\\nMH  - Computational Biology\\nMH  - *DNA, Complementary\\nMH  - Humans\\nMH  - Open Reading Frames\\nMH  - RNA, Messenger\\nMH  - *Sequence Analysis, DNA\\nEDAT- 2004/01/01 05:00\\nMHDA- 2004/02/03 05:00\\nCRDT- 2004/01/01 05:00\\nPHST- 2003/10/07 00:00 [received]\\nPHST- 2003/12/01 00:00 [accepted]\\nPHST- 2004/01/01 05:00 [pubmed]\\nPHST- 2004/02/03 05:00 [medline]\\nPHST- 2004/01/01 05:00 [entrez]\\nAID - 10.1038/ng1285 [doi]\\nAID - ng1285 [pii]\\nPST - ppublish\\nSO  - Nat Genet. 2004 Jan;36(1):40-5. doi: 10.1038/ng1285. Epub 2003 Dec 21.\", \"PMID- 12477932\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20030121\\nLR  - 20161025\\nIS  - 0027-8424 (Print)\\nIS  - 0027-8424 (Linking)\\nVI  - 99\\nIP  - 26\\nDP  - 2002 Dec 24\\nTI  - Generation and initial analysis of more than 15,000 full-length human and mouse\\n      cDNA sequences.\\nPG  - 16899-903\\nAB  - The National Institutes of Health Mammalian Gene Collection (MGC) Program is a\\n      multiinstitutional effort to identify and sequence a cDNA clone containing a\\n      complete ORF for each human and mouse gene. ESTs were generated from libraries\\n      enriched for full-length cDNAs and analyzed to identify candidate full-ORF\\n      clones, which then were sequenced to high accuracy. The MGC has currently\\n      sequenced and verified the full ORF for a nonredundant set of >9,000 human and\\n      >6,000 mouse genes. Candidate full-ORF clones for an additional 7,800 human and\\n      3,500 mouse genes also have been identified. All MGC sequences and clones are\\n      available without restriction through public databases and clone distribution\\n      networks (see http:mgc.nci.nih.gov).\\nFAU - Strausberg, Robert L\\nAU  - Strausberg RL\\nAD  - National Cancer Institute, Bethessda, MD 20892-2580, USA. rls@nih.gov\\nFAU - Feingold, Elise A\\nAU  - Feingold EA\\nFAU - Grouse, Lynette H\\nAU  - Grouse LH\\nFAU - Derge, Jeffery G\\nAU  - Derge JG\\nFAU - Klausner, Richard D\\nAU  - Klausner RD\\nFAU - Collins, Francis S\\nAU  - Collins FS\\nFAU - Wagner, Lukas\\nAU  - Wagner L\\nFAU - Shenmen, Carolyn M\\nAU  - Shenmen CM\\nFAU - Schuler, Gregory D\\nAU  - Schuler GD\\nFAU - Altschul, Stephen F\\nAU  - Altschul SF\\nFAU - Zeeberg, Barry\\nAU  - Zeeberg B\\nFAU - Buetow, Kenneth H\\nAU  - Buetow KH\\nFAU - Schaefer, Carl F\\nAU  - Schaefer CF\\nFAU - Bhat, Narayan K\\nAU  - Bhat NK\\nFAU - Hopkins, Ralph F\\nAU  - Hopkins RF\\nFAU - Jordan, Heather\\nAU  - Jordan H\\nFAU - Moore, Troy\\nAU  - Moore T\\nFAU - Max, Steve I\\nAU  - Max SI\\nFAU - Wang, Jun\\nAU  - Wang J\\nFAU - Hsieh, Florence\\nAU  - Hsieh F\\nFAU - Diatchenko, Luda\\nAU  - Diatchenko L\\nFAU - Marusina, Kate\\nAU  - Marusina K\\nFAU - Farmer, Andrew A\\nAU  - Farmer AA\\nFAU - Rubin, Gerald M\\nAU  - Rubin GM\\nFAU - Hong, Ling\\nAU  - Hong L\\nFAU - Stapleton, Mark\\nAU  - Stapleton M\\nFAU - Soares, M Bento\\nAU  - Soares MB\\nFAU - Bonaldo, Maria F\\nAU  - Bonaldo MF\\nFAU - Casavant, Tom L\\nAU  - Casavant TL\\nFAU - Scheetz, Todd E\\nAU  - Scheetz TE\\nFAU - Brownstein, Michael J\\nAU  - Brownstein MJ\\nFAU - Usdin, Ted B\\nAU  - Usdin TB\\nFAU - Toshiyuki, Shiraki\\nAU  - Toshiyuki S\\nFAU - Carninci, Piero\\nAU  - Carninci P\\nFAU - Prange, Christa\\nAU  - Prange C\\nFAU - Raha, Sam S\\nAU  - Raha SS\\nFAU - Loquellano, Naomi A\\nAU  - Loquellano NA\\nFAU - Peters, Garrick J\\nAU  - Peters GJ\\nFAU - Abramson, Rick D\\nAU  - Abramson RD\\nFAU - Mullahy, Sara J\\nAU  - Mullahy SJ\\nFAU - Bosak, Stephanie A\\nAU  - Bosak SA\\nFAU - McEwan, Paul J\\nAU  - McEwan PJ\\nFAU - McKernan, Kevin J\\nAU  - McKernan KJ\\nFAU - Malek, Joel A\\nAU  - Malek JA\\nFAU - Gunaratne, Preethi H\\nAU  - Gunaratne PH\\nFAU - Richards, Stephen\\nAU  - Richards S\\nFAU - Worley, Kim C\\nAU  - Worley KC\\nFAU - Hale, Sarah\\nAU  - Hale S\\nFAU - Garcia, Angela M\\nAU  - Garcia AM\\nFAU - Gay, Laura J\\nAU  - Gay LJ\\nFAU - Hulyk, Stephen W\\nAU  - Hulyk SW\\nFAU - Villalon, Debbie K\\nAU  - Villalon DK\\nFAU - Muzny, Donna M\\nAU  - Muzny DM\\nFAU - Sodergren, Erica J\\nAU  - Sodergren EJ\\nFAU - Lu, Xiuhua\\nAU  - Lu X\\nFAU - Gibbs, Richard A\\nAU  - Gibbs RA\\nFAU - Fahey, Jessica\\nAU  - Fahey J\\nFAU - Helton, Erin\\nAU  - Helton E\\nFAU - Ketteman, Mark\\nAU  - Ketteman M\\nFAU - Madan, Anuradha\\nAU  - Madan A\\nFAU - Rodrigues, Stephanie\\nAU  - Rodrigues S\\nFAU - Sanchez, Amy\\nAU  - Sanchez A\\nFAU - Whiting, Michelle\\nAU  - Whiting M\\nFAU - Madan, Anup\\nAU  - Madan A\\nFAU - Young, Alice C\\nAU  - Young AC\\nFAU - Shevchenko, Yuriy\\nAU  - Shevchenko Y\\nFAU - Bouffard, Gerard G\\nAU  - Bouffard GG\\nFAU - Blakesley, Robert W\\nAU  - Blakesley RW\\nFAU - Touchman, Jeffrey W\\nAU  - Touchman JW\\nFAU - Green, Eric D\\nAU  - Green ED\\nFAU - Dickson, Mark C\\nAU  - Dickson MC\\nFAU - Rodriguez, Alex C\\nAU  - Rodriguez AC\\nFAU - Grimwood, Jane\\nAU  - Grimwood J\\nFAU - Schmutz, Jeremy\\nAU  - Schmutz J\\nFAU - Myers, Richard M\\nAU  - Myers RM\\nFAU - Butterfield, Yaron S N\\nAU  - Butterfield YS\\nFAU - Krzywinski, Martin I\\nAU  - Krzywinski MI\\nFAU - Skalska, Ursula\\nAU  - Skalska U\\nFAU - Smailus, Duane E\\nAU  - Smailus DE\\nFAU - Schnerch, Angelique\\nAU  - Schnerch A\\nFAU - Schein, Jacqueline E\\nAU  - Schein JE\\nFAU - Jones, Steven J M\\nAU  - Jones SJ\\nFAU - Marra, Marco A\\nAU  - Marra MA\\nCN  - Mammalian Gene Collection Program Team\\nLA  - eng\\nGR  - U54 HG003273/HG/NHGRI NIH HHS/United States\\nPT  - Journal Article\\nPT  - Research Support, U.S. Gov't, P.H.S.\\nDEP - 20021211\\nPL  - United States\\nTA  - Proc Natl Acad Sci U S A\\nJT  - Proceedings of the National Academy of Sciences of the United States of America\\nJID - 7505876\\nRN  - 0 (DNA, Complementary)\\nSB  - IM\\nMH  - Algorithms\\nMH  - Animals\\nMH  - DNA, Complementary/analysis/*chemistry\\nMH  - Gene Library\\nMH  - Humans\\nMH  - Mice\\nMH  - Open Reading Frames\\nMH  - *Sequence Analysis, DNA\\nPMC - PMC139241\\nOID - NLM: PMC139241\\nEDAT- 2002/12/13 04:00\\nMHDA- 2003/01/22 04:00\\nCRDT- 2002/12/13 04:00\\nPHST- 2002/12/13 04:00 [pubmed]\\nPHST- 2003/01/22 04:00 [medline]\\nPHST- 2002/12/13 04:00 [entrez]\\nAID - 10.1073/pnas.242603899 [doi]\\nAID - 242603899 [pii]\\nPST - ppublish\\nSO  - Proc Natl Acad Sci U S A. 2002 Dec 24;99(26):16899-903. doi:\\n      10.1073/pnas.242603899. Epub 2002 Dec 11.\", \"PMID- 12421765\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20030114\\nLR  - 20140611\\nIS  - 1088-9051 (Print)\\nIS  - 1088-9051 (Linking)\\nVI  - 12\\nIP  - 11\\nDP  - 2002 Nov\\nTI  - Protein-protein interactions between large proteins: two-hybrid screening using a\\n      functionally classified library composed of long cDNAs.\\nPG  - 1773-84\\nAB  - Large proteins have multiple domains that are potentially capable of binding many\\n      kinds of partners. It is conceivable, therefore, that such proteins could\\n      function as an intricate framework of assembly protein complexes. To\\n      comprehensively study protein-protein interactions between large KIAA proteins,\\n      we have constructed a library composed of 1087 KIAA cDNA clones based on prior\\n      functional classifications done in silico. We were guided by two principles that \\n      raise the success rate for detecting interactions per tested combination: we\\n      avoided testing low-probability combinations, and reduced the number of potential\\n      false negatives that arise from the fact that large proteins cannot reliably be\\n      expressed in yeast. The latter was addressed by constructing a cDNA library\\n      comprised of random fragments encoding large proteins. Cytoplasmic domains of\\n      KIAA transmembrane proteins (>1000 amino acids) were used as bait for yeast\\n      two-hybrid screening. Our analyses reveal that several KIAA proteins bearing a\\n      transmembrane region have the capability of binding to other KIAA proteins\\n      containing domains (e.g., PDZ, SH3, rhoGEF, and spectrin) known to be localized\\n      to highly specialized submembranous sites, indicating that they participate in\\n      cellular junction formation, receptor or channel clustering, and intracellular\\n      signaling events. Our representative library should be a very useful resource for\\n      detecting previously unidentified interactions because it complements\\n      conventional expression libraries, which seldom contain large cDNAs.\\nFAU - Nakayama, Manabu\\nAU  - Nakayama M\\nAD  - Department of Human Gene Research, Kazusa DNA Research Institute, Kisarazu, Chiba\\n      292-0818, Japan. nmanabu@kazusa.or.jp\\nFAU - Kikuno, Reiko\\nAU  - Kikuno R\\nFAU - Ohara, Osamu\\nAU  - Ohara O\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov't\\nPL  - United States\\nTA  - Genome Res\\nJT  - Genome research\\nJID - 9518021\\nRN  - 0 (DNA, Complementary)\\nRN  - 0 (Peptides)\\nRN  - 0 (Proteins)\\nSB  - IM\\nMH  - Brain Chemistry/genetics\\nMH  - Cloning, Molecular\\nMH  - Computational Biology/instrumentation/methods\\nMH  - Cytoplasm/chemistry\\nMH  - DNA, Complementary/chemistry/*genetics\\nMH  - *Gene Library\\nMH  - Humans\\nMH  - Leukemia, Myeloid/genetics/pathology\\nMH  - Peptide Mapping/methods\\nMH  - Peptides/genetics/physiology\\nMH  - Protein Interaction Mapping/*methods\\nMH  - Protein Structure, Tertiary/physiology\\nMH  - Proteins/chemistry/genetics/physiology\\nMH  - Software\\nMH  - Tumor Cells, Cultured\\nMH  - *Two-Hybrid System Techniques\\nMH  - src Homology Domains/genetics/physiology\\nPMC - PMC187542\\nOID - NLM: PMC187542\\nEDAT- 2002/11/08 04:00\\nMHDA- 2003/01/15 04:00\\nCRDT- 2002/11/08 04:00\\nPHST- 2002/11/08 04:00 [pubmed]\\nPHST- 2003/01/15 04:00 [medline]\\nPHST- 2002/11/08 04:00 [entrez]\\nAID - 10.1101/gr.406902 [doi]\\nPST - ppublish\\nSO  - Genome Res. 2002 Nov;12(11):1773-84. doi: 10.1101/gr.406902.\", \"PMID- 9455484\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 19980227\\nLR  - 20061115\\nIS  - 1340-2838 (Print)\\nIS  - 1340-2838 (Linking)\\nVI  - 4\\nIP  - 5\\nDP  - 1997 Oct 31\\nTI  - Characterization of cDNA clones in size-fractionated cDNA libraries from human\\n      brain.\\nPG  - 345-9\\nAB  - To evaluate the size-fractionated cDNA libraries of human brain previously\\n      constructed (O. O'hara et al. DNA Research, 4, 53-59, 1997), the occurrence of\\n      chimeric clones and the content of clones with coding potentiality were analyzed \\n      using the randomly sampled clones with insert sizes of 5 to 7 kb. When the\\n      chromosomal location of 30 clones was determined by the radiation-hybrid mapping \\n      method, the map positions assigned from the 3'- and 5'-end sequences separately\\n      were coincident for 29 clones, suggesting that the occurrence of chimeric clones \\n      is at most 1/30. Using 91 clones mapped to chromosome 1, the content of clones\\n      that have the potentiality coding for proteins larger than 100 amino acid\\n      residues was estimated to be approximately 50% (46 out of 91 clones) on the basis\\n      of nucleotide sequence analysis and coding potentiality assay in vitro. No\\n      significant open reading frames were detected in the remaining clones. Although\\n      the clones coding for short peptides may not have been included in the above\\n      estimation, the libraries constructed from the whole brain mRNA fraction appear\\n      to contain a considerable amount of clones corresponding to the 5'-truncated\\n      transcripts in an unprocessed form and/or those with long 3'-untranslated\\n      regions.\\nFAU - Seki, N\\nAU  - Seki N\\nAD  - Kazusa DNA Research Institute, Chiba, Japan. nseki@kazusa.or.jp\\nFAU - Ohira, M\\nAU  - Ohira M\\nFAU - Nagase, T\\nAU  - Nagase T\\nFAU - Ishikawa, K\\nAU  - Ishikawa K\\nFAU - Miyajima, N\\nAU  - Miyajima N\\nFAU - Nakajima, D\\nAU  - Nakajima D\\nFAU - Nomura, N\\nAU  - Nomura N\\nFAU - Ohara, O\\nAU  - Ohara O\\nLA  - eng\\nSI  - GENBANK/AB007913\\nSI  - GENBANK/AB007914\\nSI  - GENBANK/AB007915\\nSI  - GENBANK/AB007916\\nSI  - GENBANK/AB007917\\nSI  - GENBANK/AB007918\\nSI  - GENBANK/AB007919\\nSI  - GENBANK/AB007920\\nSI  - GENBANK/AB007921\\nSI  - GENBANK/AB007922\\nSI  - GENBANK/AB007923\\nSI  - GENBANK/AB007924\\nSI  - GENBANK/AB007925\\nSI  - GENBANK/AB007926\\nSI  - GENBANK/AB007927\\nSI  - GENBANK/AB007928\\nSI  - GENBANK/AB007929\\nSI  - GENBANK/AB007930\\nSI  - GENBANK/AB007931\\nSI  - GENBANK/AB007932\\nSI  - GENBANK/AB007933\\nSI  - GENBANK/AB007934\\nSI  - GENBANK/AB007935\\nSI  - GENBANK/AB007936\\nSI  - GENBANK/AB007937\\nSI  - GENBANK/AB007938\\nSI  - GENBANK/AB007939\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov't\\nPL  - England\\nTA  - DNA Res\\nJT  - DNA research : an international journal for rapid publication of reports on genes\\n      and genomes\\nJID - 9423827\\nRN  - 0 (DNA, Complementary)\\nRN  - 0 (DNAJC6 protein, human)\\nRN  - 0 (HSP40 Heat-Shock Proteins)\\nRN  - 0 (RNA, Messenger)\\nSB  - IM\\nMH  - Animals\\nMH  - Brain Chemistry/*genetics\\nMH  - Chimera\\nMH  - *Chromosome Mapping\\nMH  - *Chromosomes, Human, Pair 1\\nMH  - Cloning, Molecular\\nMH  - DNA, Complementary/*chemistry/genetics/*isolation & purification\\nMH  - Electrophoresis, Polyacrylamide Gel\\nMH  - Gene Library\\nMH  - HSP40 Heat-Shock Proteins\\nMH  - Humans\\nMH  - Open Reading Frames\\nMH  - Polymerase Chain Reaction\\nMH  - RNA, Messenger/genetics\\nMH  - Repetitive Sequences, Nucleic Acid\\nMH  - Sequence Homology, Nucleic Acid\\nMH  - Sequence Tagged Sites\\nMH  - Transcription, Genetic\\nEDAT- 1998/02/10 00:00\\nMHDA- 1998/02/10 00:01\\nCRDT- 1998/02/10 00:00\\nPHST- 1998/02/10 00:00 [pubmed]\\nPHST- 1998/02/10 00:01 [medline]\\nPHST- 1998/02/10 00:00 [entrez]\\nPST - ppublish\\nSO  - DNA Res. 1997 Oct 31;4(5):345-9.\", \"PMID- 9110174\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 19970617\\nLR  - 20161019\\nIS  - 1088-9051 (Print)\\nIS  - 1088-9051 (Linking)\\nVI  - 7\\nIP  - 4\\nDP  - 1997 Apr\\nTI  - Large-scale concatenation cDNA sequencing.\\nPG  - 353-8\\nAB  - A total of 100 kb of DNA derived from 69 individual human brain cDNA clones of\\n      0.7-2.0 kb were sequenced by concatenated cDNA sequencing (CCS), whereby multiple\\n      individual DNA fragments are sequenced simultaneously in a single shotgun\\n      library. The method yielded accurate sequences and a similar efficiency compared \\n      with other shotgun libraries constructed from single DNA fragments (> 20 kb).\\n      Computer analyses were carried out on 65 cDNA clone sequences and their\\n      corresponding end sequences to examine both nucleic acid and amino acid sequence \\n      similarities in the databases. Thirty-seven clones revealed no DNA database\\n      matches, 12 clones generated exact matches (> or = 98% identity), and 16 clones\\n      generated nonexact matches (57%-97% identity) to either known human or other\\n      species genes. Of those 28 matched clones, 8 had corresponding end sequences that\\n      failed to identify similarities. In a protein similarity search, 27 clone\\n      sequences displayed significant matches, whereas only 20 of the end sequences had\\n      matches to known protein sequences. Our data indicate that full-length cDNA\\n      insert sequences provide significantly more nucleic acid and protein sequence\\n      similarity matches than expressed sequence tags (ESTs) for database searching.\\nFAU - Yu, W\\nAU  - Yu W\\nFAU - Andersson, B\\nAU  - Andersson B\\nFAU - Worley, K C\\nAU  - Worley KC\\nFAU - Muzny, D M\\nAU  - Muzny DM\\nFAU - Ding, Y\\nAU  - Ding Y\\nFAU - Liu, W\\nAU  - Liu W\\nFAU - Ricafrente, J Y\\nAU  - Ricafrente JY\\nFAU - Wentland, M A\\nAU  - Wentland MA\\nFAU - Lennon, G\\nAU  - Lennon G\\nFAU - Gibbs, R A\\nAU  - Gibbs RA\\nLA  - eng\\nSI  - GENBANK/AF007128\\nSI  - GENBANK/AF007129\\nSI  - GENBANK/AF007130\\nSI  - GENBANK/AF007131\\nSI  - GENBANK/AF007132\\nSI  - GENBANK/AF007133\\nSI  - GENBANK/AF007134\\nSI  - GENBANK/AF007135\\nSI  - GENBANK/AF007136\\nSI  - GENBANK/AF007137\\nSI  - GENBANK/AF007138\\nSI  - GENBANK/AF007139\\nSI  - GENBANK/AF007140\\nSI  - GENBANK/AF007141\\nSI  - GENBANK/AF007142\\nSI  - GENBANK/AF007143\\nSI  - GENBANK/AF007144\\nSI  - GENBANK/AF007145\\nSI  - GENBANK/AF007146\\nSI  - GENBANK/AF007147\\nSI  - GENBANK/AF007148\\nSI  - GENBANK/AF007149\\nSI  - GENBANK/AF007150\\nSI  - GENBANK/AF007151\\nSI  - GENBANK/AF007152\\nSI  - GENBANK/AF007153\\nSI  - GENBANK/AF007154\\nGR  - F33 HG000210/HG/NHGRI NIH HHS/United States\\nGR  - U54 HG003273/HG/NHGRI NIH HHS/United States\\nGR  - F32 HG000169/HG/NHGRI NIH HHS/United States\\nGR  - P30 HG00210-05/HG/NHGRI NIH HHS/United States\\nGR  - R01 HG00823/HG/NHGRI NIH HHS/United States\\nGR  - 1F32 HG00169-01/HG/NHGRI NIH HHS/United States\\nPT  - Letter\\nPT  - Research Support, U.S. Gov't, Non-P.H.S.\\nPT  - Research Support, U.S. Gov't, P.H.S.\\nPL  - United States\\nTA  - Genome Res\\nJT  - Genome research\\nJID - 9518021\\nRN  - 0 (DNA Transposable Elements)\\nRN  - 0 (DNA, Complementary)\\nRN  - 0 (Proteins)\\nSB  - IM\\nMH  - DNA Transposable Elements\\nMH  - DNA, Complementary/chemistry/*genetics\\nMH  - Databases, Factual\\nMH  - Gene Library\\nMH  - Humans\\nMH  - Molecular Sequence Data\\nMH  - Proteins/chemistry/*genetics\\nMH  - Sequence Alignment/*methods\\nMH  - Sequence Analysis, DNA/*methods\\nMH  - Sequence Homology, Amino Acid\\nMH  - Sequence Homology, Nucleic Acid\\nMH  - Software\\nPMC - PMC139146\\nOID - NLM: PMC139146\\nEDAT- 1997/04/01 00:00\\nMHDA- 1997/04/01 00:01\\nCRDT- 1997/04/01 00:00\\nPHST- 1997/04/01 00:00 [pubmed]\\nPHST- 1997/04/01 00:01 [medline]\\nPHST- 1997/04/01 00:00 [entrez]\\nPST - ppublish\\nSO  - Genome Res. 1997 Apr;7(4):353-8.\", \"PMID- 8619474\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 19960613\\nLR  - 20071114\\nIS  - 0003-2697 (Print)\\nIS  - 0003-2697 (Linking)\\nVI  - 236\\nIP  - 1\\nDP  - 1996 Apr 5\\nTI  - A \\\"double adaptor\\\" method for improved shotgun library construction.\\nPG  - 107-13\\nAB  - The efficiency of shotgun DNA sequencing depends to a great extent on the quality\\n      of the random-subclone libraries used. We here describe a novel \\\"double adaptor\\\" \\n      strategy for efficient construction of high-quality shotgun libraries. In this\\n      method, randomly sheared and end-repaired fragments are ligated to\\n      oligonucleotide adaptors creating 12-base overhangs. Nonphosphorylated\\n      oligonucleotides are used, which prevents formation of adaptor dimers and ensures\\n      efficient ligation of insert to adaptor. The vector is prepared from a modified\\n      M13 vector, by KpnI/PstI digestion followed by ligation to oligonucleotides with \\n      ends complementary to the overhangs created in the digest. These adaptors create \\n      5'-overhangs complementary to those on the inserts. Following annealing of insert\\n      to vector, the DNA is directly used for transformation without a ligation step.\\n      This protocol is robust and shows three- to fivefold higher yield of clones\\n      compared to previous protocols. No chimeric clones can be detected and the\\n      background of clones without an insert is <1%. The procedure is rapid and shows\\n      potential for automation.\\nFAU - Andersson, B\\nAU  - Andersson B\\nAD  - Department of Molecular and Human Genetics, Baylor College of Medicine, One\\n      Baylor Plaza, Houston, Texas, 77030, USA.\\nFAU - Wentland, M A\\nAU  - Wentland MA\\nFAU - Ricafrente, J Y\\nAU  - Ricafrente JY\\nFAU - Liu, W\\nAU  - Liu W\\nFAU - Gibbs, R A\\nAU  - Gibbs RA\\nLA  - eng\\nSI  - GENBANK/AF007128\\nSI  - GENBANK/AF007129\\nSI  - GENBANK/AF007130\\nSI  - GENBANK/AF007131\\nSI  - GENBANK/AF007132\\nSI  - GENBANK/AF007133\\nSI  - GENBANK/AF007134\\nSI  - GENBANK/AF007135\\nSI  - GENBANK/AF007136\\nSI  - GENBANK/AF007137\\nSI  - GENBANK/AF007138\\nSI  - GENBANK/AF007139\\nSI  - GENBANK/AF007140\\nSI  - GENBANK/AF007141\\nSI  - GENBANK/AF007142\\nSI  - GENBANK/AF007143\\nSI  - GENBANK/AF007144\\nSI  - GENBANK/AF007145\\nSI  - GENBANK/AF007146\\nSI  - GENBANK/AF007147\\nSI  - GENBANK/AF007148\\nSI  - GENBANK/AF007149\\nSI  - GENBANK/AF007150\\nSI  - GENBANK/AF007151\\nSI  - GENBANK/AF007152\\nSI  - GENBANK/AF007153\\nSI  - GENBANK/AF007154\\nGR  - R01 HG00823/HG/NHGRI NIH HHS/United States\\nPT  - Journal Article\\nPT  - Research Support, U.S. Gov't, P.H.S.\\nPL  - United States\\nTA  - Anal Biochem\\nJT  - Analytical biochemistry\\nJID - 0370535\\nRN  - 0 (Oligonucleotides)\\nSB  - IM\\nMH  - Base Sequence\\nMH  - Cloning, Molecular/*methods\\nMH  - Genetic Vectors\\nMH  - *Genomic Library\\nMH  - Humans\\nMH  - Molecular Sequence Data\\nMH  - Oligonucleotides/chemistry\\nMH  - Polymerase Chain Reaction\\nMH  - X Chromosome\\nEDAT- 1996/04/05 00:00\\nMHDA- 1996/04/05 00:01\\nCRDT- 1996/04/05 00:00\\nPHST- 1996/04/05 00:00 [pubmed]\\nPHST- 1996/04/05 00:01 [medline]\\nPHST- 1996/04/05 00:00 [entrez]\\nAID - S0003-2697(96)90138-4 [pii]\\nAID - 10.1006/abio.1996.0138 [doi]\\nPST - ppublish\\nSO  - Anal Biochem. 1996 Apr 5;236(1):107-13. doi: 10.1006/abio.1996.0138.\", \"PMID- 26344197\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20151014\\nLR  - 20170220\\nIS  - 1476-4687 (Electronic)\\nIS  - 0028-0836 (Linking)\\nVI  - 525\\nIP  - 7569\\nDP  - 2015 Sep 17\\nTI  - Panorama of ancient metazoan macromolecular complexes.\\nPG  - 339-44\\nLID - 10.1038/nature14877 [doi]\\nAB  - Macromolecular complexes are essential to conserved biological processes, but\\n      their prevalence across animals is unclear. By combining extensive biochemical\\n      fractionation with quantitative mass spectrometry, here we directly examined the \\n      composition of soluble multiprotein complexes among diverse metazoan models.\\n      Using an integrative approach, we generated a draft conservation map consisting\\n      of more than one million putative high-confidence co-complex interactions for\\n      species with fully sequenced genomes that encompasses functional modules present \\n      broadly across all extant animals. Clustering reveals a spectrum of conservation,\\n      ranging from ancient eukaryotic assemblies that have probably served cellular\\n      housekeeping roles for at least one billion years, ancestral complexes that have \\n      accrued contemporary components, and rarer metazoan innovations linked to\\n      multicellularity. We validated these projections by independent co-fractionation \\n      experiments in evolutionarily distant species, affinity purification and\\n      functional analyses. The comprehensiveness, centrality and modularity of these\\n      reconstructed interactomes reflect their fundamental mechanistic importance and\\n      adaptive value to animal cell systems.\\nFAU - Wan, Cuihong\\nAU  - Wan C\\nAD  - Donnelly Centre for Cellular and Biomolecular Research, University of Toronto,\\n      Toronto, Ontario M5S 3E1, Canada.\\nAD  - Center for Systems and Synthetic Biology, Institute for Cellular and Molecular\\n      Biology, University of Texas at Austin, Austin, Texas 78712, USA.\\nFAU - Borgeson, Blake\\nAU  - Borgeson B\\nAD  - Center for Systems and Synthetic Biology, Institute for Cellular and Molecular\\n      Biology, University of Texas at Austin, Austin, Texas 78712, USA.\\nFAU - Phanse, Sadhna\\nAU  - Phanse S\\nAD  - Donnelly Centre for Cellular and Biomolecular Research, University of Toronto,\\n      Toronto, Ontario M5S 3E1, Canada.\\nFAU - Tu, Fan\\nAU  - Tu F\\nAD  - Center for Systems and Synthetic Biology, Institute for Cellular and Molecular\\n      Biology, University of Texas at Austin, Austin, Texas 78712, USA.\\nFAU - Drew, Kevin\\nAU  - Drew K\\nAD  - Center for Systems and Synthetic Biology, Institute for Cellular and Molecular\\n      Biology, University of Texas at Austin, Austin, Texas 78712, USA.\\nFAU - Clark, Greg\\nAU  - Clark G\\nAD  - Department of Medical Biophysics, Toronto, Ontario M5G 1L7, Canada.\\nFAU - Xiong, Xuejian\\nAU  - Xiong X\\nAD  - Department of Molecular Genetics, University of Toronto, Toronto, Ontario M5S\\n      1A8, Canada.\\nAD  - Hospital for Sick Children, Toronto, Ontario M5G 1X8, Canada.\\nFAU - Kagan, Olga\\nAU  - Kagan O\\nAD  - Donnelly Centre for Cellular and Biomolecular Research, University of Toronto,\\n      Toronto, Ontario M5S 3E1, Canada.\\nFAU - Kwan, Julian\\nAU  - Kwan J\\nAD  - Donnelly Centre for Cellular and Biomolecular Research, University of Toronto,\\n      Toronto, Ontario M5S 3E1, Canada.\\nAD  - Department of Molecular Genetics, University of Toronto, Toronto, Ontario M5S\\n      1A8, Canada.\\nFAU - Bezginov, Alexandr\\nAU  - Bezginov A\\nAD  - Department of Medical Biophysics, Toronto, Ontario M5G 1L7, Canada.\\nFAU - Chessman, Kyle\\nAU  - Chessman K\\nAD  - Department of Molecular Genetics, University of Toronto, Toronto, Ontario M5S\\n      1A8, Canada.\\nAD  - Hospital for Sick Children, Toronto, Ontario M5G 1X8, Canada.\\nFAU - Pal, Swati\\nAU  - Pal S\\nAD  - Hospital for Sick Children, Toronto, Ontario M5G 1X8, Canada.\\nFAU - Cromar, Graham\\nAU  - Cromar G\\nAD  - Department of Molecular Genetics, University of Toronto, Toronto, Ontario M5S\\n      1A8, Canada.\\nAD  - Hospital for Sick Children, Toronto, Ontario M5G 1X8, Canada.\\nFAU - Papoulas, Ophelia\\nAU  - Papoulas O\\nAD  - Center for Systems and Synthetic Biology, Institute for Cellular and Molecular\\n      Biology, University of Texas at Austin, Austin, Texas 78712, USA.\\nFAU - Ni, Zuyao\\nAU  - Ni Z\\nAD  - Donnelly Centre for Cellular and Biomolecular Research, University of Toronto,\\n      Toronto, Ontario M5S 3E1, Canada.\\nFAU - Boutz, Daniel R\\nAU  - Boutz DR\\nAD  - Center for Systems and Synthetic Biology, Institute for Cellular and Molecular\\n      Biology, University of Texas at Austin, Austin, Texas 78712, USA.\\nFAU - Stoilova, Snejana\\nAU  - Stoilova S\\nAD  - Donnelly Centre for Cellular and Biomolecular Research, University of Toronto,\\n      Toronto, Ontario M5S 3E1, Canada.\\nFAU - Havugimana, Pierre C\\nAU  - Havugimana PC\\nAD  - Donnelly Centre for Cellular and Biomolecular Research, University of Toronto,\\n      Toronto, Ontario M5S 3E1, Canada.\\nFAU - Guo, Xinghua\\nAU  - Guo X\\nAD  - Donnelly Centre for Cellular and Biomolecular Research, University of Toronto,\\n      Toronto, Ontario M5S 3E1, Canada.\\nFAU - Malty, Ramy H\\nAU  - Malty RH\\nAD  - Department of Biochemistry, University of Regina, Regina, Saskatchewan S4S 0A2,\\n      Canada.\\nFAU - Sarov, Mihail\\nAU  - Sarov M\\nAD  - Max Planck Institute of Molecular Cell Biology and Genetics, 01307 Dresden,\\n      Germany.\\nFAU - Greenblatt, Jack\\nAU  - Greenblatt J\\nAD  - Donnelly Centre for Cellular and Biomolecular Research, University of Toronto,\\n      Toronto, Ontario M5S 3E1, Canada.\\nAD  - Department of Molecular Genetics, University of Toronto, Toronto, Ontario M5S\\n      1A8, Canada.\\nFAU - Babu, Mohan\\nAU  - Babu M\\nAD  - Department of Biochemistry, University of Regina, Regina, Saskatchewan S4S 0A2,\\n      Canada.\\nFAU - Derry, W Brent\\nAU  - Derry WB\\nAD  - Department of Molecular Genetics, University of Toronto, Toronto, Ontario M5S\\n      1A8, Canada.\\nAD  - Hospital for Sick Children, Toronto, Ontario M5G 1X8, Canada.\\nFAU - Tillier, Elisabeth R\\nAU  - Tillier ER\\nAD  - Department of Medical Biophysics, Toronto, Ontario M5G 1L7, Canada.\\nFAU - Wallingford, John B\\nAU  - Wallingford JB\\nAD  - Center for Systems and Synthetic Biology, Institute for Cellular and Molecular\\n      Biology, University of Texas at Austin, Austin, Texas 78712, USA.\\nAD  - Department of Molecular Biosciences, University of Texas at Austin, Austin, Texas\\n      78712, USA.\\nFAU - Parkinson, John\\nAU  - Parkinson J\\nAD  - Department of Molecular Genetics, University of Toronto, Toronto, Ontario M5S\\n      1A8, Canada.\\nAD  - Hospital for Sick Children, Toronto, Ontario M5G 1X8, Canada.\\nFAU - Marcotte, Edward M\\nAU  - Marcotte EM\\nAD  - Center for Systems and Synthetic Biology, Institute for Cellular and Molecular\\n      Biology, University of Texas at Austin, Austin, Texas 78712, USA.\\nAD  - Department of Molecular Biosciences, University of Texas at Austin, Austin, Texas\\n      78712, USA.\\nFAU - Emili, Andrew\\nAU  - Emili A\\nAD  - Donnelly Centre for Cellular and Biomolecular Research, University of Toronto,\\n      Toronto, Ontario M5S 3E1, Canada.\\nAD  - Department of Molecular Genetics, University of Toronto, Toronto, Ontario M5S\\n      1A8, Canada.\\nLA  - eng\\nGR  - F32GM112495/GM/NIGMS NIH HHS/United States\\nGR  - R01 GM076536/GM/NIGMS NIH HHS/United States\\nGR  - DP1 GM106408/GM/NIGMS NIH HHS/United States\\nGR  - R01 GM067779/GM/NIGMS NIH HHS/United States\\nGR  - Canadian Institutes of Health Research/Canada\\nGR  - F32 GM112495/GM/NIGMS NIH HHS/United States\\nPT  - Journal Article\\nPT  - Research Support, N.I.H., Extramural\\nPT  - Research Support, Non-U.S. Gov't\\nPT  - Research Support, U.S. Gov't, Non-P.H.S.\\nDEP - 20150907\\nPL  - England\\nTA  - Nature\\nJT  - Nature\\nJID - 0410462\\nRN  - 0 (Multiprotein Complexes)\\nSB  - IM\\nCIN - Nat Methods. 2015 Nov;12(11):1011. PMID: 26824107\\nCIN - Trends Biochem Sci. 2016 Jan;41(1):4-6. PMID: 26682499\\nMH  - Animals\\nMH  - Datasets as Topic\\nMH  - *Evolution, Molecular\\nMH  - Humans\\nMH  - Multiprotein Complexes/*chemistry/*metabolism\\nMH  - Protein Interaction Mapping\\nMH  - *Protein Interaction Maps\\nMH  - Reproducibility of Results\\nMH  - Systems Biology\\nMH  - Tandem Mass Spectrometry\\nPMC - PMC5036527\\nMID - NIHMS704400\\nOID - NLM: NIHMS704400\\nOID - NLM: PMC5036527\\nEDAT- 2015/09/08 06:00\\nMHDA- 2015/10/16 06:00\\nCRDT- 2015/09/08 06:00\\nPHST- 2014/12/15 00:00 [received]\\nPHST- 2015/06/30 00:00 [accepted]\\nPHST- 2015/09/08 06:00 [entrez]\\nPHST- 2015/09/08 06:00 [pubmed]\\nPHST- 2015/10/16 06:00 [medline]\\nAID - nature14877 [pii]\\nAID - 10.1038/nature14877 [doi]\\nPST - ppublish\\nSO  - Nature. 2015 Sep 17;525(7569):339-44. doi: 10.1038/nature14877. Epub 2015 Sep 7.\", \"PMID- 25896417\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20160620\\nLR  - 20150828\\nIS  - 1662-8128 (Electronic)\\nIS  - 1662-811X (Linking)\\nVI  - 7\\nIP  - 5\\nDP  - 2015\\nTI  - VPS13D Gene Variant Is Associated with Altered IL-6 Production and Mortality in\\n      Septic Shock.\\nPG  - 545-53\\nLID - 10.1159/000381265 [doi]\\nAB  - BACKGROUND: Genetic variations contribute to septic shock mortality. To discover \\n      a novel locus, we performed in vitro genome-wide association studies (GWAS) and\\n      further tested the result in a cohort of septic shock patients. METHODS: Two in\\n      vitro GWAS using a quantitative trait locus analysis of stimulated IL-6\\n      production in lymphoblastoid cells from 60 individuals of European ancestry were \\n      performed. VPS13D rs6685273 was genotyped in European ancestry patients (n =\\n      498). The VPS13D gene was silenced in vitro. RESULTS: Two GWAS using\\n      lymphoblastoid cells identified the locus of VPS13D rs6685273 that was\\n      significant in the same direction in both GWAS. The VPS13D rs6685273 C allele was\\n      associated with increased IL-6 production. Patients with septic shock who had the\\n      VPS13D rs6685273 CC genotype had an increased 28-day mortality (p = 0.023) and\\n      more organ failure (p < 0.05) compared to the CT/TT genotypes. VPS13D in vitro\\n      gene silencing in the HeLa cell line increased IL-6 production. Furthermore, the \\n      rs6685273 genotype was associated with differential VPS13D splice variant\\n      expression. CONCLUSIONS: The VPS13D rs6685273 C allele was associated with\\n      increased IL-6 production in vitro. The patients with the VPS13D rs6685273 CC\\n      genotype had increased 28-day mortality and increased organ failure. VPS13D\\n      appears to regulate IL-6 production.\\nCI  - (c) 2015 S. Karger AG, Basel.\\nFAU - Nakada, Taka-aki\\nAU  - Nakada TA\\nAD  - Critical Care Research Laboratories, Centre for Heart Lung Innovation, St. Paul's\\n      Hospital, University of British Columbia, Vancouver, B.C., Canada.\\nFAU - Boyd, John H\\nAU  - Boyd JH\\nFAU - Russell, James A\\nAU  - Russell JA\\nFAU - Aguirre-Hernandez, Rosalia\\nAU  - Aguirre-Hernandez R\\nFAU - Wilkinson, Mark D\\nAU  - Wilkinson MD\\nFAU - Thair, Simone A\\nAU  - Thair SA\\nFAU - Nakada, Emiri\\nAU  - Nakada E\\nFAU - McConechy, Melissa K\\nAU  - McConechy MK\\nFAU - Fjell, Christopher D\\nAU  - Fjell CD\\nFAU - Walley, Keith R\\nAU  - Walley KR\\nLA  - eng\\nGR  - Canadian Institutes of Health Research/Canada\\nPT  - Journal Article\\nPT  - Multicenter Study\\nPT  - Randomized Controlled Trial\\nPT  - Research Support, Non-U.S. Gov't\\nDEP - 20150417\\nPL  - Switzerland\\nTA  - J Innate Immun\\nJT  - Journal of innate immunity\\nJID - 101469471\\nRN  - 0 (Interleukin-6)\\nRN  - 0 (Proteins)\\nRN  - 0 (VPS13D protein, human)\\nSB  - IM\\nMH  - Aged\\nMH  - Cohort Studies\\nMH  - European Continental Ancestry Group/genetics\\nMH  - Female\\nMH  - Gene Frequency\\nMH  - Genetic Predisposition to Disease\\nMH  - Genome-Wide Association Study\\nMH  - Genotype\\nMH  - HeLa Cells\\nMH  - Humans\\nMH  - Interleukin-6/genetics/*metabolism\\nMH  - Lymphocyte Activation/genetics\\nMH  - Lymphocytes/*physiology\\nMH  - Male\\nMH  - Middle Aged\\nMH  - Polymorphism, Single Nucleotide\\nMH  - Proteins/*genetics\\nMH  - Shock, Septic/*genetics/immunology/*mortality\\nMH  - Spain\\nMH  - Survival Analysis\\nEDAT- 2015/04/22 06:00\\nMHDA- 2016/06/21 06:00\\nCRDT- 2015/04/22 06:00\\nPHST- 2014/12/17 00:00 [received]\\nPHST- 2015/02/26 00:00 [accepted]\\nPHST- 2015/04/22 06:00 [entrez]\\nPHST- 2015/04/22 06:00 [pubmed]\\nPHST- 2016/06/21 06:00 [medline]\\nAID - 000381265 [pii]\\nAID - 10.1159/000381265 [doi]\\nPST - ppublish\\nSO  - J Innate Immun. 2015;7(5):545-53. doi: 10.1159/000381265. Epub 2015 Apr 17.\", \"PMID- 21182205\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20110407\\nLR  - 20101224\\nIS  - 1615-9861 (Electronic)\\nIS  - 1615-9853 (Linking)\\nVI  - 11\\nIP  - 1\\nDP  - 2011 Jan\\nTI  - Identification of proteins associated with ligand-activated estrogen receptor\\n      alpha in human breast cancer cell nuclei by tandem affinity purification and nano\\n      LC-MS/MS.\\nPG  - 172-9\\nLID - 10.1002/pmic.201000217 [doi]\\nAB  - Estrogen receptor alpha (ER-alpha) is a key mediator of estrogen actions in\\n      breast cancer (BC) cells. Understanding the effects of ligand-activated ER-alpha \\n      in target cells requires identification of the molecular partners acting in\\n      concert with this nuclear receptor to transduce the hormonal signal. We applied\\n      tandem affinity purification (TAP), glycerol gradient centrifugation and MS\\n      analysis to isolate and identify proteins interacting with ligand-activated\\n      ER-alpha in MCF-7 cell nuclei. This led to the identification of 264\\n      ER-associated proteins, whose functions highlight the hinge role of ER-alpha in\\n      the coordination of multiple hormone-regulated nuclear processes in BC cells.\\nCI  - Copyright (c) 2011 WILEY-VCH Verlag GmbH & Co. KGaA, Weinheim.\\nFAU - Tarallo, Roberta\\nAU  - Tarallo R\\nAD  - Department of General Pathology, Second University of Naples, Napoli, Italy.\\nFAU - Bamundo, Angela\\nAU  - Bamundo A\\nFAU - Nassa, Giovanni\\nAU  - Nassa G\\nFAU - Nola, Ernesto\\nAU  - Nola E\\nFAU - Paris, Ornella\\nAU  - Paris O\\nFAU - Ambrosino, Concetta\\nAU  - Ambrosino C\\nFAU - Facchiano, Angelo\\nAU  - Facchiano A\\nFAU - Baumann, Marc\\nAU  - Baumann M\\nFAU - Nyman, Tuula A\\nAU  - Nyman TA\\nFAU - Weisz, Alessandro\\nAU  - Weisz A\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov't\\nDEP - 20101201\\nPL  - Germany\\nTA  - Proteomics\\nJT  - Proteomics\\nJID - 101092707\\nRN  - 0 (Estrogen Receptor alpha)\\nSB  - IM\\nMH  - Breast Neoplasms/*metabolism\\nMH  - Cell Line, Tumor\\nMH  - Chromatography, Affinity\\nMH  - Chromatography, Liquid/*methods\\nMH  - Estrogen Receptor alpha/*metabolism\\nMH  - Female\\nMH  - Humans\\nMH  - Tandem Mass Spectrometry/*methods\\nEDAT- 2010/12/25 06:00\\nMHDA- 2011/04/08 06:00\\nCRDT- 2010/12/25 06:00\\nPHST- 2010/04/02 00:00 [received]\\nPHST- 2010/08/03 00:00 [revised]\\nPHST- 2010/10/04 00:00 [accepted]\\nPHST- 2010/12/25 06:00 [entrez]\\nPHST- 2010/12/25 06:00 [pubmed]\\nPHST- 2011/04/08 06:00 [medline]\\nAID - 10.1002/pmic.201000217 [doi]\\nPST - ppublish\\nSO  - Proteomics. 2011 Jan;11(1):172-9. doi: 10.1002/pmic.201000217. Epub 2010 Dec 1.\", \"PMID- 20610895\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20110308\\nLR  - 20120216\\nIS  - 1880-3873 (Electronic)\\nIS  - 1340-3478 (Linking)\\nVI  - 17\\nIP  - 10\\nDP  - 2010 Oct 27\\nTI  - Identification of evidence suggestive of an association with peripheral arterial \\n      disease at the OSBPL10 locus by genome-wide investigation in the Japanese\\n      population.\\nPG  - 1054-62\\nAB  - AIM: Peripheral arterial disease (PAD) is a common cause of cardiovascular\\n      morbidity and an independent predictor of cardiovascular mortality. However,\\n      little is known about the genetic basis of PAD. To elucidate this, we performed a\\n      two-staged genome-wide association study in Japanese individuals. METHODS: We\\n      initially tested 222,285 single-nucleotide polymorphisms (SNPs). After the first \\n      screening in a panel of 195 PAD cases and 1,358 controls, 2,696 SNPs (1.2%) were \\n      further genotyped in the second screening using another panel of 699 PAD cases\\n      and 1540 controls. In both screenings, controls were subjects affected with some \\n      diseases other than PAD. RESULTS: When analyzed in the combined panel, the\\n      strongest signal of PAD association was observed at rs1902341 in the intron of\\n      OSBPL10 (p=4.7E-7 for trend test; OR=1.31, 95% CI 1.18-1.46). Also, PAD was\\n      modestly associated at several other loci such as rs2554503 in CSMD1 (p=5.7E-5;\\n      OR=1.32, 95% CI 1.15-1.51) or rs235243 in VSP13D (p=0.04; OR=1.18, 95% CI\\n      1.01-1.37). CONCLUSION: Our genome-wide exploration identified suggestive\\n      evidence of PAD association at the OSBPL10 locus. Because the association has not\\n      reached a genome-wide significant level, further replication study is warranted\\n      for verification in the Japanese population.\\nFAU - Koriyama, Hiroshi\\nAU  - Koriyama H\\nAD  - Department of Geriatric Medicine and Nephrology, Osaka University Graduate School\\n      of Medicine, 2-2 Yamada-oka, Suita, Japan.\\nFAU - Nakagami, Hironori\\nAU  - Nakagami H\\nFAU - Katsuya, Tomohiro\\nAU  - Katsuya T\\nFAU - Sugimoto, Ken\\nAU  - Sugimoto K\\nFAU - Yamashita, Hidetoshi\\nAU  - Yamashita H\\nFAU - Takami, Yoichi\\nAU  - Takami Y\\nFAU - Maeda, Shiro\\nAU  - Maeda S\\nFAU - Kubo, Michiaki\\nAU  - Kubo M\\nFAU - Takahashi, Atsushi\\nAU  - Takahashi A\\nFAU - Nakamura, Yusuke\\nAU  - Nakamura Y\\nFAU - Ogihara, Toshio\\nAU  - Ogihara T\\nFAU - Rakugi, Hiromi\\nAU  - Rakugi H\\nFAU - Kaneda, Yasufumi\\nAU  - Kaneda Y\\nFAU - Morishita, Ryuichi\\nAU  - Morishita R\\nLA  - eng\\nPT  - Comparative Study\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov't\\nDEP - 20100702\\nPL  - Japan\\nTA  - J Atheroscler Thromb\\nJT  - Journal of atherosclerosis and thrombosis\\nJID - 9506298\\nRN  - 0 (Receptors, Steroid)\\nRN  - 0 (oxysterol binding protein)\\nSB  - IM\\nMH  - Aged\\nMH  - Asian Continental Ancestry Group/*genetics\\nMH  - Case-Control Studies\\nMH  - Female\\nMH  - Genetic Predisposition to Disease\\nMH  - Genome, Human\\nMH  - *Genome-Wide Association Study\\nMH  - Genotype\\nMH  - Humans\\nMH  - Japan/epidemiology\\nMH  - Linkage Disequilibrium\\nMH  - Male\\nMH  - Middle Aged\\nMH  - Peripheral Vascular Diseases/epidemiology/*genetics\\nMH  - Polymorphism, Single Nucleotide/*genetics\\nMH  - Receptors, Steroid/*genetics\\nEDAT- 2010/07/09 06:00\\nMHDA- 2011/03/09 06:00\\nCRDT- 2010/07/09 06:00\\nPHST- 2010/07/09 06:00 [entrez]\\nPHST- 2010/07/09 06:00 [pubmed]\\nPHST- 2011/03/09 06:00 [medline]\\nAID - JST.JSTAGE/jat/4291 [pii]\\nPST - ppublish\\nSO  - J Atheroscler Thromb. 2010 Oct 27;17(10):1054-62. Epub 2010 Jul 2.\", \"PMID- 20379614\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20101108\\nLR  - 20170318\\nIS  - 1528-3658 (Electronic)\\nIS  - 1076-1551 (Linking)\\nVI  - 16\\nIP  - 7-8\\nDP  - 2010 Jul-Aug\\nTI  - Personalized smoking cessation: interactions between nicotine dose, dependence\\n      and quit-success genotype score.\\nPG  - 247-53\\nLID - 10.2119/molmed.2009.00159 [doi]\\nAB  - Improving and targeting nicotine replacement therapy (NRT) are cost-effective\\n      strategies for reducing adverse health consequences for smokers. Treatment\\n      studies document the efficacy of precessation NRT and support important roles for\\n      level of nicotine dependence and precessation smoking reduction in successful\\n      quitting. However, prior work has not identified the optimal precessation dose or\\n      means for personalizing NRT. Genome-wide association has identified groups of\\n      genomic markers associated with successful quitting, allowing us to develop a\\n      v1.0 \\\"quit-success\\\" genotype score. We now report influences of v1.0 quit-success\\n      genotype score, level of dependence and precessation smoking reduction in a\\n      smoking cessation trial that examined effects of 21 versus 42 mg/24 h\\n      precessation NRT. Four hundred seventy-nine smokers were randomized to 21 or 42\\n      mg NRT, initiated 2 wks prior to target quit dates. We monitored self-reported\\n      abstinence and end-expired air carbon monoxide (CO). Genotyping used Affymetrix\\n      arrays (Santa Clara, CA, USA). The primary outcome was 10-wk continuous smoking\\n      abstinence. NRT dose, level of nicotine dependence and genotype scores displayed \\n      significant interactive effects on successful quitting. Successful abstinence\\n      also was predicted by CO reductions during precessation NRT. These results\\n      document ways in which smoking cessation strategies can be personalized based on \\n      levels of nicotine dependence, genotype scores and CO monitoring. These\\n      assessments, taken together, can help match most smokers with optimal NRT doses\\n      and help rapidly identify some who may be better treated using other methods.\\nFAU - Rose, Jed E\\nAU  - Rose JE\\nAD  - Department of Psychiatry and Center for Nicotine and Smoking Cessation Research, \\n      Duke University, Durham, North Carolina 27705, USA. rose0003@mc.duke.edu\\nFAU - Behm, Frederique M\\nAU  - Behm FM\\nFAU - Drgon, Tomas\\nAU  - Drgon T\\nFAU - Johnson, Catherine\\nAU  - Johnson C\\nFAU - Uhl, George R\\nAU  - Uhl GR\\nLA  - eng\\nSI  - ClinicalTrials.gov/NCT00734617\\nGR  - P50 DA027840/DA/NIDA NIH HHS/United States\\nGR  - Intramural NIH HHS/United States\\nPT  - Journal Article\\nPT  - Randomized Controlled Trial\\nPT  - Research Support, N.I.H., Intramural\\nPT  - Research Support, Non-U.S. Gov't\\nDEP - 20100317\\nPL  - United States\\nTA  - Mol Med\\nJT  - Molecular medicine (Cambridge, Mass.)\\nJID - 9501023\\nRN  - 0 (Delayed-Action Preparations)\\nRN  - 6M3C89ZY6R (Nicotine)\\nRN  - 7U1EE4V452 (Carbon Monoxide)\\nSB  - IM\\nEIN - Mol Med. 2012;18(1):729\\nMH  - Administration, Cutaneous\\nMH  - Adult\\nMH  - Carbon Monoxide/analysis\\nMH  - Delayed-Action Preparations\\nMH  - Double-Blind Method\\nMH  - Female\\nMH  - Genome-Wide Association Study\\nMH  - Genotype\\nMH  - Humans\\nMH  - Logistic Models\\nMH  - Male\\nMH  - Middle Aged\\nMH  - Nicotine/*administration & dosage\\nMH  - Polymorphism, Single Nucleotide\\nMH  - Precision Medicine/*methods\\nMH  - Smoking Cessation/*methods\\nMH  - Tobacco Use Disorder/*drug therapy/*genetics\\nPMC - PMC2896464\\nOID - NLM: PMC2896464\\nEDAT- 2010/04/10 06:00\\nMHDA- 2010/11/09 06:00\\nCRDT- 2010/04/10 06:00\\nPHST- 2009/10/31 00:00 [received]\\nPHST- 2010/03/16 00:00 [accepted]\\nPHST- 2010/04/10 06:00 [entrez]\\nPHST- 2010/04/10 06:00 [pubmed]\\nPHST- 2010/11/09 06:00 [medline]\\nAID - molmed.2009.00159 [pii]\\nAID - 10.2119/molmed.2009.00159 [doi]\\nPST - ppublish\\nSO  - Mol Med. 2010 Jul-Aug;16(7-8):247-53. doi: 10.2119/molmed.2009.00159. Epub 2010\\n      Mar 17.\", \"PMID- 19056867\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20090226\\nLR  - 20170730\\nIS  - 1533-3450 (Electronic)\\nIS  - 1046-6673 (Linking)\\nVI  - 20\\nIP  - 2\\nDP  - 2009 Feb\\nTI  - Large-scale proteomics and phosphoproteomics of urinary exosomes.\\nPG  - 363-79\\nLID - 10.1681/ASN.2008040406 [doi]\\nAB  - Normal human urine contains large numbers of exosomes, which are 40- to 100-nm\\n      vesicles that originate as the internal vesicles in multivesicular bodies from\\n      every renal epithelial cell type facing the urinary space. Here, we used LC-MS/MS\\n      to profile the proteome of human urinary exosomes. Overall, the analysis\\n      identified 1132 proteins unambiguously, including 177 that are represented on the\\n      Online Mendelian Inheritance in Man database of disease-related genes, suggesting\\n      that exosome analysis is a potential approach to discover urinary biomarkers. We \\n      extended the proteomic analysis to phosphoproteomic profiling using neutral loss \\n      scanning, and this yielded multiple novel phosphorylation sites, including\\n      serine-811 in the thiazide-sensitive Na-Cl co-transporter, NCC. To demonstrate\\n      the potential use of exosome analysis to identify a genetic renal disease, we\\n      carried out immunoblotting of exosomes from urine samples of patients with a\\n      clinical diagnosis of Bartter syndrome type I, showing an absence of the\\n      sodium-potassium-chloride co-transporter 2, NKCC2. The proteomic data are\\n      publicly accessible at http://dir.nhlbi.nih.gov/papers/lkem/exosome/.\\nFAU - Gonzales, Patricia A\\nAU  - Gonzales PA\\nAD  - Laboratory of Kidney and Electrolyte Metabolism, National Heart, Lung, and Blood \\n      Institute, National Institutes of Health, Bethesda, MD 20892, USA.\\nFAU - Pisitkun, Trairak\\nAU  - Pisitkun T\\nFAU - Hoffert, Jason D\\nAU  - Hoffert JD\\nFAU - Tchapyjnikov, Dmitry\\nAU  - Tchapyjnikov D\\nFAU - Star, Robert A\\nAU  - Star RA\\nFAU - Kleta, Robert\\nAU  - Kleta R\\nFAU - Wang, Nam Sun\\nAU  - Wang NS\\nFAU - Knepper, Mark A\\nAU  - Knepper MA\\nLA  - eng\\nGR  - Z01 HL001285-21/Intramural NIH HHS/United States\\nGR  - Z99 HL999999/Intramural NIH HHS/United States\\nPT  - Journal Article\\nDEP - 20081203\\nPL  - United States\\nTA  - J Am Soc Nephrol\\nJT  - Journal of the American Society of Nephrology : JASN\\nJID - 9013836\\nRN  - 0 (Phosphoproteins)\\nRN  - 0 (Proteome)\\nRN  - 0 (Thiazides)\\nRN  - EC 3.6.1.- (Adenosine Triphosphatases)\\nSB  - IM\\nMH  - Adenosine Triphosphatases/chemistry\\nMH  - Adult\\nMH  - Bartter Syndrome/urine\\nMH  - Chromatography, Liquid/methods\\nMH  - Exosomes/metabolism\\nMH  - Female\\nMH  - Humans\\nMH  - Male\\nMH  - Mass Spectrometry/methods\\nMH  - Phosphoproteins/chemistry\\nMH  - Phosphorylation\\nMH  - Proteome\\nMH  - Proteomics/*methods\\nMH  - Thiazides/chemistry\\nMH  - *Urine\\nPMC - PMC2637050\\nOID - NLM: PMC2637050\\nEDAT- 2008/12/06 09:00\\nMHDA- 2009/02/27 09:00\\nCRDT- 2008/12/06 09:00\\nPHST- 2008/12/06 09:00 [pubmed]\\nPHST- 2009/02/27 09:00 [medline]\\nPHST- 2008/12/06 09:00 [entrez]\\nAID - ASN.2008040406 [pii]\\nAID - 10.1681/ASN.2008040406 [doi]\\nPST - ppublish\\nSO  - J Am Soc Nephrol. 2009 Feb;20(2):363-79. doi: 10.1681/ASN.2008040406. Epub 2008\\n      Dec 3.\", \"PMID- 16710414\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20060614\\nLR  - 20170922\\nIS  - 1476-4687 (Electronic)\\nIS  - 0028-0836 (Linking)\\nVI  - 441\\nIP  - 7091\\nDP  - 2006 May 18\\nTI  - The DNA sequence and biological annotation of human chromosome 1.\\nPG  - 315-21\\nAB  - The reference sequence for each human chromosome provides the framework for\\n      understanding genome function, variation and evolution. Here we report the\\n      finished sequence and biological annotation of human chromosome 1. Chromosome 1\\n      is gene-dense, with 3,141 genes and 991 pseudogenes, and many coding sequences\\n      overlap. Rearrangements and mutations of chromosome 1 are prevalent in cancer and\\n      many other diseases. Patterns of sequence variation reveal signals of recent\\n      selection in specific genes that may contribute to human fitness, and also in\\n      regions where no function is evident. Fine-scale recombination occurs in hotspots\\n      of varying intensity along the sequence, and is enriched near genes. These and\\n      other studies of human biology and disease encoded within chromosome 1 are made\\n      possible with the highly accurate annotated sequence, as part of the completed\\n      set of chromosome sequences that comprise the reference human genome.\\nFAU - Gregory, S G\\nAU  - Gregory SG\\nAD  - The Wellcome Trust Sanger Institute, The Wellcome Trust Genome Campus, Hinxton,\\n      Cambridgeshire CB10 1SA, UK. sgregory@chg.duhs.duke.edu\\nFAU - Barlow, K F\\nAU  - Barlow KF\\nFAU - McLay, K E\\nAU  - McLay KE\\nFAU - Kaul, R\\nAU  - Kaul R\\nFAU - Swarbreck, D\\nAU  - Swarbreck D\\nFAU - Dunham, A\\nAU  - Dunham A\\nFAU - Scott, C E\\nAU  - Scott CE\\nFAU - Howe, K L\\nAU  - Howe KL\\nFAU - Woodfine, K\\nAU  - Woodfine K\\nFAU - Spencer, C C A\\nAU  - Spencer CC\\nFAU - Jones, M C\\nAU  - Jones MC\\nFAU - Gillson, C\\nAU  - Gillson C\\nFAU - Searle, S\\nAU  - Searle S\\nFAU - Zhou, Y\\nAU  - Zhou Y\\nFAU - Kokocinski, F\\nAU  - Kokocinski F\\nFAU - McDonald, L\\nAU  - McDonald L\\nFAU - Evans, R\\nAU  - Evans R\\nFAU - Phillips, K\\nAU  - Phillips K\\nFAU - Atkinson, A\\nAU  - Atkinson A\\nFAU - Cooper, R\\nAU  - Cooper R\\nFAU - Jones, C\\nAU  - Jones C\\nFAU - Hall, R E\\nAU  - Hall RE\\nFAU - Andrews, T D\\nAU  - Andrews TD\\nFAU - Lloyd, C\\nAU  - Lloyd C\\nFAU - Ainscough, R\\nAU  - Ainscough R\\nFAU - Almeida, J P\\nAU  - Almeida JP\\nFAU - Ambrose, K D\\nAU  - Ambrose KD\\nFAU - Anderson, F\\nAU  - Anderson F\\nFAU - Andrew, R W\\nAU  - Andrew RW\\nFAU - Ashwell, R I S\\nAU  - Ashwell RI\\nFAU - Aubin, K\\nAU  - Aubin K\\nFAU - Babbage, A K\\nAU  - Babbage AK\\nFAU - Bagguley, C L\\nAU  - Bagguley CL\\nFAU - Bailey, J\\nAU  - Bailey J\\nFAU - Beasley, H\\nAU  - Beasley H\\nFAU - Bethel, G\\nAU  - Bethel G\\nFAU - Bird, C P\\nAU  - Bird CP\\nFAU - Bray-Allen, S\\nAU  - Bray-Allen S\\nFAU - Brown, J Y\\nAU  - Brown JY\\nFAU - Brown, A J\\nAU  - Brown AJ\\nFAU - Buckley, D\\nAU  - Buckley D\\nFAU - Burton, J\\nAU  - Burton J\\nFAU - Bye, J\\nAU  - Bye J\\nFAU - Carder, C\\nAU  - Carder C\\nFAU - Chapman, J C\\nAU  - Chapman JC\\nFAU - Clark, S Y\\nAU  - Clark SY\\nFAU - Clarke, G\\nAU  - Clarke G\\nFAU - Clee, C\\nAU  - Clee C\\nFAU - Cobley, V\\nAU  - Cobley V\\nFAU - Collier, R E\\nAU  - Collier RE\\nFAU - Corby, N\\nAU  - Corby N\\nFAU - Coville, G J\\nAU  - Coville GJ\\nFAU - Davies, J\\nAU  - Davies J\\nFAU - Deadman, R\\nAU  - Deadman R\\nFAU - Dunn, M\\nAU  - Dunn M\\nFAU - Earthrowl, M\\nAU  - Earthrowl M\\nFAU - Ellington, A G\\nAU  - Ellington AG\\nFAU - Errington, H\\nAU  - Errington H\\nFAU - Frankish, A\\nAU  - Frankish A\\nFAU - Frankland, J\\nAU  - Frankland J\\nFAU - French, L\\nAU  - French L\\nFAU - Garner, P\\nAU  - Garner P\\nFAU - Garnett, J\\nAU  - Garnett J\\nFAU - Gay, L\\nAU  - Gay L\\nFAU - Ghori, M R J\\nAU  - Ghori MR\\nFAU - Gibson, R\\nAU  - Gibson R\\nFAU - Gilby, L M\\nAU  - Gilby LM\\nFAU - Gillett, W\\nAU  - Gillett W\\nFAU - Glithero, R J\\nAU  - Glithero RJ\\nFAU - Grafham, D V\\nAU  - Grafham DV\\nFAU - Griffiths, C\\nAU  - Griffiths C\\nFAU - Griffiths-Jones, S\\nAU  - Griffiths-Jones S\\nFAU - Grocock, R\\nAU  - Grocock R\\nFAU - Hammond, S\\nAU  - Hammond S\\nFAU - Harrison, E S I\\nAU  - Harrison ES\\nFAU - Hart, E\\nAU  - Hart E\\nFAU - Haugen, E\\nAU  - Haugen E\\nFAU - Heath, P D\\nAU  - Heath PD\\nFAU - Holmes, S\\nAU  - Holmes S\\nFAU - Holt, K\\nAU  - Holt K\\nFAU - Howden, P J\\nAU  - Howden PJ\\nFAU - Hunt, A R\\nAU  - Hunt AR\\nFAU - Hunt, S E\\nAU  - Hunt SE\\nFAU - Hunter, G\\nAU  - Hunter G\\nFAU - Isherwood, J\\nAU  - Isherwood J\\nFAU - James, R\\nAU  - James R\\nFAU - Johnson, C\\nAU  - Johnson C\\nFAU - Johnson, D\\nAU  - Johnson D\\nFAU - Joy, A\\nAU  - Joy A\\nFAU - Kay, M\\nAU  - Kay M\\nFAU - Kershaw, J K\\nAU  - Kershaw JK\\nFAU - Kibukawa, M\\nAU  - Kibukawa M\\nFAU - Kimberley, A M\\nAU  - Kimberley AM\\nFAU - King, A\\nAU  - King A\\nFAU - Knights, A J\\nAU  - Knights AJ\\nFAU - Lad, H\\nAU  - Lad H\\nFAU - Laird, G\\nAU  - Laird G\\nFAU - Lawlor, S\\nAU  - Lawlor S\\nFAU - Leongamornlert, D A\\nAU  - Leongamornlert DA\\nFAU - Lloyd, D M\\nAU  - Lloyd DM\\nFAU - Loveland, J\\nAU  - Loveland J\\nFAU - Lovell, J\\nAU  - Lovell J\\nFAU - Lush, M J\\nAU  - Lush MJ\\nFAU - Lyne, R\\nAU  - Lyne R\\nFAU - Martin, S\\nAU  - Martin S\\nFAU - Mashreghi-Mohammadi, M\\nAU  - Mashreghi-Mohammadi M\\nFAU - Matthews, L\\nAU  - Matthews L\\nFAU - Matthews, N S W\\nAU  - Matthews NS\\nFAU - McLaren, S\\nAU  - McLaren S\\nFAU - Milne, S\\nAU  - Milne S\\nFAU - Mistry, S\\nAU  - Mistry S\\nFAU - Moore, M J F\\nAU  - Moore MJ\\nFAU - Nickerson, T\\nAU  - Nickerson T\\nFAU - O'Dell, C N\\nAU  - O'Dell CN\\nFAU - Oliver, K\\nAU  - Oliver K\\nFAU - Palmeiri, A\\nAU  - Palmeiri A\\nFAU - Palmer, S A\\nAU  - Palmer SA\\nFAU - Parker, A\\nAU  - Parker A\\nFAU - Patel, D\\nAU  - Patel D\\nFAU - Pearce, A V\\nAU  - Pearce AV\\nFAU - Peck, A I\\nAU  - Peck AI\\nFAU - Pelan, S\\nAU  - Pelan S\\nFAU - Phelps, K\\nAU  - Phelps K\\nFAU - Phillimore, B J\\nAU  - Phillimore BJ\\nFAU - Plumb, R\\nAU  - Plumb R\\nFAU - Rajan, J\\nAU  - Rajan J\\nFAU - Raymond, C\\nAU  - Raymond C\\nFAU - Rouse, G\\nAU  - Rouse G\\nFAU - Saenphimmachak, C\\nAU  - Saenphimmachak C\\nFAU - Sehra, H K\\nAU  - Sehra HK\\nFAU - Sheridan, E\\nAU  - Sheridan E\\nFAU - Shownkeen, R\\nAU  - Shownkeen R\\nFAU - Sims, S\\nAU  - Sims S\\nFAU - Skuce, C D\\nAU  - Skuce CD\\nFAU - Smith, M\\nAU  - Smith M\\nFAU - Steward, C\\nAU  - Steward C\\nFAU - Subramanian, S\\nAU  - Subramanian S\\nFAU - Sycamore, N\\nAU  - Sycamore N\\nFAU - Tracey, A\\nAU  - Tracey A\\nFAU - Tromans, A\\nAU  - Tromans A\\nFAU - Van Helmond, Z\\nAU  - Van Helmond Z\\nFAU - Wall, M\\nAU  - Wall M\\nFAU - Wallis, J M\\nAU  - Wallis JM\\nFAU - White, S\\nAU  - White S\\nFAU - Whitehead, S L\\nAU  - Whitehead SL\\nFAU - Wilkinson, J E\\nAU  - Wilkinson JE\\nFAU - Willey, D L\\nAU  - Willey DL\\nFAU - Williams, H\\nAU  - Williams H\\nFAU - Wilming, L\\nAU  - Wilming L\\nFAU - Wray, P W\\nAU  - Wray PW\\nFAU - Wu, Z\\nAU  - Wu Z\\nFAU - Coulson, A\\nAU  - Coulson A\\nFAU - Vaudin, M\\nAU  - Vaudin M\\nFAU - Sulston, J E\\nAU  - Sulston JE\\nFAU - Durbin, R\\nAU  - Durbin R\\nFAU - Hubbard, T\\nAU  - Hubbard T\\nFAU - Wooster, R\\nAU  - Wooster R\\nFAU - Dunham, I\\nAU  - Dunham I\\nFAU - Carter, N P\\nAU  - Carter NP\\nFAU - McVean, G\\nAU  - McVean G\\nFAU - Ross, M T\\nAU  - Ross MT\\nFAU - Harrow, J\\nAU  - Harrow J\\nFAU - Olson, M V\\nAU  - Olson MV\\nFAU - Beck, S\\nAU  - Beck S\\nFAU - Rogers, J\\nAU  - Rogers J\\nFAU - Bentley, D R\\nAU  - Bentley DR\\nFAU - Banerjee, R\\nAU  - Banerjee R\\nFAU - Bryant, S P\\nAU  - Bryant SP\\nFAU - Burford, D C\\nAU  - Burford DC\\nFAU - Burrill, W D H\\nAU  - Burrill WD\\nFAU - Clegg, S M\\nAU  - Clegg SM\\nFAU - Dhami, P\\nAU  - Dhami P\\nFAU - Dovey, O\\nAU  - Dovey O\\nFAU - Faulkner, L M\\nAU  - Faulkner LM\\nFAU - Gribble, S M\\nAU  - Gribble SM\\nFAU - Langford, C F\\nAU  - Langford CF\\nFAU - Pandian, R D\\nAU  - Pandian RD\\nFAU - Porter, K M\\nAU  - Porter KM\\nFAU - Prigmore, E\\nAU  - Prigmore E\\nLA  - eng\\nSI  - RefSeq/NC_000001\\nGR  - G0000107/Medical Research Council/United Kingdom\\nGR  - Wellcome Trust/United Kingdom\\nPT  - Journal Article\\nPT  - Research Support, N.I.H., Extramural\\nPT  - Research Support, Non-U.S. Gov't\\nPL  - England\\nTA  - Nature\\nJT  - Nature\\nJID - 0410462\\nSB  - IM\\nEIN - Nature. 2006 Oct 26;443(7114):1013. Banerjee, R [added]; Bryant, SP [added];\\n      Burford, DC [added]; Burrill, WDH [added]; Clegg, SM [added]; Dhami, P [added];\\n      Dovey, O [added]; Faulkner, LM [added]; Gribble, SM [added]; Langford, CF\\n      [added]; Pandian, RD [added]; Porter, KM [added]; Prigmore, E [added]\\nMH  - Base Sequence\\nMH  - Chromosomes, Human, Pair 1/*genetics\\nMH  - DNA Replication Timing\\nMH  - Disease\\nMH  - Gene Duplication\\nMH  - Genes/genetics\\nMH  - Genetic Variation/genetics\\nMH  - Genomics\\nMH  - Humans\\nMH  - Molecular Sequence Data\\nMH  - Open Reading Frames/genetics\\nMH  - Pseudogenes/genetics\\nMH  - Recombination, Genetic/genetics\\nMH  - Selection, Genetic\\nMH  - Sequence Analysis, DNA\\nEDAT- 2006/05/20 09:00\\nMHDA- 2006/06/15 09:00\\nCRDT- 2006/05/20 09:00\\nPHST- 2005/12/24 00:00 [received]\\nPHST- 2006/03/13 00:00 [accepted]\\nPHST- 2006/05/20 09:00 [pubmed]\\nPHST- 2006/06/15 09:00 [medline]\\nPHST- 2006/05/20 09:00 [entrez]\\nAID - nature04727 [pii]\\nAID - 10.1038/nature04727 [doi]\\nPST - ppublish\\nSO  - Nature. 2006 May 18;441(7091):315-21. doi: 10.1038/nature04727.\", \"PMID- 15498460\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20050124\\nLR  - 20070516\\nIS  - 0888-7543 (Print)\\nIS  - 0888-7543 (Linking)\\nVI  - 84\\nIP  - 3\\nDP  - 2004 Sep\\nTI  - Analysis of the human VPS13 gene family.\\nPG  - 536-49\\nAB  - The gene mutated in chorea-acanthocytosis (CHAC; approved gene symbol VPS13A)\\n      encodes chorein, a protein similar to yeast Vps13p. We detected several similar\\n      putative human proteins by BLAST analysis of chorein. We characterized the\\n      structure of three new genes encoding these CHAC-similar proteins, located on\\n      chromosomes 1p36, 8q22, and 15q21. The most similar gene in yeast to all four\\n      human genes is Vps13, and therefore the human genes were named VPS13A (CHAC,\\n      9q21), VPS13B (8q22), VPS13C (15q21), and VPS13D (1p36). VPS13B has recently been\\n      reported as COH1, altered in Cohen syndrome. For each gene, we describe several\\n      alternative splicing variants; at least two transcripts per gene are major forms.\\n      The expression pattern of these genes is ubiquitous, with some tissue-specific\\n      differences between several transcript variants. Protein sequence comparisons\\n      suggest that intramolecular duplications have played an important role in the\\n      evolution of this gene family.\\nFAU - Velayos-Baeza, Antonio\\nAU  - Velayos-Baeza A\\nAD  - Wellcome Trust Centre for Human Genetics, University of Oxford, Headington, OX3\\n      7BN Oxford, UK.\\nFAU - Vettori, Andrea\\nAU  - Vettori A\\nFAU - Copley, Richard R\\nAU  - Copley RR\\nFAU - Dobson-Stone, Carol\\nAU  - Dobson-Stone C\\nFAU - Monaco, A P\\nAU  - Monaco AP\\nLA  - eng\\nSI  - GENBANK/AJ608769\\nSI  - GENBANK/AJ608770\\nSI  - GENBANK/AJ608771\\nSI  - GENBANK/AJ608772\\nSI  - GENBANK/AJ608773\\nSI  - GENBANK/AJ608774\\nSI  - GENBANK/AJ608775\\nSI  - GENBANK/AJ626859\\nSI  - GENBANK/AJ626860\\nSI  - GENBANK/AJ626861\\nPT  - Comparative Study\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov't\\nPL  - United States\\nTA  - Genomics\\nJT  - Genomics\\nJID - 8800135\\nRN  - 0 (Membrane Proteins)\\nRN  - 0 (Proteins)\\nRN  - 0 (VPS13A protein, human)\\nRN  - 0 (VPS13B protein, human)\\nRN  - 0 (VPS13C protein, human)\\nRN  - 0 (VPS13D protein, human)\\nRN  - 0 (Vesicular Transport Proteins)\\nSB  - IM\\nMH  - Alternative Splicing/*genetics\\nMH  - Amino Acid Sequence\\nMH  - Base Sequence\\nMH  - Blotting, Northern\\nMH  - Evolution, Molecular\\nMH  - Gene Components\\nMH  - Gene Duplication\\nMH  - Gene Expression Profiling\\nMH  - Humans\\nMH  - Membrane Proteins/*genetics\\nMH  - Molecular Sequence Data\\nMH  - Multigene Family/*genetics\\nMH  - Phylogeny\\nMH  - Proteins/*genetics\\nMH  - Reverse Transcriptase Polymerase Chain Reaction\\nMH  - Sequence Alignment\\nMH  - Sequence Analysis, DNA\\nMH  - Sequence Homology\\nMH  - Vesicular Transport Proteins\\nEDAT- 2004/10/23 09:00\\nMHDA- 2005/01/26 09:00\\nCRDT- 2004/10/23 09:00\\nPHST- 2004/02/13 00:00 [received]\\nPHST- 2004/04/19 00:00 [accepted]\\nPHST- 2004/10/23 09:00 [pubmed]\\nPHST- 2005/01/26 09:00 [medline]\\nPHST- 2004/10/23 09:00 [entrez]\\nAID - S0888754304001156 [pii]\\nAID - 10.1016/j.ygeno.2004.04.012 [doi]\\nPST - ppublish\\nSO  - Genomics. 2004 Sep;84(3):536-49. doi: 10.1016/j.ygeno.2004.04.012.\", \"PMID- 15489334\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20041116\\nLR  - 20170219\\nIS  - 1088-9051 (Print)\\nIS  - 1088-9051 (Linking)\\nVI  - 14\\nIP  - 10B\\nDP  - 2004 Oct\\nTI  - The status, quality, and expansion of the NIH full-length cDNA project: the\\n      Mammalian Gene Collection (MGC).\\nPG  - 2121-7\\nAB  - The National Institutes of Health's Mammalian Gene Collection (MGC) project was\\n      designed to generate and sequence a publicly accessible cDNA resource containing \\n      a complete open reading frame (ORF) for every human and mouse gene. The project\\n      initially used a random strategy to select clones from a large number of cDNA\\n      libraries from diverse tissues. Candidate clones were chosen based on 5'-EST\\n      sequences, and then fully sequenced to high accuracy and analyzed by algorithms\\n      developed for this project. Currently, more than 11,000 human and 10,000 mouse\\n      genes are represented in MGC by at least one clone with a full ORF. The random\\n      selection approach is now reaching a saturation point, and a transition to\\n      protocols targeted at the missing transcripts is now required to complete the\\n      mouse and human collections. Comparison of the sequence of the MGC clones to\\n      reference genome sequences reveals that most cDNA clones are of very high\\n      sequence quality, although it is likely that some cDNAs may carry missense\\n      variants as a consequence of experimental artifact, such as PCR, cloning, or\\n      reverse transcriptase errors. Recently, a rat cDNA component was added to the\\n      project, and ongoing frog (Xenopus) and zebrafish (Danio) cDNA projects were\\n      expanded to take advantage of the high-throughput MGC pipeline.\\nFAU - Gerhard, Daniela S\\nAU  - Gerhard DS\\nFAU - Wagner, Lukas\\nAU  - Wagner L\\nFAU - Feingold, Elise A\\nAU  - Feingold EA\\nFAU - Shenmen, Carolyn M\\nAU  - Shenmen CM\\nFAU - Grouse, Lynette H\\nAU  - Grouse LH\\nFAU - Schuler, Greg\\nAU  - Schuler G\\nFAU - Klein, Steven L\\nAU  - Klein SL\\nFAU - Old, Susan\\nAU  - Old S\\nFAU - Rasooly, Rebekah\\nAU  - Rasooly R\\nFAU - Good, Peter\\nAU  - Good P\\nFAU - Guyer, Mark\\nAU  - Guyer M\\nFAU - Peck, Allison M\\nAU  - Peck AM\\nFAU - Derge, Jeffery G\\nAU  - Derge JG\\nFAU - Lipman, David\\nAU  - Lipman D\\nFAU - Collins, Francis S\\nAU  - Collins FS\\nFAU - Jang, Wonhee\\nAU  - Jang W\\nFAU - Sherry, Steven\\nAU  - Sherry S\\nFAU - Feolo, Mike\\nAU  - Feolo M\\nFAU - Misquitta, Leonie\\nAU  - Misquitta L\\nFAU - Lee, Eduardo\\nAU  - Lee E\\nFAU - Rotmistrovsky, Kirill\\nAU  - Rotmistrovsky K\\nFAU - Greenhut, Susan F\\nAU  - Greenhut SF\\nFAU - Schaefer, Carl F\\nAU  - Schaefer CF\\nFAU - Buetow, Kenneth\\nAU  - Buetow K\\nFAU - Bonner, Tom I\\nAU  - Bonner TI\\nFAU - Haussler, David\\nAU  - Haussler D\\nFAU - Kent, Jim\\nAU  - Kent J\\nFAU - Kiekhaus, Mark\\nAU  - Kiekhaus M\\nFAU - Furey, Terry\\nAU  - Furey T\\nFAU - Brent, Michael\\nAU  - Brent M\\nFAU - Prange, Christa\\nAU  - Prange C\\nFAU - Schreiber, Kirsten\\nAU  - Schreiber K\\nFAU - Shapiro, Nicole\\nAU  - Shapiro N\\nFAU - Bhat, Narayan K\\nAU  - Bhat NK\\nFAU - Hopkins, Ralph F\\nAU  - Hopkins RF\\nFAU - Hsie, Florence\\nAU  - Hsie F\\nFAU - Driscoll, Tom\\nAU  - Driscoll T\\nFAU - Soares, M Bento\\nAU  - Soares MB\\nFAU - Casavant, Tom L\\nAU  - Casavant TL\\nFAU - Scheetz, Todd E\\nAU  - Scheetz TE\\nFAU - Brown-stein, Michael J\\nAU  - Brown-stein MJ\\nFAU - Usdin, Ted B\\nAU  - Usdin TB\\nFAU - Toshiyuki, Shiraki\\nAU  - Toshiyuki S\\nFAU - Carninci, Piero\\nAU  - Carninci P\\nFAU - Piao, Yulan\\nAU  - Piao Y\\nFAU - Dudekula, Dawood B\\nAU  - Dudekula DB\\nFAU - Ko, Minoru S H\\nAU  - Ko MS\\nFAU - Kawakami, Koichi\\nAU  - Kawakami K\\nFAU - Suzuki, Yutaka\\nAU  - Suzuki Y\\nFAU - Sugano, Sumio\\nAU  - Sugano S\\nFAU - Gruber, C E\\nAU  - Gruber CE\\nFAU - Smith, M R\\nAU  - Smith MR\\nFAU - Simmons, Blake\\nAU  - Simmons B\\nFAU - Moore, Troy\\nAU  - Moore T\\nFAU - Waterman, Richard\\nAU  - Waterman R\\nFAU - Johnson, Stephen L\\nAU  - Johnson SL\\nFAU - Ruan, Yijun\\nAU  - Ruan Y\\nFAU - Wei, Chia Lin\\nAU  - Wei CL\\nFAU - Mathavan, S\\nAU  - Mathavan S\\nFAU - Gunaratne, Preethi H\\nAU  - Gunaratne PH\\nFAU - Wu, Jiaqian\\nAU  - Wu J\\nFAU - Garcia, Angela M\\nAU  - Garcia AM\\nFAU - Hulyk, Stephen W\\nAU  - Hulyk SW\\nFAU - Fuh, Edwin\\nAU  - Fuh E\\nFAU - Yuan, Ye\\nAU  - Yuan Y\\nFAU - Sneed, Anna\\nAU  - Sneed A\\nFAU - Kowis, Carla\\nAU  - Kowis C\\nFAU - Hodgson, Anne\\nAU  - Hodgson A\\nFAU - Muzny, Donna M\\nAU  - Muzny DM\\nFAU - McPherson, John\\nAU  - McPherson J\\nFAU - Gibbs, Richard A\\nAU  - Gibbs RA\\nFAU - Fahey, Jessica\\nAU  - Fahey J\\nFAU - Helton, Erin\\nAU  - Helton E\\nFAU - Ketteman, Mark\\nAU  - Ketteman M\\nFAU - Madan, Anuradha\\nAU  - Madan A\\nFAU - Rodrigues, Stephanie\\nAU  - Rodrigues S\\nFAU - Sanchez, Amy\\nAU  - Sanchez A\\nFAU - Whiting, Michelle\\nAU  - Whiting M\\nFAU - Madari, Anup\\nAU  - Madari A\\nFAU - Young, Alice C\\nAU  - Young AC\\nFAU - Wetherby, Keith D\\nAU  - Wetherby KD\\nFAU - Granite, Steven J\\nAU  - Granite SJ\\nFAU - Kwong, Peggy N\\nAU  - Kwong PN\\nFAU - Brinkley, Charles P\\nAU  - Brinkley CP\\nFAU - Pearson, Russell L\\nAU  - Pearson RL\\nFAU - Bouffard, Gerard G\\nAU  - Bouffard GG\\nFAU - Blakesly, Robert W\\nAU  - Blakesly RW\\nFAU - Green, Eric D\\nAU  - Green ED\\nFAU - Dickson, Mark C\\nAU  - Dickson MC\\nFAU - Rodriguez, Alex C\\nAU  - Rodriguez AC\\nFAU - Grimwood, Jane\\nAU  - Grimwood J\\nFAU - Schmutz, Jeremy\\nAU  - Schmutz J\\nFAU - Myers, Richard M\\nAU  - Myers RM\\nFAU - Butterfield, Yaron S N\\nAU  - Butterfield YS\\nFAU - Griffith, Malachi\\nAU  - Griffith M\\nFAU - Griffith, Obi L\\nAU  - Griffith OL\\nFAU - Krzywinski, Martin I\\nAU  - Krzywinski MI\\nFAU - Liao, Nancy\\nAU  - Liao N\\nFAU - Morin, Ryan\\nAU  - Morin R\\nFAU - Palmquist, Diana\\nAU  - Palmquist D\\nFAU - Petrescu, Anca S\\nAU  - Petrescu AS\\nFAU - Skalska, Ursula\\nAU  - Skalska U\\nFAU - Smailus, Duane E\\nAU  - Smailus DE\\nFAU - Stott, Jeff M\\nAU  - Stott JM\\nFAU - Schnerch, Angelique\\nAU  - Schnerch A\\nFAU - Schein, Jacqueline E\\nAU  - Schein JE\\nFAU - Jones, Steven J M\\nAU  - Jones SJ\\nFAU - Holt, Robert A\\nAU  - Holt RA\\nFAU - Baross, Agnes\\nAU  - Baross A\\nFAU - Marra, Marco A\\nAU  - Marra MA\\nFAU - Clifton, Sandra\\nAU  - Clifton S\\nFAU - Makowski, Kathryn A\\nAU  - Makowski KA\\nFAU - Bosak, Stephanie\\nAU  - Bosak S\\nFAU - Malek, Joel\\nAU  - Malek J\\nCN  - MGC Project Team\\nLA  - eng\\nGR  - N01-C0-12400/PHS HHS/United States\\nPT  - Comparative Study\\nPT  - Journal Article\\nPT  - Research Support, U.S. Gov't, P.H.S.\\nPL  - United States\\nTA  - Genome Res\\nJT  - Genome research\\nJID - 9518021\\nRN  - 0 (DNA Primers)\\nRN  - 0 (DNA, Complementary)\\nSB  - IM\\nEIN - Genome Res. 2006 Jun;16(6):804. Morrin, Ryan [corrected to Morin, Ryan]\\nMH  - Animals\\nMH  - Cloning, Molecular/*methods\\nMH  - Computational Biology\\nMH  - DNA Primers\\nMH  - *DNA, Complementary/genetics/metabolism\\nMH  - *Gene Library\\nMH  - Humans\\nMH  - Mice\\nMH  - National Institutes of Health (U.S.)\\nMH  - Open Reading Frames/*physiology\\nMH  - Rats\\nMH  - United States\\nMH  - Xenopus laevis/genetics\\nMH  - Zebrafish/genetics\\nPMC - PMC528928\\nOID - NLM: PMC528928\\nEDAT- 2004/10/19 09:00\\nMHDA- 2004/11/17 09:00\\nCRDT- 2004/10/19 09:00\\nPHST- 2004/10/19 09:00 [pubmed]\\nPHST- 2004/11/17 09:00 [medline]\\nPHST- 2004/10/19 09:00 [entrez]\\nAID - 14/10b/2121 [pii]\\nAID - 10.1101/gr.2596504 [doi]\\nPST - ppublish\\nSO  - Genome Res. 2004 Oct;14(10B):2121-7. doi: 10.1101/gr.2596504.\", \"PMID- 14702039\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20040202\\nLR  - 20061115\\nIS  - 1061-4036 (Print)\\nIS  - 1061-4036 (Linking)\\nVI  - 36\\nIP  - 1\\nDP  - 2004 Jan\\nTI  - Complete sequencing and characterization of 21,243 full-length human cDNAs.\\nPG  - 40-5\\nAB  - As a base for human transcriptome and functional genomics, we created the\\n      \\\"full-length long Japan\\\" (FLJ) collection of sequenced human cDNAs. We determined\\n      the entire sequence of 21,243 selected clones and found that 14,490 cDNAs (10,897\\n      clusters) were unique to the FLJ collection. About half of them (5,416) seemed to\\n      be protein-coding. Of those, 1,999 clusters had not been predicted by\\n      computational methods. The distribution of GC content of nonpredicted cDNAs had a\\n      peak at approximately 58% compared with a peak at approximately 42%for predicted \\n      cDNAs. Thus, there seems to be a slight bias against GC-rich transcripts in\\n      current gene prediction procedures. The rest of the cDNAs unique to the FLJ\\n      collection (5,481) contained no obvious open reading frames (ORFs) and thus are\\n      candidate noncoding RNAs. About one-fourth of them (1,378) showed a clear pattern\\n      of splicing. The distribution of GC content of noncoding cDNAs was narrow and had\\n      a peak at approximately 42%, relatively low compared with that of protein-coding \\n      cDNAs.\\nFAU - Ota, Toshio\\nAU  - Ota T\\nAD  - Helix Research Institute, 1532-3 Yana, Kisarazu, Chiba 292-0812, Japan.\\nFAU - Suzuki, Yutaka\\nAU  - Suzuki Y\\nFAU - Nishikawa, Tetsuo\\nAU  - Nishikawa T\\nFAU - Otsuki, Tetsuji\\nAU  - Otsuki T\\nFAU - Sugiyama, Tomoyasu\\nAU  - Sugiyama T\\nFAU - Irie, Ryotaro\\nAU  - Irie R\\nFAU - Wakamatsu, Ai\\nAU  - Wakamatsu A\\nFAU - Hayashi, Koji\\nAU  - Hayashi K\\nFAU - Sato, Hiroyuki\\nAU  - Sato H\\nFAU - Nagai, Keiichi\\nAU  - Nagai K\\nFAU - Kimura, Kouichi\\nAU  - Kimura K\\nFAU - Makita, Hiroshi\\nAU  - Makita H\\nFAU - Sekine, Mitsuo\\nAU  - Sekine M\\nFAU - Obayashi, Masaya\\nAU  - Obayashi M\\nFAU - Nishi, Tatsunari\\nAU  - Nishi T\\nFAU - Shibahara, Toshikazu\\nAU  - Shibahara T\\nFAU - Tanaka, Toshihiro\\nAU  - Tanaka T\\nFAU - Ishii, Shizuko\\nAU  - Ishii S\\nFAU - Yamamoto, Jun-ichi\\nAU  - Yamamoto J\\nFAU - Saito, Kaoru\\nAU  - Saito K\\nFAU - Kawai, Yuri\\nAU  - Kawai Y\\nFAU - Isono, Yuko\\nAU  - Isono Y\\nFAU - Nakamura, Yoshitaka\\nAU  - Nakamura Y\\nFAU - Nagahari, Kenji\\nAU  - Nagahari K\\nFAU - Murakami, Katsuhiko\\nAU  - Murakami K\\nFAU - Yasuda, Tomohiro\\nAU  - Yasuda T\\nFAU - Iwayanagi, Takao\\nAU  - Iwayanagi T\\nFAU - Wagatsuma, Masako\\nAU  - Wagatsuma M\\nFAU - Shiratori, Akiko\\nAU  - Shiratori A\\nFAU - Sudo, Hiroaki\\nAU  - Sudo H\\nFAU - Hosoiri, Takehiko\\nAU  - Hosoiri T\\nFAU - Kaku, Yoshiko\\nAU  - Kaku Y\\nFAU - Kodaira, Hiroyo\\nAU  - Kodaira H\\nFAU - Kondo, Hiroshi\\nAU  - Kondo H\\nFAU - Sugawara, Masanori\\nAU  - Sugawara M\\nFAU - Takahashi, Makiko\\nAU  - Takahashi M\\nFAU - Kanda, Katsuhiro\\nAU  - Kanda K\\nFAU - Yokoi, Takahide\\nAU  - Yokoi T\\nFAU - Furuya, Takako\\nAU  - Furuya T\\nFAU - Kikkawa, Emiko\\nAU  - Kikkawa E\\nFAU - Omura, Yuhi\\nAU  - Omura Y\\nFAU - Abe, Kumi\\nAU  - Abe K\\nFAU - Kamihara, Kumiko\\nAU  - Kamihara K\\nFAU - Katsuta, Naoko\\nAU  - Katsuta N\\nFAU - Sato, Kazuomi\\nAU  - Sato K\\nFAU - Tanikawa, Machiko\\nAU  - Tanikawa M\\nFAU - Yamazaki, Makoto\\nAU  - Yamazaki M\\nFAU - Ninomiya, Ken\\nAU  - Ninomiya K\\nFAU - Ishibashi, Tadashi\\nAU  - Ishibashi T\\nFAU - Yamashita, Hiromichi\\nAU  - Yamashita H\\nFAU - Murakawa, Katsuji\\nAU  - Murakawa K\\nFAU - Fujimori, Kiyoshi\\nAU  - Fujimori K\\nFAU - Tanai, Hiroyuki\\nAU  - Tanai H\\nFAU - Kimata, Manabu\\nAU  - Kimata M\\nFAU - Watanabe, Motoji\\nAU  - Watanabe M\\nFAU - Hiraoka, Susumu\\nAU  - Hiraoka S\\nFAU - Chiba, Yoshiyuki\\nAU  - Chiba Y\\nFAU - Ishida, Shinichi\\nAU  - Ishida S\\nFAU - Ono, Yukio\\nAU  - Ono Y\\nFAU - Takiguchi, Sumiyo\\nAU  - Takiguchi S\\nFAU - Watanabe, Susumu\\nAU  - Watanabe S\\nFAU - Yosida, Makoto\\nAU  - Yosida M\\nFAU - Hotuta, Tomoko\\nAU  - Hotuta T\\nFAU - Kusano, Junko\\nAU  - Kusano J\\nFAU - Kanehori, Keiichi\\nAU  - Kanehori K\\nFAU - Takahashi-Fujii, Asako\\nAU  - Takahashi-Fujii A\\nFAU - Hara, Hiroto\\nAU  - Hara H\\nFAU - Tanase, Tomo-o\\nAU  - Tanase TO\\nFAU - Nomura, Yoshiko\\nAU  - Nomura Y\\nFAU - Togiya, Sakae\\nAU  - Togiya S\\nFAU - Komai, Fukuyo\\nAU  - Komai F\\nFAU - Hara, Reiko\\nAU  - Hara R\\nFAU - Takeuchi, Kazuha\\nAU  - Takeuchi K\\nFAU - Arita, Miho\\nAU  - Arita M\\nFAU - Imose, Nobuyuki\\nAU  - Imose N\\nFAU - Musashino, Kaoru\\nAU  - Musashino K\\nFAU - Yuuki, Hisatsugu\\nAU  - Yuuki H\\nFAU - Oshima, Atsushi\\nAU  - Oshima A\\nFAU - Sasaki, Naokazu\\nAU  - Sasaki N\\nFAU - Aotsuka, Satoshi\\nAU  - Aotsuka S\\nFAU - Yoshikawa, Yoko\\nAU  - Yoshikawa Y\\nFAU - Matsunawa, Hiroshi\\nAU  - Matsunawa H\\nFAU - Ichihara, Tatsuo\\nAU  - Ichihara T\\nFAU - Shiohata, Namiko\\nAU  - Shiohata N\\nFAU - Sano, Sanae\\nAU  - Sano S\\nFAU - Moriya, Shogo\\nAU  - Moriya S\\nFAU - Momiyama, Hiroko\\nAU  - Momiyama H\\nFAU - Satoh, Noriko\\nAU  - Satoh N\\nFAU - Takami, Sachiko\\nAU  - Takami S\\nFAU - Terashima, Yuko\\nAU  - Terashima Y\\nFAU - Suzuki, Osamu\\nAU  - Suzuki O\\nFAU - Nakagawa, Satoshi\\nAU  - Nakagawa S\\nFAU - Senoh, Akihiro\\nAU  - Senoh A\\nFAU - Mizoguchi, Hiroshi\\nAU  - Mizoguchi H\\nFAU - Goto, Yoshihiro\\nAU  - Goto Y\\nFAU - Shimizu, Fumio\\nAU  - Shimizu F\\nFAU - Wakebe, Hirokazu\\nAU  - Wakebe H\\nFAU - Hishigaki, Haretsugu\\nAU  - Hishigaki H\\nFAU - Watanabe, Takeshi\\nAU  - Watanabe T\\nFAU - Sugiyama, Akio\\nAU  - Sugiyama A\\nFAU - Takemoto, Makoto\\nAU  - Takemoto M\\nFAU - Kawakami, Bunsei\\nAU  - Kawakami B\\nFAU - Yamazaki, Masaaki\\nAU  - Yamazaki M\\nFAU - Watanabe, Koji\\nAU  - Watanabe K\\nFAU - Kumagai, Ayako\\nAU  - Kumagai A\\nFAU - Itakura, Shoko\\nAU  - Itakura S\\nFAU - Fukuzumi, Yasuhito\\nAU  - Fukuzumi Y\\nFAU - Fujimori, Yoshifumi\\nAU  - Fujimori Y\\nFAU - Komiyama, Megumi\\nAU  - Komiyama M\\nFAU - Tashiro, Hiroyuki\\nAU  - Tashiro H\\nFAU - Tanigami, Akira\\nAU  - Tanigami A\\nFAU - Fujiwara, Tsutomu\\nAU  - Fujiwara T\\nFAU - Ono, Toshihide\\nAU  - Ono T\\nFAU - Yamada, Katsue\\nAU  - Yamada K\\nFAU - Fujii, Yuka\\nAU  - Fujii Y\\nFAU - Ozaki, Kouichi\\nAU  - Ozaki K\\nFAU - Hirao, Maasa\\nAU  - Hirao M\\nFAU - Ohmori, Yoshihiro\\nAU  - Ohmori Y\\nFAU - Kawabata, Ayako\\nAU  - Kawabata A\\nFAU - Hikiji, Takeshi\\nAU  - Hikiji T\\nFAU - Kobatake, Naoko\\nAU  - Kobatake N\\nFAU - Inagaki, Hiromi\\nAU  - Inagaki H\\nFAU - Ikema, Yasuko\\nAU  - Ikema Y\\nFAU - Okamoto, Sachiko\\nAU  - Okamoto S\\nFAU - Okitani, Rie\\nAU  - Okitani R\\nFAU - Kawakami, Takuma\\nAU  - Kawakami T\\nFAU - Noguchi, Saori\\nAU  - Noguchi S\\nFAU - Itoh, Tomoko\\nAU  - Itoh T\\nFAU - Shigeta, Keiko\\nAU  - Shigeta K\\nFAU - Senba, Tadashi\\nAU  - Senba T\\nFAU - Matsumura, Kyoka\\nAU  - Matsumura K\\nFAU - Nakajima, Yoshie\\nAU  - Nakajima Y\\nFAU - Mizuno, Takae\\nAU  - Mizuno T\\nFAU - Morinaga, Misato\\nAU  - Morinaga M\\nFAU - Sasaki, Masahide\\nAU  - Sasaki M\\nFAU - Togashi, Takushi\\nAU  - Togashi T\\nFAU - Oyama, Masaaki\\nAU  - Oyama M\\nFAU - Hata, Hiroko\\nAU  - Hata H\\nFAU - Watanabe, Manabu\\nAU  - Watanabe M\\nFAU - Komatsu, Takami\\nAU  - Komatsu T\\nFAU - Mizushima-Sugano, Junko\\nAU  - Mizushima-Sugano J\\nFAU - Satoh, Tadashi\\nAU  - Satoh T\\nFAU - Shirai, Yuko\\nAU  - Shirai Y\\nFAU - Takahashi, Yukiko\\nAU  - Takahashi Y\\nFAU - Nakagawa, Kiyomi\\nAU  - Nakagawa K\\nFAU - Okumura, Koji\\nAU  - Okumura K\\nFAU - Nagase, Takahiro\\nAU  - Nagase T\\nFAU - Nomura, Nobuo\\nAU  - Nomura N\\nFAU - Kikuchi, Hisashi\\nAU  - Kikuchi H\\nFAU - Masuho, Yasuhiko\\nAU  - Masuho Y\\nFAU - Yamashita, Riu\\nAU  - Yamashita R\\nFAU - Nakai, Kenta\\nAU  - Nakai K\\nFAU - Yada, Tetsushi\\nAU  - Yada T\\nFAU - Nakamura, Yusuke\\nAU  - Nakamura Y\\nFAU - Ohara, Osamu\\nAU  - Ohara O\\nFAU - Isogai, Takao\\nAU  - Isogai T\\nFAU - Sugano, Sumio\\nAU  - Sugano S\\nLA  - eng\\nPT  - Comparative Study\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov't\\nDEP - 20031221\\nPL  - United States\\nTA  - Nat Genet\\nJT  - Nature genetics\\nJID - 9216904\\nRN  - 0 (DNA, Complementary)\\nRN  - 0 (RNA, Messenger)\\nSB  - IM\\nMH  - Chromosomes, Human, 21-22 and Y\\nMH  - Chromosomes, Human, Pair 20\\nMH  - Computational Biology\\nMH  - *DNA, Complementary\\nMH  - Humans\\nMH  - Open Reading Frames\\nMH  - RNA, Messenger\\nMH  - *Sequence Analysis, DNA\\nEDAT- 2004/01/01 05:00\\nMHDA- 2004/02/03 05:00\\nCRDT- 2004/01/01 05:00\\nPHST- 2003/10/07 00:00 [received]\\nPHST- 2003/12/01 00:00 [accepted]\\nPHST- 2004/01/01 05:00 [pubmed]\\nPHST- 2004/02/03 05:00 [medline]\\nPHST- 2004/01/01 05:00 [entrez]\\nAID - 10.1038/ng1285 [doi]\\nAID - ng1285 [pii]\\nPST - ppublish\\nSO  - Nat Genet. 2004 Jan;36(1):40-5. doi: 10.1038/ng1285. Epub 2003 Dec 21.\", \"PMID- 12477932\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20030121\\nLR  - 20161025\\nIS  - 0027-8424 (Print)\\nIS  - 0027-8424 (Linking)\\nVI  - 99\\nIP  - 26\\nDP  - 2002 Dec 24\\nTI  - Generation and initial analysis of more than 15,000 full-length human and mouse\\n      cDNA sequences.\\nPG  - 16899-903\\nAB  - The National Institutes of Health Mammalian Gene Collection (MGC) Program is a\\n      multiinstitutional effort to identify and sequence a cDNA clone containing a\\n      complete ORF for each human and mouse gene. ESTs were generated from libraries\\n      enriched for full-length cDNAs and analyzed to identify candidate full-ORF\\n      clones, which then were sequenced to high accuracy. The MGC has currently\\n      sequenced and verified the full ORF for a nonredundant set of >9,000 human and\\n      >6,000 mouse genes. Candidate full-ORF clones for an additional 7,800 human and\\n      3,500 mouse genes also have been identified. All MGC sequences and clones are\\n      available without restriction through public databases and clone distribution\\n      networks (see http:mgc.nci.nih.gov).\\nFAU - Strausberg, Robert L\\nAU  - Strausberg RL\\nAD  - National Cancer Institute, Bethessda, MD 20892-2580, USA. rls@nih.gov\\nFAU - Feingold, Elise A\\nAU  - Feingold EA\\nFAU - Grouse, Lynette H\\nAU  - Grouse LH\\nFAU - Derge, Jeffery G\\nAU  - Derge JG\\nFAU - Klausner, Richard D\\nAU  - Klausner RD\\nFAU - Collins, Francis S\\nAU  - Collins FS\\nFAU - Wagner, Lukas\\nAU  - Wagner L\\nFAU - Shenmen, Carolyn M\\nAU  - Shenmen CM\\nFAU - Schuler, Gregory D\\nAU  - Schuler GD\\nFAU - Altschul, Stephen F\\nAU  - Altschul SF\\nFAU - Zeeberg, Barry\\nAU  - Zeeberg B\\nFAU - Buetow, Kenneth H\\nAU  - Buetow KH\\nFAU - Schaefer, Carl F\\nAU  - Schaefer CF\\nFAU - Bhat, Narayan K\\nAU  - Bhat NK\\nFAU - Hopkins, Ralph F\\nAU  - Hopkins RF\\nFAU - Jordan, Heather\\nAU  - Jordan H\\nFAU - Moore, Troy\\nAU  - Moore T\\nFAU - Max, Steve I\\nAU  - Max SI\\nFAU - Wang, Jun\\nAU  - Wang J\\nFAU - Hsieh, Florence\\nAU  - Hsieh F\\nFAU - Diatchenko, Luda\\nAU  - Diatchenko L\\nFAU - Marusina, Kate\\nAU  - Marusina K\\nFAU - Farmer, Andrew A\\nAU  - Farmer AA\\nFAU - Rubin, Gerald M\\nAU  - Rubin GM\\nFAU - Hong, Ling\\nAU  - Hong L\\nFAU - Stapleton, Mark\\nAU  - Stapleton M\\nFAU - Soares, M Bento\\nAU  - Soares MB\\nFAU - Bonaldo, Maria F\\nAU  - Bonaldo MF\\nFAU - Casavant, Tom L\\nAU  - Casavant TL\\nFAU - Scheetz, Todd E\\nAU  - Scheetz TE\\nFAU - Brownstein, Michael J\\nAU  - Brownstein MJ\\nFAU - Usdin, Ted B\\nAU  - Usdin TB\\nFAU - Toshiyuki, Shiraki\\nAU  - Toshiyuki S\\nFAU - Carninci, Piero\\nAU  - Carninci P\\nFAU - Prange, Christa\\nAU  - Prange C\\nFAU - Raha, Sam S\\nAU  - Raha SS\\nFAU - Loquellano, Naomi A\\nAU  - Loquellano NA\\nFAU - Peters, Garrick J\\nAU  - Peters GJ\\nFAU - Abramson, Rick D\\nAU  - Abramson RD\\nFAU - Mullahy, Sara J\\nAU  - Mullahy SJ\\nFAU - Bosak, Stephanie A\\nAU  - Bosak SA\\nFAU - McEwan, Paul J\\nAU  - McEwan PJ\\nFAU - McKernan, Kevin J\\nAU  - McKernan KJ\\nFAU - Malek, Joel A\\nAU  - Malek JA\\nFAU - Gunaratne, Preethi H\\nAU  - Gunaratne PH\\nFAU - Richards, Stephen\\nAU  - Richards S\\nFAU - Worley, Kim C\\nAU  - Worley KC\\nFAU - Hale, Sarah\\nAU  - Hale S\\nFAU - Garcia, Angela M\\nAU  - Garcia AM\\nFAU - Gay, Laura J\\nAU  - Gay LJ\\nFAU - Hulyk, Stephen W\\nAU  - Hulyk SW\\nFAU - Villalon, Debbie K\\nAU  - Villalon DK\\nFAU - Muzny, Donna M\\nAU  - Muzny DM\\nFAU - Sodergren, Erica J\\nAU  - Sodergren EJ\\nFAU - Lu, Xiuhua\\nAU  - Lu X\\nFAU - Gibbs, Richard A\\nAU  - Gibbs RA\\nFAU - Fahey, Jessica\\nAU  - Fahey J\\nFAU - Helton, Erin\\nAU  - Helton E\\nFAU - Ketteman, Mark\\nAU  - Ketteman M\\nFAU - Madan, Anuradha\\nAU  - Madan A\\nFAU - Rodrigues, Stephanie\\nAU  - Rodrigues S\\nFAU - Sanchez, Amy\\nAU  - Sanchez A\\nFAU - Whiting, Michelle\\nAU  - Whiting M\\nFAU - Madan, Anup\\nAU  - Madan A\\nFAU - Young, Alice C\\nAU  - Young AC\\nFAU - Shevchenko, Yuriy\\nAU  - Shevchenko Y\\nFAU - Bouffard, Gerard G\\nAU  - Bouffard GG\\nFAU - Blakesley, Robert W\\nAU  - Blakesley RW\\nFAU - Touchman, Jeffrey W\\nAU  - Touchman JW\\nFAU - Green, Eric D\\nAU  - Green ED\\nFAU - Dickson, Mark C\\nAU  - Dickson MC\\nFAU - Rodriguez, Alex C\\nAU  - Rodriguez AC\\nFAU - Grimwood, Jane\\nAU  - Grimwood J\\nFAU - Schmutz, Jeremy\\nAU  - Schmutz J\\nFAU - Myers, Richard M\\nAU  - Myers RM\\nFAU - Butterfield, Yaron S N\\nAU  - Butterfield YS\\nFAU - Krzywinski, Martin I\\nAU  - Krzywinski MI\\nFAU - Skalska, Ursula\\nAU  - Skalska U\\nFAU - Smailus, Duane E\\nAU  - Smailus DE\\nFAU - Schnerch, Angelique\\nAU  - Schnerch A\\nFAU - Schein, Jacqueline E\\nAU  - Schein JE\\nFAU - Jones, Steven J M\\nAU  - Jones SJ\\nFAU - Marra, Marco A\\nAU  - Marra MA\\nCN  - Mammalian Gene Collection Program Team\\nLA  - eng\\nGR  - U54 HG003273/HG/NHGRI NIH HHS/United States\\nPT  - Journal Article\\nPT  - Research Support, U.S. Gov't, P.H.S.\\nDEP - 20021211\\nPL  - United States\\nTA  - Proc Natl Acad Sci U S A\\nJT  - Proceedings of the National Academy of Sciences of the United States of America\\nJID - 7505876\\nRN  - 0 (DNA, Complementary)\\nSB  - IM\\nMH  - Algorithms\\nMH  - Animals\\nMH  - DNA, Complementary/analysis/*chemistry\\nMH  - Gene Library\\nMH  - Humans\\nMH  - Mice\\nMH  - Open Reading Frames\\nMH  - *Sequence Analysis, DNA\\nPMC - PMC139241\\nOID - NLM: PMC139241\\nEDAT- 2002/12/13 04:00\\nMHDA- 2003/01/22 04:00\\nCRDT- 2002/12/13 04:00\\nPHST- 2002/12/13 04:00 [pubmed]\\nPHST- 2003/01/22 04:00 [medline]\\nPHST- 2002/12/13 04:00 [entrez]\\nAID - 10.1073/pnas.242603899 [doi]\\nAID - 242603899 [pii]\\nPST - ppublish\\nSO  - Proc Natl Acad Sci U S A. 2002 Dec 24;99(26):16899-903. doi:\\n      10.1073/pnas.242603899. Epub 2002 Dec 11.\", \"PMID- 12421765\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20030114\\nLR  - 20140611\\nIS  - 1088-9051 (Print)\\nIS  - 1088-9051 (Linking)\\nVI  - 12\\nIP  - 11\\nDP  - 2002 Nov\\nTI  - Protein-protein interactions between large proteins: two-hybrid screening using a\\n      functionally classified library composed of long cDNAs.\\nPG  - 1773-84\\nAB  - Large proteins have multiple domains that are potentially capable of binding many\\n      kinds of partners. It is conceivable, therefore, that such proteins could\\n      function as an intricate framework of assembly protein complexes. To\\n      comprehensively study protein-protein interactions between large KIAA proteins,\\n      we have constructed a library composed of 1087 KIAA cDNA clones based on prior\\n      functional classifications done in silico. We were guided by two principles that \\n      raise the success rate for detecting interactions per tested combination: we\\n      avoided testing low-probability combinations, and reduced the number of potential\\n      false negatives that arise from the fact that large proteins cannot reliably be\\n      expressed in yeast. The latter was addressed by constructing a cDNA library\\n      comprised of random fragments encoding large proteins. Cytoplasmic domains of\\n      KIAA transmembrane proteins (>1000 amino acids) were used as bait for yeast\\n      two-hybrid screening. Our analyses reveal that several KIAA proteins bearing a\\n      transmembrane region have the capability of binding to other KIAA proteins\\n      containing domains (e.g., PDZ, SH3, rhoGEF, and spectrin) known to be localized\\n      to highly specialized submembranous sites, indicating that they participate in\\n      cellular junction formation, receptor or channel clustering, and intracellular\\n      signaling events. Our representative library should be a very useful resource for\\n      detecting previously unidentified interactions because it complements\\n      conventional expression libraries, which seldom contain large cDNAs.\\nFAU - Nakayama, Manabu\\nAU  - Nakayama M\\nAD  - Department of Human Gene Research, Kazusa DNA Research Institute, Kisarazu, Chiba\\n      292-0818, Japan. nmanabu@kazusa.or.jp\\nFAU - Kikuno, Reiko\\nAU  - Kikuno R\\nFAU - Ohara, Osamu\\nAU  - Ohara O\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov't\\nPL  - United States\\nTA  - Genome Res\\nJT  - Genome research\\nJID - 9518021\\nRN  - 0 (DNA, Complementary)\\nRN  - 0 (Peptides)\\nRN  - 0 (Proteins)\\nSB  - IM\\nMH  - Brain Chemistry/genetics\\nMH  - Cloning, Molecular\\nMH  - Computational Biology/instrumentation/methods\\nMH  - Cytoplasm/chemistry\\nMH  - DNA, Complementary/chemistry/*genetics\\nMH  - *Gene Library\\nMH  - Humans\\nMH  - Leukemia, Myeloid/genetics/pathology\\nMH  - Peptide Mapping/methods\\nMH  - Peptides/genetics/physiology\\nMH  - Protein Interaction Mapping/*methods\\nMH  - Protein Structure, Tertiary/physiology\\nMH  - Proteins/chemistry/genetics/physiology\\nMH  - Software\\nMH  - Tumor Cells, Cultured\\nMH  - *Two-Hybrid System Techniques\\nMH  - src Homology Domains/genetics/physiology\\nPMC - PMC187542\\nOID - NLM: PMC187542\\nEDAT- 2002/11/08 04:00\\nMHDA- 2003/01/15 04:00\\nCRDT- 2002/11/08 04:00\\nPHST- 2002/11/08 04:00 [pubmed]\\nPHST- 2003/01/15 04:00 [medline]\\nPHST- 2002/11/08 04:00 [entrez]\\nAID - 10.1101/gr.406902 [doi]\\nPST - ppublish\\nSO  - Genome Res. 2002 Nov;12(11):1773-84. doi: 10.1101/gr.406902.\", \"PMID- 9455484\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 19980227\\nLR  - 20061115\\nIS  - 1340-2838 (Print)\\nIS  - 1340-2838 (Linking)\\nVI  - 4\\nIP  - 5\\nDP  - 1997 Oct 31\\nTI  - Characterization of cDNA clones in size-fractionated cDNA libraries from human\\n      brain.\\nPG  - 345-9\\nAB  - To evaluate the size-fractionated cDNA libraries of human brain previously\\n      constructed (O. O'hara et al. DNA Research, 4, 53-59, 1997), the occurrence of\\n      chimeric clones and the content of clones with coding potentiality were analyzed \\n      using the randomly sampled clones with insert sizes of 5 to 7 kb. When the\\n      chromosomal location of 30 clones was determined by the radiation-hybrid mapping \\n      method, the map positions assigned from the 3'- and 5'-end sequences separately\\n      were coincident for 29 clones, suggesting that the occurrence of chimeric clones \\n      is at most 1/30. Using 91 clones mapped to chromosome 1, the content of clones\\n      that have the potentiality coding for proteins larger than 100 amino acid\\n      residues was estimated to be approximately 50% (46 out of 91 clones) on the basis\\n      of nucleotide sequence analysis and coding potentiality assay in vitro. No\\n      significant open reading frames were detected in the remaining clones. Although\\n      the clones coding for short peptides may not have been included in the above\\n      estimation, the libraries constructed from the whole brain mRNA fraction appear\\n      to contain a considerable amount of clones corresponding to the 5'-truncated\\n      transcripts in an unprocessed form and/or those with long 3'-untranslated\\n      regions.\\nFAU - Seki, N\\nAU  - Seki N\\nAD  - Kazusa DNA Research Institute, Chiba, Japan. nseki@kazusa.or.jp\\nFAU - Ohira, M\\nAU  - Ohira M\\nFAU - Nagase, T\\nAU  - Nagase T\\nFAU - Ishikawa, K\\nAU  - Ishikawa K\\nFAU - Miyajima, N\\nAU  - Miyajima N\\nFAU - Nakajima, D\\nAU  - Nakajima D\\nFAU - Nomura, N\\nAU  - Nomura N\\nFAU - Ohara, O\\nAU  - Ohara O\\nLA  - eng\\nSI  - GENBANK/AB007913\\nSI  - GENBANK/AB007914\\nSI  - GENBANK/AB007915\\nSI  - GENBANK/AB007916\\nSI  - GENBANK/AB007917\\nSI  - GENBANK/AB007918\\nSI  - GENBANK/AB007919\\nSI  - GENBANK/AB007920\\nSI  - GENBANK/AB007921\\nSI  - GENBANK/AB007922\\nSI  - GENBANK/AB007923\\nSI  - GENBANK/AB007924\\nSI  - GENBANK/AB007925\\nSI  - GENBANK/AB007926\\nSI  - GENBANK/AB007927\\nSI  - GENBANK/AB007928\\nSI  - GENBANK/AB007929\\nSI  - GENBANK/AB007930\\nSI  - GENBANK/AB007931\\nSI  - GENBANK/AB007932\\nSI  - GENBANK/AB007933\\nSI  - GENBANK/AB007934\\nSI  - GENBANK/AB007935\\nSI  - GENBANK/AB007936\\nSI  - GENBANK/AB007937\\nSI  - GENBANK/AB007938\\nSI  - GENBANK/AB007939\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov't\\nPL  - England\\nTA  - DNA Res\\nJT  - DNA research : an international journal for rapid publication of reports on genes\\n      and genomes\\nJID - 9423827\\nRN  - 0 (DNA, Complementary)\\nRN  - 0 (DNAJC6 protein, human)\\nRN  - 0 (HSP40 Heat-Shock Proteins)\\nRN  - 0 (RNA, Messenger)\\nSB  - IM\\nMH  - Animals\\nMH  - Brain Chemistry/*genetics\\nMH  - Chimera\\nMH  - *Chromosome Mapping\\nMH  - *Chromosomes, Human, Pair 1\\nMH  - Cloning, Molecular\\nMH  - DNA, Complementary/*chemistry/genetics/*isolation & purification\\nMH  - Electrophoresis, Polyacrylamide Gel\\nMH  - Gene Library\\nMH  - HSP40 Heat-Shock Proteins\\nMH  - Humans\\nMH  - Open Reading Frames\\nMH  - Polymerase Chain Reaction\\nMH  - RNA, Messenger/genetics\\nMH  - Repetitive Sequences, Nucleic Acid\\nMH  - Sequence Homology, Nucleic Acid\\nMH  - Sequence Tagged Sites\\nMH  - Transcription, Genetic\\nEDAT- 1998/02/10 00:00\\nMHDA- 1998/02/10 00:01\\nCRDT- 1998/02/10 00:00\\nPHST- 1998/02/10 00:00 [pubmed]\\nPHST- 1998/02/10 00:01 [medline]\\nPHST- 1998/02/10 00:00 [entrez]\\nPST - ppublish\\nSO  - DNA Res. 1997 Oct 31;4(5):345-9.\", \"PMID- 9110174\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 19970617\\nLR  - 20161019\\nIS  - 1088-9051 (Print)\\nIS  - 1088-9051 (Linking)\\nVI  - 7\\nIP  - 4\\nDP  - 1997 Apr\\nTI  - Large-scale concatenation cDNA sequencing.\\nPG  - 353-8\\nAB  - A total of 100 kb of DNA derived from 69 individual human brain cDNA clones of\\n      0.7-2.0 kb were sequenced by concatenated cDNA sequencing (CCS), whereby multiple\\n      individual DNA fragments are sequenced simultaneously in a single shotgun\\n      library. The method yielded accurate sequences and a similar efficiency compared \\n      with other shotgun libraries constructed from single DNA fragments (> 20 kb).\\n      Computer analyses were carried out on 65 cDNA clone sequences and their\\n      corresponding end sequences to examine both nucleic acid and amino acid sequence \\n      similarities in the databases. Thirty-seven clones revealed no DNA database\\n      matches, 12 clones generated exact matches (> or = 98% identity), and 16 clones\\n      generated nonexact matches (57%-97% identity) to either known human or other\\n      species genes. Of those 28 matched clones, 8 had corresponding end sequences that\\n      failed to identify similarities. In a protein similarity search, 27 clone\\n      sequences displayed significant matches, whereas only 20 of the end sequences had\\n      matches to known protein sequences. Our data indicate that full-length cDNA\\n      insert sequences provide significantly more nucleic acid and protein sequence\\n      similarity matches than expressed sequence tags (ESTs) for database searching.\\nFAU - Yu, W\\nAU  - Yu W\\nFAU - Andersson, B\\nAU  - Andersson B\\nFAU - Worley, K C\\nAU  - Worley KC\\nFAU - Muzny, D M\\nAU  - Muzny DM\\nFAU - Ding, Y\\nAU  - Ding Y\\nFAU - Liu, W\\nAU  - Liu W\\nFAU - Ricafrente, J Y\\nAU  - Ricafrente JY\\nFAU - Wentland, M A\\nAU  - Wentland MA\\nFAU - Lennon, G\\nAU  - Lennon G\\nFAU - Gibbs, R A\\nAU  - Gibbs RA\\nLA  - eng\\nSI  - GENBANK/AF007128\\nSI  - GENBANK/AF007129\\nSI  - GENBANK/AF007130\\nSI  - GENBANK/AF007131\\nSI  - GENBANK/AF007132\\nSI  - GENBANK/AF007133\\nSI  - GENBANK/AF007134\\nSI  - GENBANK/AF007135\\nSI  - GENBANK/AF007136\\nSI  - GENBANK/AF007137\\nSI  - GENBANK/AF007138\\nSI  - GENBANK/AF007139\\nSI  - GENBANK/AF007140\\nSI  - GENBANK/AF007141\\nSI  - GENBANK/AF007142\\nSI  - GENBANK/AF007143\\nSI  - GENBANK/AF007144\\nSI  - GENBANK/AF007145\\nSI  - GENBANK/AF007146\\nSI  - GENBANK/AF007147\\nSI  - GENBANK/AF007148\\nSI  - GENBANK/AF007149\\nSI  - GENBANK/AF007150\\nSI  - GENBANK/AF007151\\nSI  - GENBANK/AF007152\\nSI  - GENBANK/AF007153\\nSI  - GENBANK/AF007154\\nGR  - F33 HG000210/HG/NHGRI NIH HHS/United States\\nGR  - U54 HG003273/HG/NHGRI NIH HHS/United States\\nGR  - F32 HG000169/HG/NHGRI NIH HHS/United States\\nGR  - P30 HG00210-05/HG/NHGRI NIH HHS/United States\\nGR  - R01 HG00823/HG/NHGRI NIH HHS/United States\\nGR  - 1F32 HG00169-01/HG/NHGRI NIH HHS/United States\\nPT  - Letter\\nPT  - Research Support, U.S. Gov't, Non-P.H.S.\\nPT  - Research Support, U.S. Gov't, P.H.S.\\nPL  - United States\\nTA  - Genome Res\\nJT  - Genome research\\nJID - 9518021\\nRN  - 0 (DNA Transposable Elements)\\nRN  - 0 (DNA, Complementary)\\nRN  - 0 (Proteins)\\nSB  - IM\\nMH  - DNA Transposable Elements\\nMH  - DNA, Complementary/chemistry/*genetics\\nMH  - Databases, Factual\\nMH  - Gene Library\\nMH  - Humans\\nMH  - Molecular Sequence Data\\nMH  - Proteins/chemistry/*genetics\\nMH  - Sequence Alignment/*methods\\nMH  - Sequence Analysis, DNA/*methods\\nMH  - Sequence Homology, Amino Acid\\nMH  - Sequence Homology, Nucleic Acid\\nMH  - Software\\nPMC - PMC139146\\nOID - NLM: PMC139146\\nEDAT- 1997/04/01 00:00\\nMHDA- 1997/04/01 00:01\\nCRDT- 1997/04/01 00:00\\nPHST- 1997/04/01 00:00 [pubmed]\\nPHST- 1997/04/01 00:01 [medline]\\nPHST- 1997/04/01 00:00 [entrez]\\nPST - ppublish\\nSO  - Genome Res. 1997 Apr;7(4):353-8.\", \"PMID- 8619474\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 19960613\\nLR  - 20071114\\nIS  - 0003-2697 (Print)\\nIS  - 0003-2697 (Linking)\\nVI  - 236\\nIP  - 1\\nDP  - 1996 Apr 5\\nTI  - A \\\"double adaptor\\\" method for improved shotgun library construction.\\nPG  - 107-13\\nAB  - The efficiency of shotgun DNA sequencing depends to a great extent on the quality\\n      of the random-subclone libraries used. We here describe a novel \\\"double adaptor\\\" \\n      strategy for efficient construction of high-quality shotgun libraries. In this\\n      method, randomly sheared and end-repaired fragments are ligated to\\n      oligonucleotide adaptors creating 12-base overhangs. Nonphosphorylated\\n      oligonucleotides are used, which prevents formation of adaptor dimers and ensures\\n      efficient ligation of insert to adaptor. The vector is prepared from a modified\\n      M13 vector, by KpnI/PstI digestion followed by ligation to oligonucleotides with \\n      ends complementary to the overhangs created in the digest. These adaptors create \\n      5'-overhangs complementary to those on the inserts. Following annealing of insert\\n      to vector, the DNA is directly used for transformation without a ligation step.\\n      This protocol is robust and shows three- to fivefold higher yield of clones\\n      compared to previous protocols. No chimeric clones can be detected and the\\n      background of clones without an insert is <1%. The procedure is rapid and shows\\n      potential for automation.\\nFAU - Andersson, B\\nAU  - Andersson B\\nAD  - Department of Molecular and Human Genetics, Baylor College of Medicine, One\\n      Baylor Plaza, Houston, Texas, 77030, USA.\\nFAU - Wentland, M A\\nAU  - Wentland MA\\nFAU - Ricafrente, J Y\\nAU  - Ricafrente JY\\nFAU - Liu, W\\nAU  - Liu W\\nFAU - Gibbs, R A\\nAU  - Gibbs RA\\nLA  - eng\\nSI  - GENBANK/AF007128\\nSI  - GENBANK/AF007129\\nSI  - GENBANK/AF007130\\nSI  - GENBANK/AF007131\\nSI  - GENBANK/AF007132\\nSI  - GENBANK/AF007133\\nSI  - GENBANK/AF007134\\nSI  - GENBANK/AF007135\\nSI  - GENBANK/AF007136\\nSI  - GENBANK/AF007137\\nSI  - GENBANK/AF007138\\nSI  - GENBANK/AF007139\\nSI  - GENBANK/AF007140\\nSI  - GENBANK/AF007141\\nSI  - GENBANK/AF007142\\nSI  - GENBANK/AF007143\\nSI  - GENBANK/AF007144\\nSI  - GENBANK/AF007145\\nSI  - GENBANK/AF007146\\nSI  - GENBANK/AF007147\\nSI  - GENBANK/AF007148\\nSI  - GENBANK/AF007149\\nSI  - GENBANK/AF007150\\nSI  - GENBANK/AF007151\\nSI  - GENBANK/AF007152\\nSI  - GENBANK/AF007153\\nSI  - GENBANK/AF007154\\nGR  - R01 HG00823/HG/NHGRI NIH HHS/United States\\nPT  - Journal Article\\nPT  - Research Support, U.S. Gov't, P.H.S.\\nPL  - United States\\nTA  - Anal Biochem\\nJT  - Analytical biochemistry\\nJID - 0370535\\nRN  - 0 (Oligonucleotides)\\nSB  - IM\\nMH  - Base Sequence\\nMH  - Cloning, Molecular/*methods\\nMH  - Genetic Vectors\\nMH  - *Genomic Library\\nMH  - Humans\\nMH  - Molecular Sequence Data\\nMH  - Oligonucleotides/chemistry\\nMH  - Polymerase Chain Reaction\\nMH  - X Chromosome\\nEDAT- 1996/04/05 00:00\\nMHDA- 1996/04/05 00:01\\nCRDT- 1996/04/05 00:00\\nPHST- 1996/04/05 00:00 [pubmed]\\nPHST- 1996/04/05 00:01 [medline]\\nPHST- 1996/04/05 00:00 [entrez]\\nAID - S0003-2697(96)90138-4 [pii]\\nAID - 10.1006/abio.1996.0138 [doi]\\nPST - ppublish\\nSO  - Anal Biochem. 1996 Apr 5;236(1):107-13. doi: 10.1006/abio.1996.0138.\"]"
      ]
     },
     "execution_count": 95,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "require 'net/http'\n",
    "require 'bio'\n",
    "require 'nokogiri'\n",
    "\n",
    "\n",
    "ncbi = Bio::NCBI::REST.new\n",
    "max = 5\n",
    "\n",
    "# note that you can now use spaces in your search terms - BioRuby will format the URL correctly\n",
    "ids = ncbi.esearch('\"Wilkinson MD\"[AUTH]', {\"db\"=>\"nucleotide\", \"rettype\"=>\"gb\", \"retmax\" => max})\n",
    "\n",
    "idlist = ids.join(\",\")\n",
    "\n",
    "elink = URI(\"https://eutils.ncbi.nlm.nih.gov:443/entrez/eutils/elink.fcgi\")\n",
    "request = Net::HTTP::Post.new(elink)\n",
    "https = Net::HTTP.new(elink.host,elink.port)\n",
    "https.use_ssl = true  # tell the HTTP connection to use ecryption (SSL = Secure Socket Layer)\n",
    "\n",
    "request.set_form_data(\n",
    "  'email' => 'markw@illuminae.com',\n",
    "  'dbfrom' => 'nuccore', \n",
    "  'db' => 'pubmed',\n",
    "  'retmode' => 'xml',\n",
    "  'id' => idlist)\n",
    "\n",
    "output = https.request(request)   # make the request (we really should check if it is successful)\n",
    "\n",
    "linkset = output.body\n",
    "\n",
    "\n",
    "doc = Nokogiri::XML(linkset)\n",
    "\n",
    "pubmedids = []  # force it to be a string so that we can use '<<' operator\n",
    "\n",
    "doc.xpath('//LinkSet/LinkSetDb/Link').each do |node|    # the \"path\" (\"xpath\")to the data, starts with '//'\n",
    "   pubmedids << node.at_xpath('Id').content.to_s\n",
    "end\n",
    "$stderr.puts \"PUBMED IDs are: #{pubmedids}\"\n",
    "\n",
    "# we remove these lines because now we are going to output the BibTex instead of the Abstract\n",
    "#pubmedrecords = ncbi.efetch(pubmedids, {\"db\"=>\"pubmed\", \"rettype\"=>\"abstract\", \"retmode\"=>\"text\"})\n",
    "#puts pubmedrecords.force_encoding(\"utf-8\")  # NOTE - this is a useful trick for text that has international characters!\n",
    "\n",
    "Bio::PubMed.efetch(pubmedids).each do |pubmedrecord|\n",
    "  # pubmedrecord is a string of the record, in pubmed format\n",
    "  # we can use that to create a new Bio::MEDLINE object\n",
    "  medline = Bio::MEDLINE.new(pubmedrecord)\n",
    "  reference = medline.reference  # reference is a Bio::Reference object\n",
    "  puts reference.bibtex   # the .bibtex method creates the bibtex string of the Bio::Reference\n",
    "end"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "...and you can write that to a file and import it into EndNote, or Mendeley, or PaperPile, or whatever reference manager you prefer!\n",
    "\n",
    "\n",
    "DONE!\n"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Ruby 2.4.2",
   "language": "ruby",
   "name": "ruby"
  },
  "language_info": {
   "file_extension": ".rb",
   "mimetype": "application/x-ruby",
   "name": "ruby",
   "version": "2.4.2"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
